---
citation-template: v0.2.0
creation-date: 2025:02:05-19:39:41
---

%% Note Body --------------------------------------------------- %%
# Coordination of microbial metabolism

### Meta
- ** citekey **: chubukovCoordinationMicrobialMetabolism2014
- ** authors **: Victor Chubukov, Luca Gerosa, Karl Kochanowski, [[Uwe Sauer]]
- ** year **: [[2014]]
- ** doi **: https://doi.org/10.1038/nrmicro3238
- ** publication **: Nature Reviews Microbiology
- ** Web ** : [Open online](https://www.nature.com/articles/nrmicro3238)


### Abstract:


___

## View

%% Write here %%


## References

[1] [[@gerosaRegulationControlMetabolic2011]] https://doi.org/10.1016/j.copbio.2011.04.016 (2011) Gerosa, L. & Sauer, U. Regulation and control of metabolic fluxes in microbes. Curr. Opin. Biotechnol. 22, 566–575 (2011).

[2] https://doi.org/10.1016/j.mib.2010.02.005 (2010) Heinemann, M. & Sauer, U. Systems biology of microbial metabolism. Curr. Opin. Microbiol. 13, 343–337 (2010).

[3] [[@karrWholeCellComputationalModel2012]] https://doi.org/10.1016/j.cell.2012.05.044 (2012) Karr, J. R. et al. A whole-cell computational model predicts phenotype from genotype. Cell 150, 389–401 (2012).

[4] https://doi.org/10.1038/nrg1244 (2004) Wall, M. E., Hlavacek, W. S. & Savageau, M. A. Design of gene circuits: lessons from bacteria. Nature Rev. Genet. 5, 34–42 (2004).

[5] https://doi.org/10.1101/gr.186501 (2001) Bochner, B. R., Gadzinski, P. & Panomitros, E. Phenotype microarrays for high-throughput phenotypic testing and assay of gene function. Genome Res. 11, 1246–1255 (2001).

[6] [[@orthComprehensiveGenomescaleReconstruction2011]] https://doi.org/10.1038/msb.2011.65 (2011) Orth, J. D. et al. A comprehensive genome-scale reconstruction of Escherichia coli metabolism — 2011. Mol. Syst. Biol. 7, 535 (2011).

[7] https://doi.org/10.1093/dnares/4.2.161 (1997) Mizuno, T. Compilation of all genes encoding two-component phosphotransfer signal transducers in the genome of Escherichia coli. DNA Res. 4, 161–168 (1997).

[8] https://doi.org/10.1146/annurev.genet.41.042007.170548 (2007) Laub, M. T. & Goulian, M. Specificity in two-component signal transduction pathways. Annu. Rev. Genet. 41, 121–145 (2007).

[9] https://doi.org/10.1099/00221287-147-12-3345 (2001) Verhamme, D. T., Arents, J. C., Postma, P. W., Crielaard, W. & Hellingwerf, K. J. Glucose-6-phosphate-dependent phosphoryl flow through the Uhp two-component regulatory system. Microbiology 147, 3345–3352 (2001).

[10] https://doi.org/10.1016/j.tim.2005.11.002 (2006) Martinez-Antonio, A., Janga, S. C., Salgado, H. & Collado-Vides, J. Internal-sensing machinery directs the activity of the regulatory network in Escherichia coli. Trends Microbiol. 14, 22–27 (2006).

[11] https://doi.org/10.1016/0022-2836(72)90253-7 (1972) Jobe, A. & Bourgeois, S. lac repressor–operator interaction: VI. The natural inducer of the lac operon. J. Mol. Biol. 69, 397–408 (1972).

[12] https://doi.org/10.1016/j.tim.2004.12.006 (2005) Ulrich, L. E., Koonin, E. V. & Zhulin, I. B. One-component systems dominate signal transduction in prokaryotes. Trends Microbiol. 13, 52–56 (2005).

[13] (1958) Monod, J. Recherches sur la Croissance des Cultures Bacteriennes (in French) (Hermann, 1958).

[14] https://doi.org/10.1038/nrmicro1932 (2008) Görke, B. & Stülke, J. Carbon catabolite repression in bacteria: many ways to make the most out of nutrients. Nature Rev. Microbiol. 6, 613–624 (2008). This is an excellent review on bacterial catabolite repression and the PTS.

[15] [[@deutscherHowPhosphotransferaseSystemRelated2006]] https://doi.org/10.1128/MMBR.00024-06 (2006) Deutscher, J., Francke, C. & Postma, P. W. How phosphotransferase system-related protein phosphorylation regulates carbohydrate metabolism in bacteria. Microbiol. Mol. Biol. Rev. 70, 939–1031 (2006).

[16] https://doi.org/10.1371/journal.pone.0020081 (2011) Shimada, T., Fujita, N., Yamamoto, K. & Ishihama, A. Novel roles of cAMP receptor protein (CRP) in regulation of transport and metabolism of carbon sources. PLoS ONE 6, e20081 (2011).

[17] https://doi.org/10.1016/j.molcel.2008.01.021 (2008) Kaplan, S., Bren, A., Zaslaver, A., Dekel, E. & Alon, U. Diverse two-dimensional input functions control bacterial sugar genes. Mol. Cell 29, 786–792 (2008). This paper shows the integration of regulation by Crp and carbon source-specific transcription factors in uptake systems.

[18] https://doi.org/10.1128/JB.00819-07 (2007) Bettenbrock, K. et al. Correlation between growth rates, EIIACrr phosphorylation, and intracellular cyclic AMP levels in Escherichia coli K-12. J. Bacteriol. 189, 6891–6900 (2007).

[19] https://doi.org/10.1046/j.1365-2958.1998.01053.x (1998) Hogema, B. M. et al. Inducer exclusion in Escherichia coli by non-PTS substrates: the role of the PEP to pyruvate ratio in determining the phosphorylation state of enzyme IIAGlc. Mol. Microbiol. 30, 487–498 (1998).

[20] [[@youCoordinationBacterialProteome2013]] https://doi.org/10.1038/nature12446 (2013) You, C. et al. Coordination of bacterial proteome with metabolism by cyclic AMP signalling. Nature 500, 301–306 (2013). This is groundbreaking work that unravels the regulatory logic of Crp-dependent catabolite repression in E. coli.

[21] https://doi.org/10.1371/journal.pcbi.1000802 (2010) Goyal, S., Yuan, J., Chen, T., Rabinowitz, J. D. & Wingreen, N. S. Achieving optimal growth through product feedback inhibition in metabolism. PLoS Comput. Biol. 6, e1000802 (2010).

[22] https://doi.org/10.1128/JB.187.22.7826-7839.2005 (2005) Lorca, G. L. et al. Catabolite repression and activation in Bacillus subtilis: dependency on CcpA, HPr, and HprK. J. Bacteriol. 187, 7826–7839 (2005).

[23] https://doi.org/10.1128/JB.00848-08 (2008) Singh, K. D., Schmalisch, M. H., Stülke, J. & Görke, B. Carbon catabolite repression in Bacillus subtilis: quantitative analysis of repression exerted by different carbon sources. J. Bacteriol. 190, 7275–7284 (2008).

[24] https://doi.org/10.1074/jbc.275.3.1773 (2000) Jault, J.-M. et al. The HPr kinase from Bacillus subtilis is a homo-oligomeric enzyme which exhibits strong positive cooperativity for nucleotide and fructose 1,6-bisphosphate binding. J. Biol. Chem. 275, 1773–1780 (2000).

[25] https://doi.org/10.1038/msb.2013.66 (2013) Chubukov, V. et al. Transcriptional regulation is insufficient to explain substrate-induced flux changes in Bacillus subtilis. Mol. Syst. Biol. 9, 709 (2013).

[26] https://doi.org/10.1074/jbc.273.37.24102 (1998) Meijer, M. M. C., Boonstra, J., Verkleij, A. J. & Verrips, C. T. Glucose repression in Saccharomyces cerevisiae is related to the glucose concentration rather than the glucose flux. J. Biol. Chem. 273, 24102–24107 (1998).

[27] https://doi.org/10.1128/MMBR.70.1.253-282.2006 (2006) Santangelo, G. M. Glucose signaling in Saccharomyces cerevisiae. Microbiol. Mol. Biol. Rev. 70, 253–282 (2006).

[28] https://doi.org/10.1038/nature08653 (2009) Youk, H. & van Oudenaarden, A. Growth landscape formed by perception and import of glucose in yeast. Nature 462, 875–879 (2009).

[29] https://doi.org/10.1016/j.febslet.2009.10.071 (2009) Levy, S. & Barkai, N. Coordination of gene expression with growth rate: a feedback or a feed-forward strategy? FEBS Lett. 583, 3974–3978 (2009). This paper brings forwards the distinction between feedforward and feedback strategies of growth regulation.

[30] https://doi.org/10.1016/j.copbio.2013.03.014 (2013) Kochanowski, K., Sauer, U. & Chubukov, V. Somewhat in control — the role of transcription in regulating microbial metabolic fluxes. Curr. Opin. Biotechnol. 24, 987–993 (2013).

[31] https://doi.org/10.1038/msb.2012.55 (2012) Oliveira, A. P. et al. Regulation of yeast central metabolism by enzyme phosphorylation. Mol. Syst. Biol. 8, 623 (2012).

[32] https://doi.org/10.1126/science.1179687 (2010) Wang, Q. et al. Acetylation of metabolic enzymes coordinates carbon source utilization and metabolic flux. Science 327, 1004–1007 (2010).

[33] https://doi.org/10.1038/nbt.2489 (2013) Link, H., Kochanowski, K. & Sauer, U. Systematic identification of allosteric protein–metabolite interactions that control enzyme activity in vivo. Nature Biotech. 31, 357–361 (2013). This paper outlines an approach to systematically map relevant allosteric interactions.

[34] https://doi.org/10.1038/nchembio816 (2006) Yuan, J., Fowler, W. U., Kimball, E., Lu, W. & Rabinowitz, J. D. Kinetic flux profiling of nitrogen assimilation in Escherichia coli. Nature Chem. Biol. 2, 529–530 (2006).

[35] https://doi.org/10.1016/0923-2508(96)84003-4 (1996) Ramseier, T. M. Cra and the control of carbon flux via metabolic pathways. Res. Microbiol. 147, 489–493 (1996).

[36] https://doi.org/10.1128/JB.01214-10 (2011) Shimada, T., Yamamoto, K. & Ishihama, A. Novel members of the Cra regulon involved in carbon metabolism in Escherichia coli. J Bacteriol. 193, 649–659 (2011).

[37] https://doi.org/10.1073/pnas.1202582110 (2013) Kochanowski, K. et al. Functioning of a metabolic flux sensor in Escherichia coli. Proc. Natl Acad. Sci. USA 110, 1130–1135 (2013). This study experimentally demonstrates the potential of FBP to report glycolytic flux in E. coli.

[38] https://doi.org/10.1021/bi00647a006 (1976) Waygood, E. B., Mort, J. S. & Sanwal, B. D. The control of pyruvate kinase of Escherichia coli. Binding of substrate and allosteric effectors to the enzyme activated by fructose 1,6-bisphosphate. Biochemistry 15, 277–282 (1976).

[39] https://doi.org/10.1038/nchembio.941 (2012) Xu, Y.-F., Amador-Noguez, D., Reaves, M. L., Feng, X.-J. & Rabinowitz, J. D. Ultrasensitive regulation of anapleurosis via allosteric activation of PEP carboxylase. Nature Chem. Biol. 8, 562–568 (2012).

[40] https://doi.org/10.1371/journal.pcbi.1000812 (2010) Baldazzi, V. et al. The carbon assimilation network in Escherichia coli is densely connected and largely sign-determined by directions of metabolic fluxes. PLoS Comput. Biol. 6, e1000812 (2010).

[41] https://doi.org/10.1016/j.copbio.2008.06.008 (2008) Daniels, B. C., Chen, Y. J., Sethna, J. P., Gutenkunst, R. N. & Myers, C. R. Sloppiness, robustness, and evolvability in systems biology. Curr. Opin. Biotechnol. 19, 389–395 (2008).

[42] https://doi.org/10.1016/S0021-9258(18)93376-4 (1968) Carminatti, H., Asúa, L. J. de, Recondo, E., Passeron, S. & Rozengurt, E. Some kinetic properties of liver pyruvate kinase (Type L). J. Biol. Chem. 243, 3051–3056 (1968).

[43] https://doi.org/10.1016/S0969-2126(98)00021-5 (1998) Jurica, M. S. et al. The allosteric regulation of pyruvate kinase by fructose-1,6-bisphosphate. Structure 6, 195–210 (1998).

[44] https://doi.org/10.1016/0005-2744(72)90332-4 (1972) Diesterhaft, M. & Freese, E. Pyruvate kinase of Bacillus subtilis. Biochim. Biophys. Acta 268, 373–380 (1972).

[45] https://doi.org/10.1111/j.1365-2958.1995.tb02280.x (1995) Deutscher, J. et al. Protein kinase-dependent HPr/CcpA interaction links glycolytic activity to carbon catabolite repression in Gram-positive bacteria. Mol. Microbiol. 15, 1049–1053 (1995).

[46] https://doi.org/10.1046/j.1365-2958.2003.03404.x (2003) Doan, T. & Aymerich, S. Regulation of the central glycolytic genes in Bacillus subtilis: binding of the repressor CggR to its single DNA target sequence is modulated by fructose-1,6-bisphosphate. Mol. Microbiol. 47, 1709–1721 (2003).

[47] https://doi.org/10.1016/j.femsre.2004.11.002 (2005) Sauer, U. & Eikmanns, B. J. The PEP–pyruvate–oxaloacetate node as the switch point for carbon flux distribution in bacteria. FEMS Microbiol. Rev. 29, 765–794 (2005).

[48] https://doi.org/10.1111/j.1567-1364.2010.00713.x (2011) Christen, S. & Sauer, U. Intracellular characterization of aerobic glucose metabolism in seven yeast species by 13C flux analysis and metabolomics. FEMS Yeast Res. 11, 263–272 (2011).

[49] https://doi.org/10.1111/j.1567-1364.2011.00767.x (2011) Huberts, D. H. E. W., Niebel, B. & Heinemann, M. A flux-sensing mechanism could regulate the switch between respiration and fermentation. FEMS Yeast Res. 12, 118–128 (2011).

[50] https://doi.org/10.1016/j.molcel.2012.07.013 (2012) Xu, Y.-F. et al. Regulation of yeast pyruvate kinase by ultrasensitive allostery independent of phosphorylation. Mol. Cell 48, 52–62 (2012).

[51] https://doi.org/10.1074/jbc.M800408200 (2008) Díaz-Ruiz, R. et al. Mitochondrial oxidative phosphorylation is regulated by fructose 1,6-bisphosphate. A possible role in crabtree effect induction? J. Biol. Chem. 283, 26948–26955 (2008).

[52] https://doi.org/10.1073/pnas.0609508103 (2006) Brauer, M. J. et al. Conservation of the metabolomic response to starvation across two divergent microbes. Proc. Natl Acad. Sci. USA 103, 19302–19307 (2006).

[53] https://doi.org/10.1073/pnas.0603337103 (2006) Voit, E., Neves, A. R. & Santos, H. The intricate side of systems biology. Proc. Natl Acad. Sci. USA 103, 9452–9457 (2006).

[54] https://doi.org/10.1016/j.cmet.2011.06.017 (2011) Grüning, N.M. et al. Pyruvate kinase triggers a metabolic feedback loop that controls redox metabolism in respiring cells. Cell. Metab. 14, 415–427 (2011). This paper identifies the regulatory circuit in yeast that is responsible for the allosteric upregulation of NAPDH production in the pentose phosphate pathway following oxidative stress.

[55] https://doi.org/10.1093/nar/gks1027 (2012) Keseler, I. M. et al. EcoCyc: fusing model organism databases with systems biology. Nucleic Acids Res. 41, D605–D612 (2012).

[56] https://doi.org/10.1074/jbc.M610838200 (2007) Lorca, G. L. et al. Glyoxylate and pyruvate are antagonistic effectors of the Escherichia coli IclR transcriptional regulator. J. Biol. Chem. 282, 16476–16491 (2007).

[57] https://doi.org/10.1186/1752-0509-5-197 (2011) Göhler, A.-K. et al. More than just a metabolic regulator — elucidation and validation of new targets of PdhR in Escherichia coli. BMC Syst. Biol. 5, 197 (2011).

[58] [[@schuetzSystematicEvaluationObjective2007]] https://doi.org/10.1038/msb4100162 (2007) Schuetz, R., Kuepfer, L. & Sauer, U. Systematic evaluation of objective functions for predicting intracellular fluxes in Escherichia coli. Mol. Syst. Biol. 3, 119 (2007).

[59] [[@priceGenomescaleModelsMicrobial2004]] https://doi.org/10.1038/nrmicro1023 (2004) Price, N. D., Reed, J. L. & Palsson, B. Genome-scale models of microbial cells: evaluating the consequences of constraints. Nature Rev. Microbiol. 2, 886–897 (2004).

[60] https://doi.org/10.1038/msb.2009.60 (2009) Yuan, J. et al. Metabolomics-driven quantitative analysis of ammonia assimilation in E. coli. Mol. Syst. Biol. 5, 302 (2009).

[61] https://doi.org/10.1038/msb.2010.87 (2010) Gallego, O. et al. A systematic screen for protein–lipid interactions in Saccharomyces cerevisiae. Mol. Syst. Biol. 6, 430 (2010).

[62] https://doi.org/10.1016/j.cell.2010.09.048 (2010) Li, X., Gianoulis, T. A., Yip, K. Y., Gerstein, M. & Snyder, M. Extensive in vivo metabolite–protein interactions revealed by large-scale systematic analyses. Cell 143, 639–50 (2010).

[63] https://doi.org/10.1021/bi8000566 (2008) Rabinowitz, J. D. et al. Dissecting enzyme regulation by multiple allosteric effectors: nucleotide regulation of aspartate transcarbamoylase. Biochemistry 47, 5881–5888 (2008).

[64] https://doi.org/10.1007/978-1-4612-1072-6 (1986) Gottschalk, G. Bacterial Metabolism (Springer, 1986).

[65] https://doi.org/10.1128/JB.184.14.3909-3916.2002 (2002) Koebmann, B. J., Westerhoff, H. V., Snoep, J. L., Nilsson, D. & Jensen, P. R. The glycolytic flux in Escherichia coli is controlled by the demand for ATP. J. Bacteriol. 184, 3909 (2002).

[66] https://doi.org/10.1074/jbc.M109.095570 (2010) Holm, A. K. et al. Metabolic and transcriptional response to cofactor perturbations in Escherichia coli. J. Biol. Chem. 285, 17498–17506 (2010). References 65 and 66 provide experimental evidence that glycolysis may be controlled by ATP demand.

[67] https://doi.org/10.1016/j.febslet.2013.07.013 (2013) Mensonides, F. I. C. et al. A new regulatory principle for in vivo biochemistry: pleiotropic low affinity regulation by the adenine nucleotides — illustrated for the glycolytic enzymes of Saccharomyces cerevisiae. FEBS Lett. 587, 2860–2867 (2013).

[68] https://doi.org/10.1111/j.1432-1033.1981.tb06375.x (1981) Kotlarz, D. & Buc, H. Regulatory properties of phosphofructokinase 2 from Escherichia coli. Eur. J. Biochem. 117, 569–574 (1981).

[69] https://doi.org/10.1016/0003-9861(83)90109-1 (1983) Babul, J. & Guixé, V. Fructose bisphosphatase from Escherichia coli. Purification and characterization. Arch. Biochem. Biophys. 225, 944–949 (1983).

[70] (2002) Berg, J. M., Tymoczko, J. L. & Stryer, L. in Biochemistry (W H Freeman, 2002).

[71] https://doi.org/10.1074/jbc.275.6.3931 (2000) Petersen, C. & Møller, L. B. Invariance of the nucleoside triphosphate pools of Escherichia coli with growth rate. J. Biol. Chem. 275, 3931–3935 (2000).

[72] [[@schneiderRelationshipGrowthRate2004]] https://doi.org/10.1074/jbc.M311996200 (2004) Schneider, D. A. & Gourse, R. L. Relationship between growth rate and ATP concentration in Escherichia coli: a bioassay for available cellular ATP. J. Biol. Chem. 279, 8262–8268 (2004).

[73] https://doi.org/10.1016/0923-2508(94)90092-2 (1994) Gunsalus, R. & Park, S. Aerobic–anaerobic gene regulation in Escherichia coli: control by the ArcAB and Fnr regulons. Res. Microbiol. 145, 437–450 (1994).

[74] https://doi.org/10.1016/S0005-2728(97)00034-0 (1997) Unden, G. & Bongaerts, J. Alternative respiratory pathways of Escherichia coli: energetics and transcriptional regulation in response to electron acceptors. Biochim. Biophys. Acta 1320, 217–234 (1997).

[75] https://doi.org/10.1046/j.1365-2958.1997.4731841.x (1997) Unden, G. & Schirawski, J. The oxygen-responsive transcriptional regulator FNR of Escherichia coli: the search for signals and reactions. Mol. Microbiol. 25, 205–210 (1997).

[76] https://doi.org/10.1016/j.mib.2009.01.008 (2009) Green, J., Crack, J. C., Thomson, A. J. & LeBrun, N. E. Bacterial sensors of oxygen. Curr. Opin. Microbiol. 12, 145–151 (2009).

[77] https://doi.org/10.1126/science.1059361 (2001) Georgellis, D., Kwon, O. & Lin, E. C. Quinones as the redox signal for the arc two-component system of bacteria. Science 292, 2314–2316 (2001).

[78] https://doi.org/10.1099/mic.0.2007/006098-0 (2007) Bekker, M. et al. Changes in the redox state and composition of the quinone pool of Escherichia coli during aerobic batch-culture growth. Microbiology 153, 1974–1980 (2007).

[79] [[@nanchenNonlinearDependencyIntracellular2006]] https://doi.org/10.1128/AEM.72.2.1164-1172.2006 (2006) Nanchen, A., Schicker, A. & Sauer, U. Nonlinear dependency of intracellular fluxes on growth rate in miniaturized continuous cultures of Escherichia coli. Appl. Environ. Microbiol. 72, 1164–1172 (2006).

[80] https://doi.org/10.1016/S0021-9258(18)69186-0 (1988) Anderson, D. H. & Duckworth, H. W. In vitro mutagenesis of Escherichia coli citrate synthase to clarify the locations of ligand binding sites. J. Biol. Chem. 263, 2163–2169 (1988).

[81] https://doi.org/10.1073/pnas.0707476104 (2007) Daran-Lapujade, P. et al. The fluxes through glycolytic enzymes in Saccharomyces cerevisiae are predominantly regulated at posttranscriptional levels. Proc. Natl Acad. Sci. 104, 15753–15758 (2007).

[82] https://doi.org/10.1186/1752-0509-4-12 (2010) Fendt, S.-M. & Sauer, U. Transcriptional regulation of respiration in yeast metabolizing differently repressive carbon substrates. BMC Syst. Biol. 4, 12 (2010).

[83] [[@scottInterdependenceCellGrowth2010]] https://doi.org/10.1126/science.1192588 (2010) Scott, M., Gunderson, C. W., Mateescu, E. M., Zhang, Z. & Hwa, T. Interdependence of cell growth and gene expression: origins and consequences. Science 330, 1099–1102 (2010).

[84] [[@molenaarShiftsGrowthStrategies2009]] https://doi.org/10.1038/msb.2009.82 (2009) Molenaar, D., van Berlo, R., de Ridder, D. & Teusink, B. Shifts in growth strategies reflect tradeoffs in cellular economics. Mol. Syst. Biol. 5, 323 (2009).

[85] [[@valgepeaSystemsBiologyApproach2010]] https://doi.org/10.1186/1752-0509-4-166 (2010) Valgepea, K. et al. Systems biology approach reveals that overflow metabolism of acetate in Escherichia coli is triggered by carbon catabolite repression of acetyl-CoA synthetase. BMC Syst. Biol. 4, 166 (2010).

[86] https://doi.org/10.1146/annurev.micro.57.030502.090820 (2003) Reitzer, L. Nitrogen assimilation and global regulation in Escherichia coli. Annu. Rev. Microbiol. 57, 155–176 (2003).

[87] https://doi.org/10.1074/jbc.M411114200 (2005) Kurihara, S. et al. A novel putrescine utilization pathway involves γ-glutamylated intermediates of Escherichia coli K-12. J. Biol. Chem. 280, 4602–4608 (2005).

[88] https://doi.org/10.1073/pnas.0405821101 (2004) Sohanpal, B. K., El-Labany, S., Lahooti, M., Plumbridge, J. A. & Blomfield, I. C. Integrated regulatory responses of fimB to N-acetylneuraminic (sialic) acid and GlcNAc in Escherichia coli K-12. Proc. Natl Acad. Sci. USA 101, 16322–16327 (2004).

[89] https://doi.org/10.1146/annurev.micro.61.080706.093409 (2007) Leigh, J. A. & Dodsworth, J. A. Nitrogen regulation in bacteria and archaea. Annu. Rev. Microbiol. 61, 349–377 (2007).

[90] https://doi.org/10.1128/MMBR.00025-13 (2013) Heeswijk, W. C. van, Westerhoff, H. V. & Boogerd, F. C. Nitrogen assimilation in Escherichia coli: putting molecular data into a systems perspective. Microbiol. Mol. Biol. Rev. 77, 628–695 (2013).

[91] https://doi.org/10.1021/bi901158h (2009) Jiang, P. & Ninfa, A. J. α-ketoglutarate controls the ability of the Escherichia coli PII signal transduction protein to regulate the activities of NRII (NtrB) but does not control the binding of PII to NRII. Biochemistry 48, 11514–11521 (2009).

[92] https://doi.org/10.1128/mBio.00881-13 (2013) Schumacher, J. et al. Nitrogen and carbon status are integrated at the transcriptional level by the nitrogen regulator NtrC in vivo. mBio 4, e00881-13 (2013).

[93] https://doi.org/10.1016/j.mib.2005.02.011 (2005) Ninfa, A. J. & Jiang, P. PII signal transduction proteins: sensors of α-ketoglutarate that regulate nitrogen metabolism. Curr. Opin. Microbiol. 8, 168–173 (2005).

[94] https://doi.org/10.1021/bi0620510 (2007) Jiang, P., Mayo, A. E. & Ninfa, A. J. Escherichia coli gutamine synthetase adenylyltransferase (ATase, EC 2.7.7.49): kinetic characterization of regulation by PII, PII-UMP, glutamine, and α-Ketoglutarate. Biochemistry 46, 4133–4146 (2007).

[95] https://doi.org/10.1038/nchembio.685 (2011) Doucette, C. D., Schwab, D. J., Wingreen, N. S. & Rabinowitz, J. D. α-Ketoglutarate coordinates carbon and nitrogen utilization via enzyme I inhibition. Nature Chem. Biol. 7, 894–901 (2011).

[96] https://doi.org/10.1074/jbc.M110.153908 (2010) Radchenko, M. V., Thornton, J. & Merrick, M. Control of AmtB–GlnK complex formation by intracellular levels of ATP, ADP, and 2-Oxoglutarate. J. Biol. Chem. 285, 31037–31045 (2010).

[97] https://doi.org/10.1038/msb.2012.46 (2012) Kim, M. et al. Need-based activation of ammonium uptake in Escherichia coli. Mol. Syst. Biol. 8, 616 (2012). This study shows how E. coli prevents the wasteful uptake of ammonium in presence of ample external ammonium.

[98] https://doi.org/10.1016/S0092-8674(01)00572-4 (2001) Wray, L. V. J., Zalieckas, J. M. & Fisher, S. H. Bacillus subtilis glutamine synthetase controls gene expression through a protein–protein interaction with transcription factor TnrA. Cell 107, 427–435 (2001).

[99] https://doi.org/10.1016/S0378-1119(02)00558-9 (2002) Magasanik, B. & Kaiser, C. A. Nitrogen regulation in Saccharomyces cerevisiae. Gene 290, 1–18 (2002).

[100] https://doi.org/10.1111/j.1574-6976.2000.tb00533.x (2000) Ter Schure, E. G., van Riel, N. A. & Verrips, C. T. The role of ammonia metabolism in nitrogen catabolite repression in Saccharomyces cerevisiae. FEMS Microbiol. Rev. 24, 67–83 (2000).

[101] https://doi.org/10.1073/pnas.1304386110 (2013) Radchenko, M. V., Thornton, J. & Merrick, M. PII signal transduction proteins are ATPases whose activity is regulated by 2-oxoglutarate. Proc. Natl Acad. Sci. 110, 12948–12953 (2013).

[102] https://doi.org/10.1016/S0006-3495(00)76475-7 (2000) Alves, R. & Savageau, M. A. Effect of overall feedback inhibition in unbranched biosynthetic pathways. Biophys. J. 79, 2290–2304 (2000).

[103] https://doi.org/10.1016/S0378-1119(03)00839-4 (2003) Pardee, A. Beginnings of feedback inhibition, allostery, and multi-protein complexes. Gene 321, 17–23 (2003).

[104] https://doi.org/10.1038/nchembio.710 (2012) Cho, B.-K., Federowicz, S., Park, Y.-S., Zengler, K. & Palsson, B. Deciphering the transcriptional regulatory logic of amino acid metabolism. Nature Chem. Biol. 8, 65–71 (2012).

[105] https://doi.org/10.1038/289751a0 (1981) Yanofsky, C. Attenuation in the control of expression of bacterial operons. Nature 289, 751–758 (1981).

[106] https://doi.org/10.1073/pnas.1114235109 (2012) Chubukov, V., Zuleta, I. A. & Li, H. Regulatory architecture determines optimal regulation of gene expression in metabolic pathways. Proc. Natl Acad. Sci. USA 109, 5127–5132 (2012).

[107] https://doi.org/10.1038/msb.2013.14 (2013) Gerosa, L., Kochanowski, K., Heinemann, M. & Sauer, U. Dissecting specific and global transcriptional regulation of bacterial gene expression. Mol. Syst. Biol. 9, 658 (2013). This paper unravels the regulatory logic of the arginine biosynthesis pathway in E. coli by combining experimental and computational efforts.

[108] https://doi.org/10.1128/JB.184.11.2940-2950.2002 (2002) Kiupakis, A. K. & Reitzer, L. ArgR-independent induction and ArgR-dependent superinduction of the astCADBE operon in Escherichia coli. J. Bacteriol. 184, 2940–2950 (2002).

[109] https://doi.org/10.1128/MMBR.43.1.42-58.1979 (1979) De Felice, M., Levinthal, M., Iaccarino, M. & Guardiola, J. Growth inhibition as a consequence of antagonism between related amino acids: effect of valine in Escherichia coli K-12. Microbiol. Rev. 43, 42–58 (1979).

[110] https://doi.org/10.1128/MMBR.58.3.466-490.1994 (1994) Calvo, J. M. & Matthews, R. G. The leucine-responsive regulatory protein, a global regulator of metabolism in Escherichia coli. Microbiol. Rev. 58, 466–490 (1994).

[111] https://doi.org/10.1073/pnas.0807227105 (2008) Cho, B.-K., Barrett, C. L., Knight, E. M., Park, Y. S. & Palsson, B. Genome-scale reconstruction of the Lrp regulatory network in Escherichia coli. Proc. Natl Acad. Sci. USA 105, 19462–19467 (2008).

[112] https://doi.org/10.1111/j.1365-2958.2004.04135.x (2004) Shivers, R. P. & Sonenshein, A. L. Activation of the Bacillus subtilis global regulator CodY by direct interaction with branched-chain amino acids. Mol. Microbiol. 53, 599–611 (2004).

[113] https://doi.org/10.1016/j.molcel.2009.06.033 (2009) Binda, M. et al. The Vam6 GEF controls TORC1 by activating the EGO Complex. Mol. Cell 35, 563–573 (2009).

[114] (1996) Bremer, H. & Dennis, P. P. in Escherichia coli and Salmonella typhimurium: cellular and molecular biology (eds Neidhardt F. C. et al.) (ASM Press, 1996).

[115] https://doi.org/10.1016/S0968-0004(99)01460-7 (1999) Warner, J. R. The economics of ribosome biosynthesis in yeast. Trends Biochem. Sci. 24, 437–440 (1999).

[116] https://doi.org/10.1073/pnas.132285199 (2002) Schneider, D. a, Gaal, T. & Gourse, R. L. NTP-sensing by rRNA promoters in Escherichia coli is direct. Proc. Natl Acad. Sci. USA 99, 8602–8607 (2002). This paper demonstrates the regulation of rRNA expression by nucleotide triphosphates in E. coli.

[117] https://doi.org/10.1101/gad.183517.111 (2012) Peterson, C. N., Levchenko, I., Rabinowitz, J. D., Baker, T. A & Silhavy, T. J. RpoS proteolysis is controlled directly by ATP levels in Escherichia coli. Genes Dev. 26, 548–553 (2012).

[118] https://doi.org/10.1146/annurev.micro.62.081307.162903 (2008) Potrykus, K. & Cashel, M. (p)ppGpp: still magical? Annu. Rev. Microbiol. 62, 35–51 (2008).

[119] https://doi.org/10.1006/jmbi.2000.4327 (2001) Barker, M. M., Gaal, T., Josaitis, C. A. & Gourse, R. L. Mechanism of regulation of transcription initiation by ppGpp. I. Effects of ppGpp on transcription initiation in vivo and in vitro. J. Mol. Biol. 305, 673–688 (2001).

[120] https://doi.org/10.1101/gad.1745409 (2009) Rutherford, S. T., Villers, C. L., Lee, J. H., Ross, W. & Gourse, R. L. Allosteric control of Escherichia coli rRNA promoter complexes by DksA. Genes Dev. 23, 236–248 (2009).

[121] https://doi.org/10.1073/pnas.1019383108 (2011) Lemke, J. J. et al. Direct regulation of Escherichia coli ribosomal protein promoters by the transcription factors ppGpp and DksA. Proc. Natl Acad. Sci. USA 108, 5712–5717 (2011).

[122] https://doi.org/10.1128/JB.146.2.605-613.1981 (1981) Lopez, J. M., Dromerick, A. & Freese, E. Response of guanosine 5′-triphosphate concentration to nutritional changes and its significance for Bacillus subtilis sporulation. J. Bacteriol. 146, 605–613 (1981).

[123] https://doi.org/10.1016/j.molcel.2012.08.009 (2012) Kriel, A. et al. Direct regulation of GTP homeostasis by (p)ppGpp: a critical component of viability and stress resistance. Mol. Cell 48, 231–241 (2012).

[124] https://doi.org/10.1038/sj.emboj.7600423 (2004) Krásný, L. & Gourse, R. L. An alternative strategy for bacterial ribosome synthesis: Bacillus subtilis rRNA transcription regulation. EMBO J. 23, 4473–4483 (2004).

[125] https://doi.org/10.1128/JB.187.22.7554-7560.2005 (2005) Zhang, S. & Haldenwang, W. G. Contributions of ATP, GTP, and redox state to nutritional stress activation of the Bacillus subtilis σB transcription factor. J. Bacteriol. 187, 7554–7560 (2005).

[126] https://doi.org/10.1146/annurev.micro.59.031805.133833 (2005) Hinnebusch, A. G. Translational regulation of Gcn4 and the general amino acid control of yeast. Annu. Rev. Microbiol. 59, 407–450 (2005).

[127] https://doi.org/10.1128/EC.00334-07 (2008) Slattery, M. G., Liko, D. & Heideman, W. Protein kinase A, TOR, and glucose transport control the response to nutrient repletion in Saccharomyces cerevisiae. Eukaryot. Cell 7, 358–367 (2008).

[128] https://doi.org/10.1534/genetics.111.133363 (2011) Loewith, R. & Hall, M. N. Target of Rapamycin (TOR) in nutrient signaling and growth control. Genetics 189, 1177–1201 (2011).

[129] https://doi.org/10.1046/j.1365-2958.2000.02125.x (2000) Rolland, F. et al. Glucose-induced cAMP signalling in yeast requires both a G-protein coupled receptor system for extracellular glucose detection and a separable hexose kinase-dependent sensing process. Mol. Microbiol. 38, 348–358 (2000).

[130] https://doi.org/10.1186/1471-2164-9-102 (2008) Ishihama, Y. et al. Protein abundance profiling of the Escherichia coli cytosol. BMC Genomics 9, 102 (2008).

[131] https://doi.org/10.1093/nar/gkg210 (2003) Babu, M. M. & Teichmann, S. A. Evolution of transcription factors and the gene regulatory network in Escherichia coli. Nucleic Acids Res. 31, 1234–1244 (2003).

[132] https://doi.org/10.1186/1752-0509-2-20 (2008) Goelzer, A. et al. Reconstruction and analysis of the genetic and metabolic regulatory networks of the central metabolism of Bacillus subtilis. BMC Syst. Biol. 2, 20 (2008).

[133] https://doi.org/10.1016/j.tibs.2005.05.007 (2005) Sellick, C. A. & Reece, R. J. Eukaryotic transcription factors as direct nutrient sensors. Trends Biochem. Sci. 30, 405–412 (2005).

[134] https://doi.org/10.1016/j.ceb.2008.09.009 (2008) Dechant, R. & Peter, M. Nutrient signals driving cell growth. Curr. Opin. Cell Biol. 20, 678–687 (2008).

[135] https://doi.org/10.1091/mbc.e07-08-0779 (2008) Brauer, M. J. et al. Coordination of growth rate, cell cycle, stress response, and metabolic activity in yeast. Mol. Biol. Cell 19, 352–367 (2008).

[136] [[@airoldiPredictingCellularGrowth2009]] https://doi.org/10.1371/journal.pcbi.1000257 (2009) Airoldi, E. M. et al. Predicting cellular growth from gene expression signatures. PLoS Comput. Biol. 5, e1000257 (2009).

[137] https://doi.org/10.1038/msb.2012.70 (2013) Berthoumieux, S. et al. Shared control of gene expression in bacteria by transcription factors and global physiology of the cell. Mol. Syst. Biol. 9, 634 (2013).

[138] https://doi.org/10.1038/msb.2013.59 (2013) Keren, L. et al. Promoters maintain their relative activity levels under different growth conditions. Mol. Syst. Biol. 9, 701 (2013).

[139] [[@schellenbergerQuantitativePredictionCellular2011]] https://doi.org/10.1038/nprot.2011.308 (2011) Schellenberger, J. et al. Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox v2.0. Nature Protoc. 6, 1290–1307 (2011).

[140] https://doi.org/10.1038/nprot.2009.58 (2009) Zamboni, N., Fendt, S.-M., Rühl, M. & Sauer, U. 13C-based metabolic flux analysis. Nature Protoc. 4, 878–892 (2009).

[141] https://doi.org/10.1038/nprot.2008.131 (2008) Yuan, J., Bennett, B. D. & Rabinowitz, J. D. Kinetic flux profiling for quantitation of cellular metabolic fluxes. Nature Protoc. 3, 1328–1340 (2008).

[142] https://doi.org/10.1021/ac100101d (2010) Buescher, J. M., Moco, S., Sauer, U., Zamboni, N. & Chemistry, A. Ultra-high performance liquid chromatography-tandem mass spectrometry method for fast and robust quantification of anionic and aromatic metabolites. Anal. Chem. 82, 4403–4412 (2010).

[143] https://doi.org/10.1021/ac201267k (2011) Fuhrer, T., Heer, D., Begemann, B. & Zamboni, N. High-throughput, accurate mass metabolome profiling of cellular extracts by flow injection-time-of-flight mass spectrometry. Anal. Chem. 83, 7074–7080 (2011).

[144] https://doi.org/10.1038/nprot.2011.391 (2011) Bermejo, C., Haerizadeh, F., Takanaga, H., Chermak, D. & Frommer, W. B. Optical sensors for measuring dynamic changes of cytosolic metabolite levels in yeast. Nature Protoc. 6, 1806–1817 (2011).

[145] [[@wangRNASeqRevolutionaryTool2009]] https://doi.org/10.1038/nrg2484 (2009) Wang, Z., Gerstein, M. & Snyder, M. RNA-Seq: a revolutionary tool for transcriptomics. Nature Rev. Genet. 10, 57–63 (2009).

[146] https://doi.org/10.1038/nprot.2012.086 (2012) Ingolia, N. T., Brar, G. a, Rouskin, S., McGeachy, A. M. & Weissman, J. S. The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments. Nature Protoc. 7, 1534–1550 (2012).

[147] https://doi.org/10.1038/nmeth895 (2006) Zaslaver, A. et al. A comprehensive library of fluorescent transcriptional reporters for Escherichia coli. Nature Methods 3, 623–628 (2006).

[148] https://doi.org/10.1038/nrm2973 (2010) Ahrens, C. H., Brunner, E., Qeli, E., Basler, K. & Aebersold, R. Generating and navigating proteome maps using mass spectrometry. Nature Rev. Mol. Cell. Biol. 11, 789–801 (2010).

[149] https://doi.org/10.1016/j.mib.2012.02.005 (2012) Otto, A., Bernhardt, J., Hecker, M. & Becher, D. Global relative and absolute quantitation in microbial proteomics. Curr. Opin. Microbiol. 15, 364–372 (2012).

[150] https://doi.org/10.1038/nrg3306 (2012) Furey, T. S. ChIP–seq and beyond: new and improved methodologies to detect and characterize protein–DNA interactions. Nature Rev. Genet. 13, 840–852 (2012).

[151] https://doi.org/10.1038/nature04187 (2005) Ptacek, J. et al. Global analysis of protein phosphorylation in yeast. Nature 438, 679–684 (2005).

[152] https://doi.org/10.1021/pr300912q (2013) Zhang, K., Zheng, S., Yang, J. S., Chen, Y. & Cheng, Z. Comprehensive profiling of protein lysine acetylation in Escherichia coli. J. Proteome Res. 12, 844–851 (2013).

[153] https://doi.org/10.1016/j.molcel.2013.06.003 (2013) Weinert, B. T. et al. Acetyl-phosphate is a critical determinant of lysine acetylation in E. coli. Mol. Cell 51, 265–272 (2013).

[154] https://doi.org/10.1126/science.1238327 (2013) Moellering, R. E. & Cravatt, B. F. Functional lysine modification by an intrinsically reactive primary glycolytic metabolite. Science 341, 549–553 (2013).

[155] https://doi.org/10.1038/msb.2010.10 (2010) Kotte, O., Zaugg, J. B. & Heinemann, M. Bacterial adaptation through distributed sensing of metabolic fluxes. Mol. Syst. Biol. 6, 355 (2010).


___
%% Tags  ------------------------------------------------------- %%
#Review/ToRead
#Priority/5
#Vault/MetXVault 