---
citation-template: v0.2.0
creation-date: 2025:02:12-10:11:28
---

%% Note Body --------------------------------------------------- %%
# A global resource allocation strategy governs growth transition kinetics of Escherichia coli

### Meta
- ** citekey **: ericksonGlobalResourceAllocation2017
- ** authors **: David W. Erickson, Severin J. Schink, Vadim Patsalo, James R. Williamson, Ulrich Gerland, [[Terence Hwa]]
- ** year **: [[2017]]
- ** doi **: https://doi.org/10.1038/nature24299
- ** publication **: Nature
- ** Web ** : [Open online](https://www.nature.com/articles/nature24299)


### Abstract:


___

## View

%% Write here %%

## References

[1] (1947) Monod, J. The phenomenon of enzymatic adaptation and its bearings on problems of genetics and cellular differentiation. Growth 11, 223–289 (1947)

[2] https://doi.org/10.1099/00221287-19-3-607 (1958) Kjeldgaard, N. O., Maaløe, O. & Schaechter, M. The transition between different physiological states during balanced growth of Salmonella typhimurium. J. Gen. Microbiol. 19, 607–616 (1958)

[3] https://doi.org/10.1016/0022-2836(73)90535-4 (1973) Dennis, P. P. & Bremer, H. Regulation of ribonucleic acid synthesis in Escherichia coli: An analysis of a shift-up. J. Mol. Biol. 75, 145–159 (1973)

[4] https://doi.org/10.1016/S0006-3495(03)74517-2 (2003) Thattai, M. & Shraiman, B. I. Metabolic switching in the sugar phosphotransferase system of Escherichia coli. Biophys. J. 85, 744–754 (2003)

[5] https://doi.org/10.1126/science.1114383 (2005) Kussell, E. & Leibler, S. Phenotypic diversity, population growth, and information in fluctuating environments. Science 309, 2075–2078 (2005)

[6] https://doi.org/10.1038/ng.110 (2008) Acar, M., Mettetal, J. T. & van Oudenaarden, A. Stochastic switching as a survival strategy in fluctuating environments. Nat. Genet. 40, 471–475 (2008)

[7] https://doi.org/10.1371/journal.pone.0061686 (2013) Boulineau, S. et al. Single-cell dynamics reveals sustained growth during diauxic shifts. PLoS One 8, e61686 (2013)

[8] https://doi.org/10.15252/msb.20135022 (2014) Kotte, O., Volkmer, B., Radzikowski, J. L. & Heinemann, M. Phenotypic bistability in Escherichia coli’s central carbon metabolism. Mol. Syst. Biol. 10, 736 (2014)

[9] https://doi.org/10.1126/science.1245114 (2014) van Heerden, J. H. et al. Lost in transition: start-up of glycolysis yields subpopulations of nongrowing cells. Science 343, 1245114 (2014)

[10] https://doi.org/10.1371/journal.pbio.1002042 (2015) Venturelli, O. S., Zuleta, I., Murray, R. M. & El-Samad, H. Population diversification in a yeast metabolic program promotes anticipation of environmental shifts. PLoS Biol. 13, e1002042 (2015)

[11] https://doi.org/10.1038/nmeth.3584 (2015) Link, H., Fuhrer, T., Gerosa, L., Zamboni, N. & Sauer, U. Real-time metabolome profiling of the metabolic switch between starvation and growth. Nat. Methods 12, 1091–1097 (2015)

[12] https://doi.org/10.1146/annurev.bi.62.070193.003533 (1993) Kolb, A., Busby, S., Buc, H., Garges, S. & Adhya, S. Transcriptional regulation by cAMP and its receptor protein. Annu. Rev. Biochem. 62, 749–795 (1993)

[13] https://doi.org/10.1146/annurev.micro.62.081307.162903 (2008) Potrykus, K. & Cashel, M. (p)ppGpp: still magical? Annu. Rev. Microbiol. 62, 35–51 (2008)

[14] https://doi.org/10.1073/pnas.1019383108 (2011) Lemke, J. J. et al. Direct regulation of Escherichia coli ribosomal protein promoters by the transcription factors ppGpp and DksA. Proc. Natl Acad. Sci. USA 108, 5712–5717 (2011)

[15] [[@youCoordinationBacterialProteome2013]] https://doi.org/10.1038/nature12446 (2013) You, C. et al. Coordination of bacterial proteome with metabolism by cyclic AMP signalling. Nature 500, 301–306 (2013)

[16] [[@scottEmergenceRobustGrowth2014]] https://doi.org/10.15252/msb.20145379 (2014) Scott, M., Klumpp, S., Mateescu, E. M. & Hwa, T. Emergence of robust growth laws from optimal regulation of ribosome synthesis. Mol. Syst. Biol. 10, 747 (2014)

[17] https://doi.org/10.1038/nrmicro3448 (2015) Hauryliuk, V., Atkinson, G. C., Murakami, K. S., Tenson, T. & Gerdes, K. Recent functional insights into the role of (p)ppGpp in bacterial physiology. Nat. Rev. Microbiol. 13, 298–309 (2015)

[18] [[@scottInterdependenceCellGrowth2010]] https://doi.org/10.1126/science.1192588 (2010) Scott, M., Gunderson, C. W., Mateescu, E. M., Zhang, Z. & Hwa, T. Interdependence of cell growth and gene expression: origins and consequences. Science 330, 1099–1102 (2010)

[19] [[@huiQuantitativeProteomicAnalysis2015]] https://doi.org/10.15252/msb.20145697 (2015) Hui, S. et al. Quantitative proteomic analysis reveals a simple strategy of global resource allocation in bacteria. Mol. Syst. Biol. 11, 784 (2015)

[20] [[@basanOverflowMetabolismEscherichia2015]] https://doi.org/10.1038/nature15765 (2015) Basan, M. et al. Overflow metabolism in Escherichia coli results from efficient proteome allocation. Nature 528, 99–104 (2015)

[21] [[@hermsenGrowthRateComposition2015]] https://doi.org/10.15252/msb.20145537 (2015) Hermsen, R., Okano, H., You, C., Werner, N. & Hwa, T. A growth-rate composition formula for the growth of E. coli on co-utilized carbon substrates. Mol. Syst. Biol. 11, 801 (2015)

[22] [[@chubukovCoordinationMicrobialMetabolism2014]] https://doi.org/10.1038/nrmicro3238 (2014) Chubukov, V., Gerosa, L., Kochanowski, K. & Sauer, U. Coordination of microbial metabolism. Nat. Rev. Microbiol. 12, 327–340 (2014)

[23] https://doi.org/10.1146/annurev.micro.57.030502.090820 (2003) Reitzer, L. Nitrogen assimilation and global regulation in Escherichia coli. Annu. Rev. Microbiol. 57, 155–176 (2003)

[24] [[@daiReductionTranslatingRibosomes2016]] https://doi.org/10.1038/nmicrobiol.2016.231 (2016) Dai, X. et al. Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth. Nat. Microbiol. 2, 16231 (2016)

[25] Gausing, K. Ribosomes: Structure, Function and Genetics 693–718 (Univ. Park Press, 1980)

[26] https://doi.org/10.1128/MMBR.57.3.543-594.1993 (1993) Postma, P. W., Lengeler, J. W. & Jacobson, G. R. Phosphoenolpyruvate: carbohydrate phosphotransferase systems of bacteria. Microbiol. Rev. 57, 543–594 (1993)

[27] https://doi.org/10.1073/pnas.1309356110 (2013) Pavlov, M. Y. & Ehrenberg, M. Optimal control of gene expression for fast proteome adaptation to environmental change. Proc. Natl Acad. Sci. USA 110, 20527–20532 (2013)

[28] https://doi.org/10.1371/journal.pcbi.1004802 (2016) Giordano, N., Mairet, F., Gouzé, J.-L., Geiselmann, J. & de Jong, H. Dynamical allocation of cellular resources as an optimal control problem: novel insights into microbial growth strategies. PLoS Comput. Biol. 12, e1004802 (2016)

[29] https://doi.org/10.1128/jb.176.20.6324-6333.1994 (1994) Csonka, L. N., Ikeda, T. P., Fletcher, S. A. & Kustu, S. The accumulation of glutamate is necessary for optimal growth of Salmonella typhimurium in media of high osmolality but not induction of the proU operon. J. Bacteriol. 176, 6324–6333 (1994)

[30] https://doi.org/10.1128/JB.185.18.5611-5626.2003 (2003) Soupene, E. et al. Physiological studies of Escherichia coli strain MG1655: growth defects and apparent cross-regulation of gene expression. J. Bacteriol. 185, 5611–5626 (2003)

[31] https://doi.org/10.1073/pnas.120163297 (2000) Datsenko, K. A. & Wanner, B. L. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc. Natl Acad. Sci. USA 97, 6640–6645 (2000)

[32] Miller, J. H. Experiments in Molecular Genetics (Cold Spring Harbor Laboratory Press, 1972)

[33] https://doi.org/10.1007/BF00152753 (1991) Benthin, S., Nielsen, J. & Villadsen, J. A simple and reliable method for the determination of cellular RNA content. Biotechnol. Tech. 5, 39–42 (1991)

[34] https://doi.org/10.1002/pmic.200900375 (2010) Deutsch, E. W. et al. A guided tour of the Trans-Proteomic Pipeline. Proteomics 10, 1150–1159 (2010)

[35] https://doi.org/10.1093/bioinformatics/bth092 (2004) Craig, R. & Beavis, R. C. TANDEM: matching proteins with tandem mass spectra. Bioinformatics 20, 1466–1467 (2004)

[36] https://doi.org/10.1021/ac800080v (2008) Sperling, E., Bunner, A. E., Sykes, M. T. & Williamson, J. R. Quantitative analysis of isotope distributions in proteomic mass spectrometry using least-squares Fourier transform convolution. Anal. Chem. 80, 4906–4917 (2008)

[37] https://doi.org/10.1021/ac9028664 (2010) Sykes, M. T., Sperling, E., Chen, S. S. & Williamson, J. R. Quantitation of the ribosomal protein autoregulatory network using mass spectrometry. Anal. Chem. 82, 5038–5045 (2010)

[38] https://doi.org/10.1099/00221287-19-3-592 (1958) Schaechter, M., Maaløe, O. & Kjeldgaard, N. O. Dependency on medium and temperature of cell size and chemical composition during balanced grown of Salmonella typhimurium. J. Gen. Microbiol. 19, 592–606 (1958)

[39] [[@basanInflatingBacterialCells2015]] https://doi.org/10.15252/msb.20156178 (2015) Basan, M. et al. Inflating bacterial cells by increased protein synthesis. Mol. Syst. Biol. 11, 836 (2015)

[40] https://doi.org/10.1128/jb.178.11.3260-3269.1996 (1996) Tong, S., Porco, A., Isturiz, T. & Conway, T. Cloning and molecular genetic characterization of the Escherichia coli gntR, gntK, and gntU genes of GntI, the main system for gluconate metabolism. J. Bacteriol. 178, 3260–3269 (1996)

[41] https://doi.org/10.1128/jb.175.4.1087-1094.1993 (1993) Voegele, R. T., Sweet, G. D. & Boos, W. Glycerol kinase of Escherichia coli is activated by interaction with the glycerol facilitator. J. Bacteriol. 175, 1087–1094 (1993)

[42] https://doi.org/10.1016/0022-2836(74)90449-5 (1974) Dennis, P. P. & Bremer, H. Differential rate of ribosomal protein synthesis in Escherichia coli B/r. J. Mol. Biol. 84, 407–422 (1974)



___
%% Tags  ------------------------------------------------------- %%
#review/ToRead
#Vault/MetXVault 