---
citation-template: v0.2.0
creation-date: "2024:10:01-12:33:32"
---

%% Note Body --------------------------------------------------- %%
# Pseudo-transition Analysis Identifies the Key Regulators of Dynamic Metabolic Adaptations from Steady-State Data

### Meta
- ** citekey **: gerosaPseudotransitionAnalysisIdentifies2015
- ** authors **: Luca Gerosa, Bart R. B. Haverkorn van Rijsewijk, Dimitris Christodoulou, Karl Kochanowski, Thomas S. B. Schmidt, [[Elad Noor]], [[Uwe Sauer]]
- ** year **: [[2015]]
- ** doi **: https://doi.org/10.1016/j.cels.2015.09.008
- ** publication **: Cell Systems
- ** Web ** : [Open online]()

### Abstract:
Hundreds of molecular-level changes within central metabolism allow a cell to adapt to the changing environment. A primary challenge in cell physiology is to identify which of these molecular-level changes are active regulatory events. Here, we introduce pseudo-transition analysis, an approach that uses multiple steady-state observations of (13)C-resolved fluxes, metabolites, and transcripts to infer which regulatory events drive metabolic adaptations following environmental transitions. Pseudo-transition analysis recapitulates known biology and identifies an unexpectedly sparse, transition-dependent regulatory landscape: typically a handful of regulatory events drive adaptation between carbon sources, with transcription mainly regulating TCA cycle flux and reactants regulating EMP pathway flux. We verify these observations using time-resolved measurements of the diauxic shift, demonstrating that some dynamic transitions can be approximated as monotonic shifts between steady-state extremes. Overall, we show that pseudo-transition analysis can explore the vast regulatory landscape of dynamic transitions using relatively few steady-state data, thereby guiding time-consuming, hypothesis-driven molecular validations.

___

## View

%% Write here %%

### References

[1] https://doi.org/10.1091/mbc.E07-08-0779 (2008) missing

[2] https://doi.org/10.1021/ac100101d (2010) missing

[3] https://doi.org/10.1126/science.1206871 (2012) missing

[4] https://doi.org/10.1038/msb.2013.66 (2013) missing

[5] [[@chubukovCoordinationMicrobialMetabolism2014]] https://doi.org/10.1038/nrmicro3238 (2014) missing

[6] [[@covertIntegratingHighthroughputComputational2004]] https://doi.org/10.1038/nature02456 (2004) missing

[7] https://doi.org/10.1073/pnas.0707476104 (2007) missing

[8] https://doi.org/10.1038/msb.2010.91 (2010) missing

[9] [[@gaschGenomicExpressionPrograms2000]] https://doi.org/10.1091/mbc.11.12.4241 (2000) missing

[10] [[@gerosaRegulationControlMetabolic2011]] https://doi.org/10.1016/j.copbio.2011.04.016 (2011) missing

[11] https://doi.org/10.1038/msb.2013.14 (2013) missing

[12] https://doi.org/10.1016/j.cell.2010.02.011 (2010) missing

[13] https://doi.org/10.1038/nrmicro1932 (2008) missing

[14] https://doi.org/10.1371/journal.pcbi.1000802 (2010) missing

[15] [[@brLargescale13CfluxAnalysis2011]] https://doi.org/10.1038/msb.2011.9 (2011) missing

[16] https://doi.org/10.1016/j.mib.2010.02.005 (2010) missing

[17] [[@huiQuantitativeProteomicAnalysis2015]] https://doi.org/10.15252/msb.20145697 (2015) missing

[18] [[@ishiiMultipleHighthroughputAnalyses2007]] https://doi.org/10.1126/science.1132067 (2007) missing

[19] https://doi.org/10.1073/pnas.0305287101 (2004) missing

[20] https://doi.org/10.1016/j.molcel.2008.01.021 (2008) missing

[21] https://doi.org/10.1038/msb.2013.59 (2013) missing

[22] https://doi.org/10.1074/jbc.M109.061747 (2010) missing

[23] https://doi.org/10.1073/pnas.1202582110 (2013) missing

[24] https://doi.org/10.1042/BJ20140984 (2015) missing

[25] https://doi.org/10.1073/pnas.2136632100 (2003) missing

[26] https://doi.org/10.1038/nbt.2489 (2013) missing

[27] https://doi.org/10.1016/j.copbio.2014.01.015 (2014) missing

[28] https://doi.org/10.1016/j.febslet.2013.07.013 (2013) missing

[29] https://doi.org/10.1016/j.febslet.2013.07.028 (2013) missing

[30] https://doi.org/10.1371/journal.pcbi.1003483 (2014) missing

[31] https://doi.org/10.1038/msb.2012.55 (2012) missing

[32] [[@patilUncoveringTranscriptionalRegulation2005]] https://doi.org/10.1073/pnas.0406811102 (2005) missing

[33] https://doi.org/10.1016/j.mib.2014.12.006 (2015) missing

[34] https://doi.org/10.1038/msb.2013.16 (2013) missing

[35] [[@rossellUnravelingComplexityFlux2006]] https://doi.org/10.1073/pnas.0509831103 (2006) missing

[36] https://doi.org/10.1371/journal.pcbi.1003602 (2014) missing

[37] https://doi.org/10.1093/nar/gks1201 (2013) missing

[38] [[@sauerMetabolicNetworksMotion2006]] https://doi.org/10.1038/msb4100109 (2006) missing

[39] [[@schellenbergerQuantitativePredictionCellular2011]] https://doi.org/10.1038/nprot.2011.308 (2011) missing

[40] https://doi.org/10.1016/j.molcel.2012.07.013 (2012) missing

[41] [[@youCoordinationBacterialProteome2013]] https://doi.org/10.1038/nature12446 (2013) missing

[42] https://doi.org/10.1038/nprot.2009.58 (2009) missing

[43] https://doi.org/10.1038/nmeth895 (2006) missing



___
%% Tags  ------------------------------------------------------- %%
#review/ToRead
#Vault/MetXVault 