---
citation-template: v0.2.0
creation-date: "2024:10:27-01:07:34"
---

%% Note Body --------------------------------------------------- %%
# Quantitative proteomic analysis reveals a simple strategy of global resource allocation in bacteria

### Meta
- ** citekey **: huiQuantitativeProteomicAnalysis2015
- ** authors **: Sheng Hui, Josh M. Silverman, Stephen S. Chen, [[David W. Erickson]], [[Markus Basan]], Jilong Wang, [[Terence Hwa]], [[James R. Williamson]]
- ** year **: [[2015]]
- ** doi **: https://doi.org/10.15252/msb.20145697
- ** publication **: Molecular Systems Biology
- ** Web ** : [Open on line]()


### Abstract:
A central aim of cell biology was to understand the strategy of gene expression in response to the environment. Here, we study gene expression response to metabolic challenges in exponentially growing Escherichia coli using mass spectrometry. Despite enormous complexity in the details of the underlying regulatory network, we find that the proteome partitions into several coarse-grained sectors, with each sector's total mass abundance exhibiting positive or negative linear relations with the growth rate. The growth rate-dependent components of the proteome fractions comprise about half of the proteome by mass, and their mutual dependencies can be characterized by a simple flux model involving only two effective parameters. The success and apparent generality of this model arises from tight coordination between proteome partition and metabolism, suggesting a principle for resource allocation in proteome economy of the cell. This strategy of global gene regulation should serve as a basis for future studies on gene expression and constructing synthetic biological circuits. Coarse graining may be an effective approach to derive predictive phenomenological models for other 'omics' studies.

___

## View

%% Write here %%


[1] https://doi.org/10.1038/nature01511 missing

[2] [[@airoldiPredictingCellularGrowth2009]] https://doi.org/10.1371/journal.pcbi.1000257 missing

[3] https://doi.org/10.1007/BF01739257 missing

[4] https://doi.org/10.1007/s00216-007-1486-6 missing

[5] https://doi.org/10.1016/0022-2836(74)90234-4 missing

[6] https://doi.org/10.1038/nmeth774 missing

[7] [[@boerGenomewideTranscriptionalResponses2003]] https://doi.org/10.1074/jbc.M209759200 missing

[8] https://doi.org/10.1091/mbc.E07-08-0779 missing

[9] (2009) missing

[10] https://doi.org/10.1038/4462 missing

[11] https://doi.org/10.1186/jbiol54 missing

[12] https://doi.org/10.1128/jb.171.7.3597-3602.1989 (1989) missing

[13] (2014) missing

[14] [[@dekelOptimalityEvolutionaryTuning2005]] https://doi.org/10.1038/nature03842 missing

[15] https://doi.org/10.1016/0022-2836(74)90449-5 missing

[16] https://doi.org/10.1016/S0022-2836(76)80135-0 missing

[17] https://doi.org/10.1126/science.1237435 missing

[18] https://doi.org/10.1016/S0378-1097(03)00203-9 missing

[19] [[@flamholzGlycolyticStrategyTradeoff2013]] https://doi.org/10.1073/pnas.1215283110 missing

[20] https://doi.org/10.1038/nature02046 missing

[21] [[@goelzerBacterialGrowthRate2011]] https://doi.org/10.1016/j.bbagen.2011.05.014 missing

[22] https://doi.org/10.1016/j.cbpa.2008.07.024 missing

[23] (1945) missing

[24] https://doi.org/10.1111/j.1469-185X.1944.tb00309.x missing

[25] https://doi.org/10.1016/S0014-5793(00)01668-9 missing

[26] https://doi.org/10.1126/science.1168978 missing

[27] https://doi.org/10.1074/mcp.M500061-MCP200 missing

[28] https://doi.org/10.1186/1471-2164-9-102 missing

[29] https://doi.org/10.1099/00221287-19-3-607 missing

[30] [[@klumppGrowthRateDependentGlobal2009]] https://doi.org/10.1016/j.cell.2009.12.001 missing

[31] [[@klumppMolecularCrowdingLimits2013]] https://doi.org/10.1073/pnas.1310377110 missing

[32] https://doi.org/10.1016/0022-2836(71)90336-6 missing

[33] https://doi.org/10.1046/j.1365-2958.1996.5901313.x missing

[34] https://doi.org/10.1126/science.1061315 missing

[35] https://doi.org/10.1371/journal.pone.0000250 missing

[36] [[@liQuantifyingAbsoluteProtein2014]] https://doi.org/10.1016/j.cell.2014.02.033 missing

[37] https://doi.org/10.1142/5523 missing

[38] https://doi.org/10.1038/nbt1270 missing

[39] (1979) missing

[40] https://doi.org/10.1016/j.febslet.2009.10.036 missing

[41] https://doi.org/10.1038/nature08184 missing

[42] (2011) missing

[43] https://doi.org/10.1016/j.bpj.2013.04.049 missing

[44] [[@molenaarShiftsGrowthStrategies2009]] https://doi.org/10.1038/msb.2009.82 missing

[45] https://doi.org/10.1074/mcp.M113.032631 missing

[46] [[@obrienGenomescaleModelsMetabolism2013]] https://doi.org/10.1038/msb.2013.52 missing

[47] https://doi.org/10.1073/pnas.96.12.6591 missing

[48] https://doi.org/10.1016/S0021-9258(19)41496-8 missing

[49] https://doi.org/10.1038/nchembio736 missing

[50] https://doi.org/10.1016/0092-8674(78)90312-4 missing

[51] https://doi.org/10.1186/gb-2006-7-11-r107 missing

[52] https://doi.org/10.1146/annurev.micro.57.030502.090820 missing

[53] https://doi.org/10.1016/j.molcel.2013.03.021 missing

[54] https://doi.org/10.1093/bioinformatics/bth349 missing

[55] (1958) missing

[56] https://doi.org/10.1038/msb.2011.37 missing

[57] [[@scottInterdependenceCellGrowth2010]] https://doi.org/10.1126/science.1192588 missing

[58] https://doi.org/10.1016/j.molcel.2010.04.015 missing

[59] https://doi.org/10.1074/mcp.M500230-MCP200 missing

[60] https://doi.org/10.1021/ac800080v missing

[61] [[@taniguchiQuantifyingColiProteome2010]] https://doi.org/10.1126/science.1188308 missing

[62] https://doi.org/10.1128/JB.181.20.6425-6440.1999 missing

[63] https://doi.org/10.1038/nbt1094-994 missing

[64] https://doi.org/10.1038/nbt.2839 missing

[65] https://doi.org/10.1038/nprot.2008.132 missing

[66] https://doi.org/10.1038/msb.2010.59 missing

[67] https://doi.org/10.1038/msb.2011.46 missing

[68] [[@youCoordinationBacterialProteome2013]] https://doi.org/10.1038/nature12446 missing

[69] https://doi.org/10.1371/journal.pcbi.1000545 missing


___
%% Tags  ------------------------------------------------------- %%
#review/ToRead
#Vault/MetXVault 