---
citation-template: v0.2.0
creation-date: 2025:02:05-18:56:29
---

%% Note Body --------------------------------------------------- %%
# Cellular perception of growth rate and the mechanistic origin of bacterial growth law

### Meta
- ** citekey **: wuCellularPerceptionGrowth2022
- ** authors **: Chenhao Wu, Rohan Balakrishnan, Nathan Braniff, [[Matteo Mori]], Gabriel Manzanarez, Zhongge Zhang, [[Terence Hwa]]
- ** year **: [[2022]]
- ** doi **: https://doi.org/10.1073/pnas.2201585119
- ** publication **: Proceedings of the National Academy of Sciences
- ** Web ** : [Open online](https://pnas.org/doi/full/10.1073/pnas.2201585119)


### Abstract:
Significance Cell signaling involves interpreting information from the environment and implementing the appropriate regulatory control. The ppGpp signaling system is known to exert a fundamental control in bacteria, coordinating ribosome synthesis with the rate of cellular growth, but how this system can sense the cellular growth rate in different environments is unclear. Here we establish a striking linear relation between ppGpp level and the reciprocal of the translation elongation rate (ER) and show how bacteria can process complex nutritional information by monitoring this rate and using it to control ribosome synthesis. Importantly, the sensing of ER through ppGpp quantitatively and mechanistically explains bacterial growth laws which robustly relate the ribosome content and translation ERs to the growth rate. , Many cellular activities in bacteria are organized according to their growth rate. The notion that ppGpp measures the cell’s growth rate is well accepted in the field of bacterial physiology. However, despite decades of interrogation and the identification of multiple molecular interactions that connects ppGpp to some aspects of cell growth, we lack a system-level, quantitative picture of how this alleged “measurement” is performed. Through quantitative experiments, we show that the ppGpp pool responds inversely to the rate of translational elongation in Escherichia coli . Together with its roles in inhibiting ribosome biogenesis and activity, ppGpp closes a key regulatory circuit that enables the cell to perceive and control the rate of its growth across conditions. The celebrated linear growth law relating the ribosome content and growth rate emerges as a consequence of keeping a supply of ribosome reserves while maintaining elongation rate in slow growth conditions. Further analysis suggests the elongation rate itself is detected by sensing the ratio of dwelling and translocating ribosomes, a strategy employed to collapse the complex, high-dimensional dynamics of the molecular processes underlying cell growth to perceive the physiological state of the whole.

___

## View

%% Write here %%

## References

[1] [[@molenaarShiftsGrowthStrategies2009]] https://doi.org/10.1038/msb.2009.82 missing

[2] [[@scottInterdependenceCellGrowth2010]] https://doi.org/10.1126/science.1192588 missing

[3] [[@huiQuantitativeProteomicAnalysis2015]] https://doi.org/10.15252/msb.20145697 missing

[4] [[@schmidtQuantitativeConditiondependentEscherichia2016]] https://doi.org/10.1038/nbt.3418 missing

[5] [[@bruggemanSearchingPrinciplesMicrobial2020]] https://doi.org/10.1093/femsre/fuaa034 missing

[6] https://doi.org/10.15252/msb.20209536 missing

[7] https://doi.org/10.1099/00221287-19-3-592 missing

[8] https://doi.org/10.1016/0006-3002(60)90757-5 missing

[9] https://doi.org/10.1007/978-1-4684-3417-0_12 missing

[10] [[@scottEmergenceRobustGrowth2014]] https://doi.org/10.15252/msb.20145379 missing

[11] https://doi.org/10.1042/bj1600185 missing

[12] https://doi.org/10.1128/jb.129.2.1020-1033.1977 missing

[13] (1980) A. L. Koch, The inefficiency of ribosomes functioning in Escherichia coli growing at moderate rates. J. Gen. Microbiol. 116, 165–171 (1980).

[14] [[@moriQuantifyingBenefitProteome2017]] https://doi.org/10.1038/s41467-017-01242-8 missing

[15] [[@klumppMolecularCrowdingLimits2013]] https://doi.org/10.1073/pnas.1310377110 missing

[16] https://doi.org/10.1038/221838a0 missing

[17] https://doi.org/10.1111/j.1462-2920.2010.02357.x missing

[18] https://doi.org/10.1038/nrmicro3448 missing

[19] https://doi.org/10.1038/s41579-020-00470-y missing

[20] https://doi.org/10.3389/fmicb.2020.592718 missing

[21] https://doi.org/10.1016/j.tim.2005.03.008 missing

[22] https://doi.org/10.1146/annurev.micro.62.081307.162903 missing

[23] https://doi.org/10.1016/j.mib.2008.02.001 missing

[24] https://doi.org/10.1093/femsre/fuz009 missing

[25] https://doi.org/10.1016/0167-4781(95)00042-F missing

[26] https://doi.org/10.1111/j.1365-2958.2008.06229.x missing

[27] https://doi.org/10.1073/pnas.1019383108 missing

[28] https://doi.org/10.1073/pnas.1819682116 missing

[29] https://doi.org/10.1016/S0021-9258(18)69108-2 missing

[30] [[@paulRRNATranscriptionEscherichia2004]] https://doi.org/10.1146/annurev.genet.38.072902.091347 missing

[31] https://doi.org/10.1021/bi00797a027 missing

[32] [[@okanoRegulationUnderlyingHierarchical2019]] https://doi.org/10.1038/s41564-019-0610-7 missing

[33] [[@daiReductionTranslatingRibosomes2016]] https://doi.org/10.1038/nmicrobiol.2016.231 missing

[34] https://doi.org/10.1128/mBio.02375-17 missing

[35] https://doi.org/10.1038/s41589-018-0063-y missing

[36] https://doi.org/10.1016/j.ymeth.2019.02.004 missing

[37] https://doi.org/10.1128/jb.151.3.1261-1268.1982 missing

[38] https://doi.org/10.1007/BF00339584 missing

[39] https://doi.org/10.1146/annurev.micro.50.1.645 missing

[40] https://doi.org/10.1128/JB.181.24.7405-7408.1999 missing

[41] https://doi.org/10.1146/annurev.bi.45.070176.003531 missing

[42] https://doi.org/10.1016/S0021-9258(19)45938-3 missing

[43] https://doi.org/10.1046/j.1365-2443.2001.00457.x missing

[44] https://doi.org/10.1093/jb/mvm243 missing

[45] https://doi.org/10.1126/science.1218538 missing

[46] https://doi.org/10.1073/pnas.0606384103 missing

[47] https://doi.org/10.1073/pnas.1920013117 missing

[48] https://doi.org/10.1371/journal.pbio.3000593 missing

[49] [[@siInvarianceInitiationMass2017]] https://doi.org/10.1016/j.cub.2017.03.022 missing

[50] [[@ericksonGlobalResourceAllocation2017]] https://doi.org/10.1038/nature24299 missing

[51] https://doi.org/10.3389/fmicb.2020.619430 missing

[52] https://doi.org/10.1186/s12964-020-00642-6 missing

[53] [[@moriYieldcostTradeoffGoverns2019]] https://doi.org/10.1038/s41540-019-0093-4 missing

[54] https://doi.org/10.1038/nmeth.3584 missing

[55] https://doi.org/10.7554/eLife.17029 missing

[56] https://doi.org/10.1093/nar/gkw470 missing

[57] https://doi.org/10.1016/S1097-2765(02)00656-1 missing

[58] https://doi.org/10.1038/nature17675 missing

[59] https://doi.org/10.1073/pnas.70.5.1564 missing

[60] https://doi.org/10.1016/S0021-9258(19)67694-5 missing

[61] https://doi.org/10.1128/JB.00315-07 missing

[62] N. M. Belliveau Fundamental limits on the rate of bacterial growth. bioRxiv [Preprint] (2020). https://www.biorxiv.org/content/10.1101/2020.10.18.344382v1. Accessed 18 October 2020.

[63] [[@youCoordinationBacterialProteome2013]] https://doi.org/10.1038/nature12446 missing

[64] [[@liEscherichiaColiTranslation2018]] https://doi.org/10.1038/s41564-018-0199-2 missing

[65] https://doi.org/10.1042/bj1760305 missing

[66] https://doi.org/10.1016/0022-2836(74)90234-4 missing

[67] https://doi.org/10.1016/j.cub.2017.05.076 missing

[68] https://doi.org/10.1146/annurev-genet-120215-035130 missing

[69] https://doi.org/10.1074/jbc.REV118.002814 missing

[70] https://doi.org/10.3389/fmicb.2018.01802 missing

[71] https://doi.org/10.1128/JB.186.16.5249-5257.2004 missing

[72] (2020) T. Hwa, “Cell behavior and control” in The Physics of Living Matter: Space, Time and Information, D. J. Gross, A. Sevrin, B. Shraiman, Eds. (World Scientific, 2020), pp. 85–134.

[73] https://doi.org/10.1016/0022-2836(91)90212-O missing

[74] https://doi.org/10.1128/JB.185.18.5611-5626.2003 missing

[75] https://doi.org/10.1128/genomeA.00879-15 missing

[76] https://doi.org/10.1073/pnas.120163297 missing

[77] https://doi.org/10.1128/jb.115.1.9-14.1973 missing

[78] https://doi.org/10.1016/S0021-9258(18)93106-6 missing

[79] [[@basanInflatingBacterialCells2015]] https://doi.org/10.15252/msb.20156178 missing

[80] https://doi.org/10.1101/2021.05.24.445329 R. Balakrishnan Principles of gene regulation quantitatively connect DNA to RNA and proteins in bacteria. bioRxiv [Preprint] (2021). https://www.biorxiv.org/content/10.1101/2021.05.24.445329v2. Accessed 29 April 2022.

[81] https://doi.org/10.1007/BF00152753 missing

[82] https://doi.org/10.1016/S0580-9517(08)70641-X missing



___
%% Tags  ------------------------------------------------------- %%
#review/ToRead
#Vault/MetXVault 