![](_page_0_Picture_0.jpeg)

![](_page_0_Picture_2.jpeg)

# [How Important Is Protein Diffusion in](https://www.frontiersin.org/articles/10.3389/fmolb.2018.00093/full) Prokaryotes?

[Paul E. Schavemaker,](http://loop.frontiersin.org/people/603576/overview) Arnold J. Boersma and [Bert Poolman*](http://loop.frontiersin.org/people/561534/overview)

*Department of Biochemistry, University of Groningen, Groningen, Netherlands*

That diffusion is important for the proper functioning of cells is without question. The extent to which the diffusion coefficient is important is explored here for prokaryotic cells. We discuss the principles of diffusion focusing on diffusion-limited reactions, summarize the known values for diffusion coefficients in prokaryotes and in *in vitro* model systems, and explain a number of cases where diffusion coefficients are either limiting for reaction rates or necessary for the existence of phenomena. We suggest a number of areas that need further study including expanding the range of organism growth temperatures, direct measurements of diffusion limitation, expanding the range of cell sizes, diffusion limitation for membrane proteins, and taking into account cellular context when assessing the possibility of diffusion limitation.

Keywords: protein diffusion, crowding, prokaryote, reaction rate, diffusion limitation

### Edited by:

*Allen P. Minton, National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK), United States*

#### Reviewed by:

*James Carl Weisshaar, University of Wisconsin-Madison, United States Matthias Weiss, University of Bayreuth, Germany*

> *Correspondence: *Bert Poolman [b.poolman@rug.nl](mailto:b.poolman@rug.nl)*

#### Specialty section:

*This article was submitted to Cellular Biochemistry, a section of the journal Frontiers in Molecular Biosciences*

Received: *09 May 2018* Accepted: *22 October 2018* Published: *13 November 2018*

#### Citation:

*Schavemaker PE, Boersma AJ and Poolman B (2018) How Important Is Protein Diffusion in Prokaryotes?. Front. Mol. Biosci. 5:93. doi: [10.3389/fmolb.2018.00093](https://doi.org/10.3389/fmolb.2018.00093)*

## INTRODUCTION

In a cell everything moves and interactions between (macro) molecules are dynamic! This is one of the foremost facts about cells that any student of biochemistry and biology should know. The moving around of components allows proteins and cells to sample different states and provides meaning to the concept of entropy. The molecules move around without the need of work, which is called diffusion. Other types of motion that occur within cells, for example protein transport over membranes, do require work. Diffusion allows molecules to find one another in a cell: substrates need to find enzymes, transcription factors need to find sites on the DNA, membrane proteins need to find the membrane, etc. Thus, it is clear that diffusion is essential, but it is less clear to what degree diffusion is important, which is what we will discuss here. We focus on translational diffusion, which is the displacement of the center of mass of an object, while rotational diffusion is the rotation of an object around its center of mass. We will first discuss general principles of diffusion, including diffusion limitation of reactions and the effect of the intracellular environment on diffusion coefficients; we will provide a summary of experimentally determined protein diffusion coefficients in prokaryotes, and give examples of the consequences of these diffusion coefficients for the cell. Finally, we suggest some principles and experimental directions for further study. We conclude that the importance of protein diffusion coefficients should be assessed in the context of the various layers of complexity in the cell, and that protein diffusion plays, in some instances, a determining role in the physiology and biochemical organization of the cell.

### OVERVIEW OF DIFFUSION RATES AND THEIR CONSEQUENCES IN PROKARYOTES

### General Principles of Diffusion

Perpetual collisions cause molecules to move around inside of cells. Following the behavior of such a molecule, by noting its position every so often, reveals that the direction of travel of a molecule is random (barring structural asymmetries in the medium surrounding the molecule), and the trajectory is that of a random walk (**[Figure 1A](#page-2-0)**). On the other hand, the step-size of the molecule in each time interval is not random. The step size is determined by the size of the molecule, its interactions with the solvent, and the temperature. The step size (or, more accurately, the step size distribution) is captured by a single parameter, the diffusion coefficient (D). The diffusion coefficient can however also be dependent on length scale, which is called anomalous diffusion [(Dix and Verkman, 2008)](#page-13-0).

A distribution of molecules over space and its evolution in time are described by the diffusion equation [(Phillips et al., 2009)](#page-14-0):

$$\frac{\partial \mathcal{c}(\mathbf{x}, t)}{\partial t} = D \frac{\partial^2 \mathcal{c}(\mathbf{x}, t)}{\partial \mathbf{x}^2} \tag{1}$$

Here c(x, t) is the concentration of the molecule at position x and time t. D is the diffusion coefficient. This equation describes diffusion in only one dimension. The movement of the particle in one dimension is completely independent of its movement in the other dimensions. One of the solutions of this equation describes how a group of molecules localized to a point spreads out over time (**[Figure 1B](#page-2-0)**) [(Phillips et al., 2009)](#page-14-0):

$$c\left(\mathbf{x},t\right) = \frac{N}{\sqrt{4\pi Dt}}e^{-\frac{x^2}{4Dt}}\tag{2}$$

Here N represents the number of molecules. This equation can also be interpreted as the probability that a single molecule of the diffusing species is going to end up at position x after time t, assuming that diffusing particles do not influence each other. (Note that the molecules may influence each other, say by changing local viscosity, but then we are not dealing with normal diffusion anymore.) Taking the weighted mean over the distances in equation 2 and taking into account multiple dimensions (using Pythagoras' theorem) yields:

$$d = \sqrt{2nDt} \tag{3}$$

Here d is the distance and n is the number of dimensions considered. See **[Figure 1C](#page-2-0)**. Equations 2, 3 do not take into account boundaries such as a cell membrane and are therefore most accurate when length scales are short compared to cell size. Over longer length scales it is better to perform a detailed simulation to take into account the dimensions and shape of the cell (see **[Figure 1D](#page-2-0)**).

For two particles with diffusion coefficients of 10 µm2 /s to find each other in a 1 µm3 cell takes about 1 s. This can easily be calculated from the bimolecular reaction rate equation:

$$rate = k_{on} \left[ mol1 \right] \left[ mol2 \right] \tag{4}$$

The diffusion limited on-rate constant, kon, is about 109 M−1 s −1 (calculated with Equation 6, assuming that the two particles are reactive over their entire surface), and the molecule concentrations are about 1 nM for 1 molecule in 1 µm3 . This yields a rate of 10−9 Ms−1 , so one molecule reacts in 1 s. A similar result is obtained from a simulation of an association reaction (**[Figure 1D](#page-2-0)**). Here the reaction was carried out in a spherocylinder of 1.5µm in length and 1µm in width. Reaction times are distributed between 0 and 3 s, with the mean at 0.6 s.

A protein can stick to slower diffusing components in the cytoplasm, which can reduce its diffusion coefficient. An effective diffusion coefficient can then be calculated as follows [(Schavemaker et al., 2017)](#page-14-1):

$$D_{\rm eff} = f_{\rm free} \, D_{\rm free} + \left(1 - f_{\rm free}\right) \, D_{\rm bound} \tag{5}$$

Here, Deff is the effective diffusion coefficient, ffree is the fraction of the protein of interest that is free (unbound) at equilibrium, Dfree is the diffusion coefficient of free protein, and Dbound is the diffusion coefficient when it is bound. The results are shown in **[Figure 1E](#page-2-0)**.

### Diffusion Limited Reactions

Diffusion coefficients influence reaction rates when reactions are diffusion limited. The rate of an association reaction between two molecules depends on their concentrations and the onrate constant, kon, as shown in equation (4). When a reaction is diffusion limited every encounter between the two reactant molecules leads to reaction, so that the kon depends only on the diffusion coefficients of the two proteins. The diffusion-limited kon is given by equation (6) for two spherical proteins with a completely reactive surface area [(Schreiber et al., 2009)](#page-14-2):

$$k_{on,\ diff} = 4\pi (D1 + D2)(R1 + R2) \tag{6}$$

Here, D1 and D2 are the diffusion coefficients of proteins with radii R1 and R2. For proteins with a radius of 0.005µm, for which the diffusion coefficient is ∼10 µm2 /s in the E. coli cytoplasm and ∼100 µm2 /s in dilute solution, the kon,diff is ∼108 M−1 s −1 in the cytoplasm and ∼109 M−1 s −1 in dilute solution. Because the right side of the equation deals with single molecules and the left side with moles, the right side has units µm3 s −1 and the left side M−1 s −1 . You can convert µm3 /s into M−1 s −1 by dividing by 1015 to convert the volume, and then multiplying by Avogadro's number (6 × 1023). This equation is valid only for diffusion in 3D and as such it cannot be used for reactions involving membrane proteins. Most proteins are not reactive over their entire surface and more realistic diffusion limited kon values are 105 -106 M−1 s −1 [(Schlosshauer and Baker, 2004;](#page-14-3) [Schreiber et al., 2009)](#page-14-2). Having multiple binding sites on a protein or electrostatic interactions [(Schreiber and Fersht, 1993;](#page-14-4) Wallis et al., [1995;](#page-15-0) [Alsallaq and Zhou, 2008)](#page-13-1) can however increase the kon beyond 105 -106 M−1 s −1 . Hence, the value of the diffusion limit depends on the proteins involved. Note that we here deal with two definitions of diffusion-limited kon. One is the hard limit given by equation 6, which can only be broken by making motion non-diffusive, for instance by electrostatic attraction. In this definition a reaction is diffusion limited when the kon is according to equation 6 and the whole surface of the molecule is reactive, which is hardly ever (if ever) the case for molecules in a cell. In the second definition of the diffusion limit only part of the surface is reactive. In this case an increase of the rotational diffusion coefficient could cause the limit to be crossed. Here a

![](_page_2_Figure_2.jpeg)

<span id="page-2-0"></span>FIGURE 1 | Illustration of diffusion principles. (A) Three molecules, each in a different color, undergoing a random walk in two dimensions. Each trajectory consists of 400 steps, and beginning and end are indicated by colored spheres. All three molecules started at position (0,0). (B) Probability density in one dimension for the position of a particle after 1 s. Shown are densities for three different diffusion coefficients, which are computed with equation 2. (C) The mean distance of a molecule in time. Shown for three different diffusion coefficients. Computed from equation 3 with the number of dimensions set to one. (D) Simulation of biomolecular reaction times in a spherocylinder of 1.5µm in length and 1µm in width. In 1,000 separate simulations two particles were positioned randomly in the spherocylinder "cytoplasm" and allowed to diffuse with a diffusion coefficient of 10 µm2 /s and react with a *kon* of 109 M−1 s −1 . Simulations were performed in Smoldyn (Andrews et al., [2010)](#page-13-2). (E) The effective diffusion coefficient of a complex forming protein as a function of bound diffusion coefficient and free fraction. It was plotted using equation 5, with Dfree = 10 µm2 /s. Note that upon binding the free protein takes on the diffusion coefficient of the object it binds to. This means that the top right corner of the graph is somewhat inaccurate.

reaction is said to be diffusion limited when the coming together of proteins is the slowest step in the reaction, i.e., any necessary conformational changes are very fast. Complications arising from these two different definitions can be bypassed by considering diffusion limitation in the context of a cell. Which is what we do next.

In the cell, the diffusion limitation depends on other processes, because the two proteins may not be constantly present and reactive. These processes include protein synthesis, post-translational modifications, release of proteins from other complexes, transport over membranes, or the cycling through conformational states of one of the binding partners.

If two proteins, A and B, are reactive over their entire surface, they will form a complex as soon as they hit. If on the other hand they have small reactive patches on the surface, they will have to hit each other more often to form a complex. In both these cases, a higher collision rate leads to faster complex formation. When protein A cycles through two states however, of which only one is able to form the complex, the magnitude of the diffusion coefficient is less important: We let A spend an average of 10 s in the inactive state and 10 s in the active state, and the average time for A and B to find each other is 1 s. The inactive A is hit by B on average 10 times before it switches to the active state. When protein A finally does switch to the active state, B binds on average in 1 s. This gives a reaction time of 11 s. If B were to diffuse twice as fast this reaction time would only go down to 10.5 s, while if B were to diffuse twice as slow the reaction time would be 12 s. Thus, the diffusion coefficient of B is relatively inconsequential and the reaction is not diffusion limited. This argument only holds if the active period of A is significantly longer than the time for A and B to bump into one another. If the active period of A is 0.5 s instead of 10 s, B has less successful collisions with A, making the reaction-time sensitive to the diffusion coefficient of B, even though the reaction time will be much longer than the time A and B need to find each other. A real world example of the diminished importance of the cytoplasmic diffusion coefficient is the binding of the transcription inhibitor LacI to its DNA target site. In the search process for its proper binding site, LacI first binds the DNA nonspecifically and subsequently scans it, which takes ∼90% of the search time [(Li and Xie, 2011)](#page-13-3). If the LacI would diffuse much faster through the cytoplasm this could only reduce the total search time down to 90% of the actually measured time.

Another example where diffusion limitation depends on the context is the formation of gradients during catalysis or protein-protein interactions. Diffusion limitation can result in concentration gradients of reactants [(Berg and von Hippel,](#page-13-4) [1985)](#page-13-4) when a diffusion limited enzymatic reaction depletes its surroundings. Gradients do not form when the "enzyme" is occupied by its "substrate" (for example in a protein-protein interaction), albeit that the site where the enzyme is synthesized could in principle become the sink. The use of the gradient description depends on the biological context and could work well for the process of translation where association between molecules leads to a reaction [(Zhang et al., 2010;](#page-15-1) [Klumpp et al.,](#page-13-5) [2013)](#page-13-5), or when a membrane has the function of a sink (Schulz and Jorgensen, [2001)](#page-14-5).

### Diffusion Limitation Depends on the Intracellular Environment

How does the intracellular environment determine the diffusion coefficients? The intracellular environment is not a homogeneous medium with a single diffusion coefficient for a given protein; many factors may retard the diffusion of a protein in a crowded cell, increasing the fuzziness of diffusion limits (**[Figure 2](#page-4-0)**). Moreover, the thermodynamic non-ideality of the cytoplasm makes the diffusion coefficient not simply a sum of its contributors. We will discuss some of the most important contributors below.

The cell is highly crowded with macromolecules that provide steric barriers for a diffusing protein (**[Figure 2A](#page-4-0)**). The simplest systems in which to study the effect of crowding on diffusion are solutions of a single type of macromolecule at varying concentrations. Such experiments have shown a marked decrease in the diffusion coefficient with concentration (or volume fraction), with the fold changes getting bigger at higher concentrations [(Tokuyama et al., 2011)](#page-15-2). As for the magnitude of the decrease we will compare the relative diffusion coefficients (D/D0) at a volume fraction of 0.2, which is similar to what has been found for the E. coli cytoplasm [(Konopka et al.,](#page-13-6) [2009)](#page-13-6). In some cases data is expressed in g/L and we have converted the values by using the protein specific volume. From this we find that D/D0 is 0.3 for Barstar (10 kDa) (Nesmelova et al., [2002)](#page-14-6), 0.14 for SH3 (10 kDa) [(Rothe et al., 2016)](#page-14-7), 0.41 for Myoglobin (16 kDa) [(Nesmelova et al., 2002)](#page-14-6), 0.25 for Hemoglobin (Homo sapiens, 64 kDa) [(Keller et al., 1971)](#page-13-7), 0.14 for BSA (66 KDa) [(Nesmelova et al., 2002)](#page-14-6), and finally 0.1 for Hemoglobin (Lumbricus terrestris, 3700 kDa) [(Gros, 1978)](#page-13-8). This limited set of data shows a decrease in the relative diffusion coefficient with molecular weight, but also a rather large spread in relative diffusion coefficient at each molecular weight. This decrease in relative diffusion coefficient with molecular weight is also seen in the E. coli cytoplasm (see below). However, the relative diffusion coefficient is bigger in the in vitro systems (average of 0.2 or higher for proteins of 10–66 kDa) than in the E. coli cytoplasm (average <0.1 from 27–1,000 kDa). This difference may be related to a difference in the surface properties between proteins that are often used in in vitro experiments and those that predominate the cytoplasm. This could be a reduced binding affinity between the proteins, after all most proteins cannot be concentrated to the level needed for these in vitro diffusion measurements. These studies on single macromolecule solutions are simple but they have the limitation that you always change both the background macromolecules and the one you measure the diffusion coefficient of. This complicates interpretations of molecular weight vs. diffusion coefficient data. The diffusion coefficient of hemoglobin(CO), myoglobin(CN), BSA, and aldolase in a background of aldolase, BSA, ovalbumin, or ribonuclease also shows a bigger drop in diffusion coefficient for bigger proteins, but here also the trend is not perfect [(Muramatsu and Minton, 1988)](#page-14-8). This finding is contradicted by a study on the diffusion coefficients of rhodamine green, dextran, differently-sized DNAs, albumin, and differently-sized nanospheres in the presence of the artificial crowder Ficoll 70. Here, the fold drop in diffusion coefficient is similar for objects of varying sizes [(Dauty and Verkman, 2004)](#page-13-9). Likely, the use of a polymer crowder instead of a protein crowder induces different behavior. Concentrated polymers entangle and their effects are dictated by the monomer concentration and not their size, in contrast to strictly globular proteins. Also, the objects tested are of a rather different nature making interpretation difficult. That the nature of the macromolecule matters for its diffusion coefficient, and its response to crowding, is shown in a study on the intrinsically disordered protein α-synuclein. The diffusion coefficient (determined by NMR) of α-synuclein is slower than that of chymotrypsin inhibitor 2 under dilute conditions, but is faster in the presence of crowders (BSA, lysozyme, Ficoll 70, or PVP) [(Wang et al., 2012)](#page-15-3).

Another steric effect is the presence of immobile barriers that preclude the long distance movement of a molecule (**[Figure 2D](#page-4-0)**). For example, proteins in the periplasm experience confinement from the cytoplasmic and outer membrane. The nucleoid can become a barrier for larger, ribosome-sized, proteins or

![](_page_4_Figure_2.jpeg)

<span id="page-4-0"></span>different biopolymers induces spatial heterogeneity, leading to location-dependent diffusion coefficients of the tracer protein.

protein-aggregates that are too large to diffuse through the meshwork presented by the DNA (**[Figure 2E](#page-4-0)**) [(Bakshi et al.,](#page-13-10) [2012)](#page-13-10). In a model of LacI diffusion in the nucleoid it was concluded that DNA dynamics determines the motion of LacI to a large extent [(Chow and Skolnick, 2017)](#page-13-11). During an osmotic upshift, next to the increased crowding, invaginations provide temporal barriers for proteins [(Mika et al., 2010)](#page-14-9) until the cell recovers its volume and resumes growth. Small organic molecules experience much less reduction in their diffusion, which allows nutrients and compatible solutes to probe the entirety of the cell even under extreme conditions of osmotic stress and aid the recovery from osmotic stress [(Mika et al., 2010)](#page-14-9). Another example of immobile barriers that depend on the physiological state of the cell occurs during energy depletion where a decrease in diffusion occurs only for very large particles, i.e., viral nanoparticles, plasmids, and aggregates. Here, the cytoplasm of E. coli and yeast transits to a colloidal-glassy or gel-like state that sieves larger particles [(Parry et al., 2014;](#page-14-10) [Joyner et al., 2016;](#page-13-12) [Munder et al., 2016)](#page-14-11). In eukaryotes membrane proteins appear to encounter barriers to diffusion such as cytoskeletal elements that temporarily confine them [(Saxton and Jacobson, 1997)](#page-14-12). The consequences of such confinement for (apparent) diffusion coefficients has been studied theoretically [(Saxton, 1995)](#page-14-13). It is unclear how well these findings can be transferred to prokaryotes but it appears that at least some outer membrane proteins in E. coli are confined (**[Table 1](#page-5-0)**).

In addition, non-covalent chemical interactions with the surrounding proteins change the diffusion of a protein (**[Figure 2C](#page-4-0)**); these interactions can be hydrophobic, van der Waals, electrostatic or hydrogen-bonding, and thus strongly depend on the properties of both the protein and the proteome. Again we look first at the simplest studies of diffusion with single types of protein at different concentrations. Lysozyme changes its affinity for dimerization depending on the pH. When diffusion measurements are performed at a pH that favors the dimer the diffusion coefficient drops faster with lysozyme volume fraction than at a pH that favors the monomer (Nesmelova and Fedotov, [1998)](#page-14-14). We see similar behavior when more than one protein is in solution. For example, the diffusion coefficients of BSA and aldolase drop much faster with background protein concentration if this background protein is ribonuclease rather than aldolase, BSA, or ovalbumin [(Muramatsu and Minton,](#page-14-8) [1988)](#page-14-8). The influence of interactions is seen also in a diffusion study of a mixture of SH3 and BSA at varying concentrations. Whereas, these proteins when studied independently follow the Stokes-Einstein relation they fail to do so when in a mixture [(Rothe et al., 2016)](#page-14-7). We are unaware of in vitro studies that use more complex mixtures for studying diffusion, with the exception of diffusion measurements on proteins in cell lysates [(Wang et al.,](#page-15-4) [2010)](#page-15-4). The reason for this is presumably that complex mixtures of proteins cannot be concentrated to cellular levels. And even if you could do so you would probably not be able to avoid potential aggregation or phase separations. Computational studies can be carried out on complex mixtures and have the advantage that you can see exactly what is going on. For example, it was possible to study the diffusion of proteins in a simulated E. coli cytoplasm that contained the 50 most abundant macromolecules at appropriate concentrations. However, there was no attempt to study the effect of changes in total macromolecule concentration [(McGuffee and Elcock, 2010)](#page-14-15). A limit of such computational

#### <span id="page-5-0"></span>TABLE 1 | Overview of experimentally determined diffusion coefficients.

| Molecule                                     | Organism                 | Diffusion coefficient<br>(D; µm2<br>/s) | Comments                                                                                                         | References                                         |
|----------------------------------------------|--------------------------|-----------------------------------------|------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
| NBD-glucose                                  | Escherichia coli         | 50                                      | 0.423 kDa                                                                                                        | Mika et al., 2010                                  |
| GFP                                          | Dilute solution          | 87                                      | 27 kDa                                                                                                           | Potma et al., 2001                                 |
| GFP                                          | Dictyostelium discoideum | 24                                      | Cytoplasm, 27 kDa                                                                                                | Potma et al., 2001                                 |
| GFP                                          | Mus musculus             | 27                                      | Fibroblast cytoplasm, 27 kDa                                                                                     | Swaminathan et al.,<br>1997                        |
| GFP                                          | Escherichia coli         | 3-14                                    | Cytoplasm, 27 kDa                                                                                                | Konopka et al., 2009;<br>Mika and Poolman,<br>2011 |
| GFP                                          | Lactococcus lactis       | 7                                       | Cytoplasm, 27 kDa                                                                                                | Mika et al., 2014                                  |
| GFP                                          | Bacillus subtilis        | >1                                      | Cytoplasm, germinated spores, 27 kDa                                                                             | Cowan et al., 2003                                 |
| GFP                                          | Bacillus subtilis        | ∼0.0001                                 | Spore cytoplasm, 27 kDa                                                                                          | Cowan et al., 2003                                 |
| GFP                                          | Caulobacter crescentus   | 8                                       | Cytoplasm, 27 kDa                                                                                                | Llopis et al., 2012                                |
| GFP                                          | Haloferax volcanii       | 5.5                                     | Cytoplasm, 27 kDa                                                                                                | Schavemaker et al.,<br>2017                        |
| mCherry                                      | Pseudomonas aeruginosa   | 4                                       | Cytoplasm, 27 kDa                                                                                                | Guillon et al., 2013                               |
| TorA-GFP                                     | Escherichia coli         | 9                                       | Cytoplasm, 30 kDa, in 1tatABCDE strain                                                                           | Mullineaux et al., 2006                            |
| PtsH-YFP                                     | Escherichia coli         | 3.8                                     | Cytoplasm, 36 kDa, some degradation of the protein                                                               | Kumar et al., 2010                                 |
| CheY-GFP                                     | Escherichia coli         | 4.6                                     | Cytoplasm, 41 kDa                                                                                                | Cluzel et al., 2000                                |
| Crr-YFP                                      | Escherichia coli         | 2.0                                     | Cytoplasm, 45 kDa, some degradation of the protein                                                               | Kumar et al., 2010                                 |
| NlpAnoLB-GFP                                 | Escherichia coli         | 2.7                                     | Cytoplasm, 55 kDa                                                                                                | Nenninger et al., 2010                             |
| TorA-GFP2                                    | Escherichia coli         | 8.3                                     | Cytoplasm, 57 kDa, 2x GFP in tandem                                                                              | Nenninger et al., 2010                             |
| AmiAnoSP-GFP                                 | Escherichia coli         | 7.1                                     | Cytoplasm, 58 kDa                                                                                                | Nenninger et al., 2010                             |
| CFP-CheW-YFP                                 | Escherichia coli         | 1.5                                     | Cytoplasm, 72 kDa, some degradation of the protein                                                               | Kumar et al., 2010                                 |
| MBP-GFP                                      | Escherichia coli         | 2.5                                     | Cytoplasm, 72 kDa                                                                                                | Elowitz et al., 1999                               |
| torA-GFP3                                    | Escherichia coli         | 6.3                                     | Cytoplasm, 84 kDa, 3x GFP in tandem                                                                              | Nenninger et al., 2010                             |
| CFP-CheR-YFP                                 | Escherichia coli         | 1.7                                     | Cytoplasm, 86 kDa, some degradation of the protein                                                               | Kumar et al., 2010                                 |
| DnaK-YFP                                     | Escherichia coli         | 0.67                                    | Cytoplasm, 96 kDa, some degradation of the protein                                                               | Kumar et al., 2010                                 |
| torA-GFP4                                    | Escherichia coli         | 5.5                                     | Cytoplasm, 111 kDa, 4x GFP in tandem                                                                             | Nenninger et al., 2010                             |
| torA-GFP5                                    | Escherichia coli         | 2.8                                     | Cytoplasm, 138 kDa, 5x GFP in tandem                                                                             | Nenninger et al., 2010                             |
| HtpG-YFP                                     | Escherichia coli         | 1.7                                     | Cytoplasm, dimer of 198 kDa                                                                                      | Kumar et al., 2010                                 |
| CFP-CheA-YFP                                 | Escherichia coli         | 0.44                                    | Cytoplasm, 250 kDa, some degradation of the protein                                                              | Kumar et al., 2010                                 |
| LacI-Venus                                   | Escherichia coli         | 3                                       | Cytoplasm, tetramer of ∼260 kDa, freely diffusing, when<br>DNA binding is included D = 0.4 µm2<br>/s             | Elf et al., 2007                                   |
| β-galactosidase                              | Dilute solution          | 31                                      | Tetramer of 466 kDa                                                                                              | Hahn and Aragon,<br>2006                           |
| β-galactosidase-GFP                          | Escherichia coli         | 0.7                                     | Cytoplasm, tetramer of 582 kDa                                                                                   | Mika et al., 2010                                  |
| β-galactosidase-GFP                          | Lactococcus lactis       | 0.8                                     | Cytoplasm, tetramer of 582 kDa                                                                                   | Mika et al., 2014                                  |
| Ribosome                                     | Escherichia coli         | 0.04                                    | Cytoplasm, fully active, includes all states of translation                                                      | Bakshi et al., 2012                                |
| Ribosome (free, 30S)                         | Escherichia coli         | 0.6                                     | Cytoplasm, freely diffusing, 1 MDa                                                                               | Bakshi et al., 2012                                |
| Ribosome (bound)                             | Escherichia coli         | 0.055                                   | Cytoplasm, bound fraction                                                                                        | Sanamrad et al., 2014                              |
| Ribosome (free, 30S or 50S) Escherichia coli |                          | 0.4                                     | Cytoplasm, free fraction                                                                                         | Sanamrad et al., 2014                              |
| Ribosome (bound)                             | Caulobacter crescentus   | 0.0002–<0.0011                          | Cytoplasm, obtained from model that includes a bound<br>and free fraction                                        | Llopis et al., 2012                                |
| Ribosome (free, 50S)                         | Caulobacter crescentus   | 0.018–0.042                             | Cytoplasm, obtained from model that includes a bound<br>and free fraction                                        | Llopis et al., 2012                                |
| Ribosome (free, 50S)                         | Caulobacter crescentus   | 0.36–0.39                               | Cytoplasm, after cells were treated with rifampicin or<br>kasugamycin                                            | Llopis et al., 2012                                |
| Carboxysome                                  | Synechococcus elongatus  | 0.000046                                | Cytoplasm, constrained movement; consists of ∼5000<br>monomers of shell protein and ∼2000 monomers of<br>rubisco | Savage et al., 2010                                |
| mRNA                                         | Escherichia coli         | 0.001-0.03                              | Cytoplasm, diffusion is anomalous, mRNA in complex<br>with many copies of MS2-GFP                                | Golding and Cox,<br>2004, 2006                     |

*(Continued)*

#### TABLE 1 | Continued

| Molecule           | Organism               | Diffusion coefficient<br>(D; µm2<br>/s) | Comments                                                                                                       | References                                                             |
|--------------------|------------------------|-----------------------------------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------|
| DNA                | Escherichia coli       | 0.0004-0.0007                           | Chromosomal loci, apparent D as DNA doesn't move<br>freely                                                     | Reyes-Lamothe et al.,<br>2008                                          |
| PvdS-eYFP          | Pseudomonas aeruginosa | 1                                       | Cytoplasm, PvdS is a sigma factor, 48 kDa                                                                      | Guillon et al., 2013                                                   |
| PvdA-eYFP          | Pseudomonas aeruginosa | 0.5                                     | Cytoplasm, 76 kDa                                                                                              | Guillon et al., 2013                                                   |
| PvdQ-mCherry       | Pseudomonas aeruginosa | 0.2                                     | Periplasm, 111 kDa                                                                                             | Guillon et al., 2013                                                   |
| FpvF-mCherry       | Pseudomonas aeruginosa | 0.2                                     | Periplasm, 59 kDa                                                                                              | Guillon et al., 2013                                                   |
| GFP                | Escherichia coli       | 2.6                                     | Periplasm; TorA signal sequence removed upon export<br>to periplasm                                            | Mullineaux et al., 2006                                                |
| MotB-GFP           | Escherichia coli       | 0.0075-0.0088                           | Plasma membrane, freely diffusing, dimer                                                                       | Leake et al., 2006                                                     |
| TatA-GFP           | Escherichia coli       | 0.13                                    | Plasma membrane                                                                                                | Mullineaux et al., 2006                                                |
| Tar(1-397)-YFP     | Escherichia coli       | 0.22                                    | Plasma membrane, 4 transmembrane helices                                                                       | Kumar et al., 2010                                                     |
| Tsr(1-218)-YFP     | Escherichia coli       | 0.18                                    | Plasma membrane, 4 transmembrane helices                                                                       | Kumar et al., 2010                                                     |
| LacY-YFP           | Escherichia coli       | 0.027                                   | Plasma membrane, 12 transmembrane helices                                                                      | Kumar et al., 2010                                                     |
| MtlA-YFP           | Escherichia coli       | 0.028                                   | Plasma membrane, 12 transmembrane helices                                                                      | Kumar et al., 2010                                                     |
| Tar-YFP            | Escherichia coli       | 0.017                                   | Plasma membrane, 12 transmembrane helices                                                                      | Kumar et al., 2010                                                     |
| TetA-YFP           | Escherichia coli       | 0.09                                    | Plasma membrane, 12 transmembrane helices                                                                      | Chow et al., 2012, see<br>also discussion in ref.<br>Mika et al., 2014 |
| NagE-YFP           | Escherichia coli       | 0.020                                   | Plasma membrane, 16 transmembrane helices                                                                      | Kumar et al., 2010                                                     |
| FliG-GFP           | Escherichia coli       | 0.0049                                  | Attached to flagellum basal body                                                                               | Fukuoka et al., 2007                                                   |
| BcaP-GFP           | Lactococcus lactis     | 0.02                                    | Plasma membrane, 12 transmembrane helices                                                                      | Mika et al., 2014                                                      |
| LacS1IIA-GFP       | Lactococcus lactis     | 0.02                                    | Plasma membrane, 12 transmembrane helices                                                                      | Mika et al., 2014                                                      |
| PleC-eYFP          | Caulobacter crescentus | 0.012                                   | Plasma membrane, freely diffusing, 4 transmembrane<br>helices                                                  | Deich et al., 2004                                                     |
| Lipopolysaccharide | Salmonella typhimurium | 0.02                                    | Outer membrane                                                                                                 | Schindler et al., 1980                                                 |
| BtuB               | Escherichia coli       | 0.05-0.10                               | Outer membrane, 22-stranded β-barrel, when<br>disconnected from its binding partner TonB D = 0.27<br>µm2<br>/s | Spector et al., 2010                                                   |
| OmpF               | Escherichia coli       | 0.006                                   | Outer membrane, trimer, 16-stranded β-barrel, diffusion<br>is restricted to an area with a diameter of 100 nm  | Spector et al., 2010                                                   |
| LamB (λ-receptor)  | Escherichia coli       | 0.15                                    | Outer membrane, LamB appears to be tethered and<br>diffusion is restricted to area with 50 nm diameter         | Oddershede et al.,<br>2002                                             |

studies is the limited time window over which diffusion can be monitored. That interactions between macromolecules affect diffusion behavior in vivo is shown in a study of differently charged versions of GFP in the cytoplasm of three prokaryotes, which revealed that positive GFPs can diffuse up to 100-fold slower due to their interactions with ribosomes (Schavemaker et al., [2017)](#page-14-1).

Hydrodynamic interactions, which are caused by the flow field of other diffusing particles, have been proposed to have strong long-range effects that slow down proteins (**[Figure 2B](#page-4-0)**), and such interactions are needed to simulate the lower diffusion constants of GFP in the cell [(Ando and Skolnick, 2010)](#page-13-27). Although this phenomenon is well studied in colloidal physics, hydrodynamic interactions of proteins inside cells have only been studied by molecular dynamics simulations.

The above-mentioned associative and repulsive effects act on all the biopolymers in the cell, and likely induce spatial heterogeneity in cells (**[Figure 2F](#page-4-0)**) [(Spitzer and Poolman, 2009,](#page-14-28) [2013;](#page-14-29) [Yu et al., 2016;](#page-15-5) [van den Berg et al., 2017)](#page-15-6). Spatial heterogeneity is most pronounced where proteins phase separate from the cytoplasmic pool of proteins; these effects are most often associated with eukaryotic cells, but microdomains also occur in bacteria, for example as inclusion bodies, the nucleoid, or the cell-polarity inducing assemblies in Caulobacter crescentus [(Perez et al., 2017)](#page-14-30). Such domains likely alter the mobility of macromolecules in the cell. The rate of assembly of phase separations may be diffusion limited and can give rise to spatial pattern formation in complex reaction-diffusion networks when competing binding partners are present [(Saha et al., 2016)](#page-14-31). Stress conditions may increase spatial heterogeneity, in particular in energy-starved cells. It has been suggested that ATP at physiological concentrations (5–10 mM) acts as a biological hydrotope in cells, preventing phase separation of proteins (Patel et al., [2017)](#page-14-32). When ATP levels decreased, the disordered proteins increasingly self-associate, giving rise to regions that are more crowded than others are. As we proposed previously, it is likely that small molecules mediate the protein organization, also in cells [(Spitzer and Poolman, 2005,](#page-14-33) [2009;](#page-14-28) [van den Berg et al., 2017)](#page-15-6). This implies that in the cell there may be physiology-dependent regions with lowered diffusion coefficients for a given protein, and here reactions could become rate limiting.

In summary, the diffusion coefficient and the occurrence of diffusion-limited reactions depend on the location of the protein in the cell, the physiological state of the cell itself, and the size and chemical nature of the diffusing species.

### What Are the Diffusion Coefficients of Proteins in Cells?

A large set of diffusion coefficients has been determined in prokaryotes, of which we provide a comprehensive overview in **[Table 1](#page-5-0)**. For comparison we include data on a small molecule in Escherichia coli, proteins in some eukaryotes, and proteins in dilute solution. Diffusion coefficients have been measured for proteins in the cytoplasm, plasma membrane, periplasm, and outer membrane. Most diffusion coefficients have been determined for proteins in E. coli, but a decent amount of data is also available for the bacteria Caulobacter crescentus, Pseudomonas aeruginosa, and Lactococcus lactis. Most in vivo diffusion coefficients have been determined with fluorescence recovery after photo-bleaching (FRAP), some are determined by single particle tracking (SPT) or fluorescence correlation spectroscopy (FCS). For a short description of these techniques we refer to [(Mika and Poolman, 2011)](#page-14-18). For in vitro determinations of diffusion coefficients you can use NMR (Wang et al., [2012)](#page-15-3), analytical ultracentrifugation [(Tyn and Gusek, 1990)](#page-15-7), the rate of transfer of protein over a porous membrane, or dynamic light scattering [(van Holde et al., 2006)](#page-15-8).

The values represented in **[Table 1](#page-5-0)** are means or medians over populations of cells. Typically, there is considerable variation in the diffusion coefficient between cells [(Konopka et al., 2009;](#page-13-6) [Mika et al., 2014)](#page-14-19). We illustrate this in **[Figure 3A](#page-8-0)** where we show histograms of the diffusion coefficients of GFP, a large enzyme complex (β-galactosidase-GFP), and a membrane protein (LacS1IIA-GFP) in L. lactis. In addition, not all diffusive processes can be described by a single diffusion coefficient. In some cases, the molecules are confined [(Fukuoka et al., 2007)](#page-13-25) or exhibit anomalous diffusion [(Golding and Cox, 2006)](#page-13-22).

The diffusion coefficients show differences for the same molecule (GFP or mCherry) in the cytoplasms of C. crescentus (8 µm2 /s), P. aeruginosa (4 µm2 /s), L. lactis (7 µm2 /s), and the archaeon Hfx. volcanii (5.5 µm2 /s), which all fall within the range that is measured for E. coli (3–14 µm2 /s). Hence, it is not clear whether these differences are real and could be due to measurement error, the method used, or different growth and measurement conditions. E. coli and L. lactis give roughly equivalent diffusion coefficient of β-galactosidase-GFP, as well as for membrane proteins with 12 transmembrane helices. There are differences between E. coli and C. crescentus, with diffusion coefficients of 0.04 µm2 /s vs. 0.0002–<0.0011 µm2 /s for ribosome diffusion, respectively, and 0.18–0.22 µm2 /s vs. 0.012 µm2 /s for membrane proteins with 4 transmembrane helices. It is currently not clear what causes the differences in mobility in E. coli and C. crescentus.

It is not just the isolated values listed in **[Table 1](#page-5-0)** that matter, we also need to consider how diffusion values vary in different contexts (e.g., physicochemical state of the cell) and with protein properties (**[Figures 3B–G](#page-8-0)**). Protein diffusion coefficients go down with molecular weight of the protein, both in dilute solution and in the E. coli cytoplasm (**[Figure 3B](#page-8-0)**). This is also seen for membrane proteins in relation to their (membraneembedded) radius in giant unilamellar vesicles (GUVs) and the E. coli plasma membrane (**[Figure 3C](#page-8-0)**). Increasing the salt concentration of the outside medium reduces the water content of E. coli cells and increases the volume fraction that is excluded by macromolecules. When cells are allowed to adapt to the increased salt concentration, the diffusion coefficient drops less fast with excluded volume fraction than when this is done swiftly (shocked) (**[Figure 3D](#page-8-0)**). The drop in diffusion coefficient with osmotic shock is less for L. lactis than for E. coli (**[Figure 3E](#page-8-0)**), while the drop in diffusion coefficient with relative cell volume (after shock) is much larger in L. lactis than in E. coli (**[Figure 3F](#page-8-0)**). The reason for this difference between L. lactis and E. coli is not clear but could have something to do with different levels of crowding or different co-solvents in the cytoplasm. Finally, the diffusion coefficient of different surface-modified variants of GFP depends on their net charge, with positive proteins diffusing up to a 100-fold slower. This effect is strongest in E. coli but is also present in L. lactis and the archaeon Haloferax volcanii (**[Figure 3G](#page-8-0)**). L. lactis and H. volcanii have a (much) higher ionic strength than E. coli, which explains the smaller impact of a cationic surface on the slowing of diffusion in these microbes. Thus, "electrostatic lubrication" seems important in keeping the macromolecules moving and may have been a driver in the evolution of the cellular proteomes. In summary, the measured diffusion coefficients depend strongly on the protein size, surface chemistry, as well as the intracellular environment.

### Examples of Diffusion Limitation in Prokaryotes

Lowering the diffusion coefficient of a protein indefinitely would cause any reaction to become diffusion limited. Therefore, a study of diffusion coefficients and diffusion limitation of processes is pertinent. There are not many examples where the importance of the diffusion coefficient actually has been demonstrated. Below, we summarize cases where diffusion limitation appears to occur. Some more discussion of diffusion processes in prokaryotes can be found in [(Soh et al., 2010)](#page-14-34).

#### The On-Rate Constant of Barnase-Barstar Goes Beyond the Diffusion Limit

The diffusion limited kon starts at 105 -106 M−1 s −1 (Schlosshauer and Baker, [2004;](#page-14-3) [Alsallaq and Zhou, 2008)](#page-13-1), but protein pairs such as Barnase-Barstar from Bacillus amyloliquefaciens manage to have a kon of 108 -1010 M−1 s −1 [(Schreiber and Fersht,](#page-14-4) [1993;](#page-14-4) [Wallis et al., 1995;](#page-15-0) [Alsallaq and Zhou, 2008)](#page-13-1). Barnase is an extracellular ribonuclease that is bound by Barstar in the cytoplasm to prevent damage of endogenous RNA [(Buckle et al.,](#page-13-28) [1994)](#page-13-28). The fact that the reaction is electrostatically steered, and that the on-rate constant is two orders of magnitude higher than the non-electrostatic diffusion limit, suggests that the diffusion

![](_page_8_Figure_2.jpeg)

<span id="page-8-0"></span>FIGURE 3 | Systematic variation of diffusion coefficients with protein and environment properties. (A) Variation of diffusion coefficient within a population of cells for the proteins GFP and β*-*galactosidase-GFP (tetramer) in the cytoplasm, and LacS1IIA-GFP in the membrane of *Lactococcus lactis* [(Mika et al., 2014)](#page-14-19). (B) The dependence of diffusion coefficient on molecular weight in dilute solution [(Tyn and Gusek, 1990)](#page-15-7) and the *Escherichia coli* cytoplasm [(Elowitz et al., 1999;](#page-13-18) van den Bogaart et al., [2007;](#page-15-9) [Konopka et al., 2009;](#page-13-6) [Kumar et al., 2010;](#page-13-16) [Mika et al., 2010;](#page-14-9) [Nenninger et al., 2010;](#page-14-21) [Bakshi et al., 2012)](#page-13-10). (C) The dependence of diffusion coefficient on radius of the membrane spanning part of membrane proteins in giant unilamellar vesicles (GUVs) [(Ramadurai et al., 2009)](#page-14-35) and in the *Escherichia coli* plasma membrane [(Kumar et al., 2010)](#page-13-16). The radii for the proteins studied in the *E. coli* membrane are calculated from the number of transmembrane helices (Kumar et al., [2010)](#page-13-16) and the radius of a single helix peptide reported in [(Ramadurai et al., 2009)](#page-14-35). (D) The dependence of diffusion coefficient of cytoplasmic GFP on excluded volume fraction in adapted and shocked *Escherichia coli* cells [(Konopka et al., 2009)](#page-13-6). (E) The dependence of the diffusion coefficient of cytoplasmic GFP on medium osmolality after osmotic shock for *Escherichia coli* and *Lactococcus lactis* [(Konopka et al., 2009;](#page-13-6) [Mika et al., 2014)](#page-14-19). The growth medium had the same osmolality as the first points on the graph. (F) The dependence of the diffusion coefficient of cytoplasmic GFP on the relative cell volume after osmotic shock in *Escherichia coli* and *L. lactis* [(Mika et al., 2014)](#page-14-19). (G) The dependence of the cytoplasmic diffusion coefficient of GFP variants on their net charge in *Escherichia coli*, *Lactococcus lactis* and *Haloferax volcanii* [(Schavemaker et al., 2017)](#page-14-1). There is no data for −30 GFP in *L. lactis*.

coefficient is important for this reaction. (Note that we use the second definition of diffusion limitation as outlined in section Diffusion Limited Reactions.) Another protein pair with a very high on-rate constant is ColicinE9-Im9. ColicinE9 is a secreted toxin with DNase activity. Again, its binding partner, Im9, is used to prevent damage in the cytoplasm where ColicinE9 is made [(Wallis et al., 1995)](#page-15-0). Note that the increase in on-rate constant could be there to make the complex bind more tightly rather than increase the on-rate per se. A direct determination of diffusion limitation has not been carried out. As a final qualifier we add that the kon measurements were carried out on dilute samples and it isn't clear how well these results transfer to the in vivo (crowded) situation.

There are several other bacterial proteins that form complexes with high kon values, although this is not always demonstrated with the physiological binding partner: SecB from E. coli was shown to interact with BPTI (Bovine pancreatic trypsin inhibitor) with a kon of 5 × 109 M−1 s −1 [(Fekkes et al.,](#page-13-29) [1995)](#page-13-29), and the chaperone complex GroEL interacts with various proteins with a high kon values, including the unfolded state of barnase with a kon of 0.35–1.8 × 108 M−1 s −1 (Gray and Fersht, [1993;](#page-13-30) [Perrett et al., 1997)](#page-14-36), MBP (Maltose binding protein) with a kon of 0.9–7.0 × 107 M−1 s −1 [(Sparrer et al.,](#page-14-37) [1996)](#page-14-37), and DHFR with a kon of 3 × 107 M−1 s −1 (Clark and Frieden, [1997)](#page-13-31). It is however not clear whether these GroEL interactions are really diffusion limited because the unfolded proteins provide many more interaction opportunities than folded proteins, so the limit of 105 -106 M−1 s −1 may not apply.

#### Translation and Cell Growth Rate Are Limited by Charged tRNA Availability

Protein production could limit the growth rate and is set by the number of ribosomes, how fast they can start and end the production of one protein, and how fast they can elongate the proteins. In individual cases protein production can be limited by ribosome binding site strength rather than elongation rate. Elongation consists of the arrival of ternary complex, a complex that consists of amino acyl-tRNA, EF-Tu and GTP, and its processing by the ribosome. Using a computational model of the translation process it was found that if many ribosomes (100) are synthesizing the same protein and thus using the same amino acids, the rate per codon was decreased because of diffusion limitation. The effect was exacerbated when the diffusion coefficient was decreased after simulating an osmotic shock [(Zhang et al., 2010)](#page-15-1). It is not clear whether this diffusion limitation is present at actual cellular conditions and amino acid sequences.

In another study [(Klumpp et al., 2013)](#page-13-5) the rate of translation was also found to be diffusion limited. In the calculations, Michaelis-Menten kinetics was assumed for amino acid incorporation. The KM was calculated under the assumption that the reaction is diffusion limited. They estimated a diffusion coefficient of 1 µm2 /s for the ternary complex, from which they determine the kon for binding of ternary complex to ribosome to be 107 M−1 s −1 . The rate of going from the ternary complex being bound to the ribosome to amino acid chain elongation, kelong , was set at 30 s−1 . From this they calculated that the diffusion limited KM is 3µM, which compares to the concentrations of tRNA in E. coli, 3–30µM. The finding that the concentrations of tRNA are equal or higher than the diffusion limited KM is taken as evidence that the process operates at diffusion limited rate. The estimate of the diffusion limited kon is made on the condition that Equation 6 is valid, which assumes that the molecules that react can have any orientation upon collision and react immediately. This is unlikely to be the case. Diffusion limited kon's are also not necessarily single values as electrostatic interactions may steer the interaction and make the reaction faster. Next, they made a model that takes into account allocation of resources to different parts of the proteome. The translation speed is limited by the association rate of the ternary complex to the ribosome, which depends on both kon and concentration. Allocating resources to increasing the concentration of ternary complex will limit the resources that can be put into ribosome production. The cell growth rate is a function of both translation speed and ribosome concentration, and thus cell growth rate and allocation of resources are influenced by the diffusion coefficient of the ternary complex.

#### The Combination of Cell Size and Protein Concentration in Prokaryotic and Eukaryotic Cells Is Optimized to Facilitate Rapid Diffusion

Say you hold the number of proteins in an E. coli cell constant but would decrease cell size, then the distances that need to be overcome by diffusion are smaller, but the crowding increase leads to slower diffusion. If you make the cell bigger, the distances become larger but diffusion becomes faster. This scenario has been turned into a quantitative model, which shows that for prokaryotes the cell diameter is predicted to be 1.1µm, and for eukaryotes 15.7µm at the smallest characteristic diffusion times [(Soh et al., 2013)](#page-14-38). It is claimed that these diameters are comparable to the typical sizes of the prokaryotic and eukaryotic cells, indicating that the combination of cell size and macromolecule concentration is optimized for rapid diffusion, and that there are diffusion-limited processes in these cells. This prediction of cell size depends on the number of proteins in these cells, which is 3 × 106 for prokaryotes and 8 × 109 for eukaryotes. In the study, it is mentioned that the model provides an argument for determining what the sizes of prokaryotic and eukaryotic cells should be, yet no argument is provided that states why the number of proteins ought to be 3 × 106 and 8 × 109 . Furthermore, the characteristic distance that diffusion needs to bridge is taken to be the size of the cell. For many reactions the targets are probably much closer.

### Differences in Diffusion Coefficients Leading to Functional Differences

We discussed several cases where diffusion could limit rates of other processes in the cell. These consequences of the diffusion coefficients are essentially efficiency improvements; they do not arbitrate on the existence of phenomena. Here we will give two examples in which diffusion makes a functional difference, which are phenomena that would not exist were it not for certain diffusion coefficients.

#### The Min System Oscillation in E. coli

Cell division in E. coli creates two equal-sized daughter cells. A key protein in cell division is FtsZ, which forms a ring in the middle of the cell that helps to pull the cell envelope inward. The position of the FtsZ ring is partially determined by the Min system [(Loose et al., 2011)](#page-13-32). The Min system consists of the proteins MinC, MinD, and MinE. MinC inhibits FtsZ ring formation and does so only when bound to MinD. MinD and E form an oscillator that moves MinC, D, and E from one cell pole (bound to the membrane) to the other with a periodicity of about a minute. Because of this oscillator, MinC spends the least time in the mid cell region so that the FtsZ ring can form. An important feature necessary to create oscillations in space is the fact that when MinD is membrane bound, it has a lower diffusion coefficient than when it is free in solution to move to the other cell pole. Hence, diffusion coefficients determine whether the spatiotemporal oscillation can exist.

#### Stable Cytoplasmic Protein Gradients in Small Cells

A group of proteins can spread within seconds through a cell of several micrometers in length. Because of this, it is not likely that stable protein gradients can form over the length of the cell. However, it has been shown theoretically that protein gradients can form under special circumstances [(Lipkow and Odde, 2008)](#page-13-33). Consider three proteins in a cell: a kinase at one of the cell poles, a phosphatase throughout the cytoplasm, and a substrate protein that can cycle between a phosphorylated and unphosphorylated state. For the system to be able to form a gradient of the substrate protein, the diffusion coefficient of its two states must be different. In a 5µm long cell, with a kinase rate constant of 10 µm/s (the system is one-dimensional hence the m rather than m3 ), a phosphatase rate constant of 1 s−1 , and diffusion coefficients of 0.3 µm2 /s and 10 µm2 /s for phosphorylated and unphosphorylated forms yields a 10 fold concentration gradient of the substrate protein over the length of the cell. Again, the difference in diffusion coefficients allows the phenomenon to exist.

### NEW HORIZONS AND OUTSTANDING QUESTIONS

Tremendous progress in the determination and understanding of diffusion in a select group of prokaryotes has been made in the last decades. This research has led to the emergence of novel questions, which would lead to improved understanding of the role and importance of diffusion coefficients. In this second part of the review, we will summarize these outstanding questions.

### Consequences of Electrostatic Steering and Ionic Strength on Diffusion Limitation

Earlier we presented the case of the barnase-barstar complex formation. The diffusion limitation that this reaction labors under has been stretched by electrostatic interactions. Yet it is well-known that the on-rate of this particular electrostatic interaction, and others, diminishes with increased ionic strength [(Stone et al., 1989;](#page-14-39) [Schreiber and Fersht, 1993;](#page-14-4) [Wallis et al.,](#page-15-0) [1995)](#page-15-0). This means that organisms with relatively low internal ion concentrations, such as E. coli [(Shabala et al., 2009)](#page-14-40), are less affected by diffusion limitation than organisms with high internal ion concentrations, such as Haloferax volcanii (Pérez-Fillol and Rodriguez-Vallera, [1986)](#page-14-41). Does this mean that organisms such as Hfx. volcanii are unable to use toxin-antitoxin systems like barnase-barstar? How does this affect transcription factor binding to DNA, or the assembly of ribosomes?

### The Effect of Temperature on Diffusion Coefficients

All prokaryotes for which protein diffusion coefficients are known function in a small range of temperatures. How does the diffusion coefficient change if you go from 0 to 100◦C ? For proteins in dilute solution we can get an estimate from the Stokes-Einstein equation:

$$D = \frac{k_B}{6\pi R} \times \frac{T}{\eta(T)}\tag{7}$$

Here D is the diffusion coefficient, kB is the Boltzmann constant, R is the Stokes radius of the protein, T is the absolute temperature, and η(T) is the viscosity at temperature T. We want to know how D changes from 0 to 100◦C, and thus need to consider only T η(T) . For 0◦C we fill in T = 273 K and η(273) = 1.8 × 10−3 kg s−1m−1 , for 100◦C we fill in T = 373 K and η(373) = 0.28 × 10−3 kg s−1m−1 . This yields a ∼9 fold faster diffusion coefficient at 100 ◦C. In this calculation we used the viscosity of water. It is unlikely that the Stokes-Einstein equation holds for proteins in the cytoplasm. Firstly, the viscosity is different and not uniform in the cytoplasm, and secondly, and perhaps more importantly, diffusion in cells is probably more affected by excluded volume than by viscosity. It has also been shown that the Stokes-Einstein equation does not hold in the cytoplasm for the relation between diffusion coefficient and Stokes radius [(Mika and Poolman, 2011)](#page-14-18). Nonetheless, the impact of temperature on the diffusion coefficient in cells still needs to be experimentally tested. If there is an increase in diffusion coefficient with temperature, which seems likely, we can make the (conditional) prediction that cells at higher temperatures could have higher cytoplasmic concentrations of macromolecules before essential processes get diffusion limited.

### Direct Measurements of Diffusion Limitation

All examples of diffusion limitation discussed above are based on indirect observations, and rely heavily on modeling (parameters). It would be helpful to have a method for directly determining the diffusion limitation of various processes. That is to vary the diffusion coefficient of one of the actors in the process and then observing whether the rate of the process changes. This is difficult to do because changing the diffusion coefficient can also change other aspects of the cell. Take the example of an osmotic shock which indeed changes the diffusion coefficient [(Konopka et al.,](#page-13-6) [2009;](#page-13-6) [Mika et al., 2014)](#page-14-19), but firstly it does so for all big molecules, secondly it increases ion concentrations of the cytoplasm, and thirdly it increases the excluded volume and therefore affects rates and equilibria of all kinds of processes.

#### A Note on the Use of d= √ 2nDt

This equation indicates the distance over which a process can act in a given timeframe. Yet this reflects an ensemble of molecules and thus ignores the key characteristic of diffusion: variation of diffusion times for individual proteins. A cell could exploit this variation by using more proteins to send a signal. If you need concentration x at point A for a signal to be effective, you could increase the rate by having more signaling proteins start at point B. It would be interesting to see if this principle could in part explain, for example, the concentrations of two component signaling systems [(Capra and Laub, 2012)](#page-13-34) in the membranes of bacteria.

### Cell Size and Diffusion Length Scales

The enormous panoply of prokaryotic species has within itself also a great range of cell sizes. With on the smaller end the Archaeon Thermodiscus, with a volume of 3 × 10−3 µm3 , and the bacterium Mycoplasma pneumoniae, 5 × 10−3 µm3 . On the larger end we have the bacteria Epulopiscium fishelsoni, 3 × 106 µm3 , and Thiomargarita namibiensis, 2 × 108 µm3 (Schulz and Jorgensen, [2001)](#page-14-5). Somewhat counterintuitively both small and large sized could pose challenges for diffusion. For large size, the challenge is obvious; nutrients have to reach parts of the cell from outside of the cell, and proteins have to reach parts of the cell from the chromosome (via mRNA). In Epulopiscium fishelsoni and Thiomargarita namibiensis this appears to be solved by having many chromosomes, and having them packed against the membrane of the cell. The challenge for the small cells derives from their DNA. E. coli has 4.6 Mbp of DNA in a single chromosome [(Blattner et al., 1997)](#page-13-35) and has a volume of about 1 µm3 [(Taheri-Araghi et al., 2015)](#page-15-10); Mycoplasma genitalium has 0.58 Mbp of DNA [(Fraser et al., 1995)](#page-13-36) and has a volume of about 0.01 µm3 [(Taylor-Robinson, 1995)](#page-15-11) (here we assume spherical shape for M. genitalium, in reality the cells are pear shaped). The chromosome copy number in E. coli depends on growth rate [(Stokke et al., 2012)](#page-14-42), as does its volume [(Taheri-Araghi et al.,](#page-15-10) [2015)](#page-15-10). For the following, we are assuming that the chromosome copy numbers are the same for E. coli and M. genitalium. The M. genitalium volume is 100 times smaller than that of E. coli, whereas its genome is only 8 times smaller; leading to a 12.5 times higher concentration of DNA. In E. coli the DNA constitutes 3.1% of dry weight, compared to 55% for protein and 20.4% for RNA [(Phillips et al., 2009)](#page-14-0). DNA makes up 3.9% of the M. genitalium macromolecules. Multiplying 3.9% by 12.5 gives 49% (that is an extra 45%), so if the protein and RNA content is still the same, we have 1.45 times the amount of macromolecule in M. genitalium than in E. coli. The consequence that this (potential) difference in volume exclusion has on diffusion coefficients is unclear. For example, when excluded volume is altered by osmotic shocks the effect on the diffusion coefficient appears to be very different in E. coli than in L. lactis (see **[Figure 3F](#page-8-0)**). Of course, the distance between any point in the cytoplasm and the outside of the cell is smaller in M. genitalium than in E. coli, and therefore diffusion is more effective in delivering molecules. However, this distance benefit (in travel time) scales only with the power two (here 21 fold; see Equation 3), whereas the increase in DNA excluded volume scales with the power three (here 100-fold). No studies of diffusion coefficient in prokaryotes have looked at its variation, or lack thereof, along the cell size axis.

The travel distance of a molecule from the membrane to a location in the cytoplasm can be quantified with a characteristic value. The average distance of a point in the cytoplasm to the cell membrane is somewhat less than half the radius. Other distances to consider are for example the average distance between a gene and the membrane or a point in the cytoplasm; the average distance between ternary complex and ribosomes; or the average distance between some position in the cytoplasm and the tip of the stalk of C. crescentus [(Young, 2006)](#page-15-12). All these various distances, and the travel times associated with them could be limiting for some process. Consider a Thiovulum majus cell that has a diameter of 18µm [(Schulz and Jorgensen, 2001)](#page-14-5): If the limiting factor was the distance from a gene to a location in the cytoplasm, T. majus could just increase its number of chromosomes. Many bacteria are known to have increased numbers of chromosomes [(Pecoraro et al., 2011)](#page-14-43), which would be inconsequential if transport from cell membrane to a point in the cytoplasm is important. Hence, the characteristic distances should be taken into account when dealing with diffusion limitation in prokaryotes.

### Diffusion Limitation and Membrane Proteins

In E. coli both plasma- and outer membrane proteins diffuse with much lower diffusion coefficients than cytoplasmic proteins. The same is seen for plasma membrane proteins in L. lactis and C. crescentus (**[Table 1](#page-5-0)**). It is unclear if this leads to more diffusion limitation in membrane processes than in cytoplasmic ones. Unlike cytoplasmic proteins membrane proteins can rotate only along one axis which makes it easier for interaction interfaces to find one another. This rotational effect can lower the dissociation constant for a dimerization reaction by orders of magnitude [(Grasberger et al., 1986)](#page-13-37). A similar effect probably also occurs for rates.

### The Importance of Diffusion Coefficients in a Cellular Context

The rate of a reaction depends on both the concentration of reactants and the on-rate constant (Equation 4). Thus, for diffusion-limited reactions, the rate can be tuned by changing either the concentration or the diffusion coefficient. This means that slower proteins can increase their copy number to have the same interaction rate as smaller proteins. For example, the association rate of ternary complex with ribosomes is capped by limitations on the amount of ternary complex that can be made by a cell, before other processes are adversely affected. Hence, when a process requires the assembly of many proteins, such as ternary complex supplying the amino acids to the ribosome for use in translation, the impact of increasing copy number to increase association rate is tremendous. On the other hand, a change in the association rate for transcription factor binding to a site on the DNA, which needs only one copy (if there is one target site), can be done without much cost. For each protein in the cell one may ask to what degree its copy number is determined by association rate.

From the foregoing paragraph we are led into another question. Is it possible for a cell to have no diffusion limitation? Say we have x amount of protein molecules in a cell and no reaction is diffusion limited. With more protein molecules the cell is able to do more things and, for example, grow faster. So you would expect there to be evolutionary pressure to increase the amount of molecules in cells, and in so doing use up the free, inconsequential, space along the diffusion coefficient axis. Increasing the amount of protein molecules in a cell would continue up to the point that some reactions start to become limiting. This is rather similar to the previous discussion on the relation between cell size and protein concentration, but looked at from a different angle. If true, this means that there will always be diffusion limitation in cells. We can also turn the argument around and ask whether it is possible to have more than one process diffusion limited.

If it would be beneficial to increase the rate of all reactions in a cell, why not make them all steered by electrostatic interactions like the protein interaction pair, barnase-barstar? First, one cannot always change a protein's surface because it could affect its function directly or its stability. Secondly, is it even possible to make electrostatic interactions specific enough so that steering could be done independently for a thousand different interactions? Here the cellular context provides limitations on protein diffusion limited reactions.

### Diffusion May Affect Different Parts of Cells Differently

Any cell consists of a great number of interlocking and overlapping processes. Protein folding, protein-protein binding, nutrient transport to the cytoplasm, transcription factor binding, structuring the nucleoid, inserting membrane proteins, formation of the Z-ring, Min system cycling, chromosome segregation, cell size maintenance, converting the proteome in response to environmental stress, cell cycle time, etc. For each of these processes we can ask whether they are affected, either in rate or in functional form, by the diffusion coefficients of their constituent proteins. There are bound to be differences between processes in their susceptibility to diffusion changes. Cell cycle time is dependent on the diffusion coefficient of the ternary complex, whereas the cycling rate of the Min system is independent of the cytoplasmic diffusion coefficients of the Min proteins. Processes that require bigger proteins may suffer more from diffusion limitation than processes with small proteins (see **[Figure 3B](#page-8-0)**). Objects that have a size in the tens of nanometers may also experience other types of mobility [(Parry et al., 2014)](#page-14-10), and processes involving them may thus also be affected. Whether a protein is folded or disordered also seems to have an effect on its diffusion coefficient, with (unexpectedly) a disordered protein diffusing faster than a folded protein in the presence of artificial crowders [(Wang et al., 2012)](#page-15-3). Something discussed earlier relates to the different ranges over which diffusion occurs: translation happens at many places in the cytoplasm with shorter distances between ternary complex and ribosome than, for example, for a two component signaling system that needs to cross the distance between the membrane and a site on the DNA. Different processes are made up of such basic elements in different proportions and may thus be differently affected by changes in diffusion coefficient. Changes in diffusion coefficients can happen in real life situations for example after an osmotic upshift that reduces the cytoplasmic water content. These events could be transient as the uptake of for example K+ and compatible solutes counteracts the osmotic imbalance and restores the cell volume [(Wood, 2011)](#page-15-13). To know what the impact of an osmotic upshift is we have to know which processes are vulnerable to a reduction in diffusion coefficient. More generally, we can ask for each process by how many fold the diffusion coefficient needs to go down before this process becomes diffusion limited.

### The Reach of Diffusion Limitation

Cellular processes are layered: (i) The association rate of ternary complex binding to a ribosome is involved in the time of incorporation of a single amino acid into a polypeptide chain (chain elongation); (ii) the rate of chain elongation figures in the rate of protein production; (iii) this in turn determines the rate of accumulation of biomass and cell volume growth, and (iv) together with other processes this sets the cell cycle time. At each layer, the diffusion limitation that sets the rate of a reaction could lose its significance by a slower process in higher layers. How far a diffusion-limited reaction is affecting processes in higher layers is the reach of the diffusion limitation, which should be considered when determining the importance of a diffusion limited protein-protein interactions.

### CONCLUSION

Protein diffusion coefficients have been determined in vitro and in the prokaryotes E. coli, L. lactis, C. crescentus, P. aeruginosa, Hfx. volcanii, and others. E. coli is the best-studied prokaryote by far. The in vivo protein diffusion coefficients have been measured in the cytoplasm, periplasm, plasma membrane, and outer membrane. Various parameters of both proteins and their environment have been compared systematically to the diffusion coefficient, such as protein size, protein surface charge, cytoplasmic ionic strength, and level of excluded volume both in vitro and in the cytoplasm. Multiple studies have also been carried out on the importance of the diffusion coefficient in the context of protein toxins, translation, and the level of excluded volume in cells. Yet despite these achievements, the role of diffusion coefficients in prokaryotic cells is still murky. In the future we may look, among other things, into the relation between diffusion coefficient, excluded volume, and temperature; the relation between diffusion coefficient, excluded volume, and cell size; the effect of different diffusion length scales on the impact of diffusion coefficients on various physiological processes; and the complex relation between reaction rates, diffusion coefficients, and protein concentrations. We may also want to try and determine diffusion limitation of processes directly by altering the diffusion coefficient of particular proteins and monitoring the rate of whatever process these proteins function in.

### AUTHOR CONTRIBUTIONS

All authors listed have made substantial, direct, and intellectual contribution to the work and approved it for publication.

### REFERENCES

- <span id="page-13-1"></span>Alsallaq, R., and Zhou, H. (2008). Electrostatic rate enhancement and transient complex of protein-protein association. Proteins Struct. Func. Bioinform. 71, 320–335. doi: [10.1002/prot.21679](https://doi.org/10.1002/prot.21679)
- <span id="page-13-27"></span>Ando, T., and Skolnick, J. (2010). Crowding and hydrodynamic interactions likely dominate in vivo macromolecular motion. Proc. Natl. Acad. Sci. U. S. A. 107, 18457–18462. doi: [10.1073/pnas.1011354107](https://doi.org/10.1073/pnas.1011354107)
- <span id="page-13-2"></span>Andrews, S. S., Addy, N. J., Brent, R., and Arkin, A. P. (2010). Detailed simulations of cell biology with smoldyn 2.1. PLoS Comput. Biol. 6:e1000705. doi: [10.1371/journal.pcbi.1000705](https://doi.org/10.1371/journal.pcbi.1000705)
- <span id="page-13-10"></span>Bakshi, S., Siryaporn, A., Goulian, M., and Weisshaar, J. C. (2012). Superresolution imaging of ribosomes and RNA polymerase in live Escherichia coli cells. Mol. Microbiol. 85, 21–38. doi: [10.1111/j.1365-2958.2012.08081.x](https://doi.org/10.1111/j.1365-2958.2012.08081.x)
- <span id="page-13-4"></span>Berg, O., and von Hippel, P. (1985). Diffusion-controlled macromolecular interactions. Annu. Rev. Biophys. Biophys. Chem. 14, 131–160. doi: [10.1146/annurev.biophys.14.1.131](https://doi.org/10.1146/annurev.biophys.14.1.131)
- <span id="page-13-35"></span>Blattner, F., Plunkett, G., Bloch, C., Perna, N., Burland, V., Riley, M., et al. (1997). The complete genome sequence of Escherichia coli K-12. Science 277, 1453–1462. doi: [10.1126/science.277.5331.1453](https://doi.org/10.1126/science.277.5331.1453)
- <span id="page-13-28"></span>Buckle, A., Schreiber, G., and Fersht, A. (1994). Protein-protein recognition - crystal structural-analysis of a barnase barstar complex at 2.0-angstrom resolution. Biochemistry 33, 8878–8889. doi: [10.1021/bi00196a004](https://doi.org/10.1021/bi00196a004)
- <span id="page-13-34"></span>Capra, E. J., and Laub, M. T. (2012). Evolution of two-component signal transduction systems. Annu. Rev. Microbiol. 66, 325–347. doi: [10.1146/annurev-micro-092611-150039](https://doi.org/10.1146/annurev-micro-092611-150039)
- <span id="page-13-24"></span>Chow, D., Guo, L., Gai, F., and Goulian, M. (2012). Fluorescence correlation spectroscopy measurements of the membrane protein TetA in Escherichia coli suggest rapid diffusion at short length scales. PLoS ONE 7:e48600. doi: [10.1371/journal.pone.0048600](https://doi.org/10.1371/journal.pone.0048600)
- <span id="page-13-11"></span>Chow, E., and Skolnick, J. (2017). DNA Internal motion likely accelerates protein target search in a packed nucleoid. Biophys. J. 112, 2261–2270. doi: [10.1016/j.bpj.2017.04.049](https://doi.org/10.1016/j.bpj.2017.04.049)
- <span id="page-13-31"></span>Clark, A., and Frieden, C. (1997). GroEL-mediated folding of structurally homologous dihydrofolate reductases. J. Mol. Biol. 268, 512–525. doi: [10.1006/jmbi.1997.0969](https://doi.org/10.1006/jmbi.1997.0969)
- <span id="page-13-17"></span>Cluzel, P., Surette, M., and Leibler, S. (2000). An ultrasensitive bacterial motor revealed by monitoring signaling proteins in single cells. Science 287, 1652–1655. doi: [10.1126/science.287.5458.1652](https://doi.org/10.1126/science.287.5458.1652)
- <span id="page-13-13"></span>Cowan, A. E., Koppel, D. E., Setlow, B., and Setlow, P. (2003). A soluble protein is immobile in dormant spores of Bacillus subtilis but is mobile in germinated spores: implications for spore dormancy. Proc. Natl. Acad. Sci. U. S. A. 100, 4209–4214. doi: [10.1073/pnas.0636762100](https://doi.org/10.1073/pnas.0636762100)
- <span id="page-13-9"></span>Dauty, E., and Verkman, A. S. (2004). Molecular crowding reduces to a similar extent the diffusion of small solutes and macromolecules: measurement by fluorescence correlation spectroscopy. J. Mol. Recogn. 17, 441–447. doi: [10.1002/jmr.709](https://doi.org/10.1002/jmr.709)
- <span id="page-13-26"></span>Deich, J., Judd, E. M., McAdams, H. H., and Moerner, W. E. (2004). Visualization of the movement of single histidine kinase molecules in live Caulobacter cells. Proc. Natl. Acad. Sci. U. S. A. 101, 15921–15926. doi: [10.1073/pnas.0404200101](https://doi.org/10.1073/pnas.0404200101)
- <span id="page-13-0"></span>Dix, J. A., and Verkman, A. S. (2008). Crowding effects on diffusion in solutions and cells. Annu. Rev. Biophys. 37, 247–263. doi: [10.1146/annurev.biophys.37.032807.125824](https://doi.org/10.1146/annurev.biophys.37.032807.125824)
- <span id="page-13-19"></span>Elf, J., Li, G., and Xie, X. S. (2007). Probing transcription factor dynamics at the single-molecule level in a living cell. Science 316, 1191–1194. doi: [10.1126/science.1141967](https://doi.org/10.1126/science.1141967)

### ACKNOWLEDGMENTS

The work was funded by a NWO TOP-PUNT (grant number 718.014.001) and an ERC Advanced grant (ABCVolume, grant number 670578) awarded to BP, and a NWO VIDI grant awarded to AB (grant number 723.015. 002).

- <span id="page-13-18"></span>Elowitz, M. B., Surette, M. G., Wolf, P. E., Stock, J. B., and Leibler, S. (1999). Protein mobility in the cytoplasm of Escherichia coli. J. Bacteriol. 181, 197–203.
- <span id="page-13-29"></span>Fekkes, P., den Blaauwen, T., and Driessen, A. (1995). Diffusion-limited interaction between unfolded polypeptides and the Escherichia coli chaperone Secb, Biochemistry 34, 10078–10085. doi: [10.1021/bi00031a032](https://doi.org/10.1021/bi00031a032)
- <span id="page-13-36"></span>Fraser, C., Gocayne, J., White, O., Adams, M., Clayton, R., Fleischmann, R., et al. (1995). The minimal gene complement of mycoplasma-genitalium. Science 270, 397–403. doi: [10.1126/science.270.5235.397](https://doi.org/10.1126/science.270.5235.397)
- <span id="page-13-25"></span>Fukuoka, H., Sowa, Y., Kojima, S., Ishijima, A., and Homma, M. (2007). Visualization of functional rotor proteins of the bacterial flagellar motor in the cell membrane. J. Mol. Biol. 367, 692–701. doi: [10.1016/j.jmb.2007.01.015](https://doi.org/10.1016/j.jmb.2007.01.015)
- <span id="page-13-21"></span>Golding, I., and Cox, E. (2004). RNA dynamics in live Escherichia coli cells. Proc. Natl. Acad. Sci. U. S. A. 101, 11310–11315. doi: [10.1073/pnas.0404443101](https://doi.org/10.1073/pnas.0404443101)
- <span id="page-13-22"></span>Golding, I., and Cox, E. (2006). Physical nature of bacterial cytoplasm. Phys. Rev. Lett. 96:098102. doi: [10.1103/PhysRevLett.96.098102](https://doi.org/10.1103/PhysRevLett.96.098102)
- <span id="page-13-37"></span>Grasberger, B., Minton, A., Delisi, C., and Metzger, H. (1986). Interaction between proteins localized in membranes. Proc. Natl. Acad. Sci. U. S. A. 83, 6258–6262. doi: [10.1073/pnas.83.17.6258](https://doi.org/10.1073/pnas.83.17.6258)
- <span id="page-13-30"></span>Gray, T., and Fersht, A. (1993). Refolding of barnase in the presence of GroE. J. Mol. Biol. 232, 1197–1207. doi: [10.1006/jmbi.1993.1471](https://doi.org/10.1006/jmbi.1993.1471)
- <span id="page-13-8"></span>Gros, G. (1978). Concentration-dependence of self-diffusion of human and lumbricus-terrestris hemoglobin. Biophys. J. 22, 453–468. doi: [10.1016/S0006-3495(78)85499-X](https://doi.org/10.1016/S0006-3495(78)85499-X)
- <span id="page-13-15"></span>Guillon, L., Altenburger, S., Graumann, P. L., and Schalk, I. J. (2013). Deciphering protein dynamics of the siderophore pyoverdine pathway in Pseudomonas aeruginosa. PLoS ONE 8:e79111. doi: [10.1371/journal.pone.0079111](https://doi.org/10.1371/journal.pone.0079111)
- <span id="page-13-20"></span>Hahn, D. K., and Aragon, S. R. (2006). Intrinsic viscosity of proteins and platonic solids by boundary element methods. J. Chem. Theor. Comput. 2, 1416–1428. doi: [10.1021/ct600062y](https://doi.org/10.1021/ct600062y)
- <span id="page-13-12"></span>Joyner, R. P., Tang, J. H., Helenius, J., Dultz, E., Brune, C., Holt, L. J., et al. (2016). A glucose-starvation response regulates the diffusion of macromolecules. Elife 5:e09376. doi: [10.7554/eLife.09376](https://doi.org/10.7554/eLife.09376)
- <span id="page-13-7"></span>Keller, K., Canales, E., and Yum, S. (1971). Tracer and mutual diffusion coefficients of proteins. J. Phys. Chem. 75, 379–387. doi: [10.1021/j100673a015](https://doi.org/10.1021/j100673a015)
- <span id="page-13-5"></span>Klumpp, S., Scott, M., Pedersen, S., and Hwa, T. (2013). Molecular crowding limits translation and cell growth. Proc. Natl. Acad. Sci. U. S. A. 110, 16754–16759. doi: [10.1073/pnas.1310377110](https://doi.org/10.1073/pnas.1310377110)
- <span id="page-13-6"></span>Konopka, M. C., Sochacki, K. A., Bratton, B. P., Shkel, I. A., Record, M. T., and Weisshaar, J. C. (2009). Cytoplasmic protein mobility in osmotically stressed Escherichia coli. J. Bacteriol. 191, 231–237. doi: [10.1128/JB.00536-08](https://doi.org/10.1128/JB.00536-08)
- <span id="page-13-16"></span>Kumar, M., Mommer, M. S., and Sourjik, V. (2010). Mobility of cytoplasmic, membrane, and DNA-binding proteins in Escherichia coli. Biophys. J. 98, 552–559. doi: [10.1016/j.bpj.2009.11.002](https://doi.org/10.1016/j.bpj.2009.11.002)
- <span id="page-13-23"></span>Leake, M. C., Chandler, J. H., Wadhams, G. H., Bai, F., Berry, R. M., and Armitage, J. P. (2006). Stoichiometry and turnover in single, functioning membrane protein complexes. Nature 443, 355–358. doi: [10.1038/nature05135](https://doi.org/10.1038/nature05135)
- <span id="page-13-3"></span>Li, G., and Xie, X. S. (2011). Central dogma at the single-molecule level in living cells. Nature 475, 308–315. doi: [10.1038/nature10315](https://doi.org/10.1038/nature10315)
- <span id="page-13-33"></span>Lipkow, K., and Odde, D. J. (2008). Model for protein concentration gradients in the cytoplasm. Cell. Mol. Bioeng. 1, 84–92. doi: [10.1007/s12195-008-0008-8](https://doi.org/10.1007/s12195-008-0008-8)
- <span id="page-13-14"></span>Llopis, P. M., Sliusarenko, O., Heinritz, J., and Jacobs-Wagner, C. (2012). In vivo biochemistry in bacterial cells using FRAP: insight into the translation cycle. Biophys. J. 103, 1848–1859. doi: [10.1016/j.bpj.2012.09.035](https://doi.org/10.1016/j.bpj.2012.09.035)
- <span id="page-13-32"></span>Loose, M., Kruse, K., and Schwille, P. (2011). Protein self-organization: lessons from the min system. Annu. Rev. Biophys. 40, 315–336. doi: [10.1146/annurev-biophys-042910-155332](https://doi.org/10.1146/annurev-biophys-042910-155332)
- <span id="page-14-15"></span>McGuffee, S. R., and Elcock, A. H. (2010). Diffusion, crowding & protein stability in a dynamic molecular model of the bacterial cytoplasm RID F-4799-2010. PLoS Comput. Biol. 6:e1000694. doi: [10.1371/journal.pcbi.1000694](https://doi.org/10.1371/journal.pcbi.1000694)
- <span id="page-14-9"></span>Mika, J. T., G., van den Bogaart, Veenhoff, L., Krasnikov, V., and Poolman, B. (2010). Molecular sieving properties of the cytoplasm of Escherichia coli and consequences of osmotic stress. Mol. Microbiol. 77, 200–207. doi: [10.1111/j.1365-2958.2010.07201.x](https://doi.org/10.1111/j.1365-2958.2010.07201.x)
- <span id="page-14-18"></span>Mika, J. T., and Poolman, B. (2011). Macromolecule diffusion and confinement in prokaryotic cells. Curr. Opin. Biotechnol. 22, 117–126. doi: [10.1016/j.copbio.2010.09.009](https://doi.org/10.1016/j.copbio.2010.09.009)
- <span id="page-14-19"></span>Mika, J. T., Schavemaker, P. E., Krasnikov, V., and Poolman, B. (2014). Impact of osmotic stress on protein diffusion in Lactococcus lactis. Mol. Microbiol. 94, 857–870. doi: [10.1111/mmi.12800](https://doi.org/10.1111/mmi.12800)
- <span id="page-14-20"></span>Mullineaux, C. W., Nenninger, A., Ray, N., and Robinson, C. (2006). Diffusion of green fluorescent protein in three cell environments in Escherichia coli. J. Bacteriol. 188, 3442–3448. doi: [10.1128/JB.188.10.3442-3448.2006](https://doi.org/10.1128/JB.188.10.3442-3448.2006)
- <span id="page-14-11"></span>Munder, M. C., Midtvedt, D., Franzmann, T., Nueske, E., Otto, O., Herbig, M., et al. (2016). A pH-driven transition of the cytoplasm from a fluid- to a solid-like state promotes entry into dormancy. Elife 5:e09347. doi: [10.7554/eLife.09347](https://doi.org/10.7554/eLife.09347)
- <span id="page-14-8"></span>Muramatsu, N., and Minton, A. (1988). Tracer diffusion of globular-proteins in concentrated protein solutions. Proc. Natl. Acad. Sci. U. S. A. 85, 2984–2988. doi: [10.1073/pnas.85.9.2984](https://doi.org/10.1073/pnas.85.9.2984)
- <span id="page-14-21"></span>Nenninger, A., Mastroianni, G., and Mullineaux, C. W. (2010). Size dependence of protein diffusion in the cytoplasm of Escherichia coli. J. Bacteriol. 192, 4535–4540. doi: [10.1128/JB.00284-10](https://doi.org/10.1128/JB.00284-10)
- <span id="page-14-14"></span>Nesmelova, I., and Fedotov, V. (1998). Self-diffusion and self-association of lysozyme molecules in solution. Biochim. Et Biophys. Acta Protein Struct. Mol. Enzymol. 1383, 311–316. doi: [10.1016/S0167-4838(97)00224-0](https://doi.org/10.1016/S0167-4838(97)00224-0)
- <span id="page-14-6"></span>Nesmelova, I., Skirda, V., and Fedotov, V. (2002). Generalized concentration dependence of globular protein self-diffusion coefficients in aqueous solutions. Biopolymers 63, 132–140. doi: [10.1002/bip.10023](https://doi.org/10.1002/bip.10023)
- <span id="page-14-27"></span>Oddershede, L., Dreyer, J., Grego, S., Brown, S., and Berg-Sorenson, K. (2002). The motion of a single molecule, the lambda-receptor, in the bacterial outer membrane. Biophys. J. [83, 3152–3161. doi: 10.1016/S0006-3495(02)](https://doi.org/10.1016/S0006-3495(02)75318-6) 75318-6
- <span id="page-14-10"></span>Parry, B. R., Surovtsev, I. V., Cabeen, M. T., O'Hem, C. S., Dufresne, E. R., and Jacobs-Wagner, C. (2014). The bacterial cytoplasm has glasslike properties and is fluidized by metabolic activity. Cell 156, 183–194. doi: [10.1016/j.cell.2013.11.028](https://doi.org/10.1016/j.cell.2013.11.028)
- <span id="page-14-32"></span>Patel, A., Malinovska, L., Saha, S., Wang, J., Alberti, S., Krishnan, Y., et al. (2017). ATP as a biological hydrotrope. Science 356, 753–756. doi: [10.1126/science.aaf6846](https://doi.org/10.1126/science.aaf6846)
- <span id="page-14-43"></span>Pecoraro, V., Zerulla, K., Lange, C., and Soppa, J. (2011). Quantification of ploidy in proteobacteria revealed the existence of monoploid, (mero-)oligoploid and polyploid species. PLoS ONE [6:e16392. doi: 10.1371/journal.pone.00](https://doi.org/10.1371/journal.pone.0016392) 16392
- <span id="page-14-30"></span>Perez, A. M., Mann, T. H., Lasker, K., Ahrens, D. G., Eckart, M. R., and Shapiro, L. (2017). A localized complex of two protein oligomers controls the orientation of cell polarity. Mbio 8:e02238–16. doi: [10.1128/mBio.02238-16](https://doi.org/10.1128/mBio.02238-16)
- <span id="page-14-41"></span>Pérez-Fillol, M., and Rodriguez-Vallera, F. (1986). Potassium ion accumulation in cells of different halobacteria. Microbiologia 2, 73–80.
- <span id="page-14-36"></span>Perrett, S., Zahn, R., Stenberg, G., and Fersht, A. (1997). Importance of electrostatic interactions in the rapid binding of polypeptides to GroEL. J. Mol. Biol. 269, 892–901. doi: [10.1006/jmbi.1997.1081](https://doi.org/10.1006/jmbi.1997.1081)
- <span id="page-14-0"></span>Phillips, R., Kondev, J., and Theriot, J. (2009). Physical Biology of the Cell. New York, NY: Garland Science.
- <span id="page-14-16"></span>Potma, E., de Boeij, W., Bosgraaf, L., Roelofs, J., van Haastert, P., and Wiersma, D. (2001). Reduced protein diffusion rate by cytoskeleton in vegetative and polarized Dictyostelium cells. Biophys. J. 81, 2010–2019. doi: [10.1016/S0006-3495(01)75851-1](https://doi.org/10.1016/S0006-3495(01)75851-1)
- <span id="page-14-35"></span>Ramadurai, S., Holt, A., Krasnikov, V., G., van den Bogaart, Killian, J. A., and Poolman, B. (2009). Lateral diffusion of membrane proteins. J. Am. Chem. Soc. 131, 12650–12656. doi: [10.1021/ja902853g](https://doi.org/10.1021/ja902853g)
- <span id="page-14-24"></span>Reyes-Lamothe, R., Possoz, C., Danilova, O., and Sherratt, D. J. (2008). Independent positioning and action of Escherichia coli replisomes in live cells. Cell 133, 90–102. doi: [10.1016/j.cell.2008.01.044](https://doi.org/10.1016/j.cell.2008.01.044)
- <span id="page-14-7"></span>Rothe, M., Gruber, T., Groeger, S., Balbach, J., Saalwaechter, K., and Roos, M. (2016). Transient binding accounts for apparent violation of the generalized

Stokes-Einstein relation in crowded protein solutions. Phys. Chem. Chem. Phys. 18, 18006–18014. doi: [10.1039/c6cp01056c](https://doi.org/10.1039/c6cp01056c)

- <span id="page-14-31"></span>Saha, S., Weber, C. A., Nousch, M., Adame-Arana, O., Hoege, C., Hein, M. Y., et al. (2016). Polar positioning of phase-separated liquid compartments in cells regulated by an mRNA competition mechanism. Cell 166, 1572–1584. doi: [10.1016/j.cell.2016.08.006](https://doi.org/10.1016/j.cell.2016.08.006)
- <span id="page-14-22"></span>Sanamrad, A., Persson, F., Lundius, E. G., Fange, D., Gynna, A. H., and Elf, J. (2014). Single-particle tracking reveals that free ribosomal subunits are not excluded from the Escherichia coli nucleoid. Proc. Natl. Acad. Sci. U. S. A. 111, 11413–11418. doi: [10.1073/pnas.1411558111](https://doi.org/10.1073/pnas.1411558111)
- <span id="page-14-23"></span>Savage, D. F., Afonso, B., Chen, A. H., and Silver, P. A. (2010). Spatially ordered dynamics of the bacterial carbon fixation machinery. Science 327, 1258–1261. doi: [10.1126/science.1186090](https://doi.org/10.1126/science.1186090)
- <span id="page-14-13"></span>Saxton, M. (1995). Single-particle tracking - effects of corrals. Biophys. J. 69, 389–398. doi: [10.1016/S0006-3495(95)79911-8](https://doi.org/10.1016/S0006-3495(95)79911-8)
- <span id="page-14-12"></span>Saxton, M., and Jacobson, K. (1997). Single-particle tracking: applications to membrane dynamics. Annu. Rev. Biophys. Biomol. Struct. 26, 373–399. doi: [10.1146/annurev.biophys.26.1.373](https://doi.org/10.1146/annurev.biophys.26.1.373)
- <span id="page-14-1"></span>Schavemaker, P. E., Smigiel, W. M., and Poolman, B. (2017). Ribosome surface properties may impose limits on the nature of the cytoplasmic proteome, Elife 6:e30084. doi: [10.7554/elife.30084](https://doi.org/10.7554/elife.30084)
- <span id="page-14-25"></span>Schindler, M., Osborn, M., and Koppel, D. (1980). Lateral diffusion of lipopolysaccharide in the outer-membrane of salmonella-typhimurium. Nature 285, 261–263. doi: [10.1038/285261a0](https://doi.org/10.1038/285261a0)
- <span id="page-14-3"></span>Schlosshauer, M., and Baker, D. (2004). Realistic protein-protein association rates from a simple diffusional model neglecting long-range interactions, free energy barriers, and landscape ruggedness. Protein Sci. 13, 1660–1669. doi: [10.1110/ps.03517304](https://doi.org/10.1110/ps.03517304)
- <span id="page-14-4"></span>Schreiber, G., and Fersht, A. R. (1993). Interaction of barnase with its polypeptide inhibitor barstar studied by protein engineering. Biochemistry 32, 5145–5150.
- <span id="page-14-2"></span>Schreiber, G., Haran, G., and Zhou, H. (2009). Fundamental aspects of proteinprotein association kinetics. Chem. Rev. 109, 839–860. doi: [10.1021/cr800373w](https://doi.org/10.1021/cr800373w)
- <span id="page-14-5"></span>Schulz, H., and Jorgensen, B. (2001). Big bacteria. Annu. Rev. Microbiol. 55, 105–137. doi: [10.1146/annurev.micro.55.1.105](https://doi.org/10.1146/annurev.micro.55.1.105)
- <span id="page-14-40"></span>Shabala, L., Bowman, J., Brown, J., Ross, T., McMeekin, T., and Shabala, S. (2009). Ion transport and osmotic adjustment in Escherichia coli in response to ionic and non-ionic osmotica. Environ. Microbiol. 11, 137–148. doi: [10.1111/j.1462-2920.2008.01748.x](https://doi.org/10.1111/j.1462-2920.2008.01748.x)
- <span id="page-14-38"></span>Soh, S., Banaszak, M., Kandere-Grzybowska, K., and Grzybowski, B. A. (2013). Why cells are microscopic: a transport-time perspective. J. Phys. Chem. Lett. 4, 861–865. doi: [10.1021/jz3019379](https://doi.org/10.1021/jz3019379)
- <span id="page-14-34"></span>Soh, S., Byrska, M., Kandere-Grzybowska, K., and Grzybowski, B. A. (2010). Reaction-diffusion systems in intracellular molecular transport and control. Ange. Chem. Int. Ed. 49, 4170–4198. doi: [10.1002/anie.200905513](https://doi.org/10.1002/anie.200905513)
- <span id="page-14-37"></span>Sparrer, H., Lilie, H., and Buchner, J. (1996). Dynamics of the GroEL protein complex: Effects of nucleotides and folding mutants. J. Mol. Biol. 258, 74–87. doi: [10.1006/jmbi.1996.0235](https://doi.org/10.1006/jmbi.1996.0235)
- <span id="page-14-26"></span>Spector, J., Zakharov, S., Lill, Y., Sharma, O., Cramer, W. A., and Ritchie, K. (2010). Mobility of BtuB and OmpF in the Escherichia coli outer membrane: implications for dynamic formation of a translocon complex. Biophys. J. 99, 3880–3886. doi: [10.1016/j.bpj.2010.10.029](https://doi.org/10.1016/j.bpj.2010.10.029)
- <span id="page-14-33"></span>Spitzer, J., and Poolman, B. (2005). Electrochemical structure of the crowded cytoplasm. Trends Biochem. Sci. 30, 536–541. doi: [10.1016/j.tibs.2005.08.002](https://doi.org/10.1016/j.tibs.2005.08.002)
- <span id="page-14-28"></span>Spitzer, J., and Poolman, B. (2009). The Role of biomacromolecular crowding, ionic strength, and physicochemical gradients in the complexities of life's emergence. Microbiol. Mol. Biol. Rev. 73, 371–388. doi: [10.1128/MMBR.00010-09](https://doi.org/10.1128/MMBR.00010-09)
- <span id="page-14-29"></span>Spitzer, J., and Poolman, B. (2013). How crowded is the prokaryotic cytoplasm? FEBS Lett. 587, 2094–2098. doi: [10.1016/j.febslet.2013.05.051](https://doi.org/10.1016/j.febslet.2013.05.051)
- <span id="page-14-42"></span>Stokke, C., Flatten, I., and Skarstad, K. (2012). An easy-to-use simulation program demonstrates variations in bacterial cell cycle parameters depending on medium and temperature. PLoS ONE 7:e30981.
- <span id="page-14-39"></span>Stone, S. R., Dennis, S., and Hofsteenge, J. (1989). Quantitative evaluation of the contribution of ionic interactions to the formation of the thrombin-hirudin complex. Biochemistry 28, 6857–6863.
- <span id="page-14-17"></span>Swaminathan, R., Hoang, C., and Verkman, A. (1997). Photobleaching recovery and anisotropy decay of green fluorescent protein GFP-S65T in solution and cells: Cytoplasmic viscosity probed by green fluorescent protein translational and rotational diffusion. Biophys. J. 72, 1900–1907.
- <span id="page-15-10"></span>Taheri-Araghi, S., Bradde, S., Sauls, J. T., Hill, N. S., Levin, P. A., Paulsson, J., et al. (2015). Cell-size control and homeostasis in bacteria. Curr. Biol. 25, 385–391. doi: [10.1016/j.cub.2014.12.009](https://doi.org/10.1016/j.cub.2014.12.009)
- <span id="page-15-11"></span>Taylor-Robinson, D. (1995). The history and role of mycoplasma-genitalium in sexually-transmitted diseases. Genitourin Med. 71, 1–8.
- <span id="page-15-2"></span>Tokuyama, M., Moriki, T., and Kimura, Y. (2011). Self-diffusion of biomolecules in solution. Phys. Rev. E 83:051402. doi: [10.1103/PhysRevE.83.051402](https://doi.org/10.1103/PhysRevE.83.051402)
- <span id="page-15-7"></span>Tyn, M., and Gusek, T. (1990). Prediction of diffusion-coefficients of proteins. Biotechnol. Bioeng. [35, 327–338. doi: 10.1002/bit.](https://doi.org/10.1002/bit.260350402) 260350402
- <span id="page-15-6"></span>van den Berg, J., Boersma, A. J., and Poolman, B. (2017). Microorganisms [maintain crowding homeostasis.](https://doi.org/10.1038/nrmicro.2017.17) Nat. Rev. Micro. 15, 309–318. doi: 10.1038/ nrmicro.2017.17
- <span id="page-15-9"></span>van den Bogaart, Hermans, N., Krasnikov, V., and Poolman, B. (2007). Protein mobility and diffusive barriers in Escherichia coli: consequences of osmotic stress. Mol. Microbiol. [64, 858–871. doi: 10.1111/j.1365-2958.2007.](https://doi.org/10.1111/j.1365-2958.2007.05705.x) 05705.x
- <span id="page-15-8"></span>van Holde, K. E., Curtis Johnson, W., and Shing Ho, P. (2006). Principles of Physical Biochemistry, 2nd Edn. Upper Sadle River, NJ: Pearson Prentice Hall.
- <span id="page-15-0"></span>Wallis, R., Moore, G. R., James, R., and Kleanthous, C. (1995). Proteinprotein interactions in colicin E9 DNase-immunity protein complexes. 1. Diffusion-controlled association and femtomolar binding for the cognate complex. Biochemistry 34, 13743–13750.
- <span id="page-15-3"></span>Wang, Y., Benton, L. A., Singh, V., and Pielak, G. J. (2012). Disordered protein diffusion under crowded conditions. J. Phys. Chem. Lett. 3, 2703–2706. doi: [10.1021/jz3010915](https://doi.org/10.1021/jz3010915)
- <span id="page-15-4"></span>Wang, Y., Li, C., and Pielak, G. J. (2010). Effects of proteins on protein diffusion. J. Am. Chem. Soc. 132, 9392–9397. doi: [10.1021/ja102296k](https://doi.org/10.1021/ja102296k)
- <span id="page-15-13"></span>Wood, J. M. (2011). Bacterial osmoregulation, a paradigm for the study of cellular homeostasis. Annu. Rev. Microbiol. 65, 215–238. doi: [10.1146/annurev-micro-090110-102815](https://doi.org/10.1146/annurev-micro-090110-102815)
- <span id="page-15-12"></span>Young, K. D. (2006). The selective value of bacterial shape. Microbiol. Mol. Biol. Rev. 70, 660–703. doi: [10.1128/MMBR.00001-06](https://doi.org/10.1128/MMBR.00001-06)
- <span id="page-15-5"></span>Yu, I., Mori, T., Ando, T., Harada, R., Jung, J., Sugita, Y., et al. (2016). Biomolecular interactions modulate macromolecular structure and dynamics [in atomistic model of a bacterial cytoplasm.](https://doi.org/10.7554/eLife.19274) Elife 5:e19274. doi: 10.7554/eLife. 19274
- <span id="page-15-1"></span>Zhang, G., Fedyunin, I., Miekley, O., Valleriani, A., Moura, A., and Ignatova, Z. (2010). Global and local depletion of ternary complex limits translational elongation. Nucleic Acids Res. 38, 4778–4787. doi: [10.1093/nar/gkq196](https://doi.org/10.1093/nar/gkq196)

**Conflict of Interest Statement:** The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

Copyright © 2018 Schavemaker, Boersma and Poolman. This is an open-access article distributed under the terms of the [Creative Commons](http://creativecommons.org/licenses/by/4.0/) [Attribution License](http://creativecommons.org/licenses/by/4.0/) (CC [BY). The use, distribution or reproduction in other forums is](http://creativecommons.org/licenses/by/4.0/) permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.