![](_page_0_Picture_0.jpeg)

# Principles of cellular resource allocation revealed by condition-dependent proteome profiling

Eyal Metzl-Raz† , Moshe Kafri† , Gilad Yaakov, Ilya Soifer, Yonat Gurvich, Naama Barkai*

Department of Molecular Genetics, Weizmann Institute of Science, Rehovot, Israel

Abstract Growing cells coordinate protein translation with metabolic rates. Central to this coordination is ribosome production. Ribosomes drive cell growth, but translation of ribosomal proteins competes with production of non-ribosomal proteins. Theory shows that cell growth is maximized when all expressed ribosomes are constantly translating. To examine whether budding yeast function at this limit of full ribosomal usage, we profiled the proteomes of cells growing in different environments. We find that cells produce excess ribosomal proteins, amounting to a constant » 8% of the proteome. Accordingly, » 25% of ribosomal proteins expressed in rapidly growing cells does not contribute to translation. Further, this fraction increases as growth rate decreases and these excess ribosomal proteins are employed when translation demands unexpectedly increase. We suggest that steadily growing cells prepare for conditions that demand increased translation by producing excess ribosomes, at the expense of lower steady-state growth rate.

[DOI: https://doi.org/10.7554/eLife.28034.001](https://doi.org/10.7554/eLife.28034.001)

# Introduction

Producing ribosomes is a major biosynthetic process in cells. Rapidly growing budding yeast, for example, contain about 190,000 ribosomes ([von der Haar, 2008](#page-20-0)), each of which requires the production of over a hundred different proteins. Overall, » 30% of the proteome of rapidly growing cells encodes for ribosomal proteins and » 80% of all cellular RNA encodes for the rRNA components of the ribosomes ([Warner, 1999](#page-20-0)). It is therefore expected that cells minimize the amount of ribosomes they produce, making only the ribosomes required for protein translation in the given condition.

The coordination of ribosomal protein expression with cell growth rate is of a particular interest. Ribosomes drive protein translation and therefore cell growth. Increasing the amounts of ribosomes in cells could therefore increase growth rate. Yet, translation of ribosomal proteins competes with the production of other proteins and could therefore introduce limitations in other essential cellular processes, such as the production of metabolites needed for translation. Resource allocation models formalize this interplay by defining the optimal distribution of translation resources which maximizes growth rate. These models predict that growth rate is maximized when all expressed ribosomal proteins are fully employed in protein translation ([Bosdriesz et al., 2015](#page-18-0); [Dekel and Alon, 2005](#page-18-0); [Kafri et al., 2016a](#page-19-0); [Keren et al., 2013](#page-19-0); [Klumpp et al., 2013](#page-19-0); [Koch, 1988](#page-19-0); [Maaløe, 1979](#page-19-0); [Scott and Hwa, 2011](#page-20-0); [Scott et al., 2010](#page-19-0); [2014](#page-20-0); [Vind et al., 1993](#page-20-0)). Indeed, producing excess ribosomal proteins that are not actively translating would compete with alternative cellular processes without contributing to protein production and cell growth.

The prevailing model therefore assumes that in growing cells, all ribosomes are constantly translating. However, recent studies suggest that while this model may hold in rapidly growing bacteria,

### *For correspondence:

†

naama.barkai@weizmann.ac.il

These authors contributed equally to this work Competing interest: [See](#page-17-0)

[page 18](#page-17-0)

Funding: [See page 18](#page-17-0)

Received: 24 April 2017 Accepted: 08 August 2017 Published: 31 August 2017

Reviewing editor: Patricia J Wittkopp, University of Michigan, United States

Copyright Metzl-Raz et al. This article is distributed under the terms of the [Creative Commons](http://creativecommons.org/licenses/by/4.0/)

[Attribution License,](http://creativecommons.org/licenses/by/4.0/) which permits unrestricted use and redistribution provided that the original author and source are credited.

slow-growing bacteria do contain excess ribosomes that are not actively translating ([Dai et al.,](#page-18-0) [2016](#page-18-0)). Further, we recently reported that in rapidly growing budding yeast, protein translation is not universally limiting for protein production and growth rate ([Kafri et al., 2016b](#page-19-0)). This raises the question of whether rapidly growing cells do in fact employ all expressed ribosomes at full capacity, as predicted by resource allocation models.

Consistent with its major role in driving cell growth, the ribosome content of bacteria growing in different environments increases linearly with growth rate, independently of the specifics of each nutrient ([Brauer et al., 2008](#page-18-0); [Bremer and Ehrenberg, 1995](#page-18-0); [Nomura et al., 1984](#page-19-0); [Schaechter et al., 1958](#page-19-0); [Scott et al., 2010](#page-19-0); [Waldron and Lacroute, 1975](#page-20-0); [Warner, 1999](#page-20-0); [Zaslaver et al., 2009](#page-20-0)). This classical growth law was derived by measuring RNA content, which well approximates the amounts of ribosomes. In this work, we wished to extend these studies in three directions. First, we focus on budding yeast, asking whether the bacterial growth law connecting ribosome content and growth rate is conserved in this model eukaryote. Second, using proteomic profiling, we directly measure ribosomal protein levels and compare their abundance with that of other protein groups. Finally, we consider non-steady state conditions and in particular situations in which cells face increased translation demands, asking whether and how such conditions impact the relation between ribosome content and growth rate.

By analyzing the proteome composition of cells growing in a wide range of conditions, we show that the proteome fraction coding for ribosomal proteins scales linearly with growth rate, in a manner that is similar to that described in bacteria. Quantitative analysis of this scaling relation, coupled with polysome profiling, suggests that at each growth rate, a constant » 8% fraction of the entire proteome encodes for an excess of ribosomes that are not actively translating. Accordingly, in rapidly growing cells, » 25% of ribosomal proteins produced are not employed in translation at any given time and this inactive fraction significantly increases as the growth rate decreases. We provide evidence that this excess of ribosomes may be employed when cells are subjected to an unexpected increase in translation demands. Finally, we suggest that steadily growing cells prepare for fluctuating conditions by producing excess ribosomes. This ribosomal excess enables a faster response to upshift conditions, but comes at the expense of limiting the cells' instantaneous growth rate.

## Results

### The proteomic composition of budding yeast cells growing in different conditions

We employed mass-spectrometry analysis to define the proteomic composition of S. cerevisiae cells growing in three conditions: standard media (SC), media low in nitrogen (Low N) and media low in phosphate (Low Pi). This data was combined with published proteomic datasets of budding yeast growing on 12 different carbon sources ([Paulo et al., 2015](#page-19-0); [2016](#page-19-0)). To avoid method-specific biases, all profiles were calibrated with an external data reference defining absolute protein levels ([Wang et al., 2012](#page-20-0)).

Examining the Pearson correlation between the different profiles ([Figure 1A](#page-2-0)), we observed that profiles were classified into two dominating clusters, depending on whether cells grew in a fermentative or respiratory mode. Correlations between profiles within the same cluster were » 0.7–0.9, while correlations between profiles assigned to different clusters were lower but still substantial (0.3–0.6). The data further demonstrated the expected induction of condition-specific proteins (e.g. activation of the phosphate and nitrogen starvation pathways), as well as differential expression of proteins involved in translation and stress response ([Figure 1B](#page-2-0)).

Growth rate is a major determinant of proteome composition. Division times of cells in the 15 conditions we examined varied between 1.5 to 6.5 hr. To obtain a general overview of the effects of growth rate on the proteome composition, we classified proteins into eleven groups of related functions, which together covered » 80% of the proteome ([Supplementary file 1](#page-17-0)), and examined how the relative abundance of each group changes with growth rate. In standard media, the proteome was dominated by translation-related factors ( » 40%) and glycolytic proteins ( » 15%) ([Figure 1C](#page-2-0) and [Figure 1—figure supplement 1A](#page-3-0)). The fraction of glycolytic proteins remained largely invariant between conditions, while the translation-related fraction decreased with growth rate, reaching » 15% in slow-growing cells. This decrease was accompanied by an increased abundance of

<span id="page-2-0"></span>![](_page_2_Figure_2.jpeg)

Figure 1. Proteomic analysis of budding yeast grown in different conditions. (A) Proteome profiles in our dataset clusters into two main groups on their fermentative or respiratory growth mode: Shown is the Pearson correlation matrix between proteome compositions in the indicated conditions. (B) Condition-dependent regulation of protein expression: The expression of each protein in each condition was normalized by its mean expression over all conditions. Shown is the (Log2) protein expression of proteins in the indicated groups. See [Supplementary file 2](#page-17-0) for protein names. (C) Expression Figure 1 continued on next page

<span id="page-3-0"></span>![](_page_3_Picture_0.jpeg)

#### Figure 1 continued

of translation genes decreases in slow-growing cells: Proteins were classified into eleven groups by function, which together included » 80% of the proteome. For each condition, we calculated the fraction of the proteome coding for each of these eleven groups. Shown here are the proteome fractions of groups composed of proteins involved in translation, glycolysis or mitochondrial function, plotted as a function of Generations per Hour (generation time1 = m/ln(2)). Additional protein groups are shown in Figure 1—figure supplement 1A. Filled circles correspond to data obtained in this work while empty circles are data from [Paulo et al. (2015)](#page-19-0), [(2016)](#page-19-0), as specified in [Figure 2A](#page-4-0). The proteins assigned to each protein group are specified in [Supplementary file 1](#page-17-0). Dashed lines are the data's linear fits. Note that the translation group slope (y = 0.36x + 0.15) is almost identical to the ribosomal group slope ([Figure 2A](#page-4-0)). (D) The overall proteome composition in fast vs. slow growing cells: The fraction of proteome encoding for each protein group was compared between the fast and slow growth condition in our dataset. In the upper panel, the proteome fraction encoding each specified protein group is plotted as a function of this group in the slow growing condition. Fast growth (0.67 gen/hr) corresponds to standard (SC) conditions, while slow growth (0.15 gen/hr) is extrapolated from the linear fit of the abundance vs. growth rate relation, shown in [Figure 1C](#page-2-0) and Figure 1—figure supplement 1A. In the bottom panel, the same data is plotted as pie-charts.

[DOI: https://doi.org/10.7554/eLife.28034.002](https://doi.org/10.7554/eLife.28034.002)

The following source data and figure supplement are available for figure 1:

Source data 1. DAmP and WT LC-MS/MS proteomic data as described in Materials and methods.

[DOI: https://doi.org/10.7554/eLife.28034.004](https://doi.org/10.7554/eLife.28034.004)

Figure supplement 1. Proteome composition.

[DOI: https://doi.org/10.7554/eLife.28034.003](https://doi.org/10.7554/eLife.28034.003)

condition-specific proteins, which in our dataset were mostly respiration-related. Mitochondrial proteins, for example, double in quantity to » 20% of the proteome in the slow growing cells, in accordance with the increased reliance of these cells on respiration ([Figure 1C](#page-2-0)). We summarize the changes in the proteomic fraction devoted to the different gene groups in slow versus fast growing conditions by plotting the abundance of each group in the slowest growth condition, against its abundance in the fast growth conditions ([Figure 1D](#page-2-0), top). This is also illustrated by a pie-chart summarizing the proteome composition in those two conditions ([Figure 1D](#page-2-0), bottom).

### The proteome fraction encoding ribosomal proteins scales linearly with cell growth rate

We next focused more specifically on the relation between the expression of ribosomal proteins and the rate of cell growth. We defined the ribosomal fraction of the proteome by clustering together all proteins annotated as subunits of the ribosome ([Supplementary file 1](#page-17-0)). The proteome fraction coding for this group increased linearly with growth rate, irrespective of the specific media, from » 8% in the slowest growing cells to » 30% in rapidly growing cells ([Figure 2A](#page-4-0)). Therefore, budding yeast show a similar growth law to that previously described in bacteria.

Cells may adjust their ribosome content by modifying protein translation, protein degradation or mRNA levels. To distinguish between these possibilities, we examined the transcription profiles of cells growing in the different conditions used for the proteome profiling. Notably, plotting the fraction of mRNA transcripts that code for ribosomal proteins as a function of cell growth rate showed the same quantitative scaling as observed at the level of the proteome ([Figure 2B](#page-4-0) top). Thus, regulation of the ribosomal protein fraction with growth rate depends almost exclusively on mRNA transcription.

Next, we asked whether the scaling between ribosome content and growth rate is strain-specific and regulated by changing growth conditions, or whether it also explains differences in growth rate between different strains growing in the same condition. We recently reported that budding yeast strains show a large strain-to-strain variability in growth rate when growing on pentose as the sole carbon source ([Tamari et al., 2014](#page-20-0); [2016](#page-20-0)). Examining the transcription profiles of these strains, we find that the ribosomal fraction follows growth rate with the same qualitative scaling as observed when comparing growth rates in different conditions ([Figure 2B](#page-4-0), top). The general scaling relationship between ribosome content and growth rate is therefore conserved not only when comparing the same strain across conditions, but also when comparing different strains growing in the same condition.

We next asked whether the scaling of ribosome content and growth rate is also maintained following genetic perturbations that affect cell growth. To this end, we used a large dataset describing the gene expression profiles and growth rates of most viable yeast deletion mutants

<span id="page-4-0"></span>![](_page_4_Figure_2.jpeg)

Figure 2. Ribosome content scales linearly with cell growth rate. (A) Shown is the fraction of the proteome coding for the ribosomal proteins in each condition as a function of cell growth rate. Note that here and henceforth, growth rate is shown in units of generation per hour (Generation Time1 ) which is related to the specific growth rate m by a scaling factor, m = GT1 *ln(2). The slope of this scaling curve is = 0.35-60 min=21 min so Dr/Dm = 21/ ln(2)[min]. For a broader definition of the translational group, see [Figure 1C](#page-2-0) top panel and [Supplementary file 1](#page-17-0) for gene names. Our data (filled circles) is generated from three biological repeats. (B) The transcriptome fraction coding ribosomal proteins scales with growth rate: Shown on top is the fraction of the ribosomal protein transcripts from the full transcriptome as a function of cell growth rate. Conditions are specified in Figure 2 figure supplement 1A. Dotted black line is the linear fit from (A). Note the high agreement between the transcriptomic data and the proteomic linear fit, implying that on average, the translation-to-degradation of ribosomal proteins is the same as that of other proteins (primarily highly abundant ones). The bottom figure shows that this scaling is lost when comparing growth-affecting mutations. Mutant data was taken from [Kemmeren et al. (2014)](#page-19-0) and [O'Duibhir et al. (2014)](#page-19-0).

[DOI: https://doi.org/10.7554/eLife.28034.005](https://doi.org/10.7554/eLife.28034.005)

The following figure supplement is available for figure 2:

Figure supplement 1. The proteome profiles of budding yeast cells growing in different conditions. [DOI: https://doi.org/10.7554/eLife.28034.006](https://doi.org/10.7554/eLife.28034.006)

> ([Kemmeren et al., 2014](#page-19-0); [O'Duibhir et al., 2014](#page-19-0)). Notably, in this case, the scaling between ribosome content and growth rate was largely lost (Figure 2B, bottom); in fact, most mutants that reduced growth rate had a minor effect on the expression of genes coding for ribosomal proteins. As a result, growth-affecting mutations expressed more ribosomal protein coding genes than expected given their growth rate. We observed a similar deviation from the scaling curve in cells forced to express a large amount of unstable mRNA (Figure 2—figure supplement 1B–C): here as well, growth rate decreased but expression of ribosomal proteins remained largely unaffected. Therefore, while the scaling of ribosome content with growth rate is well conserved in wild-type strains growing in different conditions, it is not maintained upon growth-affecting genetic mutations, primarily because most growth-affecting mutants do not lead to a comparable reduction in the

ribosome content. This implies that the tight correlation between ribosome content and growth rate results from environmental signaling, rather than direct causal relationships ([Levy et al., 2007](#page-19-0)).

#### Quantitative interpretation of the scaling between ribosome content and growth rate

The scaling relation between ribosome content, r, and the specific growth rate, , is typically interpreted based on the general relationship between protein translation and growth rate during balanced growth. In balanced growth, the cellular protein content doubles at each cell division. Since proteins are translated by active ribosomes, this leads to the relationship: = g ra, where g is the rate of ribosome translation and ra the proteome fraction encoding for actively translating ribosomal proteins ([Kafri et al., 2016a](#page-19-0); [Maaløe, 1979](#page-19-0); [Scott et al., 2010](#page-19-0)). Note that g 1 is the time required for one ribosome to duplicate itself, defining a limit on the minimal cell cycle time.

Written differently, this relationship connects the two parameters we measure: the specific growth rate , and the proteome fraction encoding for the total (translating and non-translating) ribosomal proteins, r:

$$r = \gamma^{-1}\,\mu + r_0 \tag{7}$$

Here, r0 ¼ r ra, is the proteome fraction encoding for residual ribosomal proteins that are not actively translating. In principle, both parameters of this theoretical relation, g and r0, could change with nutrient conditions. The measured scaling curve, [Figure 2A](#page-4-0), however, implies that these parameters are tightly coordinated to maintain the linear relationship between ribosome content and growth rate across the different conditions.

We can estimate g using measured parameters: the translation elongation rate (10.5 aa/sec) ([Waldron et al., 1977](#page-20-0)) and the number of amino-acids composing the budding yeast ribosome (12,485 A.A). By this estimation, g 1 = 20 min. This predicted value is » 30% smaller than our measured slope Dr/Dm = 21/ln(2) min, ([Figure 2A–B](#page-4-0)). Of note, the same quantitative difference between the estimated and measured slope was seen in bacteria, where it was interpreted as inaccuracy in estimated g, stemming from the need to include additional translation factors that function within the active ribosomal unit ([Scott et al., 2010](#page-19-0)). Alternatively, this discrepancy between the estimated and measured slope may reflect inaccuracies of the simplified theory. For example, any time-delay between production of ribosomal protein and their contribution to the actively translating pool will increase the predicted g 1 . In addition, the measured slope would be precisely explained if ribosome production in each cell was effectively linear rather than exponential. In this case, the relation between the specific growth rate m and the division time Td is m = 1/Td rather than m = ln(2)/Td.

#### A constant fraction r0 » 8% of the proteome encodes for nontranslating ribosomal proteins

In our analysis above, we distinguished the fraction of ribosomal proteins that contribute to cell growth (=g ra) from the residual inactive, non-translating part (r0). The fact that inactive r0 exists is most easily appreciated at the regime of very slow, or no-growth. In this regime, ribosomal proteins still account for r0 » 8% of the proteome ([Figure 2A–B](#page-4-0)). As protein translation in arrested cells is dramatically reduced, the majority of these ribosomes remain inactive. Indeed, arrested cells were shown to maintain a pool of inactive ribosomes ([Dai et al., 2016](#page-18-0); [van den Elzen et al., 2014](#page-20-0)).

By extension, residual inactive ribosomes may also be present in proliferating cells. Using the simplified model above, Equation (1), the fraction of this inactive pool r0 can be estimated from the measured scaling curve, provided that we know the ribosome elongation rates g. If elongation rate g is constant and g is independent of growth rate, then the residual fraction also remains a constant r0 = 8% at all growth rate. By contrast, if elongation rate changes with growth rate, g = a1 , where a is a fixed growth-rate independent constant, then the residual fraction should also increase with the growth rate, r0 = ra. Similarly, the r0/r, fraction of inactive ribosomes out of the total ribosome pool, will increase with decreasing growth rate if elongation rate is constant ([Figure 3A](#page-6-0)), but will decrease with decreasing growth rate if elongation rate changes.

Previous measurements of elongation rates in different media gave conflicting results. In one report, measured elongation rates in glucose and in media lacking nitrogen remain the same despite a two-fold reduction in growth rate ([Waldron et al., 1977](#page-20-0)). By contrast, a second report suggested

<span id="page-6-0"></span>![](_page_6_Figure_2.jpeg)

Figure 3. A substantial fraction of ribosomes is not actively translating at a given time. (A) Estimating the fraction of inactive ribosomes using polysomal profiling: Cells were grown in the indicated conditions and their ribosomal content was analyzed on sucrose gradients as an indication for translational activity. Representative profiles of raw data are shown. The fraction of inactive ribosomes was estimated by the ratio (r) of monosomes (mRNAs bound by a single ribosome), the 40S and 60S, to the total ribosome density and is plotted in (B) in corresponding colors (dashed circles). (B) The fraction on inactive ribosomes increases with decreasing growth rate: Shown is the estimated fraction of inactive ribosomes (Y = 0.08/(0.35X + 0.08)) for each condition, as a function of cell generations per hour. Conditions as specified in [Figure 2A](#page-4-0). Dashed circles represent the inactive/total ribosomes ratio (r) calculated in (A) from three independent experiments. [DOI: https://doi.org/10.7554/eLife.28034.007](https://doi.org/10.7554/eLife.28034.007)

that elongation rates decrease by » 40% in media lacking acetate ([Bonven and Gulløv, 1979](#page-18-0)). These measurements used the same experimental setup but differed in the details of the analysis (See Material and methods).

To more directly examine the fraction of inactive ribosomes in growing cells, we performed polysomal profiling. In this method, the optical density of RNAs is quantified, with polysomes consisting of mRNA bound by multiple ribosomes. Most actively translating ribosomes are found in the polysomes while monosomes (mRNA with a single ribosome) are thought to contain primarily inactive ribosomes ([Aspden et al., 2014](#page-18-0); [Castelli et al., 2011](#page-18-0); Chasse´ [et al., 2017](#page-18-0); [Kapp et al., 2004](#page-19-0); [Kelen et al., 2009](#page-20-0); [Liu and Qian, 2016](#page-19-0); [Warner et al., 1963](#page-20-0)). Therefore, using the monosome to polysome ratio, we can estimate the fraction of inactive ribosomes out of the total ribosome pool and compare it to the predicted behavior, as above.

We examined the polysomal profiles in cells growing in standard medium (SC), in cells growing in low phosphate and low-nitrogen media. A substantial inactive fraction (monosomes) was observed in the fast growing cells (r = 0.18), which further increased in the slow-growing cells (Figure 3A). Notably, the relative fraction of monosomes increased with decreasing growth rate. Further, this fraction was in quantitative agreement with the predicted fraction of inactive ribosomes assuming a constant residual r0 » 8% (Figure 3B). We conclude that during steady growth, cells devote a constant, » 8% of their proteome to producing ribosomes that are not actively translating. Therefore, the fraction of ribosomes that are not actively translating ranges from » 25% in rapidly growing cells (r0 » 8% relative to the total ribosomal fraction r » 30%) and further increases with decreasing growth rate ([Figure 3B](#page-6-0)).

### The scaling between ribosome content and growth rate changes when cells prepare to enter stationary phase

Cells may use the excess ribosomes to accommodate an increase in translation demands. Entering stationary phase represents such a scenario ([Ju and Warner, 1994](#page-19-0)), as the reduced availability of nutrients requires an increased production of glycolytic enzymes ([Figure 1—figure supplement 1A](#page-3-0), right column). To examine whether the fraction of non-translating ribosomes (r0) decreases during this transition, we analyzed published proteomic profiles of batch-grown cultures entering stationary phase ([Murphy et al., 2015](#page-19-0)). Indeed, ribosomal content decreased » 2.5 generations (four hours) before growth rate dropped ([Figure 4A](#page-8-0), [Figure 4—figure supplement 1C](#page-9-0)). This could be interpreted in two ways: a transient increase in translation elongation rate, or a more efficient use of available ribosomes, leading to lower r0.

Similarly, transcription profiling of cells shifted to low phosphate medium showed an early decrease in ribosomal protein transcripts, ~5 hr before growth rate was reduced ([Figure 4—figure](#page-9-0) [supplement 1A–B](#page-9-0)). Therefore, during preparation for stationary phase, cells reduce their overall ribosomal content but maintain a stable growth rate. Again, this could be interpreted in two ways: either cells transiently increase the translation elongation rate, or they make a more efficient use of the already available ribosomes by decreasing the non-translating fraction r0.

### The scaling between ribosome content and growth rate changes during nutrient upshift

Translation demands increase when cells undergo nutrient upshifts. Under such conditions, nutrient limitation is lifted, and cells can resume fast growth practically immediately, provided that sufficient ribosomes are available to enable faster protein production. We examined the kinetics by which cells resume fast growth upon nutrient upshift by reanalyzing data we previously acquired. First, we found that continuous cell cultures growing in a phosphate-limited chemostat responded to a spike of high phosphate within minutes ([Figure 4B](#page-8-0)). Second, using microfluidic-coupled microscopy, we followed individual cells transferred from galactose to glucose and observed that cells increase their growth rate from 0.36 gen/hr to 0.54 gen/hr within minutes of the transfer ([Figure 4C](#page-8-0)). Therefore, cells increase their growth rate and thus protein production rates within a short time that appears insufficient for synthesizing new ribosomes. This is consistent with either a transient increase in translation elongation rate, or a more efficient use of available ribosomes through a decrease in the non-translating fraction r0.

To more directly correlate the ribosomal protein content and growth rate during nutrient upshift, we profiled the proteome of cells transferred from early stationary phase (OD600 » 6) to fresh media (SC). Cells resumed rapid growth of 0.55 gen/hr following a » 40 min lag. Proteome profiling revealed that ribosome content did not change during the lag time, but began to increase only after cells had attained fast growth ([Figure 4D](#page-8-0)). Since active translation in stationary cells is minimal, we can conclude that in this case, cells necessarily made use of the residual ribosomal fraction r0 for initiating growth. In fact, it appears that to account for the observed growth rate during this initial growth phase, cells need to utilize the vast majority of expressed ribosomes.

### The excess of non-translating ribosomal proteins is decreased when cells are forced to express unneeded proteins

The linear scaling between ribosome content and growth rate is adjusted when cells face increased translation demands. This implies a transient increase in translation elongation rate or employment of the residual ribosomal fraction r0. To better understand these effects, we asked whether increasing translation demands during logarithmic growth will also alter the scaling relation between ribosome content and growth rate. To this end, we engineered cells to constitutively produce high mCherry protein amounts using a system we previously described ([Kafri et al., 2016b](#page-19-0)). In this

<span id="page-8-0"></span>![](_page_8_Figure_2.jpeg)

Figure 4. The scaling between ribosome content and growth rate changes when balanced growth is perturbed. (A) The scaling between ribosome content and growth rate changes when cells prepare to enter stationary phase: Cells grown in batch culture were followed as they increase in density (top panel), and were subjected to proteome profiling at the indicated times. The bottom panel shows the fraction of proteome coding for ribosomal proteins along this time course. Data taken from [Murphy et al. (2015)](#page-19-0). Color gradient from light to dark represents increasing time. (B) Cells growing Figure 4 continued on next page

<span id="page-9-0"></span>![](_page_9_Picture_0.jpeg)

#### Figure 4 continued

in a phosphate-limited chemostat increase their growth rate immediately upon phosphate addition: Continuous cultures were grown to steady state in a phosphate-limited chemostat at a dilution rate of 4.5 hr/gen. Media containing high phosphate was then injected into the growth chamber (dotted line). Shown is the density (OD) of the culture before and after phosphate injection. (C) Cells increase their volume growth immediately upon transfer to a preferred carbon source: Cells were imaged using microfluidics-coupled live-cell microscopy, while their carbon source was changed from galactose to glucose (dotted line). Shown is the cell volume before and after the media change, averaged over 50 cells. Specific growth rate was calculated 25 min before and after the upshift. Error bars represent SEM. (D) Cells exiting stationary phase resume rapid growth before increasing the ribosome content: Cells were grown in SC until saturation (OD600 » 6) and diluted back into fresh media at time 0 (empty circle). OD measurements were taken every 15–20 min as shown in the top panel and samples for proteomic analysis were taken as indicated. The ribosomal fraction, as calculated from the proteomic data, is shown in the bottom panel as a function of cell growth rate. Each point is the median of three biological repeats. [DOI: https://doi.org/10.7554/eLife.28034.008](https://doi.org/10.7554/eLife.28034.008)

The following figure supplement is available for figure 4:

Figure supplement 1. The scaling between ribosome content and growth rate changes when balanced growth is perturbed. [DOI: https://doi.org/10.7554/eLife.28034.009](https://doi.org/10.7554/eLife.28034.009)

> analysis, a library of strains expressing increasing amounts of mCherry proteins was generated, with mCherry levels reaching » 25% of the total proteome in the highest burden cells.

> We measured the growth rate and profiled the proteome for cells with different levels of mCherry burden in three conditions: standard medium (SC), medium containing low-phosphate and medium containing low nitrogen ([Figure 5—figure supplement 1C](#page-10-0)). As expected, in all three conditions, growth rate and the ribosomal fraction r decreased with increasing mCherry levels ([Figure 5A](#page-10-0)). For each level of burden, we plotted the ribosome content as a function of the cell's growth rate in the three growth conditions examined ([Figure 5A](#page-10-0), [Figure 5—figure supplement 1A](#page-10-0)). Burden cells showed a scaling relation that was highly similar to that observed in wild-type cells: when compared across conditions, ribosome content decreased linearly with growth rate. Notably, the slope of this scaling curve was independent of the level of the burden, suggesting that ribosome translation rates g remained invariant to the burden.

> In contrast, the burden did affect the residual r0, the predicted proteome fraction coding for nontranslating ribosomes. This can be easily appreciated from the zero-growth limit. In this limit, ribosome content in high burden cells decreased from its wild-type value (r0 = 8.1%) to a lower value, r0 = 5.5% ([Figure 5—figure supplement 1B](#page-10-0)). Therefore, the burdened cells employ a higher fraction of their ribosomes, leaving a smaller pool of inactive ribosomes.

#### The smaller pool of excess ribosomes in burden cells explains their delayed recovery from starvation

Burdened cells exit starvation with a delay ([Kafri et al., 2016b](#page-19-0); [Shachrai et al., 2010](#page-20-0)). As we showed above, during this exit, cells employ a large fraction of the excess ribosomal proteins r0 to initiate protein translation and cell growth. We therefore reasoned that the delayed recovery of the burdened cells may be due to the smaller fraction of the residual ribosomes, r0, they express.

To examine this prediction, we quantified the delay introduced by the burden when cells recovered from starvation ([Figure 5B](#page-10-0); data from [Kafri et al. (2016b)](#page-19-0). As expected, this delay increases linearly with the mCherry burden. Notably, the differences in recovery times were quantitatively explained by the difference in r0; for example, the recovery time of eight-copy burden cells was prolonged by » 15%, consistent with the corresponding » 15% decrease in the measured r0.

#### The fraction of ribosomes actively translating is tuned by growth conditions, but remains largely invariant to cell growth rate

A central implication of the scaling relation between ribosome content and growth rate ([Figures 2A](#page-4-0) and [5A](#page-10-0)) is that when growth conditions change, cells tune not only the fraction of the proteome encoding for ribosomal proteins (r), but also the ratio between active and inactive ribosomes (ra/r0). As a direct consequence of the scaling law, when comparing cells that grow in different conditions, the predicted ratio ra/r0 increases linearly with growth rate ([Figure 6A](#page-11-0)). However, comparing between wild-type and burden cells, we noted that this ratio is invariant to the burden and depends mainly on the growth condition ([Figure 6A](#page-11-0)). Thus, although burden cells grow more slowly and express a smaller ribosomal protein fraction compared to wild-type cells, the fraction which is

<span id="page-10-0"></span>![](_page_10_Figure_2.jpeg)

Figure 5. Forcing unneeded protein production reduces the pool of free ribosomes. (A) Scaling of ribosomal proteins with growth rate in proteinburdened strains: Five strains expressing increasing amounts of mCherry proteins were generated and their proteome profiles and growth rates were measured in the three indicated conditions. Shown is the proteome fraction coding for ribosomal proteins in each strain and in each condition as a function of the cell growth rate. Different strains are indicated by the different shades of colors, with increased burden corresponding to a darker circle. Colors indicate the specific condition used. The two dashed lines correspond to the scaling curves defined by the no-burden and most highlyburdened strains: The top line is the same as in [Figure 2A](#page-4-0), while the bottom line describes the ribosome content of the highest burden as a function of its growth rate in the different conditions. Error bars represent the standard deviation around the median between three biological repeats. (B) The reduced pool of inactive ribosome in the burden cells quantitatively accounts for their delayed exit from starvation: Cells expressing different amounts of mCherry proteins were grown to saturation. The cells were kept in stationary for a week before diluting them back to fresh media. Recovery times (Y axis) were defined by the time at which cells' optical density increased by 50%. Solid blue line represents the predicted recovery time based on the fold-reduction in free ribosomes, dashed black line is the data's linear fit. Data is re-plotted and analyzed from [Kafri et al. (2016b)](#page-19-0). [DOI: https://doi.org/10.7554/eLife.28034.010](https://doi.org/10.7554/eLife.28034.010)

The following figure supplement is available for figure 5:

Figure supplement 1. Cells forced to produce unneeded proteins have a smaller pool of excess ribosomes. [DOI: https://doi.org/10.7554/eLife.28034.011](https://doi.org/10.7554/eLife.28034.011)

> employed in translation is predicted to remain the same as in wild-type cells, depending only on the growth conditions. To verify this prediction, we used polysomal profiling of wild-type and burdened strains growing in different conditions. As predicted, the ratio of polysomes to monosome remains largely invariant between wild-type and burden cells growing in the same condition, despite their different growth rate ([Figure 6B](#page-11-0)).

> Taken together, we propose that the tuning of ribosome content and of the fraction of ribosomes that are actively translating depend primarily on signaling from the environment with little contribution of internal growth-rate dependent feedback ([Figure 7](#page-11-0)). Thus, although both parameters show a tight correlation with growth rate, this correlation is not direct, but rather results from evolutionarily tuned signaling.

# Discussion

Ribosome production and function are the major resource-consuming processes in cells and its tight control in response to a variety of signals is well appreciated ([Nomura et al., 1984](#page-19-0); [Reuveni et al.,](#page-19-0) [2017](#page-19-0); [Schaechter et al., 1958](#page-19-0); [Schimmel, 1993](#page-19-0); [Scott et al., 2010](#page-19-0); [Warner, 1999](#page-20-0)). Our study

<span id="page-11-0"></span>![](_page_11_Figure_2.jpeg)

Figure 6. The ratio of active to inactive ribosomes remains invariant to protein burden. (A) The ratio of active to inactive ribosome predicted by proteomic data: Shown is the ratio of active to inactive ribosomes ra/r0, as predicted from our analysis of proteomic data as a function of growth rate in wild-type and high-burden cells. Note that while this ratio decreases with growth rate when comparing the same strain across different conditions, it remains almost invariant to the burden when compared between two strains within the same condition. (B) The ratio of active to inactive ribosomes predicted by polysome profiling: Low and high burden cells were grown in the indicated conditions and their ribosomal content was analyzed on sucrose gradients as an indication for translational activity. Representative profiles of raw data are shown on top. The ratio between the polysomes (active transcripts with more than one ribosome bound) to detached small and large subunits (40S and 60S) together with the monosomes (mRNAs bound by a single ribosome) is shown in a bar graph on bottom. SEM error bars are from three biological independent repeats. [DOI: https://doi.org/10.7554/eLife.28034.012](https://doi.org/10.7554/eLife.28034.012)

![](_page_11_Figure_4.jpeg)

Figure 7. Model for ribosome allocation. Cells tune their ribosome content and ribosome efficiency based on signaling from the environment. Evolutionary tuning of this signaling results in a precise scaling of ribosome content with growth rate during logarithmic growth. Growth-rate dependent feedbacks play a minor role in the tuning of ribosome content or efficiency. [DOI: https://doi.org/10.7554/eLife.28034.013](https://doi.org/10.7554/eLife.28034.013)

quantified a specific aspect of this control that is particularly relevant for understanding the overall regulation of cellular physiology: how the proteomic fraction coding for ribosomal proteins is regulated with growth rate. This question was extensively studied in theoretical models and experimentally in bacteria ([Brauer et al., 2008](#page-18-0); [Bremer and Ehrenberg, 1995](#page-18-0); [Nomura et al., 1984](#page-19-0); [Schaechter et al., 1958](#page-19-0); [Scott et al., 2010](#page-19-0); [Waldron and Lacroute, 1975](#page-20-0); [Warner, 1999](#page-20-0); [Zaslaver et al., 2009](#page-20-0)), but received relatively little attention in eukaryotic models.

We find that also in budding yeast, the proteome fraction coding for ribosomal proteins scales linearly with the specific growth rate. The value of this slope is predicted by models of balanced growth to be precisely the time required for translation elongation of a full ribosome. The quantitative agreement of our measured slope and this predicted value depends on whether we assume exponential or linear production of ribosomes at each individual cell cycle time: a precise agreement is obtained if linear growth is assumed, while the values differ by » 30% ( » ln(2)) if growth is exponential. Notably, this was also the case in bacteria: the measured slope differed by » 30% form the predicted slope calculated by assuming exponential growth. Since growth is indeed thought to be exponential, the discrepancy between the measured and predicted slopes may be explained by inaccuracies of the simplified theory, e.g. failure to include possible delays between ribosome production and usage which may effectively shift growth dynamics closer to linear growth. Alternatively, studies in bacteria attributed this difference to the need for including additional proteins for production of functional ribosomal unit, which may change the predicted value of the slope ([Klumpp et al.,](#page-19-0) [2013](#page-19-0); [Scott et al., 2010](#page-19-0)).

We provided evidences that a constant fraction ( » 8%) of the budding yeast proteome encodes for ribosomal proteins that are not actively translating at a given time. Accordingly, even rapidly growing cells still maintain » 25% of their ribosomal proteins inactive. This result is surprising in light of the prevailing notion that cells tune their proteome composition in a way that ensures that all expressed ribosomal proteins are employed at full capacity ([Bosdriesz et al., 2015](#page-18-0); [Dekel and](#page-18-0) [Alon, 2005](#page-18-0); [Kafri et al., 2016a](#page-19-0); [Keren et al., 2013](#page-19-0); [Klumpp et al., 2013](#page-19-0); [Koch, 1988](#page-19-0); [Maa](#page-19-0)[løe, 1979](#page-19-0); [Scott and Hwa, 2011](#page-20-0); [Scott et al., 2010](#page-19-0); [2014](#page-20-0); [Vind et al., 1993](#page-20-0)). Notably, while the scaling of ribosome with growth rate was maintained over a range of conditions, it was readily altered by different perturbations. These included genetic mutations, as well as transient perturbations that led to a sudden increase in translation demands. Further, forcing the cells to express unneeded proteins effectively reduced the residual fraction of unused ribosomes, leading to a more efficient ribosome usage. These results imply that the scaling curve we observed, and in particular the fraction of residual, inactive ribosomes, is not an inherent unavoidable consequence of cell physiology or the mechanism of ribosome production, but is rather a consequence of signaling which sets the ratio of active vs. inactive ribosomes as a function of growth conditions.

Why would cells express extra ribosomes more than needed for meeting translation demands? We suggest that growing cells produce extra ribosomes in order to prepare for fluctuating conditions. Consider a nutrient upshift during growth in poor media: when the nutrient limitation is lifted, cells can increase their growth rate, provided that sufficient ribosomes are available. If cells were to precisely tune their ribosomal content to express only the ribosomes needed for the slow growing conditions, their level would limit the speed by which rapid growth could be resumed upon nutrients supply. Previous studies described strategies by which cells prepare for changing conditions, including the activation of stress genes by moderate stresses ([Gasch et al., 2000](#page-18-0); [Guan et al., 2012](#page-18-0); [Levy et al., 2011](#page-19-0); [Mitchell et al., 2009](#page-19-0)) and the transition of a subpopulation into persistence ([Balaban et al., 2004](#page-18-0); [Levy et al., 2012](#page-19-0); [Soll and Kraft, 1988](#page-20-0); [Yaakov et al., 2017](#page-20-0)). Our study suggests that this requirement to be prepared for fluctuating conditions affects not only specific genes or pathways, but also the central cellular processes such as the coordination of ribosome production with growth rate.

# Materials and methods

### Media and strains

All budding yeast (S. cerevisiae) strains in this study were based on BY4742 ([Brachmann et al.,](#page-18-0) [1998](#page-18-0)) or Y8205 ([Hin Yan Tong and Boone, 2005](#page-18-0)) laboratory strains. TDH3p-mCherry (protein fusion) was made by fusing the mCherry cassette to the end of the Tdh3 protein using a standard PEG;LiAC;ssDNA transformation protocol ([Gietz and Woods, 2002](#page-18-0)). The rest of the strains were constructed previously ([Kafri et al., 2016b](#page-19-0)).

Strains were grown in SC medium or in SC medium depleted of a specific nutrient, as described in the main text. SC limiting media was prepared from YNB without the relevant nutrient (Low Phosphate medium - ForMedium, CYN0804, Low Nitrogen medium - BD 3101130). Phosphate depleted medium was prepared by adding phosphate in the form of KH2PO4 to a final concentration of 0.2 mM. The level of potassium was preserved by adding KCl (instead of KH2PO4). Nitrogen limiting medium was prepared from YNB without amino acids and ammonium sulfate (BD 3101130) by adding separately 50 mM of ammonium sulfate and the essential amino acids.

### Growth rates

Growth rates were calculated from the data reported previously ([Kafri et al., 2016b](#page-19-0)). We further validated the growth rates in the various conditions by log-phase competition assays ([Figure 5—figure](#page-10-0) [supplement 1A](#page-10-0)), as described below.

### Log-phase competition

Cells were grown overnight to stationary phase in the relevant media. GFP and mCherry strains were then co-incubated in the specified media at 30˚C. The initial OD was set to » 0.05, and the WT initial frequency was » 50% of the total population. The number of generations was calculated from the dilution factor. Frequencies of GFP versus mCherry cells were measured by flow cytometry. The cells were diluted every » 8 hr. Experiments were done with WT strains expressing GFP vs. mCherry burdened cells. A linear fit of the log2 for the WT frequency was used to calculate the slope for each competition. The relative fitness advantage is calculated from the slope divided by log2. The slope of the strains' fitness advantage and their copy number were used to calculate the growth effect per burden copy.

### Protein measurements

#### Sample preparation

All chemicals are from Sigma Aldrich, unless stated otherwise. Samples were subjected to in-solution tryptic digestion using a modified Filter Aided Sample Preparation protocol (FASP). Sodium dodecyl sulfate buffer (SDT) included: 4%(w/v) SDS, 100 mM Tris/HCl pH 7.6, 0.1M DTT. Urea buffer (UB): 8 M urea (Sigma, U5128) in 0.1 M Tris/HCl pH 8.0 and 50 mM Ammonium Bicarbonate. Cells were dissolved in 100 mL SDT buffer and lysed for 3 min at 95˚C and then centrifuged at 16,000 RCF for 10 min. 100 ug total protein were mixed with 200 mL UB and loaded onto 30 kDa molecular weight cutoff filters and centrifuged. 200 ml of UB were added to the filter unit and centrifuged at 14,000 x g for 40 min. Alkylation using 100 ml IAA, 2 washed with Ammonium Bicarbonate. Trypsin was then added and samples incubated at 37˚C overnight. Additional amount of trypsin was added and incubated for 4 hr at 37˚C. Digested proteins were then spun down to a clean collecting tube, 50 ul NaCl 0.5M was added and spun down, acidified with trifloroacetic acid, desalted using HBL Oasis, speed vac to dry and stored in 80˚C until analysis.

### Liquid chromatography

ULC/MS grade solvents were used for all chromatographic steps. Each sample was loaded using split-less nano-Ultra Performance Liquid Chromatography (10 kpsi nanoAcquity; Waters, Milford, MA, USA). The mobile phase was: (A) H2O + 0.1% formic acid and (B) acetonitrile +0.1% formic acid. Desalting of the samples was performed online using a reversed-phase C18 trapping column (180 mm internal diameter, 20 mm length, 5 mm particle size; Waters). The peptides were then separated using a T3 HSS nano-column (75 mm internal diameter, 250 mm length, 1.8 mm particle size; Waters) at 0.35 mL/min. Peptides were eluted from the column into the mass spectrometer using the following gradient: 4% to 22%B in 145 min, 22% to 90%B in 20 min, maintained at 95% for 5 min and then back to initial conditions.

### Mass spectrometry

The nanoUPLC was coupled online through a nanoESI emitter (10 mm tip; New Objective; Woburn, MA, USA) to a quadrupole orbitrap mass spectrometer (Q Exactive Plus, Thermo Fisher Scientific, Bremen, Germany) using a FlexIon nanospray apparatus (Proxeon). Data was acquired in DDA mode, using a Top20 method. MS1 resolution was set to 70,000 (at 400 m/z) and maximum injection time was set to 20msec. MS2 resolution was set to 17,500 and maximum injection time of 60 ms.

### Data processing and basic analysis

Raw data was imported into the Expressionist software (Genedata, San Francisco, CA) and processed as described previously ([Shalit et al., 2015](#page-20-0)). The software was used for retention time alignment and peak detection of precursor peptides. A master peak list was generated from all MS/MS events and sent for database searching using Mascot v2.5 (Matrix Sciences) and MSGF+ (Integrative Omics, <https://omics.pnl.gov/software/ms-gf>). Data was searched against the Saccharomyces cerevisiae (strain ATCC 204508/S288c) protein database as downloaded from UniprotKB [(http://www.uni](http://www.uniprot.org/)[prot.org/)](http://www.uniprot.org/), and appended with 125 common laboratory contaminant proteins as well as the mCherry protein sequence (Uniprot accession X5DSL3). Fixed modification was set to carbamidomethylation of cysteines and variable modifications were set to oxidation of methionines and deamidation of N or Q. Search results were then filtered using the PeptideProphet algorithm ([Keller et al., 2002](#page-19-0)) to achieve maximum false discovery rate of 1% at the protein level. Peptide identifications were imported back to Expressionist to annotate identified peaks. Quantification of proteins from the peptide data was performed using an in-house script ([Keller et al., 2002](#page-19-0)). Data was normalized based on the total ion current. Protein abundance was obtained by summing the three most intense, unique peptides per protein. A Student's t-Test, after logarithmic transformation, was used to identify significant differences across the biological replica. If not annotated differently in the figure caption, the analysis was based on the median of three biological repeats.

### Proteomic datasets

We analyzed the following external mass-spectrometry datasets: two carbon sources ([Paulo et al.,](#page-19-0) [2015](#page-19-0), [2016](#page-19-0)) and diauxic shift dataset ([Murphy et al., 2015](#page-19-0)). In each dataset, we normalized our glucose condition (SC) or time zero to the literature (median of 13 rich conditions from PaxBD ([Wang et al., 2012](#page-20-0)), see conditions in [Figure 1—figure supplement 1B](#page-3-0), 'Rich Conditions'). For the analysis, we used only the genes also present in our dataset. The carbon source data set ([Paulo et al., 2015](#page-19-0), [2016](#page-19-0)) growth rates were acquired from the papers or via personal communication. The diauxic shift data set ([Murphy et al., 2015](#page-19-0)) growth rates were calculated from [Figure 4A](#page-8-0) upper panel by applying a growth curve to the data.

### Gene groups

We divided the proteome into 12 groups, 11 of which were based on SGD GO annotations or KEGG annotations, which together account for 80% of the proteome (by protein abundance). The rest were grouped together as an additional 12th group. There is a small overlap between the groups, See [Supplementary file 1](#page-17-0) for the protein names in each group.

### RNAseq transcription protocol and analysis

As described in [Voichek et al. (2016)](#page-20-0). The analysis was based on the median of 6–8 biological repeats (SC/Low N – 8; Low Pi – 6).

### Published RNA datasets

The different carbon datasets ([Figure 2B](#page-4-0),upper panel - dark orange, [Figure 2—figure supplement](#page-4-0) [1](#page-4-0)) were obtained from ([Gasch et al., 2000](#page-18-0)). Data for strains growing on xylulose ([Figure 2B](#page-4-0), upper panel - yellow) were obtained from ([Tamari et al., 2014](#page-20-0); [2016](#page-20-0)). We compared ribosomal fractions from the RNAseq data with ChIP data between the two papers, and excluded from the analysis strains with high variability. Deletion strains ([Figure 2B](#page-4-0), bottom panel) datasets were obtained from ([Kemmeren et al., 2014](#page-19-0); [O'Duibhir et al., 2014](#page-19-0)).

## RNA data of cells growing in low Pi media (0.06 mM)

Logarithmic WT cells ( » 0.4 OD) grown in SC medium were washed in low Pi (0.06 mM) media. Cells were then diluted and inoculated into fresh low Pi media to an initial OD of 0.05. Cells were grown at 30˚C for several hours. At every time point, as indicated at [Figure 4—figure supplement 1A](#page-9-0), a sample was extracted and RNAseq was performed as described above.

### Ribosomal fraction analysis in the RNA data

The ribosomal fraction was calculated by dividing the number of mRNA reads of ribosomal proteins by the total reads. In the ChIP data, the ribosomal fraction was calculated as the relative amount normalized to the SC condition (Logarithmic mean).

### mCherry level measurements

The mCherry levels were calculated from our mass-spectrometry data. To compare the mCherry between the different copy numbers, we used the relative mass-spectrometry data. In order to measure our reference strain (one copy in SC), we used TDH3p-mCherry (protein fusion) strain as described in the Strains section. The amount of mCherry in this strain is equal to the TDH3 protein ( » 2.2% base on the iBAQ parameter). The mCherry level of our one mCherry copy strain was 14.5% lower than the mCherry fluorescence of the TDH3p-mCherry strain. Thus, the mCherry level of one copy strain is » 1.9% of the proteome. Notably, this calibration is in very good agreement with our previous fluorescence-based calibration ([Kafri et al., 2016b](#page-19-0)).

### Chemostat experiments

The chemostat experiments were performed using a DASBox Bacterial Fermentation system (DAS-GIP, Eppendorf). Yeast cells containing one copy number were grown in the fermenter at 30˚C in SC with low Pi. The fermentation started with the inoculation of an exponentially growing starter at OD6000.5–1 diluted x10. Culture was initially grown in a batch mode for 6–8 hr to an OD600 between 0.5–1. The culture was then switched to chemostat mode with final doubling time of 3 hr/doubling. When the culture reached steady state in the chemostat, we spiked in the same media containing high levels of phosphate for three hours.

### Live microscopy environment perturbation experiment

Cells were grown in a standard FCS2 flow cell (Bioptechs) improved similar to ([Charvin et al., 2010](#page-18-0)). Cells growing in the flow cell were imaged with Olympus IX81 microscope with automated stage and ZDC autofocus and cooled CCD camera (Hamamatsu). In order to grow the cells in a planar layer while simultaneously control their extra cellular environment, standard FCS2 flow cell (Bioptechs) was used and improved in the following way ([Charvin et al., 2010](#page-18-0)): the 40 mm round cover slips were coated with a thin layer of PDMS (Sylgrad 184, GE). This was done in a clean room, using a suitable spinning procedure to achieve layers of » 30 mm thick. To confine the cells and prevent their movement, while medium is flowing through the chamber ([Balaban et al., 2004](#page-18-0)), a diffusive cellulose membrane, which was cut from dialysis tubes (Sigma Aldrich, D9527) was used.

The membrane was cleaned and prepared as described in [Charvin et al., 2010](#page-18-0). The cells are confined between the PDMS layer and the membrane, while the medium is flowing above the membrane. In this way the medium reaches the cells area without exerting too much force on the cells. The flow cell is then connected to two micro-perfusion pumps (Instech), which are controlled automatically by the computer via a D/A convener (measurement computing usb-3110), enabling the exchange of medium during the experiment without perturbing the cells. The media tanks for the environment perturbation experiment were continuously stirred, to allow for better aeration of the medium. The whole system is controlled by ImagePro 6.3.1 (Media Cybernetics).

Galactose to glucose shift experiments were done by replacing synthetic complete medium based on 2% galactose (SC-Gal) with synthetic complete medium based on 2% glucose (SC-Glu). Cell volume was estimated from the bright field images assuming that the yeast cells are prolate spheroids.

### Polysome profiling

100 ml cultures of exponentially growing yeast cells (OD6000.2–0.4 with at least 6 exponential divisions since their dilution) were treated with 100 mg/ml Cycloheximide for 5 min at 30˚C, harvested, and then washed with cold Buffer AB (20 mM Tris-HCl pH 7.4, 50 mM KCl, 10 mM MgCl2, 1 mM DTT and 100 mg/ml Cycloheximide). The cells were lysed with 250 ml Buffer AB using glass beads as in ([Pospı´sek and Vala´sek, 2013](#page-19-0)). Cleared lysates were loaded onto 10–50% sucrose gradients with the same AB buffer composition containing 100 mg/ml Cycloheximide, and centrifuged at 39,000 RPM in a SW41 rotor for 2.5 hr at 4˚C. Gradients were continuously recorded using Biocomp gradient station at OD254 nm.

Profiles were aligned in the x-axis by their peak locations and in the y-axis to SC low burden. A blank sucrose gradient measurement was subtracted from all the samples. Next, the integrals of the 40S, 60S, monosomes and polysome fractions (disomes and higher) were measured. The ratio of polysomes to monosomes was defined as the (polysomes integral) / (40S + 60S + monosomes integrals). Both mRNA and rRNA are present in a given mono/polysome signal measured by the OD254 nm absorbance. The total length of rRNA in one ribosome is about 5500 bp while the average mRNA length is 1250 bp ([Milo et al., 2010](#page-19-0)). Accordingly, the rRNA fraction of total RNA in monosomes is ~82%, and in polysomes containing 5 ribosomes is ~96%. This differential proportion was taken into account when calculating the inactive ribosome fraction.

### Interpreting polysomal profiles

Our analysis of the polysomal profiles ([Figure 3](#page-6-0) and [Figure 6](#page-11-0)) assumes that the monsome fraction represents the inactive ribosomes.

The common notion is that the majority of monosomes represent inactive ribosomes that are not actively translating ([Aspden et al., 2014](#page-18-0); [Castelli et al., 2011](#page-18-0); Chasse´ [et al., 2017](#page-18-0); [Kapp et al.,](#page-19-0) [2004](#page-19-0); [Kelen et al., 2009](#page-20-0); [Liu and Qian, 2016](#page-19-0); [Warner et al., 1963](#page-20-0)). This notion was recently challenged by a study which analyzed monosomes using ribosome profiling, identifying some translation in the monosome peak, although the fraction of monosomes that is actively translating remained unclear ([Heyer and Moore, 2016](#page-18-0)). Of note, monosomal ribosomes were shown to be present primarily in the vicinity of the translation start site, with only a minor fraction spread across the full gene, in sharp contrast to the ribosomes in polysomes which were generally distributed evenly across the full ORF, as expected from a primarily elongating species (Heyer and Moore, 2016, [Fig](#page-4-0)[ure 2](#page-4-0)). This supports the common notion that, at any given instance, the majority of monosomes do not actively contribute to protein production.

### Literature values for translation elongation rates

Direct comparisons of translation elongation rates of cells growing in different conditions were reported in two papers: [Waldron et al. (1977)](#page-20-0) and [Bonven and Gulløv (1979)](#page-18-0). These papers reached different conclusions: [Waldron et al. (1977)](#page-20-0) directly measured translation elongation rates of an active translating ribosome and concluded that translation elongation rates do not change between rich (SC) and low-nitrogen media, with a constant value of 10.5 aa/sec measured in both conditions, despite significant differences in cell growth rate. [Bonven and Gulløv (1979)](#page-18-0) used basically the same method but reached a different conclusion, reporting 40% decrease in elongation rate between the growth on glucose and on acetate (with values of 9.6 aa/sec to 5.4 aa/sec, respectively). The difference between the two reports could be due to the use of different media or alternatively, details of the experiments. For example, [Waldron et al. (1977)](#page-20-0) quantified translation rate using proteins of high molecular weights (mentioning larger errors in the short-time estimates), while ([Bonven and Gulløv, 1979](#page-18-0)) estimated elongation rates of the slow-growing cells focusing on three subclasses of small molecular weight proteins, (while noting that an "increase in the uptake rate is registered at 70 s", perhaps indicating that elongation rate at the higher molecular weight regime are closer to the glucose-measured values). [Boehlke and Friesen, 1975](#page-18-0) also reported translation elongation rates, concluding that the translation elongation rate change with growth rate. These authors, however, did not measure elongation rates directly, but in fact calculated this value by measuring ribosome content and growth rate, assuming that ~90% of the expressed ribosomes are active in each media. This study, however, did not detect growth-rate dependent changes in the ribosome expression level or in the polysome profiles, in disagreement with broad available literature. The authors' estimates are therefore based on the assumptions that both the ribosome content and the active fraction are independent of growth rate, which is unlikely to hold.

### Data availability

The RNAseq data is available via accession number: SRP107059

# <span id="page-17-0"></span>Acknowledgements

We thank J Paulo for sending unpublished growth data, G Jona for assistance with the chemostats and our lab members for fruitful discussions. We thank Roni Winkler and Noam Stern-Ginossar for help and instruments used for polysome profiling. We would like to thank Dr. Yishai Levin, Dr. Alon Savidor and Dalia Elinger of the INCPM Institute for protein profiling for handling, running and initial analysis of the mass spectrometry samples. This work was supported by the ERC, the ISF and by the Minerva Center.

# Additional information

#### Competing interests

Naama Barkai: Senior editor, eLife. The other authors declare that no competing interests exist.

#### Funding

| Funder                             | Author       |
|------------------------------------|--------------|
| Israel Science Foundation          | Naama Barkai |
| H2020 European Research<br>Council | Naama Barkai |

The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.

#### Author contributions

Eyal Metzl-Raz, Conceptualization, Resources, Data curation, Software, Formal analysis, Validation, Investigation, Visulaization, Methodology, Writing—original draft, Writing—review and editing; Moshe Kafri, Conceptualization, Resources, Data curation, Software, Formal analysis, Validation, Investigation, Visualization, Methodology, Writing—original draft, Writing—review and editing; Gilad Yaakov, Resources, Data curation, Formal analysis, Methodology, Writing—review and editing; Ilya Soifer, Resources, Software, Writing—review and editing; Yonat Gurvich, Resources, Data curation; Naama Barkai, Conceptualization, Supervision, Funding acquisition, Investigation, Methodology, Writing—original draft, Project administration, Writing—review and editing

### Author ORCIDs

Eyal Metzl-Raz <https://orcid.org/0000-0002-7111-2822> Naama Barkai <https://orcid.org/0000-0002-2444-6061>

#### Decision letter and Author response

Decision letter <https://doi.org/10.7554/eLife.28034.020> Author response [https://doi.org/10.7554/eLife.28034.02](https://doi.org/10.7554/eLife.28034.021) 1

# Additional files

### Supplementary files

. Supplementary file 1. Proteome functional groups: We divided the proteome into 12 groups, 11 of which were based on SGD GO annotations or KEGG annotations, which together account for 80% of the proteome (by protein abundance). The rest were grouped together as an additional 12th group. [DOI: https://doi.org/10.7554/eLife.28034.014](https://doi.org/10.7554/eLife.28034.014)

. Supplementary file 2. Proteins group used for [Figure 1B](#page-2-0). [DOI: https://doi.org/10.7554/eLife.28034.015](https://doi.org/10.7554/eLife.28034.015)

. Supplementary file 3. One copy mCherry calculation: LC-MS/MS proteomic data for one genomic copy of mCherry and for fused mCherry (see Materials and methods). [DOI: https://doi.org/10.7554/eLife.28034.016](https://doi.org/10.7554/eLife.28034.016)

- . Transparent reporting form
#### <span id="page-18-0"></span>[DOI: https://doi.org/10.7554/eLife.28034.017](https://doi.org/10.7554/eLife.28034.017)

#### Major datasets

The following dataset was generated:

| Author(s)                                                                                        |      | Year Dataset title                                           | Dataset URL                                              | Database, license,<br>and accessibility<br>information                             |
|--------------------------------------------------------------------------------------------------|------|--------------------------------------------------------------|----------------------------------------------------------|------------------------------------------------------------------------------------|
| Eyal Metzl-Raz,<br>Moshe Kafri, Gilad<br>Yaakov, Ilya Soifer,<br>Yonat Gurvich, Naa<br>ma Barkai | 2017 | Saccharomyces cerevisiae strain:<br>Y8205 Raw sequence reads | https://www.ncbi.nlm.<br>nih.gov/sra/?term=<br>SRP107059 | Publicly available at<br>the Sequence Read<br>Archive (accession no.<br>SRP107059) |

### References

- Aspden JL, Eyre-Walker YC, Phillips RJ, Amin U, Mumtaz MA, Brocard M, Couso JP. 2014. Extensive translation of small open reading frames revealed by Poly-Ribo-Seq. eLife 3:e03528. DOI: [https://doi.org/10.7554/eLife.](https://doi.org/10.7554/eLife.03528) [03528](https://doi.org/10.7554/eLife.03528), PMID: [25144939](http://www.ncbi.nlm.nih.gov/pubmed/25144939)
- Balaban NQ, Merrin J, Chait R, Kowalik L, Leibler S. 2004. Bacterial persistence as a phenotypic switch. Science 305:1622–1625. DOI: [https://doi.org/10.1126/science.1099390,](https://doi.org/10.1126/science.1099390) PMID: [15308767](http://www.ncbi.nlm.nih.gov/pubmed/15308767)
- Boehlke KW, Friesen JD. 1975. Cellular content of ribonucleic acid and protein in Saccharomyces cerevisiae as a function of exponential growth rate: calculation of the apparent peptide chain elongation rate. Journal of Bacteriology 121:429–433. PMID: [1089627](http://www.ncbi.nlm.nih.gov/pubmed/1089627)
- Bonven B, Gulløv K. 1979. Peptide chain elongation rate and ribosomal activity in Saccharomyces cerevisiae as a function of the growth rate. MGG Molecular & General Genetics 170:225–230. DOI: [https://doi.org/10.1007/](https://doi.org/10.1007/BF00337800) [BF00337800,](https://doi.org/10.1007/BF00337800) PMID: [372763](http://www.ncbi.nlm.nih.gov/pubmed/372763)
- Bosdriesz E, Molenaar D, Teusink B, Bruggeman FJ. 2015. How fast-growing bacteria robustly tune their ribosome concentration to approximate growth-rate maximization. FEBS Journal 282:2029–2044. DOI: [https://](https://doi.org/10.1111/febs.13258) [doi.org/10.1111/febs.13258,](https://doi.org/10.1111/febs.13258) PMID: [25754869](http://www.ncbi.nlm.nih.gov/pubmed/25754869)
- Brachmann CB, Davies A, Cost GJ, Caputo E, Li J, Hieter P, Boeke JD. 1998. Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications. Yeast 14:115–132. DOI: [https://doi.org/10.1002/(SICI)1097-0061(19980130)14:2<115::](https://doi.org/10.1002/(SICI)1097-0061(19980130)14:2%3C115::AID-YEA204%3E3.0.CO;2-2) [AID-YEA204>3.0.CO;2-2](https://doi.org/10.1002/(SICI)1097-0061(19980130)14:2%3C115::AID-YEA204%3E3.0.CO;2-2), PMID: [9483801](http://www.ncbi.nlm.nih.gov/pubmed/9483801)
- Brauer MJ, Huttenhower C, Airoldi EM, Rosenstein R, Matese JC, Gresham D, Boer VM, Troyanskaya OG, Botstein D. 2008. Coordination of growth rate, cell cycle, stress response, and metabolic activity in yeast. Molecular Biology of the Cell 19:352–367. DOI:<https://doi.org/10.1091/mbc.E07-08-0779>, PMID: [17959824](http://www.ncbi.nlm.nih.gov/pubmed/17959824)
- Bremer H, Ehrenberg M. 1995. Guanosine tetraphosphate as a global regulator of bacterial RNA synthesis: a model involving RNA polymerase pausing and queuing. Biochimica et Biophysica Acta (BBA) - Gene Structure and Expression 1262:15–36. DOI: [https://doi.org/10.1016/0167-4781(95)00042-F,](https://doi.org/10.1016/0167-4781(95)00042-F) PMID: [7539631](http://www.ncbi.nlm.nih.gov/pubmed/7539631)
- Castelli LM, Lui J, Campbell SG, Rowe W, Zeef LA, Holmes LE, Hoyle NP, Bone J, Selley JN, Sims PF, Ashe MP. 2011. Glucose depletion inhibits translation initiation via eIF4A loss and subsequent 48S preinitiation complex accumulation, while the pentose phosphate pathway is coordinately up-regulated. Molecular Biology of the Cell 22:3379–3393. DOI:<https://doi.org/10.1091/mbc.E11-02-0153>, PMID: [21795399](http://www.ncbi.nlm.nih.gov/pubmed/21795399)
- Charvin G, Oikonomou C, Siggia ED, Cross FR. 2010. Origin of irreversibility of cell cycle start in budding yeast. PLoS Biology 8:e1000284. DOI: [https://doi.org/10.1371/journal.pbio.1000284,](https://doi.org/10.1371/journal.pbio.1000284) PMID: [20087409](http://www.ncbi.nlm.nih.gov/pubmed/20087409)
- Chasse´ H, Boulben S, Costache V, Cormier P, Morales J. 2017. Analysis of translation using polysome profiling. Nucleic acids research 45:e15. DOI:<https://doi.org/10.1093/nar/gkw907>, PMID: [28180329](http://www.ncbi.nlm.nih.gov/pubmed/28180329)
- Dai X, Zhu M, Warren M, Balakrishnan R, Patsalo V, Okano H, Williamson JR, Fredrick K, Wang YP, Hwa T. 2016. Reduction of translating ribosomes enables Escherichia coli to maintain elongation rates during slow growth. Nature Microbiology 2:16231. DOI: [https://doi.org/10.1038/nmicrobiol.2016.231,](https://doi.org/10.1038/nmicrobiol.2016.231) PMID: [27941827](http://www.ncbi.nlm.nih.gov/pubmed/27941827)
- Dekel E, Alon U. 2005. Optimality and evolutionary tuning of the expression level of a protein. Nature 436:588– 592. DOI:<https://doi.org/10.1038/nature03842>, PMID: [16049495](http://www.ncbi.nlm.nih.gov/pubmed/16049495)
- Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz G, Botstein D, Brown PO. 2000. Genomic expression programs in the response of yeast cells to environmental changes. Molecular Biology of the Cell 11: 4241–4257. DOI: [https://doi.org/10.1091/mbc.11.12.4241,](https://doi.org/10.1091/mbc.11.12.4241) PMID: [11102521](http://www.ncbi.nlm.nih.gov/pubmed/11102521)
- Gietz RD, Woods RA. 2002. Transformation of yeast by lithium acetate/single-stranded carrier DNA/polyethylene glycol method. Methods in Enzymology 350:87–96. DOI: [https://doi.org/10.1016/S0076-6879(02)50957-5,](https://doi.org/10.1016/S0076-6879(02)50957-5) PMID: [12073338](http://www.ncbi.nlm.nih.gov/pubmed/12073338)
- Guan Q, Haroon S, Bravo DG, Will JL, Gasch AP. 2012. Cellular memory of acquired stress resistance in Saccharomyces cerevisiae. Genetics 192:495–505. DOI: [https://doi.org/10.1534/genetics.112.143016,](https://doi.org/10.1534/genetics.112.143016) PMID: [22](http://www.ncbi.nlm.nih.gov/pubmed/22851651) [851651](http://www.ncbi.nlm.nih.gov/pubmed/22851651)
- Heyer EE, Moore MJ. 2016. Redefining the translational status of 80s monosomes. Cell 164:757–769. DOI:<https://doi.org/10.1016/j.cell.2016.01.003>, PMID: [26871635](http://www.ncbi.nlm.nih.gov/pubmed/26871635)
- Hin Yan Tong A, Boone C. 2005. High-Throughput Strain Construction and Systematic Synthetic Lethal Screening in Saccharomyces Cerevisiae. Elsevier.
- <span id="page-19-0"></span>Ju Q, Warner JR. 1994. Ribosome synthesis during the growth cycle of Saccharomyces cerevisiae. Yeast 10:151– 157. DOI: [https://doi.org/10.1002/yea.320100203,](https://doi.org/10.1002/yea.320100203) PMID: [8203157](http://www.ncbi.nlm.nih.gov/pubmed/8203157)
- Kafri M, Metzl-Raz E, Jona G, Barkai N. 2016b. The Cost of Protein Production. Cell Reports 14:22–31. DOI:<https://doi.org/10.1016/j.celrep.2015.12.015>, PMID: [26725116](http://www.ncbi.nlm.nih.gov/pubmed/26725116)
- Kafri M, Metzl-Raz E, Jonas F, Barkai N. 2016a. Rethinking cell growth models. FEMS Yeast Research 16:fow081. DOI: [https://doi.org/10.1093/femsyr/fow081,](https://doi.org/10.1093/femsyr/fow081) PMID: [27650704](http://www.ncbi.nlm.nih.gov/pubmed/27650704)
- Kapp LD, Lorsch JR, Sonenberg N, Mathews M, Kapp L, Lorsch J, Haar Tvonder, Oku Y, Ptushkina M, Moerke N. 2004. The molecular mechanics of eukaryotic translation. Annual Review of Biochemistry 73:657–704. DOI:<https://doi.org/10.1146/annurev.biochem.73.030403.080419>, PMID: [15189156](http://www.ncbi.nlm.nih.gov/pubmed/15189156)
- Keller A, Nesvizhskii AI, Kolker E, Aebersold R. 2002. An explanation of the Peptide Prophet algorithm developed. Analytical Chemistry 74:5383–5392.
- Kemmeren P, Sameith K, van de Pasch LA, Benschop JJ, Lenstra TL, Margaritis T, O'Duibhir E, Apweiler E, van Wageningen S, Ko CW, van Heesch S, Kashani MM, Ampatziadis-Michailidis G, Brok MO, Brabers NA, Miles AJ, Bouwmeester D, van Hooff SR, van Bakel H, Sluiters E, et al. 2014. Large-scale genetic perturbations reveal regulatory networks and an abundance of gene-specific repressors. Cell 157:740–752. DOI: [https://doi.org/10.](https://doi.org/10.1016/j.cell.2014.02.054) [1016/j.cell.2014.02.054](https://doi.org/10.1016/j.cell.2014.02.054), PMID: [24766815](http://www.ncbi.nlm.nih.gov/pubmed/24766815)
- Keren L, Zackay O, Lotan-Pompan M, Barenholz U, Dekel E, Sasson V, Aidelberg G, Bren A, Zeevi D, Weinberger A, Alon U, Milo R, Segal E. 2013. Promoters maintain their relative activity levels under different growth conditions. Molecular Systems Biology 9:701. DOI: [https://doi.org/10.1038/msb.2013.59,](https://doi.org/10.1038/msb.2013.59) PMID: [24169404](http://www.ncbi.nlm.nih.gov/pubmed/24169404)
- Klumpp S, Scott M, Pedersen S, Hwa T. 2013. Molecular crowding limits translation and cell growth. PNAS 110: 16754–16759. DOI:<https://doi.org/10.1073/pnas.1310377110>, PMID: [24082144](http://www.ncbi.nlm.nih.gov/pubmed/24082144)
- Koch AL. 1988. Why can't a cell grow infinitely fast? Canadian Journal of Microbiology 34:421–426. DOI: [https://](https://doi.org/10.1139/m88-074) [doi.org/10.1139/m88-074,](https://doi.org/10.1139/m88-074) PMID: [2460206](http://www.ncbi.nlm.nih.gov/pubmed/2460206)
- Levy S, Ihmels J, Carmi M, Weinberger A, Friedlander G, Barkai N. 2007. Strategy of transcription regulation in the budding yeast. PLoS One 2:e250. DOI:<https://doi.org/10.1371/journal.pone.0000250>, PMID: [17327914](http://www.ncbi.nlm.nih.gov/pubmed/17327914)
- Levy S, Kafri M, Carmi M, Barkai N. 2011. The competitive advantage of a dual-transporter system. Science 334: 1408–1412. DOI:<https://doi.org/10.1126/science.1207154>, PMID: [22158820](http://www.ncbi.nlm.nih.gov/pubmed/22158820)
- Levy SF, Ziv N, Siegal ML. 2012. Bet hedging in yeast by heterogeneous, age-correlated expression of a stress protectant. PLoS Biology 10:e1001325. DOI:<https://doi.org/10.1371/journal.pbio.1001325>, PMID: [22589700](http://www.ncbi.nlm.nih.gov/pubmed/22589700)
- Liu B, Qian SB. 2016. Characterizing inactive ribosomes in translational profiling. Translation 4:e1138018. DOI: [https://doi.org/10.1080/21690731.2015.1138018,](https://doi.org/10.1080/21690731.2015.1138018) PMID: [27335722](http://www.ncbi.nlm.nih.gov/pubmed/27335722)
- Maaløe O. 1979. Regulation of the protein-synthesizing machinery—ribosomes, tRNA, factors, and so on. Biological Regulation and Development :487–542. DOI:<https://doi.org/10.1007/978-1-4684-3417-0_12>
- Milo R, Jorgensen P, Moran U, Weber G, Springer M. 2010. BioNumbers–the database of key numbers in molecular and cell biology. Nucleic Acids Research 38:D750–D753. DOI:<https://doi.org/10.1093/nar/gkp889>, PMID: [19854939](http://www.ncbi.nlm.nih.gov/pubmed/19854939)
- Mitchell A, Romano GH, Groisman B, Yona A, Dekel E, Kupiec M, Dahan O, Pilpel Y. 2009. Adaptive prediction of environmental changes by microorganisms. Nature 460:220–224. DOI:<https://doi.org/10.1038/nature08112>, PMID: [19536156](http://www.ncbi.nlm.nih.gov/pubmed/19536156)
- Murphy JP, Stepanova E, Everley RA, Paulo JA, Gygi SP. 2015. Comprehensive Temporal Protein Dynamics during the Diauxic Shift in Saccharomyces cerevisiae. Molecular & Cellular Proteomics 14:2454–2465. DOI:<https://doi.org/10.1074/mcp.M114.045849>, PMID: [26077900](http://www.ncbi.nlm.nih.gov/pubmed/26077900)
- Nomura M, Gourse R, Baughman G. 1984. Regulation of the synthesis of ribosomes and ribosomal components. Annual Review of Biochemistry 53:75–117. DOI: [https://doi.org/10.1146/annurev.bi.53.070184.000451,](https://doi.org/10.1146/annurev.bi.53.070184.000451) PMID: [6206783](http://www.ncbi.nlm.nih.gov/pubmed/6206783)
- O'Duibhir E, Lijnzaad P, Benschop JJ, Lenstra TL, van Leenen D, Groot Koerkamp MJ, Margaritis T, Brok MO, Kemmeren P, Holstege FC. 2014. Cell cycle population effects in perturbation studies. Molecular Systems Biology 10:732. DOI:<https://doi.org/10.15252/msb.20145172>, PMID: [24952590](http://www.ncbi.nlm.nih.gov/pubmed/24952590)
- Paulo JA, O'Connell JD, Everley RA, O'Brien J, Gygi MA, Gygi SP. 2016. Quantitative mass spectrometry-based multiplexing compares the abundance of 5000 S. cerevisiae proteins across 10 carbon sources. Journal of Proteomics 148:85–93. DOI:<https://doi.org/10.1016/j.jprot.2016.07.005>, PMID: [27432472](http://www.ncbi.nlm.nih.gov/pubmed/27432472)
- Paulo JA, O'Connell JD, Gaun A, Gygi SP. 2015. Proteome-wide quantitative multiplexed profiling of protein expression: carbon-source dependency in Saccharomyces cerevisiae. Molecular Biology of the Cell 26:4063– 4074. DOI: [https://doi.org/10.1091/mbc.E15-07-0499,](https://doi.org/10.1091/mbc.E15-07-0499) PMID: [26399295](http://www.ncbi.nlm.nih.gov/pubmed/26399295)
- Pospı´sek M, Vala´ sek L. 2013. Chapter nine polysome profile analysis yeast. In: Methods in Enzymology. p. 173–181.
- Reuveni S, Ehrenberg M, Paulsson J. 2017. Ribosomes are optimized for autocatalytic production. Nature 547: 293–297. DOI: [https://doi.org/10.1038/nature22998,](https://doi.org/10.1038/nature22998) PMID: [28726822](http://www.ncbi.nlm.nih.gov/pubmed/28726822)
- Schaechter M, MaalOe O, Kjeldgaard NO. 1958. Dependency on medium and temperature of cell size and chemical composition during balanced grown of Salmonella typhimurium. Journal of General Microbiology 19: 592–606. DOI:<https://doi.org/10.1099/00221287-19-3-592>, PMID: [13611202](http://www.ncbi.nlm.nih.gov/pubmed/13611202)
- Schimmel P. 1993. GTP hydrolysis in protein synthesis: two for Tu? Science 259:1264–1265. DOI: [https://doi.org/](https://doi.org/10.1126/science.8446896) [10.1126/science.8446896](https://doi.org/10.1126/science.8446896), PMID: [8446896](http://www.ncbi.nlm.nih.gov/pubmed/8446896)
- Scott M, Gunderson CW, Mateescu EM, Zhang Z, Hwa T. 2010. Interdependence of cell growth and gene expression: origins and consequences. Science 330:1099–1102. DOI: [https://doi.org/10.1126/science.1192588,](https://doi.org/10.1126/science.1192588) PMID: [21097934](http://www.ncbi.nlm.nih.gov/pubmed/21097934)
- <span id="page-20-0"></span>Scott M, Hwa T. 2011. Bacterial growth laws and their applications. Current Opinion in Biotechnology 22:559– 565. DOI: [https://doi.org/10.1016/j.copbio.2011.04.014,](https://doi.org/10.1016/j.copbio.2011.04.014) PMID: [21592775](http://www.ncbi.nlm.nih.gov/pubmed/21592775)
- Scott M, Klumpp S, Mateescu EM, Hwa T. 2014. Emergence of robust growth laws from optimal regulation of ribosome synthesis. Molecular Systems Biology 10:747–14. DOI:<https://doi.org/10.15252/msb.20145379>, PMID: [25149558](http://www.ncbi.nlm.nih.gov/pubmed/25149558)
- Shachrai I, Zaslaver A, Alon U, Dekel E. 2010. Cost of unneeded proteins in E. coli is reduced after several generations in exponential growth. Molecular Cell 38:758–767. DOI: [https://doi.org/10.1016/j.molcel.2010.04.](https://doi.org/10.1016/j.molcel.2010.04.015) [015,](https://doi.org/10.1016/j.molcel.2010.04.015) PMID: [20434381](http://www.ncbi.nlm.nih.gov/pubmed/20434381)
- Shalit T, Elinger D, Savidor A, Gabashvili A, Levin Y. 2015. MS1-based label-free proteomics using a quadrupole orbitrap mass spectrometer. Journal of Proteome Research 14:1979–1986. DOI: [https://doi.org/10.1021/](https://doi.org/10.1021/pr501045t) [pr501045t](https://doi.org/10.1021/pr501045t), PMID: [25780947](http://www.ncbi.nlm.nih.gov/pubmed/25780947)
- Soll DR, Kraft B. 1988. A comparison of high frequency switching in the yeast Candida albicans and the slime mold Dictyostelium discoideum. Developmental Genetics 9:615–628. DOI: [https://doi.org/10.1002/dvg.](https://doi.org/10.1002/dvg.1020090438) [1020090438](https://doi.org/10.1002/dvg.1020090438), PMID: [3072134](http://www.ncbi.nlm.nih.gov/pubmed/3072134)
- Tamari Z, Rosin D, Voichek Y, Barkai N. 2014. Coordination of gene expression and growth-rate in natural populations of budding yeast. PLoS One 9:e88801. DOI: [https://doi.org/10.1371/journal.pone.0088801,](https://doi.org/10.1371/journal.pone.0088801) PMID: [24533150](http://www.ncbi.nlm.nih.gov/pubmed/24533150)
- Tamari Z, Yona AH, Pilpel Y, Barkai N. 2016. Rapid evolutionary adaptation to growth on an 'unfamiliar' carbon source. BMC Genomics 17:674. DOI:<https://doi.org/10.1186/s12864-016-3010-x>, PMID: [27552923](http://www.ncbi.nlm.nih.gov/pubmed/27552923)
- van den Elzen AM, Schuller A, Green R, Se´ raphin B. 2014. Dom34-Hbs1 mediated dissociation of inactive 80S ribosomes promotes restart of translation after stress. The EMBO Journal 33:26576. DOI: [https://doi.org/10.](https://doi.org/10.1002/embj.201386123) [1002/embj.201386123,](https://doi.org/10.1002/embj.201386123) PMID: [24424461](http://www.ncbi.nlm.nih.gov/pubmed/24424461)
- Van Der Kelen K, Beyaert R, Inze´ D, De Veylder L. 2009. Translational control of eukaryotic gene expression. Critical Reviews in Biochemistry and Molecular Biology 44:143–168. DOI: [https://doi.org/10.1080/](https://doi.org/10.1080/10409230902882090) [10409230902882090](https://doi.org/10.1080/10409230902882090), PMID: [19604130](http://www.ncbi.nlm.nih.gov/pubmed/19604130)
- Vind J, Sørensen MA, Rasmussen MD, Pedersen S. 1993. Synthesis of Proteins in Escherichia coli is Limited by the Concentration of Free Ribosomes. Journal of Molecular Biology 231:678–688. DOI: [https://doi.org/10.](https://doi.org/10.1006/jmbi.1993.1319) [1006/jmbi.1993.1319](https://doi.org/10.1006/jmbi.1993.1319)
- Voichek Y, Bar-Ziv R, Barkai N. 2016. Expression homeostasis during DNA replication. Science 351:1087–1090. DOI: [https://doi.org/10.1126/science.aad1162,](https://doi.org/10.1126/science.aad1162) PMID: [26941319](http://www.ncbi.nlm.nih.gov/pubmed/26941319)
- von der Haar T. 2008. A quantitative estimation of the global translational activity in logarithmically growing yeast cells. BMC Systems Biology 2:87. DOI: [https://doi.org/10.1186/1752-0509-2-87,](https://doi.org/10.1186/1752-0509-2-87) PMID: [18925958](http://www.ncbi.nlm.nih.gov/pubmed/18925958)
- Waldron C, Jund R, Lacroute F. 1977. Evidence for a high proportion of inactive ribosomes in slow-growing yeast cells. Biochemical Journal 168:409–415. DOI: [https://doi.org/10.1042/bj1680409,](https://doi.org/10.1042/bj1680409) PMID: [343781](http://www.ncbi.nlm.nih.gov/pubmed/343781)
- Waldron C, Lacroute F. 1975. Effect of growth rate on the amounts of ribosomal and transfer ribonucleic acids in yeast. Journal of Bacteriology 122:855–865 . PMID: [1097403](http://www.ncbi.nlm.nih.gov/pubmed/1097403)
- Wang M, Weiss M, Simonovic M, Haertinger G, Schrimpf SP, Hengartner MO, von Mering C. 2012. PaxDb, a database of protein abundance averages across all three domains of life. Molecular & Cellular Proteomics 11: 492–500. DOI: [https://doi.org/10.1074/mcp.O111.014704,](https://doi.org/10.1074/mcp.O111.014704) PMID: [22535208](http://www.ncbi.nlm.nih.gov/pubmed/22535208)
- Warner JR, Knopf PM, Rich A. 1963. A multiple ribosomal structure in protein synthesis. PNAS 49:122–129. DOI: [https://doi.org/10.1073/pnas.49.1.122,](https://doi.org/10.1073/pnas.49.1.122) PMID: [13998950](http://www.ncbi.nlm.nih.gov/pubmed/13998950)
- Warner JR. 1999. The economics of ribosome biosynthesis in yeast. Trends in Biochemical Sciences 24:437–440. DOI:<https://doi.org/10.1016/S0968-0004(99)01460-7>, PMID: [10542411](http://www.ncbi.nlm.nih.gov/pubmed/10542411)
- Yaakov G, Lerner D, Bentele K, Steinberger J, Barkai N. 2017. Coupling phenotypic persistence to DNA damage increases genetic diversity in severe stress. Nature Ecology & Evolution 11::497–500. DOI: [https://doi.org/10.](https://doi.org/10.1038/s41559-016-0016) [1038/s41559-016-0016](https://doi.org/10.1038/s41559-016-0016)
- Zaslaver A, Kaplan S, Bren A, Jinich A, Mayo A, Dekel E, Alon U, Itzkovitz S. 2009. Invariant distribution of promoter activities in Escherichia coli. PLoS Computational Biology 5:e1000545. DOI: [https://doi.org/10.1371/](https://doi.org/10.1371/journal.pcbi.1000545) [journal.pcbi.1000545](https://doi.org/10.1371/journal.pcbi.1000545), PMID: [19851443](http://www.ncbi.nlm.nih.gov/pubmed/19851443)