![](_page_0_Picture_0.jpeg)

# ARTICLE

1234567890():,;

https://doi.org/10.1038/s41467-019-09261-3 **OPEN**

# Growth strategy of microbes on mixed carbon sources

Xin Wan[g](http://orcid.org/0000-0001-6479-395X) 1,2, Kang Xi[a](http://orcid.org/0000-0002-5942-5594) [1](http://orcid.org/0000-0002-5942-5594),3, Xiaojing Yan[g](http://orcid.org/0000-0002-3397-0578) 1 & Chao Tan[g](http://orcid.org/0000-0003-1474-3705) [1](http://orcid.org/0000-0003-1474-3705)

A classic problem in microbiology is that bacteria display two types of growth behavior when cultured on a mixture of two carbon sources: the two sources are sequentially consumed one after another (diauxie) or they are simultaneously consumed (co-utilization). The search for the molecular mechanism of diauxie led to the discovery of the lac operon. However, questions remain as why microbes would bother to have different strategies of taking up nutrients. Here we show that diauxie versus co-utilization can be understood from the topological features of the metabolic network. A model of optimal allocation of protein resources quantitatively explains why and how the cell makes the choice. In case of coutilization, the model predicts the percentage of each carbon source in supplying the amino acid pools, which is quantitatively verified by experiments. Our work solves a long-standing puzzle and provides a quantitative framework for the carbon source utilization of microbes.

<sup>1</sup> Center for Quantitative Biology, School of Physics and Peking-Tsinghua Center for Life Sciences, Peking University, Beijing 100871, China. 2 Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA 02115, USA. 3 College of Life Sciences, Wuhan University, Wuhan 430072, China. These authors contributed equally: Xin Wang, Kang Xia. Correspondence and requests for materials should be addressed to C.T. (email: [tangc@pku.edu.cn)](mailto:tangc@pku.edu.cn)

<span id="page-1-0"></span>During the course of evolution, biological systems have acquired a myriad of strategies to adapt to their environments. A great challenge is to understand the rationale of these strategies on quantitative bases. It has long been discovered that the production of digestive enzymes in a microorganism depends on (adapts to) the composition of the medium[1.](#page-5-0) More precisely, in the 1940s Jacques Monod observed two distinct strategies in bacteria (E. coli and B. subtilis) to take up nutrients. He cultured these bacteria on a mixture of two carbon sources, and found that for certain mixtures the bacteria consume both nutrients simultaneously while for other mixtures they consume the two nutrients one after anothe[r2,3.](#page-5-0) The latter case resulted in a growth curve consisted of two consecutive exponentials, for which he termed this phenomenon diauxie. Subsequent studies revealed that the two types of growth behavior, diauxic- and co-utilization of carbon sources are common in microorganisms[4](#page-5-0)–[8.](#page-5-0) The regulatory mechanism responsible for diauxie, that is the molecular mechanism for the microbes to express only the enzymes for the preferred carbon source even when multiple sources are present, is commonly ascribed to catabolite repression[5](#page-5-0),[9](#page-5-0)–[13.](#page-5-0) In bacteria it is exemplified by the lac operon and the cAMP-CRP system[14](#page-5-0)–[17](#page-6-0). In yeast, the molecular implementation of catabolite repression differs, but the logic and the outcome are similar[5.](#page-5-0)

Why have microbes evolved to possess the two strategies and what are the determining factors for them to choose one versus the other? For unicellular organisms, long term survival and growth at the population level are paramount. Cells allocate their cellular resources to achieve optimal growth[18](#page-6-0)–[27](#page-6-0). In particular, it has been demonstrated that the principle of optimal protein resource allocation can quantitatively explain a large body of experimental data[20](#page-6-0),[21](#page-6-0) and that the most efficient enzyme allocation in metabolic networks corresponds to elementary flux mode[25](#page-6-0),[26](#page-6-0),[28](#page-6-0). In this paper, we extend these approaches to address the question of multiple carbon sources and show that the two growth strategies can be understood from optimal enzyme allocation further constrained by the topological features of the metabolic network.

### Results

Categorization of carbon sources. Carbon sources taken by the cell serve as substrates of the metabolic network, in which they are broken down to supply pools of amino acids and other components that make up a cell. Amino acids take up a majority of carbon supply (about 55%)[29](#page-6-0)–[31.](#page-6-0) As shown in Fig. 1, different carbon sources enter the metabolic network at different points[31.](#page-6-0) Denote those sources entering the upper part of the glycolysis Group A and those joining at other points of the metabolic network Group B (Fig. 1). Studies have shown that when mixing a carbon source of Group A with that of Group B, the bacteria tend to co-utilize both sources and the growth rate is higher than that with each individual source[6](#page-5-0),[7](#page-5-0)[,32](#page-6-0). When mixing two sources both from Group A, the bacteria usually use a preferred source (of higher growth rate) first[4](#page-5-0),[6,11,13](#page-5-0),[33](#page-6-0),[34](#page-6-0).

Precursor pools of biomass components. Based on the topology of metabolic network (Fig. 1), we classify the precursors of biomass components (amino acids and others) into seven precursor pools. Specifically, each pool is named depending on its entrance point on the metabolic network (see Supplementary Note 1.3 for details): a1 (entering from G6P/F6P), a2 (entering from GA3P), a3 (entering from 3PG), a4 (entering from PEP), b (entering from pyruvate/Acetyle-CoA), c (entering from α-Ketoglutarate) and d (entering from oxaloacetate). The Pools a1-a4 are collective called as Pool a.

![](_page_1_Figure_7.jpeg)

Fig. 1 Metabolic network of carbon source utilization. Group A substrates (in green frames) can be simultaneously utilized with Group B substrates (red frames), whereas substrates paired from Group A usually display diauxie. Only the major pathways are shown. The precursors of biomass components (amino acids marked with light brown frames and other components marked with orange frames) are classified into Pools a–d (marked with gray dashed line frames). The enzyme for the interconversion between Glucose 6-phosphate (G6P) and fructose 6-phosphate (F6P) is very efficient (Supplementary Table 1), so we approximate G6P/F6P as a single node for convenience. All Group A carbon sources enter the metabolic network through G6P/F6P, while Group B carbon sources enter the metabolic network from different points after glycolysis. Glycerol enters from the upper part of glycolysis but not G6P/F6P, thus we classify glycerol as a quasi-Group A carbon source (see Supplementary Note 1.4 for details)

Coarse-graining the metabolic network. Note that the carbon sources from Group A converge to the node (G6P/F6P) before entering various pools, while the carbon sources from Group B can take other routes (Fig. 1). In fact, the metabolic network shown in Fig. 1 can be coarse-grained (see Methods) into a model shown in Fig. [2a](#page-2-0), in which nodes A1 and A2 represent carbon sources from Group A, node B a source from Group B, and Pools 1 and 2 are some combinations of the four pools in Fig. 1. In other words, Fig. [2a](#page-2-0) is topologically equivalent to Fig. 1 as far as the carbon flux is concerned. Each coarse-grained arrow (which can contain several metabolic steps) carries a carbon flux J and is characterized by two quantities: the total enzyme cost Φ dedicated to carry the flux and a parameter κ so that J ¼ Φ κ.

Origin of diauxie for carbon sources in Group A. Let us first consider the case in which both carbon sources are from Group A. We proceed to solve the simple model of Fig. [2a](#page-2-0) with two sources A1 and A2 ([B] = 0), using the optimal protein allocation hypothesis[18](#page-6-0),[20,21,25,](#page-6-0) which maximizes the enzyme utilization efficiency.

In Fig. [2a](#page-2-0), all enzymes that carry and digest nutrient Ai (i = 1, 2) into node M are simplified to a single effective enzyme EAi of cost ΦAi (see Supplementary Note 1.2 for details). The carbon flux to the precursor pools from source Ai is proportional to ΦAi. We take the Michaelis–Menten form (see Supplementary Note 1.2 for details): JAi ¼ ΦAi κAi, where κAi ¼ kAi - ½Ai ½AiþKAi (denoted as the <span id="page-2-0"></span>substrate quality). ½Ai is the concentration of Ai. For the subsystem consist of A1, A2 and node M, Jtot ¼ ΦA1 κA1 þ ΦA2 κA2 and Φtot ¼ ΦA1 þ ΦA2. We define the efficiency of a pathway by the flux delivered per total enzyme cos[t25,26](#page-6-0):

$$
\varepsilon \equiv \frac{J_{\text{tot}}}{\Phi_{\text{tot}}}.\tag{1}
$$

The efficiency to deliver carbon flux from the two sources A1 and A2 to node M is then ε ¼ ΦA1κA1 þ ΦA2κA2 ΦA1 þ ΦA2 . If κA1 > κA2, that is, if the substrate quality of A1 is better than that of A2, then ε ¼ κA1 ΦA2 κA1κA2 ð Þ ΦA1þΦA2 κA1. It is easy to see that the optimal solution (maximum efficiency) is ΦA2 ¼ 0. This means that the cell expresses only the enzyme for A1 and thus utilizes only A1. Conversely, if κA1 < κA2, ΦA1 ¼ 0 is optimal and the cell would utilize A2 only. In either case, optimal growth would imply that cells only consume the preferable carbon source, which corresponds to the case of diauxie[2,3](#page-5-0),[6,9,11.](#page-5-0)

In the above coarse-grained model, the enzyme efficiency of the carbon source Ai is lump summed in a single effective parameter κAi. In practice, there are intermediate nodes and enzymes along the pathway as depicted in Fig. 2b, and more elaborate calculations taking into account the intermediate steps are needed to evaluate the pathway efficiency. Note that Fig. 2b is rather generic in representing a part of the metabolic pathway under consideration. X and Y can represent carbon sources coming from Group A and/or Group B, M represents the convergent node of the two sources under consideration and Pool z represents the precursor pool under consideration. We now proceed to calculate and compare the efficiencies of the two branches: X ! M and Y ! M. Using the branch X ! M as the example, denote Ej X the enzymes (of protein cost Φj X) catalyzing the intermediate nodes mj X (j ¼ 1 NX) (Fig. 2b). Define Φb X ΦX þ PNX j¼1 Φj X, which is the total protein cost for enzymes dedicated to the branch. The pathway efficiency for the branch X ! M is then εX!M ¼ JX!M=Φb X, where JX!M is the carbon flux from X to M. Assuming that the flux is conserved in each step along the branch, JX!M ¼ ΦX κX ¼ Φj X κ j X (j ¼ 1 NX), where κ j X ¼ k j X - ½mj X ½mj XþKj X k j X is the substrate quality of mj X (Supplementary Note 1.2) and the last approximation is valid with ½mj X Kj X which is generally true in bacteria[35,36](#page-6-0) and which also maximizes the flux with a given enzyme cost (Supplementary Note 1.5). It is then easy to see that

$$\varepsilon_{\mathbf{X}\to\mathbf{M}} = \frac{1}{1/\kappa_{\mathbf{X}} + \sum_{j=1}^{N_{\mathbf{X}}} 1/\kappa_{\mathbf{X}}^{j}} \approx \frac{1}{1/\kappa_{\mathbf{X}} + \sum_{j=1}^{N_{\mathbf{X}}} 1/k_{\mathbf{X}}^{j}}.\tag{2}$$

For εX!M > εY!M, the optimal solution is ΦY ¼ Φj Y ¼ 0; for εX!M < εY!M, the optimal solution is ΦX ¼ Φj X ¼ 0 (see Supplementary Note 2.2 for details). Only the nutrient with higher branch efficiency is utilized to supply the convergent node M and thus Pool z (Fig. 2b).

When X and Y both come from Group A, the convergent node M resides upstream to all precursor pools (Fig. [1](#page-1-0) and Supplementary Fig. 1c). Only the nutrient with higher efficiency is being utilized to supply all precursor pools. When the preferred nutrient is exhausted, the cell switch to the other less favorable nutrient. The actual switching point could depend on the concentrations of both nutrients. Note that the branch efficiency εAi!M (Eq. (2)) depends on the concentration of the nutrient ½Ai through the substrate quality κAi ¼ kAi - ½Ai ½AiþKAi . Thus, only with

![](_page_2_Figure_9.jpeg)

Fig. 2 Topology of the metabolic network. a Coarse-grained model of the metabolic network. Group A carbon sources merge to a common node M before reaching precursor pools. Group B sources can supply some precursor pools from other routes. b Topology of the part of the metabolic network connecting two carbon sources to a precursor pool. The two carbon sources X and Y (from Group A and/or Group B) reach a common node M through multiple intermediate nodes (metabolites) mj X and mj Y along their respective pathway, after which the flux is diverted to Pool z

saturating concentrations, one can have an absolute ranking of the nutrient quality. For concentration ½Ai < KAi, εAi!Mð½AiÞ drops fast with ½Ai. Theoretically, when the concentration of the originally preferred nutrient (A1) drops to a point ½A1 T such that εA1!Mð½A1 TÞ ¼ εA2!Mð½A2Þ, the other nutrient (A2) becomes preferrable, and the cell may switch to A2 at this point. This gives ½A1 T ¼ δ-½A2 Δþ½A2 , where δ and Δ are constant (see Supplementary Note 2.3 for details). For ½A2 Δ, the turning point is reduced to ½A1 T ¼ δ Δ - ½A2, a form of ratio sensing. Indeed, ratio sensing was recently observed in the budding yeast Saccharomyces cerevisiae cultured in glucose-galactose mixed medium[33](#page-6-0), and the experimental results agree well with the turning point equation derived above (see Supplementary Fig. 2 and Supplementary Note 2.3 for details).

Co-utilization of carbon sources. The diauxic growth is due to the topology of the metabolic network, in which Group A sources converge to a common node (G6P/F6P) before diverting to various precursor pools (Figs. [1](#page-1-0), 2a and Supplementary Fig. 1c). The situation is different if the two mixed carbon sources are from Groups A and B, respectively (denoted as A + B). (Some combinations of two Group B sources also fall into this category and can be analyzed similarly; see Supplementary Fig. 3d) Group B sources can directly supply some precursor pools without going through the common node (G6P/F6P) (Fig. [1](#page-1-0)). The topologies of the metabolic network in the cases of A + B are exemplified in Supplementary Fig. 3. All A + B cases can be mapped to a common coarse-grained model depicted in Supplementary Fig. 1d (which is also Fig. 2a with only one of the A sources present), although the actual position of nodes M and N in the metabolic network, and the contents of Pools 1 and 2 may depend on specific cases. As obvious from Fig. 2a, source A or B alone could in principle supply all precursor pools. However, because of the location of the precursor pools relative to the sources, it may be more economical for one pool to draw carbon flux from one source and the other from the other source.

<span id="page-3-0"></span>To determine which of the two carbon sources should supply which pool(s), we apply branch efficiency analysis (see Supplementary Note 3 for more details). For Pool 1, we compare the efficiency of A and B in supplying flux to node M; while for Pool 2 to node N. The criteria are simply:

$$\text{Pool 1 is supplied by} \begin{cases} A, \text{ if } \varepsilon_{A \to M} > \varepsilon_{B \to M} \\ B, \text{ if } \varepsilon_{A \to M} < \varepsilon_{B \to M} \end{cases} . \tag{3}$$

$$\text{Pool 2 is supplied by} \begin{cases} A, \text{ if } \varepsilon_{A \to N} > \varepsilon_{B \to N} \\ B, \text{ if } \varepsilon_{A \to N} < \varepsilon_{B \to N} \end{cases} . \tag{4}$$

It is easy to see from inequalities (3) and (4) that if the following condition is met

$$1/\kappa_{\rm B} - 1/\kappa_{\rm M}^{'} < 1/\kappa_{\rm A} < 1/\kappa_{\rm B} + 1/\kappa_{\rm N}^{'},\tag{5}$$

then A supplies Pool 1 and B supplies Pool 2—the two carbon sources are simultaneously consumed. In reality, there are multiple intermediate nodes between the M-N interconversion (Figs. [1,](#page-1-0) [2](#page-2-0)a and Supplementary Fig. 1d). Similar to Eq. ([2](#page-2-0)), 1=κ′ M and 1=κ′ N here actually represent summations of all intermediate terms between M and N in the metabolic network.

Pools suppliers in the mixed carbon sources. In order to apply the above analysis to the real case, we collected the available data for metabolic enzymes of E. coli from the literature (Supplementary Table 1). We calculated the branch efficiencies of different carbon sources to the metabolites F6P, GA3P, 3PG, PEP, pyruvate and oxaloacetate (see Supplementary Note 4.1 for details), which correspond to the nodes M or N in the simplified network of Fig. [2a](#page-2-0) for Pools a1–d (Fig. [1](#page-1-0) and Supplementary Fig. 3). The results are shown in Supplementary Table 2. Then using the criteria Eqs. (3) and (4), one could evaluate the carbon source supplier(s) of all pools (Supplementary Table 3). Note that Pool c is supplied by both the suppliers of Pools b and d owing to the effect of converged flux (see Supplementary Notes 4.1–4.2 for details).

However, in practice, the suppliers of Pools b-d can be different from the above evaluation due to energy production in the TCA cycle. Specifically, when oxaloacetate flow through a TCA cycle back to itself, it generates fixed amount of energy[31,](#page-6-0) with half of the carbon atoms replaced by those coming from pyruvate (see Supplementary Figs. 4b-c and Supplementary Notes 4.3-4.4 for details). By collecting relevant energy production data from literatures[37,](#page-6-0) we quantitatively analyzed the influence of this effect, and obtained the optimal carbon source supplier(s) of each precursor pool for E. coli under aerobic growth in various combinations of source mixture (see Supplementary Notes 4.3– 4.5 for details). The results are shown in Supplementary Table 4.

Comparison with experiments. To test these predictions (Supplementary Table 4), we use 13C isotope labeling methods to trace the carbon source(s) of each precursor pool (see Methods for details). Specifically, we cultured E. coli to the steady state in a mixture of two carbon sources with one source being labeled with 13C. We then measured the 13C labeling percentage of amino acids and obtained the percentage of each carbon source in supplying the synthesis of each amino acid. To ensure reliability, we obtained and compared our experimental results using two types of fragment in mass spec raw data: M-57 and M-85 (Supplementary Fig. 5) (see Methods for details).

We first examined the A + B cases (Group A source: glucose, lactose, fructose, glycerol; Group B source: pyruvate, succinate, fumarate, malate). Overall, the experimental results showed excellent agreement with our predictions (Fig. 3). A number of features are worth noting. Just as the model predicted

![](_page_3_Figure_12.jpeg)

Fig. 3 Suppliers of precursor pools in A + B cases. Vertical axes are the percentages of the carbon atoms from the first of the two sources indicated. Model predictions (in hollow bars, see Supplementary Table 4) are shown together with experimental results (in color bars). The source supplier of representative amino acids in Pools a–d was measured using 13C labeling (raw data from M-57 fragment; see Methods for details). Error bars represent standard deviations. Source data are provided as a Source Data file. a Glucose or lactose mixed with a Group B carbon source. b Fructose or glycerol mixed with a Group B carbon source

(Supplementary Table 4), two patterns of the carbon source partition (Fig. 3a, b) were observed depending on which A source was used. Glucose and lactose are both highly preferable carbon sources for E. coli, both supporting large growth rates (Supplementary Table 5); their supply patterns look almost the same (Fig. 3a). Fructose, glycerol, maltose and galactose are less preferable Group A sources with lower growth rates (Supplementary Table 5), and they showed very similar supply patterns when mixed with the same Group B source (Fig. 3b, Supplementary Fig. 6a). Group B sources succinate, fumarate and malate showed similar supply patterns when mixed with a same Group A source (Figs. 3a, b, [4a](#page-4-0) & Supplementary Fig. 6a). There was a noticeable systematic discrepancy between the experimental results and the model predictions for Pool a. This may be due to the fact that microbes reserve a portion of gluconeogenesis enzymes preparing for potential changing environment (see also Supplementary Notes 4.8–4.9).

<span id="page-4-0"></span>Next, as our model can calculate carbon source utilization and partition in any combinations of sources, we performed experiments for B + B cases (pyruvate mixed with succinate, fumarate or malate; succinate mixed with malate). In agreement with the model prediction, these B + B cases showed coutilization and the measured carbon supply percentages quantitatively agree with model predictions (Fig. 4).

# Discussion

The diauxie versus co-utilization puzzle can be understood from the topology of the metabolic network. This can be illustrated with the coarse-grained model shown in Fig. [2a](#page-2-0) (see also Supplementary Figs. 1b–d). The sources of Group A go through a common node before delivered to various precursor pools, and the most efficient source wins[22.](#page-6-0) It has been observed that there is a hierarchy among Group A sources ranked according to the single-source growth rate, and when two or more sources are present the bacteria usually use the one with highest growth rate[6,](#page-5-0)[34.](#page-6-0) This is a natural consequence of our theory. A higher growth rate commonly implies higher enzyme utilization efficiency and thus a higher priority to be utilized. Other than the lac operon, questions remain as how this priority is implemented molecularly. It has been known that in many cases the catabolite repression is not complete and that this may depend on whether the carbon sources belong to the type of Phosphotransferase system (PTS)[10,13,](#page-5-0) highlighting the potential constraints, tradeoffs and/or costs of implementing a prefect optimal solution. We have mixed glucose (a PTS sugar) with both PTS sugar fructose and non-PTS sugars maltose and glycerol, which all belong to the A + A cases. We found that while glucose showed almost perfect inhibition to the two non-PTS sugars (all precursor pools were about 100% supplied by glucose), its inhibition to the other PTS sugar fructose was not complete (glucose supplied ~83% precursor pools) (Supplementary Fig. 6b).

When Group B source is present along with Group A source, it can take a shortcut to reach some of the precursor pools (Fig. [2](#page-2-0)a, Supplementary Fig. 1d) and can be more efficient to supply these pools. Some combinations of two Group B sources also fall into this category and thus can be co-utilized. In these cases, our experimental results quantitatively agree with our model predictions. As can be seen from Figs. [3](#page-3-0) and 4, despite the various possible combinations of carbon sources, the partition of the sources among the pools fall into a few patterns. This is due to the fact that these partitions are largely determined by the topology of the metabolic network and thus are quantized. This property also relaxes the requirements of accurate enzyme parameters in determining the pool suppliers. To test the robustness of the model predictions with respect to the errors/uncertainties of the parameters extracted/estimated from the literature, we carried out a detailed analysis (Supplementary Note 4.9 and Supplementary Tables 6 and 7). The analysis showed that for any mixture of two carbon sources and for arbitrary choice of parameters, only a very few (no more than 4) partition patterns of the sources are qualitatively similar to the experimental result. Model predictions using the nominal parameter values from the literature quantitatively and consistently agree with all the experimental patterns for all the combinations of carbon sources we tested. Conversely, in order to produce a pattern that is qualitatively similar but quantitatively different from the experimental one, a very large deviation from at least one nominal value is necessary.

The present work deals with relatively stable growth conditions and the simple exponential growth behavior. In this case, there is a body of experimental evidence for optimal protein allocation[18](#page-6-0)–[24,27,38.](#page-6-0) Furthermore, our model relies only on the assumption that microbes optimize enzyme utilization efficiency,

![](_page_4_Figure_7.jpeg)

Fig. 4 Suppliers of precursor pools in B + B cases. Vertical axes are the percentages of the carbon atoms from the first of the two sources indicated. Model predictions (in hollow bars, see Supplementary Table 4) are shown together with experimental results (in color bars). The source supplier of representative amino acids in Pools a–d was measured using 13C labeling (raw data from M-57 fragment; see Methods for details). Error bars represent standard deviations. Source data are provided as a Source Data file. a Pyruvate mixed with another Group B carbon source.

b Succinate mixed with malate. In this case, the branch efficiencies of the two sources are about the same

so it may also be applicable to suboptimal growth cases[27](#page-6-0),[38](#page-6-0). The environment the microbes face can be highly variable and uncertain. Their long-term fitness of the population may not simply be determined only by the growth rate of individual cells in the exponential phase, but a result of trade-offs that best adapt to the changing environment. Strategies such as bet hedging, memory of the past and anticipation of the future are found to exist in microorganisms[39](#page-6-0)–[48](#page-6-0).

Finally, from theoretical aspects, our analysis framework is broadly applicable to more complex regulations in metabolic networks such as reversible reactions, allosteric enzymes, metabolites inhibitions, etc. (see Supplementary Notes 5–7 for details). However, there are cases, such as bi-substrates transporters or enzymes (e.g., glucose transporters in E. coli can co-transport mannose[49](#page-6-0)), for which specific care needs to be taken (see Supplementary Notes 6–7 for details). In practice, the nutrient uptake strategy or eating habit of a microbe is shaped by its environmental history. While the phenomena of diauxie versus coutilization are widely spread in microbes, there are bound to be variations and exceptions. For example, certain microbes may have different hierarchies of preferable carbon sources[4.](#page-5-0) It is still a great challenge to understand in quantitative frameworks how cells and population behave and evolve in different and changing environments.

#### Methods

Coarse graining methods. Coarse graining of the metabolic network is done in such a way as to preserve the network topology but grouping metabolites, enzymes and pathways into single representative nodes and corresponding effective enzymes. In particular, a linear pathway is lump summed into two nodes (start and end) connecting with a single effective enzyme.

Strain. The strain used in this study is E. coli K-12 strain NCM3722. Growth medium: Most of the cultures were based on the M9 minimal medium (42 mM Na2HPO4, 22 mM KH2PO4, 8.5 mM NaCl, 18.7 mM NH4Cl, 2 mM MgSO4, 0.1 mM CaCl2), and supplemented with one or two types of carbon sources. For the carbon sources in each medium, the following concentrations were applied: 0.4% (w/v) glucose, 0.4% (w/v) lactose, 0.4% (v/v) glycerol, 20 mM fructose, 20 mM fructose, 20 mM maltose, 20 mM pyruvate, 15 mM succinate, 20 mM fumarate, and 20 mM malate.

<span id="page-5-0"></span>Batch culture growth: The batch cultures were performed either in the 37 °C incubator shaker shaking at 250 rpm or in the microplate reader, which holds the temperature at 37 °C and shakes at 900 rpm. The culture volume was 200 μL in 96-well plates, 1 mL in 5 mL round-bottom tubes or 50 mL in 100 mL flasks. Every batch culture was performed as described below. Single colony from the LB agar plate was first inoculated into 50 mL LB medium and cultured overnight. Then, 0.5 mL overnight culture was inoculated into 50 mL LB medium and cultured for 2 h. Cells were centrifuged at 4000 rpm for 2 min, and the cell pellets were diluted to OD600 = 0.001 in the culture medium (M9 medium supplemented with various carbon sources). Then, the medium was cultured in microplate reader (measuring growth rate) or incubator shaker (isotope labeling).

Growth rate measurement: Each well of the 96-well plate was covered with a 200-μL culture medium (OD600 = 0.001). Cells were cultured at 37 °C in the microplate reader shaking at 720 rpm with a 2-mm diameter. The microplate reader measured the OD600 of each well at an interval of 5 min for 20 h. Growth rate λ was measured as the multiplicative inverse of doubling time:

$$
\lambda = \frac{d\log_2 \text{OD}_{\text{@0}}}{dt}.\tag{6}
$$

The interval of 2 h with the maximum Pearson correlation coefficient was defined to be in the exponential phase and the slope of this interval was the growth rate. For some cultures, the OD600 remains constant during the first 20 h. Thus the record time was extended to 72 h for these cultures.

Isotope labeling: The following 13C carbon sources were applied in the isotope labeling experiments: glucose (Product code: CLM-1396; Cambridge Isotope Laboratories, Inc.), fructose (Product number: 587621; Sigma-Aldrich) and glycerol (Product number: 489476; Sigma-Aldrich) (Group A); pyruvate (Product code: CLM-2440; Cambridge Isotope Laboratories, Inc.) and succinate (Product number: 491985; Sigma-Aldrich) (Group B). In an isotope labeling experiment, there are two types of carbon sources: one is uniformly labeled with 13C, while the other one is not labeled. In every experiment, 1 mL 13C-labeled culture medium (OD600 = 0.001) was inoculated into the 5 mL round-bottom tube and cultured in the incubator shaker until the OD600 = 0.150 to 0.250 (7–8 generations). Three independent experiments (with numerous distinct cells) were carried out for each combination of mixed carbon sources.

Extraction and derivatization of amino acids[50](#page-6-0): Cells labeled by 13C were harvested by centrifuging for 3 min at 12,000 rpm. The cell pellets were washed with 1 mL PBS and centrifuged for 2 min at 12,000 rpm twice, and then resuspended in 200 μL of 6 M HCl. The resuspended cells were transferred into sealed 1.5 mL tubes and hydrolyzed for 20 h at 105 ℃. The cell hydrolysate was dried at 65 °C under the fume hood. The dry hydrolysate was resuspended with 40 μL N,N-Dimethylformamide and 20 μL N-tert-butyldimethylsilyl-Nmethyltrifluoroacetamide and heated at 85 ℃ for 1 h so that the amino acids were derivatized (structure shown in Supplementary Fig. 7b). The solution of mixed derivatized amino acids was filtered with 13 mm syringe filter with 0.2 μm membrane.

GC-MS setup: GC-MS analysis was carried out using the Hybrid Quadrupole-Orbitrap GC-MS/MS System (Q Exactive GC, ThermoFisher). The injected sample volume was 1 μL at a carrier gas flow of 1.200 mL/min helium with a split ratio of 1:4.2. The oven temperature was initially set at 150°C and maintained for 2 min, raised to 180°C at 5°C/min and immediately to 260°C at 10 °C/min and maintained for 8 min, and then raised to 350°C/min and maintained for 5 min. The ionization mode was set as electron impact ionization. The ion source temperature was set at 230 °C. The MS transfer line temperature was set at 250 °C. The scan range was 50.0 to 650.0 m/z with a resolution at 60,000. The MS was tuned to 414.0 m/z.

GC-MS analysis of 13C labeled derivatized amino acids: The derivatized amino acids was analyzed by GC-MS with the setup described above. Different kinds of derivatized amino acids in one sample were separated in the gas chromatography (GC) according to their different retention time. The amino acids were fragmented during ionization, forming different kinds of fragments (Supplementary Fig. 7b). Different kinds of fragment of the same derivatized amino acid and their relative abundance (Supplementary Fig. 7b–c) were analyzed by mass spectrometry (MS). The labeling percentage of a certain amino acid can be inferred from the labeling percentage of its fragments. Thermo Xcalibur4.0 was used to view and process the GC-MS data. According to the relative retention time in the chromatogram (Supplementary Fig. 7a) and the corresponding mass spectrum (Supplementary Fig. 7b), 13 kinds of amino acid were detected. The integrated mass spectrum over the full peak range of every derivatized amino acid was obtained to calculate the 13C labeling percentag[e51.](#page-6-0) For a typical derivatized amino acid, 5 types of fragment, M-15, M-57, M-85, M-159 and f302 (Supplementary Fig. 7b), were detected. M-57 denotes that this fragment weighs 57 daltons less than the corresponding derivatized amino acids, and the same goes for M-15, M-85 and M-159. f302 denotes a fragment of weights 302 daltons. For a certain fragment containing N carbon atoms from the underivatized amino acids (the natural form of concern), there were (N þ 1) kinds of mass isotopomer incorporating 0 N 13C respectively. Ii denoted the intensity of mass isotopomer that had i 13C and N i 12C. The 13C labeling percentage ζ of this fragment was calculated as follows:

$$\zeta = \frac{\sum_{i=1}^{N} t \cdot I_i}{N \cdot \sum_{i=1}^{N} I_i}. \tag{7}$$

Amino acids 13C labeling data of different fragments: There are five types of fragments (M-15, M-57, M-85, M-159 and f302) formed during the ionization of amino acids. With each type of raw data, we can calculate (with Eq. (7)) a set of 13C labeling percentages for amino acids. According to the molecular structure, these results can be classified into the following three categories: using (1) M-15/M-57; (2) M-85/M-159; and (3) f302. Only in the first category, there is no carbon atom loss during the fragmentation of the amino acids of concern (Supplementary Fig. 7). This means that M-15/M-57 reflects the exact 13C labeling percentages of amino acids. Yet, the signal intensity of M-15 is faint, thus M-57 is the best choice among the fragments. However, in practice, Leu M-57 and Ile M-57 fragments share the same mass as that of f302 and thus are not applicable for analysis. Consequently, we used Leu M-15 and Ile M-15 to calculate the 13C labeling percentage of Leu and Ile, and M-57 data for other amino acids throughout our manuscript and Supplementary Information unless otherwise specified. In the second category, the carboxylic carbon atom in an amino acid was lost during the fragmentation (Supplementary Fig. 7). Thus M-85/M-159 data can reflect the 13C labeling percentages for a majority yet not all carbon atoms in amino acids, and indeed M-85 and M57 data show a very good agreement with each other (Supplementary Fig. 5). In the third category, all carbon atoms in the side chain of an amino acid (a considerable proportion) were lost during fragmentation (Supplementary Fig. 7). As a result, we did not use f302 data for calculation in this study.

Reporting summary. Further information on experimental design is available in the Nature Research Reporting Summary linked to this article.

#### Code availability

This paper does not involve computer code. Built-in functions of Origin (v9) were used for curve fitting in Supplementary Fig. 2.

#### Data availability

The data that support the findings of this study are available from the corresponding author (C.T.) upon request. The source data underlying Figs. [3](#page-3-0)–[4](#page-4-0), Supplementary Figs. 5–6, Supplementary Note 4.7 and Supplementary Table 5 are provided with the paper.

Received: 3 April 2017 Accepted: 1 March 2019

#### References

- 1. Duclaux, E. Traité de Microbiologie (Masson, Paris, 1899).
- 2. Monod, J. Recherches sur la croissance des cultures bactériennes. Thesis, Hermann et Cie, Paris (1942).
- 3. Monod, J. The phenomenon of enzymatic adaptation and its bearings on problems of genetics and cellular differentiation. Growth 11, 223–289 (1947).
- 4. Görke, B. & Stülke, J. Carbon catabolite repression in bacteria: many ways to make the most out of nutrients. Nat. Rev. Microbiol. 6, 613–624 (2008).
- 5. Gancedo, J. M. Yeast carbon catabolite repression. Microbiol. Mol. Biol. Rev. 62, 334–361 (1998).
- 6. Hermsen, R., Okano, H., You, C., Werner, N. & Hwa, T. A growth-rate composition formula for the growth of E.coli on co-utilized carbon substrates. Mol. Syst. Biol. 11, 801 (2015).
- 7. Kleijn, R. J. et al. Metabolic fluxes during strong carbon catabolite repression by malate in Bacillus subtilis. J. Biol. Chem. 285, 1587–1596 (2010).
- 8. Narang, A. & Pilyugin, S. S. Bacterial gene regulation in diauxic and nondiauxic growth. J. Theor. Biol. 244, 326–348 (2007).
- 9. Magasanik, B. Catabolite repression. Cold Spring Harb Sym. Quant. Biol. 26, 249–256 (1961).
- 10. Deutscher, J., Francke, C. & Postma, P. W. How phosphotransferase systemrelated protein phosphorylation regulates carbohydrate metabolism in bacteria. Microbiol. Mol. Biol. Rev. 70, 939–1031 (2006).
- 11. Neidhardt, F. C. & Magasanik, B. Inhibitory effect of glucose on enzyme formation. Nature 178, 801–802 (1956).
- 12. Deutscher, J. The mechanisms of carbon catabolite repression in bacteria. Curr. Opin. Microbiol. 11, 87–93 (2008).
- 13. Aidelberg, G. et al. Hierarchy of non-glucose sugars in Escherichia coli. BMC Syst. Biol. 8, 133 (2014).
- 14. Kolb, A., Busby, S., Buc, H., Garges, S. & Adhya, S. Transcriptional regulation by cAMP and its receptor protein. Annu. Rev. Biochem. 62, 749–795 (1993).
- <span id="page-6-0"></span>15. Busby, S. & Ebright, R. H. Transcription activation by catabolite activator protein (CAP). J. Mol. Biol. 293, 199–213 (1999).
- 16. Müller-Hill, B. The lac Operon (Walter de Gruyter, New York, 1996).
- 17. Jacob, F. & Monod, J. Genetic regulatory mechanisms in the synthesis of proteins. J. Mol. Biol. 3, 318–356 (1961).
- 18. You, C. et al. Coordination of bacterial proteome with metabolism by cyclic AMP signalling. Nature 500, 301–306 (2013).
- 19. Scott, M., Klumpp, S., Mateescu, E. M. & Hwa, T. Emergence of robust growth laws from optimal regulation of ribosome synthesis. Mol. Syst. Biol. 10, 747 (2014).
- 20. Scott, M., Gunderson, C. W., Mateescu, E. M., Zhang, Z. & Hwa, T. Interdependence of cell growth and gene expression: origins and consequences. Science 330, 1099–1102 (2010).
- 21. Hui, S. et al. Quantitative proteomic analysis reveals a simple strategy of global resource allocation in bacteria. Mol. Syst. Biol. 11, 784 (2015).
- 22. Thattai, M. & Shraiman, B. I. Metabolic switching in the sugar phosphotransferase system of Escherichia coli. Biophys. J. 85, 744–754 (2003).
- 23. Edwards, J. S., Ibarra, R. U. & Palsson, B. O. In silico predictions of Escherichia coli metabolic capabilities are consistent with experimental data. Nat. Biotechnol. 19, 125–130 (2001).
- 24. Dekel, E. & Alon, U. Optimality and evolutionary tuning of the expression level of a protein. Nature 436, 588–592 (2005).
- 25. Wortel, M. T., Peters, H., Hulshof, J., Teusink, B. & Bruggeman, F. J. Metabolic states with maximal specific rate carry flux through an elementary flux mode. FEBS J. 281, 1547–1555 (2014).
- 26. Müller, S., Regensburger, G. & Steuer, R. Enzyme allocation problems in kinetic metabolic networks: optimal solutions are elementary flux modes. J. Theor. Biol. 347, 182–190 (2014).
- 27. Li, S. H.-J. et al. Escherichia coli translation strategies differ across carbon, nitrogen and phosphorus limitation conditions. Nat. Microbiol. 3, 939–947 (2018).
- 28. Schuster, S. & Hilgetag, C. On elementary flux modes in biochemical reaction systems at steady state. J. Biol. Syst. 2, 165–182 (1994).
- 29. Neidhardt, F. C., Ingraham, J. L. & Schaechter, M. Physiology of the Bacterial Cell: a Molecular Approach (Sinauer, Sunderland, 1990).
- 30. Schönheit, P., Buckel, W. & Martin, W. F. On the origin of heterotrophy. Trends Microbiol. 24, 12–25 (2016).
- 31. Lehninger, A. L., Nelson, D. L. & Cox, M. M. Lehninger Principles of Biochemistry 5th edn. (W.H. Freeman, New York, 2008).
- 32. Peyraud, R., Kiefer, P., Christen, P., Portais, J.-C. & Vorholt, J. A. Coconsumption of methanol and succinate by Methylobacterium extorquens AM1. PLoS One 7, e48271 (2012).
- 33. Escalante-Chong, R. et al. Galactose metabolic genes in yeast respond to a ratio of galactose and glucose. Proc. Natl Acad. Sci. USA 112, 1636–1641 (2015).
- 34. Monod, J. The growth of bacterial cultures. Annu. Rev. Microbiol. 3, 371–394 (1949).
- 35. Park, J. O. et al. Metabolite concentrations, fluxes and free energies imply efficient enzyme usage. Nat. Chem. Biol. 12, 482 (2016).
- 36. Bennett, B. D. et al. Absolute metabolite concentrations and implied enzyme active site occupancy in Escherichia coli. Nat. Chem. Biol. 5, 593 (2009).
- 37. Phillips, R. & Milo, R. A feeling for the numbers in biology. Proc. Natl Acad. Sci. USA 106, 21465–21471 (2009).
- 38. Towbin, B. D. et al. Optimality and sub-optimality in a bacterial growth law. Nat. Commun. 8, 14123 (2017).
- 39. Kussell, E. & Leibler, S. Phenotypic diversity, population growth, and information in fluctuating environments. Science 309, 2075–2078 (2005).
- 40. de Jong, I. G., Haccou, P. & Kuipers, O. P. Bet hedging or not? A guide to proper classification of microbial survival strategies. BioEssays 33, 215–223 (2011).
- 41. Lambert, G. & Kussell, E. Memory and fitness optimization of bacteria under fluctuating environments. PLoS Genet. 10, e1004556 (2014).
- 42. Mitchell, A. et al. Adaptive prediction of environmental changes by microorganisms. Nature 460, 220–224 (2009).
- 43. New, A. M. et al. Different levels of catabolite repression optimize growth in stable and variable environments. PLoS Biol. 12, e1001764 (2014).
- 44. Siegal, M. L. Shifting sugars and shifting paradigms. PLoS Biol. 13, e1002068 (2015).
- 45. Venturelli, O. S., Zuleta, I., Murray, R. M. & El-Samad, H. Population diversification in a yeast metabolic program promotes anticipation of environmental shifts. PLoS Biol. 13, e1002042 (2015).
- 46. Wang, J. et al. Natural variation in preparation for nutrient depletion reveals a cost–benefit tradeoff. PLoS Biol. 13, e1002041 (2015).
- 47. Liu, X. et al. Reliable cell cycle commitment in budding yeast is ensured by signal integration. eLife 4, e03977 (2015).
- 48. Solopova, A. et al. Bet-hedging during bacterial diauxic shift. Proc. Natl Acad. Sci. USA 111, 7427–7432 (2014).
- 49. Postma, P. W., Lengeler, J. W. & Jacobson, G. R. Phosphoenolpyruvate: carbohydrate phosphotransferase systems of bacteria. Microbiol. Rev. 57, 543–594 (1993).
- 50. Zamboni, N., Fendt, S.-M., Rühl, M. & Sauer, U. 13 C-based metabolic flux analysis. Nat. Protoc. 4, 878 (2009).
- 51. Dauner, M. & Sauer, U. GC‐MS analysis of amino acids rapidly provides rich information for isotopomer balancing. Biotechnol. Prog. 16, 642–649 (2000).

#### Acknowledgements

We thank Yiping Wang for providing us the strain NCM3722, and Terence Hwa, Yuan Yuan, Haoyuan Sun, Xiaohui Liu, Yanjun Li, Shaoqi Zhu, and Yimiao Qu for helpful discussions. This work was supported by Chinese Ministry of Science and Technology (2015CB910300) and National Natural Science Foundation of China (91430217).

# Author contributions

X.W. and C.T. conceived and designed the project, developed the model, designed the experiments, and wrote the paper. X.W. carried out the theoretical analysis. K.X. carried out the experiments. X.W. and K.X. analyzed the experimental data. C.T. supervised the whole project. X.Y. co-supervised the experiments.

# Additional information

Supplementary Information accompanies this paper at [https://doi.org/10.1038/s41467-](https://doi.org/10.1038/s41467-019-09261-3) [019-09261-3.](https://doi.org/10.1038/s41467-019-09261-3)

Competing interests: The authors declare no competing interests.

Reprints and permission information is available online at [http://npg.nature.com/](http://npg.nature.com/reprintsandpermissions/) [reprintsandpermissions/](http://npg.nature.com/reprintsandpermissions/)

Journal peer review information: Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available.

Publisher's note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit [http://creativecommons.org/](http://creativecommons.org/licenses/by/4.0/) [licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

© The Author(s) 2019