![](_page_0_Picture_0.jpeg)

![](_page_0_Picture_2.jpeg)

# [Heterogeneity in Pure Microbial](http://journal.frontiersin.org/article/10.3389/fmicb.2017.01813/abstract) Systems: Experimental Measurements and Modeling

[Rebeca González-Cabaleiro](http://loop.frontiersin.org/people/458233/overview)1† , [Anca M. Mitchell](http://loop.frontiersin.org/people/469235/overview)1† , [Wendy Smith](http://loop.frontiersin.org/people/469209/overview)2 , Anil Wipat2 and [Irina D. Ofi¸teru](http://loop.frontiersin.org/people/387225/overview)1 *

1 School of Engineering, Chemical Engineering, Newcastle University, Newcastle upon Tyne, United Kingdom, 2 Interdisciplinary Computing and Complex BioSystems (ICOS), School of Computing, Newcastle University, Newcastle upon Tyne, United Kingdom

#### Edited by:

Clara Prats, Universitat Politecnica de Catalunya, Spain

#### Reviewed by:

Xavier Portell, Cranfield University, United Kingdom Susann Müller, Helmholtz-Zentrum für Umweltforschung (UFZ), Germany

#### *Correspondence:

Irina D. Ofi¸teru dana.ofiteru@ncl.ac.uk

†These authors have contributed equally to this work.

#### Specialty section:

This article was submitted to Systems Microbiology, a section of the journal Frontiers in Microbiology

Received: 29 June 2017 Accepted: 05 September 2017 Published: 20 September 2017

#### Citation:

González-Cabaleiro R, Mitchell AM, Smith W, Wipat A and Ofi¸teru ID (2017) Heterogeneity in Pure Microbial Systems: Experimental Measurements and Modeling. Front. Microbiol. 8:1813. doi: [10.3389/fmicb.2017.01813](https://doi.org/10.3389/fmicb.2017.01813) Cellular heterogeneity influences bioprocess performance in ways that until date are not completely elucidated. In order to account for this phenomenon in the design and operation of bioprocesses, reliable analytical and mathematical descriptions are required. We present an overview of the single cell analysis, and the mathematical modeling frameworks that have potential to be used in bioprocess control and optimization, in particular for microbial processes. In order to be suitable for bioprocess monitoring, experimental methods need to be high throughput and to require relatively short processing time. One such method used successfully under dynamic conditions is flow cytometry. Population balance and individual based models are suitable modeling options, the latter one having in particular a good potential to integrate the various data collected through experimentation. This will be highly beneficial for appropriate process design and scale up as a more rigorous approach may prevent a priori unwanted performance losses. It will also help progressing synthetic biology applications to industrial scale.

Keywords: population heterogeneity, single cell analysis, flow cytometry, population balance models, individual based models

## INTRODUCTION

Microbial populations developing in seemingly homogenous environments have been historically considered as formed by identical individuals. In reality no two cells in a pure culture are alike, even if they are derived from single clonal colonies [(Ackermann,](#page-4-0) [2015)](#page-4-0). This phenomenon is of fundamental importance in biotechnological fermentations as the yields obtained will be lower if the cells are not in the same optimal productive state [(Fernandes et al.,](#page-5-0) [2011)](#page-5-0).

In bioprocess industries the bioreactors, and in particular the stirred tanks, are the central production units. The performance of any bioreactor is the emergent property of the activity and interactions at the single cell level and therefore, variations at this level can profoundly affect the dynamics and productivity of the process. Fluctuations that affect cell metabolism in industrial fermentations are estimated to generate losses of 30% or above [(Lara et al.,](#page-5-1) [2006;](#page-5-1) [Takors,](#page-6-0) [2012)](#page-6-0). Moreover, sometimes recombinant protein production processes fail completely for reasons which are not fully understood but can be related to heterogeneity in the microbial population [(Rosano](#page-6-1) [and Ceccarelli,](#page-6-1) [2014)](#page-6-1).

The effect of cell individuality when using bacteria for obtaining useful products has been emphasized recently in several publications [(Li and You,](#page-6-2) [2013;](#page-6-2) [Wyre and Overton,](#page-7-0) [2014a,](#page-7-0)[b;](#page-7-1) [Chen et al.,](#page-5-2) [2015)](#page-5-2). Nevertheless, the true impact of microbial population heterogeneity on bioprocesses remains unknown [(Delvigne and Goffin,](#page-5-3) [2014)](#page-5-3) and therefore it is not systematically considered in design. This is partially due to the fact that experimental biological data obtained with traditional methods represents population average information [(Pasotti and Zucca,](#page-6-3) [2014)](#page-6-3) which means that the performance of individuals is masked [(Ackermann and Schreiber,](#page-4-1) [2015)](#page-4-1). Another potential reason is the relatively limited options for monitoring the heterogeneity under dynamic conditions. There are suggestions that minor subpopulations will not have significant influence on the whole population function [(Lidstrom](#page-6-4) [and Konopka,](#page-6-4) [2010)](#page-6-4), but more recent work emphasizes that non-genetic variation plays an important role in the overall biosynthetic performance of a bioprocess [(Xiao et al.,](#page-7-2) [2016)](#page-7-2).

Ultimately, industry needs to be able to engineer heterogeneity to obtain better yields and more robust processes. This requires both quantitative evaluation of the change of individual cells in time and of their interaction with the environment [(Bley,](#page-4-2) [2011;](#page-4-2) [Sauer and Mattanovich,](#page-6-5) [2012)](#page-6-5). Furthermore, this information needs to be included in mathematical frameworks used for design and control in order to have a realistic representation of the bioprocesses and to improve their performance.

In this mini-review we present an overview of the experimental methods used for characterizing the cell to cell variation in bacterial cultures and the corresponding mathematical tools for modeling them (see **[Figure 1](#page-2-0)**), with a focus on the appropriate ones for fermentation processes.

#### SOURCES OF CELL HETEROGENEITY

The sources of heterogeneity in clonal microbial populations are biological (intrinsic) or environmental (extrinsic), or both. Whereas the intrinsic heterogeneity is generated by factors as cell cycle states, age distribution or the stochasticity of gene expression and metabolic reactions, the source for the extrinsic heterogeneity are the fluctuations in the environment. Therefore the latter represents a physiological response to stress [(Lidstrom](#page-6-4) [and Konopka,](#page-6-4) [2010;](#page-6-4) [Ryall et al.,](#page-6-6) [2012)](#page-6-6) and a survival strategy developed over evolutionary times [(Booth,](#page-4-3) [2002;](#page-4-3) [Sumner and](#page-6-7) [Avery,](#page-6-7) [2002)](#page-6-7). This is the real challenge in the scaling-up of bioprocesses as poor mixing and heat transfer limitations generate concentration gradients which further influence the cells physiology.

A proposed way to overcome extrinsic heterogeneity and obtain similar performance in large scale reactors compared with laboratory reactors is to use strains specifically engineered to withstand certain environmental variability [(Löffler et al.,](#page-6-8) [2016)](#page-6-8). However, some investigations, both by modeling [(Lavric and](#page-5-4) [Graham,](#page-5-4) [2010)](#page-5-4) and experimental studies [(Chi Fru et al.,](#page-5-5) [2011;](#page-5-5) [Ofi¸teru et al.,](#page-6-9) [2012)](#page-6-9) suggest that bacterial populations display constant heterogeneity in apparently steady growth and habitat conditions, questioning the very existence of truly homogenous cultures [(Grote et al.,](#page-5-6) [2015)](#page-5-6).

### EXPERIMENTAL METHODS

The first step in single cell analysis is the isolation and/or immobilization of individuals from cell suspension. The experimental methods employed for this include serial dilutions (the traditional method), physical trapping (mechanical, hydrodynamic or dielectrophoretic), flow suspension [e.g., flow cytometry and in particular fluorescence-activated cell sorting [(Winson and Davey,](#page-6-10) [2000)](#page-6-10)] and micromanipulation [mechanical or with optical tweezers [(Landry et al.,](#page-5-7) [2013)](#page-5-7)]. Between these techniques, the use of trapping of single cells in lab-on-a-chip microfluidic devices is expanding.

Once the isolation is achieved, single cell heterogeneity can be assessed. Bioprocess monitoring requires high throughput methods which allow rapid and highly parallel experimentation, with relative fast processing time. In general, the methods for single cell analysis were primarily developed for basic research and not all of them are adapted to be used for fermentation processes [(Geiler-Samerotte et al.,](#page-5-8) [2013)](#page-5-8). At the same time, some micro tools for isolation and interrogation of single cells developed for mammalian cells need further refinement when dealing with smaller microbial cells [(Love et al.,](#page-6-11) [2013)](#page-6-11).

The experimental methods for monitoring and assessing single cell heterogeneity can be classified as: (i) biophysical characterization; (ii) gene expression; (iii) protein analysis; and (iv) metabolite analysis. Several detailed reviews exist on single cell heterogeneity analysis [(Brehm-Stecher and Johnson,](#page-5-9) [2004;](#page-5-9) [Amantonico et al.,](#page-4-4) [2010;](#page-4-4) [Fernandes et al.,](#page-5-0) [2011;](#page-5-0) [Lecault et al.,](#page-6-12) [2012;](#page-6-12) [Shi et al.,](#page-6-13) [2015;](#page-6-13) [Vasdekis et al.,](#page-6-14) [2015)](#page-6-14). We are presenting a general overview, emphasizing the ones appropriate for microbial population under dynamic process conditions.

Biophysical characterization gives information on the cell size, mass, volume, internal structure, and mechanical properties. Optical microscopy is widely used, being the simplest and quickest method but with low throughput. A high throughput method which provides information on cell size is flow cytometry. The composition of the individual cells can be obtained by Raman spectroscopy, a label free optical method that has been used for bioprocess investigation [(Huang et al.,](#page-5-10) [2004)](#page-5-10).

Gene expression methods (e.g., RT-qPCR, RNA-seq, FISH) give information on the expression state of a certain gene and protein synthesis dynamics. However, if a method involves lysis of cells, the dynamics of the gene expression in bioprocesses cannot be followed. An alternative is to use reporter systems (e.g., green fluorescent protein and its variants) that can be monitored with fluorescent time-lapse microscopy [(Young et al.,](#page-7-3) [2012)](#page-7-3). The limitation in this case is that production cultures usually do not contain fluorescent protein as marker and therefore this method is less suitable for monitoring fermentation processes.

Protein analysis at the single cell level can, in theory, provide information on protein abundance, protein secretion, or protein/protein interaction. Flow cytometry is the most commonly used method for measuring the protein content [(Wu](#page-6-15)

![](_page_2_Figure_2.jpeg)

<span id="page-2-0"></span>[and Singh,](#page-6-15) [2012)](#page-6-15). Mass spectrometry has a high sensitivity and offers high quantity of information, from simultaneous identification of proteins to their posttranslational modifications. A workflow for sorted subpopulations, involving flow cytometry and mass spectrometry, was reported by [Jahn et al.](#page-5-11) [(2013)](#page-5-11). Nevertheless, there are still significant limitations due to the complexity of the proteome, the small amount of protein and the various types of measurement to be performed.

Metabolites analysis (intracellular and extracellular) is an indirect measurement of the phenotype of the biological system. The small size of the microbial cell and the minute quantity of metabolites make their detection at the single cell level very challenging. Methods used successfully in proof-ofconcept experiments are Raman microspectroscopy, secondary ion mass spectrometry (SIMS) and Fourier transform infrared spectroscopy [(Heinemann and Zenobi,](#page-5-12) [2011;](#page-5-12) [Armitage et al.,](#page-4-5) [2013;](#page-4-5) [Rubakhin et al.,](#page-6-16) [2013)](#page-6-16). Coupling a microfluidic unit to a mass spectrometer has the highest potential to deliver relevant data. NanoSIMS is a powerful tool for revealing element distribution in nanometer-scale resolution [(Musat](#page-6-17) [et al.,](#page-6-17) [2012;](#page-6-17) [Gao et al.,](#page-5-13) [2016)](#page-5-13). However, the single cell metabolite analysis is considered to still be in its early stages.

From the reviewed methods, flow cytometry is the most suitable, relatively fast and user-friendly for measuring phenotypic single cell heterogeneity in bioprocesses and under dynamic conditions [(Want et al.,](#page-6-18) [2009;](#page-6-18) [Muller and](#page-6-19) [Nebe-von-Caron,](#page-6-19) [2010;](#page-6-19) [Ambriz-Aviña et al.,](#page-4-6) [2014;](#page-4-6) [Delvigne and](#page-5-3) [Goffin,](#page-5-3) [2014;](#page-5-3) [Baert et al.,](#page-4-7) [2016)](#page-4-7). Flow cytometry measures the distribution of a large variety of cellular parameters across a cell population by analyzing the light scattering and fluorescent signals of stained cells which flow in front of a powerful light source (e.g., a laser beam). Individual cells can be segregated based on their size, shape, intracellular properties, membrane potential, and variation in fluorescent signal. Because the large number of cells (tens of thousands) measured in a short processing time, flow cytometry offers statistically significant results and provides a quantitative measurement of heterogeneity in the sample, having the potential to identifying rare cell types [(Shapiro,](#page-6-20) [2000;](#page-6-20) [Davey and Winson,](#page-5-14) [2003)](#page-5-14). Recently, in combination with supervised machine learning techniques, flow cytometry was used also for single cell identification of populations in synthetic bacterial communities [(Rubbens et al.,](#page-6-21) [2017)](#page-6-21).

Nevertheless, the challenge of the high throughput methods is the amount of data generated, which requires rigorous quality control, together with sophisticated bioinformatics and statistics. Therefore, although automated flow-cytometry was expected to be implemented for real-time quality programs in factories [(Hewitt et al.,](#page-5-15) [1999;](#page-5-15) [Díaz et al.,](#page-5-16) [2010)](#page-5-16), to date single cell characterization is not routinely used in-process [(Royle et al.,](#page-6-22) [2013)](#page-6-22).

A complementary way to evaluate the microbial phenotypic heterogeneity under realistic bioprocess conditions is by employing scale down single cell micro-cultivation devices in which large scale reactors conditions are mimicked. Single cells in lab-on-a-chip microfluidic devices allow parallelization and high throughput experiments [(Grünberger et al.,](#page-5-17) [2014;](#page-5-17) [Dusny and Schmid,](#page-5-18) [2015;](#page-5-18) [Rosenthal et al.,](#page-6-23) [2015;](#page-6-23) [Oliveira](#page-6-24) [et al.,](#page-6-24) [2016)](#page-6-24), contributing to large-scale bioprocess improvement [(Grünberger et al.,](#page-5-19) [2012;](#page-5-19) [Ladner et al.,](#page-5-20) [2017)](#page-5-20). Sorting of different subpopulations of cells in order to understand the physiological responses in fluctuating microenvironments was also performed by microarray analysis [(Hewitt et al.,](#page-5-21) [2007)](#page-5-21).

#### MODELING OPTIONS FOR HETEROGENEOUS POPULATIONS

There is currently a gap between the new methods for single cell analysis and the availability of mathematical models which can integrate the data collected. But models are essential in the design and control of bioprocesses. Use of the complex information obtained by investigations of cells at genomic, transcriptomic, proteomic, and metabolomic level to predict bioprocesses is challenging and requires multidisciplinary analysis and significant computational efforts [(Zhang et al.,](#page-7-4) [2010)](#page-7-4).

The traditional classification of the mathematical models for cell populations is in unsegregated/segregated and unstructured/structured. The least complex is a model unstructured and unsegregated, which considers a homogenous population represented by an average unstructured cell, while the most complex is a model structured and segregated, which considers a heterogeneous population of structured cells [(Song](#page-6-25) [et al.,](#page-6-25) [2014)](#page-6-25). The behavior of an average cell is representative only for a synchronous population [(Noack et al.,](#page-6-26) [2008)](#page-6-26), but for a heterogeneous population the model needs to include at least the segregation in the biophase.

Since their initial development, single cell models were seen as a promise for connecting the macroscopic bioreactor with the microscopic one, the cell [(Shuler,](#page-6-27) [1999)](#page-6-27). Integrated multiomics predictive models can inform biological discovery but their application is in its infancy [(Brink et al.,](#page-5-22) [2016;](#page-5-22) [Kim et al.,](#page-5-23) [2016)](#page-5-23). Some authors have attempted models which involve a laborious theoretical development to account for different sources of heterogeneity [(Stamatakis and Zygourakis,](#page-6-28) [2010)](#page-6-28), though the same authors acknowledge them as being far too complex to lend themselves for practical application [(Fredrickson and Mantzaris,](#page-5-24) [2002)](#page-5-24). Therefore, so far, the distributed properties measured within cell population are not integrated in a single modeling framework appropriate to be used in design, optimization and control of bioprocesses [(Henson,](#page-5-25) [2003;](#page-5-25) [Müller et al.,](#page-6-29) [2010;](#page-6-29) [Fernandes et al.,](#page-5-0) [2011)](#page-5-0).

Here, we are presenting two options for modeling heterogeneity: population balance models (PBM) and individual based models (IBM). Both modeling approaches describe the variation in the population, but, while the PBM consider each fraction of the population as a continuous phase, in IBM the cells are discrete particles.

### Population Balance Models

In PBM cells are differentiated based on variables which characterize their intracellular state. Most commonly these variables are cell age or/and cell mass. The mass balances for substrates, biomass, and products are represented by non-linear and partial differential equations which have as independent variables time and the internal state of the cells. The different phases during the cell life cycle can be represented. The results obtained with PBM will predict the time variation of the cell number distribution, as resulted from growth and division.

Multidimensional PBM can be developed based on flow cytometry data [(Fernandes et al.,](#page-5-26) [2013;](#page-5-26) [Ramkrishna and Singh,](#page-6-30) [2014)](#page-6-30). Biological heterogeneity in bioreactors was modeled by coupling a population balance model with a biokinetic model [(Morchain et al.,](#page-6-31) [2013)](#page-6-31) and later with a hydrodynamic model [(Pigou and Morchain,](#page-6-32) [2015)](#page-6-32). One important limitation of PBM is that they are computationally demanding if they are represented more than one single internal state of the cells and this limits on line applications [(Royle et al.,](#page-6-22) [2013)](#page-6-22).

#### Individual Based Models

In IBM the cells are discrete particles which interact with each other and with the environment. Microbial characteristics are described at single cell level. This allows the study of the system behavior as a result of the properties and performance of the individual components [(Railsback and Grimm,](#page-6-33) [2012)](#page-6-33). However, it is not always possible to simulate all the individual cells of the system due to computing constraints and choices need to be made about the type of agents used (a cell or a cluster of cells or superindividual) and the level of detail for each of them.

In the last two decades IBM have gained popularity in microbiology [(Ferrer et al.,](#page-5-27) [2008;](#page-5-27) [Schuler et al.,](#page-6-34) [2011;](#page-6-34) [Hellweger](#page-5-28) [et al.,](#page-5-28) [2016)](#page-5-28) due to rapid advancement in computational technologies and the development of specialized software. Open source generic platforms are now available (e.g., [Sklar,](#page-6-35) [2007;](#page-6-35) [Lardon et al.,](#page-5-29) [2011;](#page-5-29) [Rudge et al.,](#page-6-36) [2012;](#page-6-36) [Coakley et al.,](#page-5-30) [2016)](#page-5-30). However, due to their complex structure IBM require more computing skills than other modeling approaches.

Both PBM and IBM approaches can be used for multispecies fermentation and can take into account the environmental heterogeneity in bioreactors (see Coupling the scales). However, PBM models explicitly the behavior of the population and can include only limited cell properties. They are also more restricted in representing stochastic processes as problems of closure may arise [(Ramkrishna and Mahoney,](#page-6-37) [2002)](#page-6-37). Instead, IBM models the behavior of individuals, each having its own properties, with the population behavior emerging from their interactions. Therefore, it has a higher potential to integrate the detailed data generated with single cell analysis. At the same time, IBM offers a better representation of the stochastic processes, being able to describe the average fluctuations and not only the average behavior in a population.

### Coupling the Scales

The solution for PBM and IBM involve a numerical method using discrete time steps. In biological processes there is a wide range of

relevant timescales, varying from nanoseconds to hours. The use of time steps in solving the mathematical models means that all the transformations which have a timescale smaller than the time step chosen for the numerical solution will only be approximated. Their influence on the state variables may then results as nonrealistically high. Therefore, it is important to understand the effect of the approximations on the final output of the model and how it affects its predictive capabilities [(Gameiro et al.,](#page-5-31) [2016)](#page-5-31).

For a complete mathematical representation of the bioprocess, suitable to be used in scale-up and design applications, a two-way coupling between mass transfer, hydrodynamics, and biology is required [(Wang et al.,](#page-6-38) [2015;](#page-6-38) [Morchain,](#page-6-39) [2017)](#page-6-39). These interactions are important as extracellular micro-heterogeneities may amplify the intracellular ones and place an upper limit on productivity and bioprocess reliability [(Vasdekis et al.,](#page-6-14) [2015)](#page-6-14). Local environmental conditions generated by flow streams affect the microbial metabolism and can be described by computational fluid dynamics (CFD). The Euler-Lagrange method represents the appropriate option for studying the impact of substrate gradients on the microbial metabolism in conjunction with the hydrodynamics [(Lapin et al.,](#page-5-32) [2004;](#page-5-32) [Liu et al.,](#page-6-40) [2016;](#page-6-40) [Haringa](#page-5-33) [et al.,](#page-5-33) [2017;](#page-5-33) [Kuschel et al.,](#page-5-34) [2017)](#page-5-34). However, because of the high number of individual cells involved in a fermentation, it is not feasible to directly couple IBM with CFD at the large scale. One useful approach is using statistical emulators (metamodels), which extract the significant information from microscale and are computationally much faster [(Wilkinson,](#page-6-41) [2009;](#page-6-41) [Conti and](#page-5-35) [O'Hagan,](#page-5-35) [2010)](#page-5-35). The advantage of an emulator over using a continuous model is that the former will not select a priori the information to be transmitted across scales but it will be based on a detailed mechanistic single cell model, representing therefore a simplified simulation strategy to calibrate multi-scale models. This approach was recently implemented by [Oyebamiji et al.](#page-6-42) [(2017)](#page-6-42) as an attempt to scale up a microbial system.

#### CONCLUSION

In industrial setups there is a tradeoff between cellular growth and process robustness [(Carlquist et al.,](#page-5-36) [2012)](#page-5-36). Understanding and controlling cell heterogeneity at the single cell level will generate more robust and efficient bioprocesses, as, for example, it has been proven that it is not the highest biomass concentration, but higher proportion of viable cells which gives the best productivity

### REFERENCES

- <span id="page-4-0"></span>Ackermann, M. (2015). A functional perspective on phenotypic heterogeneity in microorganisms. Nat. Rev. Microbiol. 13, 497–508. [doi: 10.1038/nrmicro](https://doi.org/10.1038/nrmicro3491) [3491](https://doi.org/10.1038/nrmicro3491)
- <span id="page-4-1"></span>Ackermann, M., and Schreiber, F. (2015). A growing focus on bacterial individuality. Environ. Microbiol. 17, 2193–2195. [doi: 10.1111/1462-2920.12877](https://doi.org/10.1111/1462-2920.12877)
- <span id="page-4-4"></span>Amantonico, A., Urban, P. L., and Zenobi, R. (2010). Analytical techniques for single-cell metabolomics: state of the art and trends. Anal. Bioanal. Chem. 398, 2493–2504. [doi: 10.1007/s00216-010-3850-1](https://doi.org/10.1007/s00216-010-3850-1)
- <span id="page-4-6"></span>Ambriz-Aviña, V., Contreras-Garduño, J. A., and Pedraza-Reyes, M. (2014). Applications of flow cytometry to characterize bacterial physiological responses. Biomed Res. Int. 2014:461941. [doi: 10.1155/2014/461941](https://doi.org/10.1155/2014/461941)

[(Want et al.,](#page-6-18) [2009)](#page-6-18). Insights into bioprocesses at single cell level are expected to contribute also to the development of more accurate mathematical models that can be applied to the prediction and control of fermentative processes [(Zhang et al.,](#page-7-5) [2015)](#page-7-5). This will be highly beneficial as appropriate process and bioreactor design, able to prevent a priori unwanted performance losses, is still missing [(Takors,](#page-6-0) [2012)](#page-6-0) and scaling up has a high degree of empiricism [(Brognaux et al.,](#page-5-37) [2013)](#page-5-37). IBM have the potential to integrate protein measurements with genomics, transcriptomics and metabolomics, and to predict the dynamics of the system across scales and in different environments [(Hellweger et al.,](#page-5-28) [2016)](#page-5-28), giving a better evaluation of the overall system performance.

This is relevant also for synthetic biology, a rapidly growing field which is limited by the lack of understanding on complex fluctuations in physiology and fitness of overall microbial populations [(Cardinale and Arkin,](#page-5-38) [2012)](#page-5-38). Therefore connecting the single cell dynamics and heterogeneity of cell population with the bioreactor performance is a strategically important objective that is vital to the translation of systems and synthetic biology into an industrial reality.

### AUTHOR CONTRIBUTIONS

All authors contributed to the writing of the manuscript. IO carried out the initial literature review and wrote the initial draft. RG-C provided insight relating to the mathematical modeling. AM and WS provided expertise relating the experimental methods. AW provided over-all guidance of the work and editing of the text.

### FUNDING

AM acknowledges the support of the EPSRC DTA scholarship. RG-C, AW, and IO acknowledge the support of the EPSRC Frontier Grant "Simulation of open engineered biological systems," led by Newcastle University, ref EP/K038648. WS and AW acknowledge the support of the EPSRC Grant "Synthetic Portabolomics: Leading the way at the crossroads of the Digital and the Bio Economies," ref EP/R003629/1. No new data were created during this study.

- <span id="page-4-5"></span>Armitage, E. G., Kotze, H. L., and Lockyer, N. P. (2013). Imaging of metabolites using secondary ion mass spectrometry. Metabolomics 9, S102–S109. [doi: 10.1007/s11306-012-0477-6](https://doi.org/10.1007/s11306-012-0477-6)
- <span id="page-4-7"></span>Baert, J., Delepierre, A., Telek, S., Fickers, P., Toye, D., Delamotte, A., et al. (2016). Microbial population heterogeneity versus bioreactor heterogeneity: evaluation of Redox Sensor Green as an exogenous metabolic biosensor. Eng. Life Sci. 16, 643–651. [doi: 10.1002/elsc.201500149](https://doi.org/10.1002/elsc.201500149)
- <span id="page-4-2"></span>Bley, T. (2011). From single cells to microbial population dynamics: modelling in biotechnology based on measurements of individual cells. Adv. Biochem. Eng. Biotechnol. 124, 211–227. [doi: 10.1007/10_2010_79](https://doi.org/10.1007/10_2010_79)
- <span id="page-4-3"></span>Booth, I. R. (2002). Stress and the single cell: intrapopulation diversity is a mechanism to ensure survival upon exposure to stress. Int. J. Food Microbiol. 78, 19–30. [doi: 10.1016/S0168-1605(02)00239-8](https://doi.org/10.1016/S0168-1605(02)00239-8)

- <span id="page-5-9"></span>Brehm-Stecher, B. F., and Johnson, E. A. (2004). Single-cell microbiology: tools, technologies, and applications. Microbiol. Mol. Biol. Rev. 68, 538–559. [doi: 10.1128/MMBR.68.3.538-559.2004](https://doi.org/10.1128/MMBR.68.3.538-559.2004)
- <span id="page-5-22"></span>Brink, B. G., Seidel, A., Kleinbolting, N., Nattkemper, T. W., and Albaum, S. P. (2016). Omics fusion - a platform for integrative analysis of omics data. J. Integr. Bioinform. 13:296. [doi: 10.2390/biecoll-jib-2016-296.](https://doi.org/10.2390/biecoll-jib-2016-296.)
- <span id="page-5-37"></span>Brognaux, A., Thonart, P., Delvigne, F., Neubauer, P., Twizere, J. C., Francis, F., et al. (2013). Direct and indirect use of GFP whole cell biosensors for the assessment of bioprocess performances: design of milliliter scale-down bioreactors. Biotechnol. Prog. 29, 48–59. [doi: 10.1002/btpr.1660](https://doi.org/10.1002/btpr.1660)
- <span id="page-5-38"></span>Cardinale, S., and Arkin, A. P. (2012). Contextualizing context for synthetic biology - identifying causes of failure of synthetic biological systems. Biotechnol. J. 7, 856–866. [doi: 10.1002/biot.201200085](https://doi.org/10.1002/biot.201200085)
- <span id="page-5-36"></span>Carlquist, M., Fernandes, R. L., Helmark, S., Heins A-L, Lundin, L., Sørensen, S. J., et al. (2012). Physiological heterogeneities in microbial populations and implications for physical stress tolerance. Microb. Cell Fact. 11:94. [doi: 10.1186/](https://doi.org/10.1186/1475-2859-11-94) [1475-2859-11-94](https://doi.org/10.1186/1475-2859-11-94)
- <span id="page-5-2"></span>Chen, Y., Kim, J. K., Hirning, A. J., Josic, K., and Bennett, M. R. (2015). Emergent ´ genetic oscillations in a synthetic microbial consortium. Science 349, 986–989. [doi: 10.1126/science.aaa3794](https://doi.org/10.1126/science.aaa3794)
- <span id="page-5-5"></span>Chi Fru, E., Ofi¸teru, I. D., Lavric, V., and Graham, D. W. (2011). Non-linear population dynamics in chemostats associated with live–dead cell cycling in Escherichia coli strain K12-MG1655. Appl. Microbiol. Biotechnol. 89, 791–798. [doi: 10.1007/s00253-010-2895-6](https://doi.org/10.1007/s00253-010-2895-6)
- <span id="page-5-30"></span>Coakley, S., Richmond, P., Gheorghe, M., Chin, S., Worth, D., Holcombe, M., et al. (2016). Large-scale simulations with FLAME, in Intelligent Agents in Dataintensive Computing, eds J. Kołodziej, L. Correia, and J. Manuel Molina (Cham: Springer International Publishing), 123–142. [doi: 10.1007/978-3-319-237](https://doi.org/10.1007/978-3-319-23742-8_6) [42-8_6](https://doi.org/10.1007/978-3-319-23742-8_6)
- <span id="page-5-35"></span>Conti, S., and O'Hagan, A. (2010). Bayesian emulation of complex multioutput and dynamic computer models. *ı*J. Stat. Plan. Inference 140, 640–651. [doi: 10.1016/j.jspi.2009.08.006](https://doi.org/10.1016/j.jspi.2009.08.006)
- <span id="page-5-14"></span>Davey, H. M., and Winson, M. K. (2003). Using flow cytometry to quantify microbial heterogeneity. Curr. Issues Mol. Biol. 5, 9–15.
- <span id="page-5-3"></span>Delvigne, F., and Goffin, P. (2014). Microbial heterogeneity affects bioprocess robustness: dynamic single-cell analysis contributes to understanding of microbial populations. Biotechnol. J. 9, 61–72. [doi: 10.1002/biot.201300119](https://doi.org/10.1002/biot.201300119)
- <span id="page-5-16"></span>Díaz, M., Herrero, M., García, L. A., and Quirós, C. (2010). Application of flow cytometry to industrial microbial bioprocesses. Biochem. Eng. J. 48, 385–407. [doi: 10.1111/1751-7915.12018](https://doi.org/10.1111/1751-7915.12018)
- <span id="page-5-18"></span>Dusny, C., and Schmid, A. (2015). Microfluidic single-cell analysis links boundary environments and individual microbial phenotypes. Environ. Microbiol. 17, 1839–1856. [doi: 10.1111/1462-2920.12667](https://doi.org/10.1111/1462-2920.12667)
- <span id="page-5-26"></span>Fernandes, R. L., Carlquist, M., Lundin, L., Heins, A. L., Dutta, A., Sorensen, S. J., et al. (2013). Cell mass and cell cycle dynamics of an asynchronous budding yeast population: experimental observations, flow cytometry data analysis, and multi-scale modeling. Biotechnol. Bioeng. 110, 812–826. [doi: 10.1002/bit.24749](https://doi.org/10.1002/bit.24749)
- <span id="page-5-0"></span>Fernandes, R. L., Nierychlo, M., Lundin, L., Pedersen, A. E., Puentes Tellez, P. E., Dutta, A., et al. (2011). Experimental methods and modeling techniques for description of cell population heterogeneity. Biotechnol. Adv. 29, 575–599. [doi: 10.1016/j.biotechadv.2011.03.007](https://doi.org/10.1016/j.biotechadv.2011.03.007)
- <span id="page-5-27"></span>Ferrer, J., Prats, C., and López, D. (2008). Individual-based modelling: an essential tool for microbiology. J. Biol. Phys. 34, 19–37. [doi: 10.1007/s10867-008-9082-3](https://doi.org/10.1007/s10867-008-9082-3)
- <span id="page-5-24"></span>Fredrickson, A. G., and Mantzaris, N. V. (2002). A new set of population balance equations for microbial and cell cultures. Chem. Eng. Sci. 57, 2265–2278. [doi: 10.1016/S0009-2509(02)00116-1](https://doi.org/10.1016/S0009-2509(02)00116-1)
- <span id="page-5-31"></span>Gameiro, D., Pérez-Pérez, M., Pérez-Rodríguez, G., Monteiro, G., Azevedo, N. F., and Lourenço, A. (2016). Computational resources and strategies to construct single-molecule metabolic models of microbial cells. Brief. Bioinform. 17, 863–876. [doi: 10.1093/bib/bbv096](https://doi.org/10.1093/bib/bbv096)
- <span id="page-5-13"></span>Gao, D. W., Huang, X. L., and Tao, Y. (2016). A critical review of NanoSIMS in analysis of microbial metabolic activities at single-cell level. Crit. Rev. Biotechnol. 36, 884–890. [doi: 10.3109/07388551.2015.1057550](https://doi.org/10.3109/07388551.2015.1057550)
- <span id="page-5-8"></span>Geiler-Samerotte, K. A., Bauer, C. R., Li, S., Ziv, N., Gresham, D., and Siegal, M. L. (2013). The details in the distributions: why and how to study phenotypic variability. Curr. Opin. Biotechnol. 24, 752–759. [doi: 10.1016/j.copbio.2013.](https://doi.org/10.1016/j.copbio.2013.03.010) [03.010](https://doi.org/10.1016/j.copbio.2013.03.010)
- <span id="page-5-6"></span>Grote, J., Krysciak, D., and Streit, W. R. (2015). Phenotypic heterogeneity, a phenomenon that may explain why quorum sensing does not always result in truly homogenous cell behavior. Appl. Environ. Microbiol. 81, 5280–5289. [doi: 10.1128/AEM.00900-15](https://doi.org/10.1128/AEM.00900-15)
- <span id="page-5-19"></span>Grünberger, A., Paczia, N., Probst, C., Schendzielorz, G., Eggeling, L., Noack, S., et al. (2012). A disposable picolitre bioreactor for cultivation and investigation of industrially relevant bacteria on the single cell level. Lab Chip 12, 2060–2068. [doi: 10.1039/c2lc40156h](https://doi.org/10.1039/c2lc40156h)
- <span id="page-5-17"></span>Grünberger, A., Wiechert, W., and Kohlheyer, D. (2014). Single-cell microfluidics: opportunity for bioprocess development. Curr. Opin. Biotechnol. 29, 15–23. [doi: 10.1016/j.copbio.2014.02.008](https://doi.org/10.1016/j.copbio.2014.02.008)
- <span id="page-5-33"></span>Haringa, C., Noorman, H. J., and Mudde, R. F. (2017). Lagrangian modeling of hydrodynamic–kinetic interactions in (bio)chemical reactors: practical implementation and setup guidelines. Chem. Eng. Sci. 157, 159–168. [doi: 10.1016/j.ces.2016.07.031](https://doi.org/10.1016/j.ces.2016.07.031)
- <span id="page-5-12"></span>Heinemann, M., and Zenobi, R. (2011). Single cell metabolomics. Curr. Opin. Biotechnol. 22, 26–31. [doi: 10.1016/j.copbio.2010.09.008](https://doi.org/10.1016/j.copbio.2010.09.008)
- <span id="page-5-28"></span>Hellweger, F. L., Clegg, R. J., Clark, J. R., Plugge, C. M., and Kreft, J. U. (2016). Advancing microbial sciences by individual-based modelling. Nat. Rev. Microbiol. 14, 461–471. [doi: 10.1038/nrmicro.2016.62](https://doi.org/10.1038/nrmicro.2016.62)
- <span id="page-5-25"></span>Henson, M. A. (2003). Dynamic modeling of microbial cell populations. Curr. Opin. Biotechnol. 14, 460–467. [doi: 10.1016/S0958-1669(03)00104-6](https://doi.org/10.1016/S0958-1669(03)00104-6)
- <span id="page-5-15"></span>Hewitt, C. J., Caron, G. N. V., Nienow, A. W., and McFarlane, C. M. (1999). The use of multi-parameter flow cytometry to compare the physiological response of Escherichia coli W3110 to glucose limitation during batch, fed-batch and continuous culture cultivations. J. Biotechnol. 75, 251–264. [doi: 10.1016/S0168-](https://doi.org/10.1016/S0168-1656(99)00168-6) [1656(99)00168-6](https://doi.org/10.1016/S0168-1656(99)00168-6)
- <span id="page-5-21"></span>Hewitt, C. J., Onyeaka, H., Lewis, G., Taylor, I. W., and Nienow, A. W. (2007). A comparison of high cell density fed-batch fermentations involving both induced and non-induced recombinant Escherichia coli under well-mixed small-scale and simulated poorly mixed large-scale conditions. Biotechnol. Bioeng. 96, 495–505. [doi: 10.1002/bit.21127](https://doi.org/10.1002/bit.21127)
- <span id="page-5-10"></span>Huang, W. E., Griffiths, R. I., Thompson, I. P., Bailey, M. J., and Whiteley, A. S. (2004). Raman microscopic analysis of single microbial cells. Anal. Chem. 76, 4452–4458. [doi: 10.1021/ac049753k](https://doi.org/10.1021/ac049753k)
- <span id="page-5-11"></span>Jahn, M., Seifert, J., von Bergen, M., Schmid, A., Buhler, B., and Muller, S. (2013). Subpopulation-proteomics in prokaryotic populations. Curr. Opin. Biotechnol. 24, 79–87. [doi: 10.1016/j.copbio.2012.10.017](https://doi.org/10.1016/j.copbio.2012.10.017)
- <span id="page-5-23"></span>Kim, M., Rai, N., Zorraquino, V., and Tagkopoulos, I. (2016). Multi-omics integration accurately predicts cellular state in unexplored conditions for Escherichia coli. Nat. Commun. 7:13090. [doi: 10.1038/ncomms](https://doi.org/10.1038/ncomms13090) [13090](https://doi.org/10.1038/ncomms13090)
- <span id="page-5-34"></span>Kuschel, M., Siebler, F., and Takors, R. (2017). Lagrangian trajectories to predict the formation of population heterogeneity in large-scale bioreactors. Bioengineering 4:27. [doi: 10.3390/bioengineering4020027](https://doi.org/10.3390/bioengineering4020027)
- <span id="page-5-20"></span>Ladner, T., Grünberger, A., Probst, C., Kohlheyer, D., Büchs, J., and Delvigne, F. (2017). "Application of mini- and micro-bioreactors for microbial bioprocesses," in Current Developments in Biotechnology and Bioengineering, eds C. Larroche, M. Á. Sanromán, G. Du and A. Pandey (Amsterdam: Elsevier), 433–461.
- <span id="page-5-7"></span>Landry, Z. C., Giovanonni, S. J., Quake, S. R., and Blainey, P. C. (2013). Optofluidic cell selection from complex microbial communities for single-genome analysis. Methods Enzymol. 531, 61–90. [doi: 10.1016/B978-0-12-407863-5.00004-6.](https://doi.org/10.1016/B978-0-12-407863-5.00004-6.)
- <span id="page-5-32"></span>Lapin, A., Muller, D., and Reuss, M. (2004). Dynamic behavior of microbial populations in stirred bioreactors simulated with Euler-Lagrange methods: traveling along the lifelines of single cells. Ind. Eng. Chem. Res. 43, 4647–4656. [doi: 10.1021/ie030786k](https://doi.org/10.1021/ie030786k)
- <span id="page-5-1"></span>Lara, A. R., Galindo, E., Ramírez, O. T., and Palomares, L. A. (2006). Living with heterogeneities in bioreactors. Mol. Biotechnol. 34, 355–381. [doi: 10.1385/MB:](https://doi.org/10.1385/MB:34:3:355) [34:3:355](https://doi.org/10.1385/MB:34:3:355)
- <span id="page-5-29"></span>Lardon, L. A., Merkey, B. V., Martins, S., Dotsch, A., Picioreanu, C., Kreft, J. U., et al. (2011). iDynoMiCS: next-generation individual-based modelling of biofilms. Environ. Microbiol. 13, 2416–2434. [doi: 10.1111/j.1462-2920.2011.](https://doi.org/10.1111/j.1462-2920.2011.02414.x) [02414.x](https://doi.org/10.1111/j.1462-2920.2011.02414.x)
- <span id="page-5-4"></span>Lavric, V., and Graham, D. W. (2010). Birth, growth and death as structuring operators in bacterial population dynamics. J. Theor. Biol. 264, 45–54. [doi: 10.1016/j.jtbi.2010.01.020](https://doi.org/10.1016/j.jtbi.2010.01.020)

- <span id="page-6-12"></span>Lecault, V., White, A. K., Singhal, A., and Hansen, C. L. (2012). Microfluidic single cell analysis: from promise to practice. Curr. Opin. Chem. Biol. 16, 381–390. [doi: 10.1016/j.cbpa.2012.03.022](https://doi.org/10.1016/j.cbpa.2012.03.022)
- <span id="page-6-2"></span>Li, B., and You, L. (2013). Predictive power of cell-to-cell variability. Quant. Biol. 1, 131–139. [doi: 10.1007/s40484-013-0013-3](https://doi.org/10.1007/s40484-013-0013-3)
- <span id="page-6-4"></span>Lidstrom, M. E., and Konopka, M. C. (2010). The role of physiological heterogeneity in microbial population behavior. Nat. Chem. Biol. 6, 705–712. [doi: 10.1038/nchembio.436](https://doi.org/10.1038/nchembio.436)
- <span id="page-6-40"></span>Liu, Y., Wang, Z.-J., Xia, J.-Y., Haringa, C., Liu, Y.-P., Chu, J et al. (2016). Application of Euler-Lagrange CFD for quantitative evaluating the effect of shear force on Carthamus tinctorius L. cell in a stirred tank bioreactor. Biochem. Eng. J. 114, 209–217. [doi: 10.1016/j.bej.2016.07.006](https://doi.org/10.1016/j.bej.2016.07.006)
- <span id="page-6-8"></span>Löffler, M., Simen, J. D., Jäger, G., Schäferhoff, K., Freund, A., and Takors, R. (2016). Engineering E. coli for large-scale production – strategies considering ATP expenses and transcriptional responses. Metab. Eng. 38, 73–85. [doi: 10.1016/j.ymben.2016.06.008](https://doi.org/10.1016/j.ymben.2016.06.008)
- <span id="page-6-11"></span>Love, K. R., Bagh, S., Choi, J., and Love, J. C. (2013). Microtools for singlecell analysis in biopharmaceutical development and manufacturing. Trends Biotechnol. 31, 280–286. [doi: 10.1016/j.tibtech.2013.03.001](https://doi.org/10.1016/j.tibtech.2013.03.001)
- <span id="page-6-39"></span>Morchain, J. (2017). "Numerical tools for scaling up bioreactors," in Current Developments in Biotechnology and Bioengineering: Bioprocesses, Bioreactors and Controls, eds M. Á. Sanromán, G. Du, and A. Pandey (Toronto, ON: Elsevier), 495–523. [doi: 10.1016/B978-0-444-63663-8.00017-3](https://doi.org/10.1016/B978-0-444-63663-8.00017-3)
- <span id="page-6-31"></span>Morchain, J., Gabelle, J. C., and Cockx, A. (2013). Coupling of biokinetic and population balance models to account for biological heterogeneity in bioreactors. Aiche J. 59, 369–379. [doi: 10.1002/aic.13820](https://doi.org/10.1002/aic.13820)
- <span id="page-6-29"></span>Müller, S., Harms, H., and Bley, T. (2010). Origin and analysis of microbial population heterogeneity in bioprocesses. Curr. Opin. Biotechnol. 21, 100–113. [doi: 10.1016/j.copbio.2010.01.002](https://doi.org/10.1016/j.copbio.2010.01.002)
- <span id="page-6-19"></span>Muller, S., and Nebe-von-Caron, G. (2010). Functional single-cell analyses: flow cytometry and cell sorting of microbial populations and communities. FEMS Microbiol. Rev. 34, 554–587. [doi: 10.1111/j.1574-6976.2010.00214.x](https://doi.org/10.1111/j.1574-6976.2010.00214.x)
- <span id="page-6-17"></span>Musat, N., Foster, R., Vagner, T., Adam, B., and Kuypers, M. M. M. (2012). Detecting metabolic activities in single cells, with emphasis on nanoSIMS. FEMS Microbiol. Rev. 36, 486–511. [doi: 10.1111/j.1574-6976.2011.00303.x](https://doi.org/10.1111/j.1574-6976.2011.00303.x)
- <span id="page-6-26"></span>Noack, S., Kloden, W., and Bley, T. (2008). Modeling synchronous growth of bacterial populations in phased cultivation. Bioprocess Biosyst. Eng. 31, 435–443. [doi: 10.1007/s00449-007-0180-6](https://doi.org/10.1007/s00449-007-0180-6)
- <span id="page-6-9"></span>Ofi¸teru, I. D., Ferde¸s, M., Knapp, C. W., Graham, D. W., and Lavric, V. (2012). Conditional confined oscillatory dynamics of Escherichia coli strain K12-MG1655 in chemostat systems. Appl. Microbiol. Biotechnol. 94, 185–192. [doi: 10.1007/s00253-011-3697-1](https://doi.org/10.1007/s00253-011-3697-1)
- <span id="page-6-24"></span>Oliveira, A. F., Pessoa, A., Bastos, R. G., and de la Torre, L. G. (2016). Microfluidic tools toward industrial biotechnology. Biotechnol. Prog. 32, 1372–1389. [doi: 10.1002/btpr.2350](https://doi.org/10.1002/btpr.2350)
- <span id="page-6-42"></span>Oyebamiji, O. K., Wilkinson, D. J., Jayathilake, P. G., Curtis, T. P., Rushton, S. P., Li, B., et al. (2017). Gaussian process emulation of an individual-based model simulation of microbial communities. J. Comput. Sci. 22, 69–84. [doi: 10.1016/j.](https://doi.org/10.1016/j.jocs.2017.08.006) [jocs.2017.08.006](https://doi.org/10.1016/j.jocs.2017.08.006)
- <span id="page-6-3"></span>Pasotti, L., and Zucca, S. (2014). Advances and computational tools towards predictable design in biological engineering. Comput. Math. Methods Med. 2014:369681. [doi: 10.1155/2014/369681](https://doi.org/10.1155/2014/369681)
- <span id="page-6-32"></span>Pigou, M., and Morchain, J. (2015). Investigating the interactions between physical and biological heterogeneities in bioreactors using compartment, population balance and metabolic models. Chem. Eng. Sci. 126, 267–282. [doi: 10.1016/j.ces.](https://doi.org/10.1016/j.ces.2014.11.035) [2014.11.035](https://doi.org/10.1016/j.ces.2014.11.035)
- <span id="page-6-33"></span>Railsback, S. F., and Grimm, V. (2012). Agent-based and Individual-based Modeling: A Practical Introduction. Princeton, NJ: Princeton University Press.
- <span id="page-6-37"></span>Ramkrishna, D., and Mahoney, A. W. (2002). Population balance modeling. Promise for the future. Chem. Eng. Sci. 57, 595–606. [doi: 10.1016/S0009-](https://doi.org/10.1016/S0009-2509(01)00386-4) [2509(01)00386-4](https://doi.org/10.1016/S0009-2509(01)00386-4)
- <span id="page-6-30"></span>Ramkrishna, D., and Singh, M. R. (2014). Population balance modeling: current status and future prospects. Annu. Rev. Chem. Biomol. Eng. 5, 123–146. [doi: 10.1146/annurev-chembioeng-060713-040241](https://doi.org/10.1146/annurev-chembioeng-060713-040241)
- <span id="page-6-1"></span>Rosano, G. L., and Ceccarelli, E. A. (2014). Recombinant protein expression in Escherichia coli: advances and challenges. Front. Microbiol. 5:172. [doi: 10.3389/](https://doi.org/10.3389/fmicb.2014.00172) [fmicb.2014.00172](https://doi.org/10.3389/fmicb.2014.00172)
- <span id="page-6-23"></span>Rosenthal, K., Falke, F., Frick, O., Dusny, C., and Schmid, A. (2015). An inert continuous microreactor for the isolation and analysis of a single microbial cell. Micromachines 6, 1836–1855. [doi: 10.3390/mi6121459](https://doi.org/10.3390/mi6121459)
- <span id="page-6-22"></span>Royle, K. E., del Val, I. J., and Kontoravdi, C. (2013). Integration of models and experimentation to optimise the production of potential biotherapeutics. Drug Discov. Today 18, 1250–1255. [doi: 10.1016/j.drudis.2013.07.002](https://doi.org/10.1016/j.drudis.2013.07.002)
- <span id="page-6-16"></span>Rubakhin, S. S., Lanni, E. J., and Sweedler, J. V. (2013). Progress toward single cell metabolomics. Curr. Opin. Biotechnol. 24, 95–104. [doi: 10.1016/j.copbio.2012.](https://doi.org/10.1016/j.copbio.2012.10.021) [10.021](https://doi.org/10.1016/j.copbio.2012.10.021)
- <span id="page-6-21"></span>Rubbens, P., Props, R., Boon, N., and Waegeman, W. (2017). Flow cytometric single-cell identification of populations in synthetic bacterial communities. PLOS ONE 12:e0169754. [doi: 10.1371/journal.pone.0169754](https://doi.org/10.1371/journal.pone.0169754)
- <span id="page-6-36"></span>Rudge, T. J., Steiner, P. J., Phillips, A., and Haseloff, J. (2012). Computational modeling of synthetic microbial biofilms. ACS Synth. Biol. 1, 345–352. [doi: 10.1021/sb300031n](https://doi.org/10.1021/sb300031n)
- <span id="page-6-6"></span>Ryall, B., Eydallin, G., and Ferenci, T. (2012). Culture history and population heterogeneity as determinants of bacterial adaptation: the adaptomics of a single environmental transition. Microbiol. Mol. Biol. Rev. 76, 597–625. [doi: 10.1128/MMBR.05028-11](https://doi.org/10.1128/MMBR.05028-11)
- <span id="page-6-5"></span>Sauer, M., and Mattanovich, D. (2012). Construction of microbial cell factories for industrial bioprocesses. J. Chem. Technol. Biotechnol. 87, 445–450. [doi: 10.1002/jctb.3711](https://doi.org/10.1002/jctb.3711)
- <span id="page-6-34"></span>Schuler, A. J., Majed, N., Bucci, V., Hellweger, F. L., Tu, Y., and Gu, A. Z. (2011). Is the whole the sumof its parts? Agent-based modelling of wastewater treatment systems. Water Sci. Technol. 63, 1590–1598. [doi: 10.2166/wst.2011.218](https://doi.org/10.2166/wst.2011.218)
- <span id="page-6-20"></span>Shapiro, H. H. (2000). Microbial analysis at the single-cell level: tasks and techniques. J. Microbiol. Methods 42, 3–16. [doi: 10.1016/S0167-7012(00)](https://doi.org/10.1016/S0167-7012(00)00167-6) [00167-6](https://doi.org/10.1016/S0167-7012(00)00167-6)
- <span id="page-6-13"></span>Shi, X., Gao, W. M., Wang, J. X., Chao, S. H., Zhang, W. W., and Meldrum, D. R. (2015). Measuring gene expression in single bacterial cells: recent advances in methods and micro-devices. Crit. Rev. Biotechnol. 35, 448–460. [doi: 10.3109/](https://doi.org/10.3109/07388551.2014.899556) [07388551.2014.899556](https://doi.org/10.3109/07388551.2014.899556)
- <span id="page-6-27"></span>Shuler, M. L. (1999). Single-cell models: promise and limitations. J. Biotechnol. 71, 225–228. [doi: 10.1016/S0168-1656(99)00024-3](https://doi.org/10.1016/S0168-1656(99)00024-3)
- <span id="page-6-35"></span>Sklar, E. (2007). Software review: NetLogo, a multi-agent simulation environment. Artif. Life 13, 303–311. [doi: 10.1162/artl.2007.13.3.303](https://doi.org/10.1162/artl.2007.13.3.303)
- <span id="page-6-25"></span>Song, H.-S., Cannon, W., Beliaev, A., and Konopka, A. (2014). Mathematical modeling of microbial community dynamics: a methodological review. Processes 2, 711–752. [doi: 10.3390/pr2040711](https://doi.org/10.3390/pr2040711)
- <span id="page-6-28"></span>Stamatakis, M., and Zygourakis, K. (2010). A mathematical and computational approach for integrating the major sources of cell population heterogeneity. J. Theor. Biol. 266, 41–61. [doi: 10.1016/j.jtbi.2010.06.002](https://doi.org/10.1016/j.jtbi.2010.06.002)
- <span id="page-6-7"></span>Sumner, E. R., and Avery, S. V. (2002). Phenotypic heterogeneity: differential stress resistance among individual cells of the yeast Saccharomyces cerevisiae. Microbiology 148, 345–351. [doi: 10.1099/00221287-148-2-345](https://doi.org/10.1099/00221287-148-2-345)
- <span id="page-6-0"></span>Takors, R. (2012). Scale-up of microbial processes: impacts, tools and open questions. J. Biotechnol. 160, 3–9. [doi: 10.1016/j.jbiotec.2011.](https://doi.org/10.1016/j.jbiotec.2011.12.010) [12.010](https://doi.org/10.1016/j.jbiotec.2011.12.010)
- <span id="page-6-14"></span>Vasdekis, A. E., Silverman, A. M., and Stephanopoulos, G. (2015). Origins of cell-to-cell bioprocessing diversity and implications of the extracellular environment revealed at the single-cell level. Sci. Rep. 5:17689. [doi: 10.1038/](https://doi.org/10.1038/srep17689) [srep17689](https://doi.org/10.1038/srep17689)
- <span id="page-6-38"></span>Wang, G., Tang, W. J., Xia, J. Y., Chu, J., Noorman, H., and van Gulik, W. M. (2015). Integration of microbial kinetics and fluid dynamics toward modeldriven scale-up of industrial bioprocesses. Eng. Life Sci. 15, 20–29. [doi: 10.1002/](https://doi.org/10.1002/elsc.201400172) [elsc.201400172](https://doi.org/10.1002/elsc.201400172)
- <span id="page-6-18"></span>Want, A., Thomas, O. R. T., Kara, B., Liddell, J., and Hewitt, C. J. (2009). Studies related to antibody fragment (fab) production in Escherichia coli W3110 fedbatch fermentation processes using multiparameter flow cytometry. Cytometry A 75, 148–154. [doi: 10.1002/cyto.a.20683](https://doi.org/10.1002/cyto.a.20683)
- <span id="page-6-41"></span>Wilkinson, D. J. (2009). Stochastic modelling for quantitative description of heterogeneous biological systems. Nat. Rev. Genet. 10, 122–133. [doi: 10.1038/](https://doi.org/10.1038/nrg2509) [nrg2509](https://doi.org/10.1038/nrg2509)
- <span id="page-6-10"></span>Winson, M. K., and Davey, H. M. (2000). Flow cytometric analysis of microorganisms. Methods 21, 231–240. [doi: 10.1006/meth.2000.1003](https://doi.org/10.1006/meth.2000.1003)
- <span id="page-6-15"></span>Wu, M. Y., and Singh, A. K. (2012). Single-cell protein analysis. Curr. Opin. Biotechnol. 23, 83–88. [doi: 10.1016/j.copbio.2011.11.023](https://doi.org/10.1016/j.copbio.2011.11.023)

- <span id="page-7-0"></span>Wyre, C., and Overton, T. (2014a). Flow cytometric analysis of E. coli on agar plates: implications for recombinant protein production. Biotechnol. Lett. 36, 1485–1494. [doi: 10.1007/s10529-014-1511-8](https://doi.org/10.1007/s10529-014-1511-8)
- <span id="page-7-1"></span>Wyre, C., and Overton, T. (2014b). Use of a stress-minimisation paradigm in high cell density fed-batch Escherichia coli fermentations to optimise recombinant protein production. J. Ind. Microbiol. Biotechnol. 41, 1391–1404. [doi: 10.1007/](https://doi.org/10.1007/s10295-014-1489-1) [s10295-014-1489-1](https://doi.org/10.1007/s10295-014-1489-1)
- <span id="page-7-2"></span>Xiao, Y., Bowen, C. H., Liu, D., and Zhang, F. (2016). Exploiting nongenetic cell-to-cell variation for enhanced biosynthesis. Nat. Chem. Biol. 12, 339–344. [doi: 10.1038/nchembio.2046](https://doi.org/10.1038/nchembio.2046)
- <span id="page-7-3"></span>Young, J. W., Locke, J. C. W., Altinok, A., Rosenfeld, N., Bacarian, T., Swain, P. S., et al. (2012). Measuring single-cell gene expression dynamics in bacteria using fluorescence time-lapse microscopy. Nat. Protoc. 7, 80–88. [doi: 10.1038/nprot.](https://doi.org/10.1038/nprot.2011.432) [2011.432](https://doi.org/10.1038/nprot.2011.432)
- <span id="page-7-5"></span>Zhang, H., Cui, X. L., Bi, J. X., Dai, S., and Ye, H. T. (2015). Single-cell analysis for bioprocessing. Eng. Life Sci. 15, 582–592. [doi: 10.1002/elsc.201400155](https://doi.org/10.1002/elsc.201400155)

- <span id="page-7-4"></span>Zhang, W. W., Li, F., and Nie, L. (2010). Integrating multiple 'omics' analysis for microbial biology: application and methodologies. Microbiology 156, 287–301. [doi: 10.1099/mic.0.034793-0](https://doi.org/10.1099/mic.0.034793-0)
**Conflict of Interest Statement:** The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

Copyright © 2017 González-Cabaleiro, Mitchell, Smith, Wipat and Ofi¸teru. This is an open-access article distributed under the terms of the [Creative](http://creativecommons.org/licenses/by/4.0/) [Commons](http://creativecommons.org/licenses/by/4.0/) [Attribution](http://creativecommons.org/licenses/by/4.0/) [License](http://creativecommons.org/licenses/by/4.0/) [(CC BY).](http://creativecommons.org/licenses/by/4.0/) The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.