# R E S EAR CH Open Access

![](_page_0_Picture_2.jpeg)

# The metabolic potential of Escherichia coli BL21 in defined and rich medium

Zhaopeng Li1,2, Manfred Nimtz1 and Ursula Rinas1,2*

## Abstract

Background: The proteome reflects the available cellular machinery to deal with nutrients and environmental challenges. The most common E. coli strain BL21 growing in different, commonly employed media was evaluated using a detailed quantitative proteome analysis.

Results: The presence of preformed biomass precursor molecules in rich media such as Luria Bertani supported rapid growth concomitant to acetate formation and apparently unbalanced abundances of central metabolic pathway enzymes, e.g. high levels of lower glycolytic pathway enzymes as well as pyruvate dehydrogenase, and low levels of TCA cycle and high levels of the acetate forming enzymes Pta and AckA. The proteome of cells growing exponentially in glucose-supplemented mineral salt medium was dominated by enzymes of amino acid synthesis pathways, contained more balanced abundances of central metabolic pathway enzymes, and a lower portion of ribosomal and other translational proteins. Entry into stationary phase led to a reconstruction of the bacterial proteome by increasing e.g. the portion of proteins required for scavenging rare nutrients and general cell protection. This proteomic reconstruction during entry into stationary phase was more noticeable in cells growing in rich medium as they have a greater reservoir of recyclable proteins from the translational machinery.

Conclusions: The proteomic comparison of cells growing exponentially in different media reflected the antagonistic and competitive regulation of central metabolic pathways through the global transcriptional regulators Cra, Crp, and ArcA. For example, the proteome of cells growing exponentially in rich medium was consistent with a dominating role of phosphorylated ArcA most likely a result from limitations in reoxidizing reduced quinones in the respiratory chain under these growth conditions. The proteomic alterations of exponentially growing cells into stationary phase cells were consistent with stringent-like and stationary phase responses and a dominating control through DksA-ppGpp and RpoS.

Keywords: Escherichia coli, Growth rate control, Metabolic balance, Overflow metabolism, Proteome, Stationary phase response, Transcriptional control, Two-dimensional gel electrophoresis

## Background

Escherichia coli is still the most utilized bacterial workhorse for the production of biomolecules, in particular the strain BL21 is the most employed host for the production of recombinant proteins. E. coli BL21 is considered a very robust strain compared to E. coli K12 strains as it produces less acetate [[1,2\]](#page-14-0). Acetate, produced as major by-product during fast growth in carbon excess conditions, negatively effects the production of proteins and other biomolecules.

For rapid and convenient lab-scale production of recombinant proteins, cells are usually grown in rich medium such as Luria Bertani or Terrific broth. However, for largescale recombinant protein production, defined mineral salt media are generally preferred as they allow the implementation of fed-batch based high-cell density cultivations [[3](#page-14-0)]. E. coli is not only used as host for recombinant protein production but is also gaining increasing importance in synthetic biology for the production of heterologous low molecular weight compounds [\[4](#page-14-0)-[6](#page-14-0)] and also for overproduction of homologous metabolites such as amino acids [[7](#page-14-0)] or for the utilization as a whole cell biocatalyst

![](_page_0_Picture_18.jpeg)

© 2014 Li et al.; licensee BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License [(http://creativecommons.org/licenses/by/2.0)](http://creativecommons.org/licenses/by/2.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver [(http://creativecommons.org/publicdomain/zero/1.0/](http://creativecommons.org/publicdomain/zero/1.0/)) applies to the data made available in this article, unless otherwise stated.

<sup>*</sup> Correspondence: [Ursula.Rinas@helmholtz-hzi.de](mailto:Ursula.Rinas@helmholtz-hzi.de) 1

Helmholtz Centre for Infection Research, Inhoffenstraße 7, D-38124 Braunschweig, Germany

<sup>2</sup> Leibniz University of Hannover, Technical Chemistry–Life Science, Callinstraße 5, D-30167 Hannover, Germany

<span id="page-1-0"></span>employed in biotransformation processes [[8\]](#page-14-0). Thus, a more profound comprehension of the metabolic potential of exponentially growing or stationary phase and resting cells is of considerable importance to understand and improve production processes but also for the pure knowledge gain to better understand bacterial physiology.

Based on a proteomic study we present here for the first time a detailed description of the metabolic potential of E. coli BL21 during growth in defined and rich medium. The quantitative proteome analysis was not only done for exponentially growing cells but also for stationary phase cells to analyze the metabolic capabilities as well as the adaptation potential to changing environmental conditions.

### Results

During growth in rich medium such as the commonly employed media Luria Bertani or Terrific Broth cells do not need to synthesize most of the precursor molecules (e.g. amino acids) as they are already present in the medium. Thus, cells can spend more resources to produce the macromolecules required for their own proliferation and can grow more rapidly compared to cells growing in defined mineral salt medium (Figure 1). Remarkably, E. coli BL21 also secretes low but detectable levels of acetate during rapid growth in rich medium even in the absence of oxygen limiting conditions (Figure 1). For a better understanding of the bacterial physiology of E. coli BL21 under these different conditions of nutrient availability a detailed comparative analysis of the bacterial proteome was carried out. A first brief visual examination revealed a less complex nature of the proteome during rapid growth in rich medium compared to the slower growth in the glucose supplemented mineral salt medium (Figure [2)](#page-2-0). Most notably, enzymes required for amino acid biosynthesis are virtually absent during rapid growth in rich medium (Figure [2](#page-2-0)). A lumped quantitative analysis of the abundance levels of all identified proteins in cells from exponential and stationary phase in defined and rich media is given in Figure [3](#page-3-0) and in more detail visualized in Additional file [1](#page-14-0): Figures S1-S9 with corresponding values shown in the Additional file [1](#page-14-0): Table S1 (values with standard deviation also in Additional file [2](#page-14-0): Table S5).

#### Central (carbon) metabolism

The comparative proteome analysis of cells growing exponentially in defined and rich medium with special attention to enzymes of the central metabolic pathways revealed a complex picture corroborating a "suboptimal" coupling of catabolic and anabolic pathways especially during rapid growth in rich medium also for E. coli BL21 (Figure [4)](#page-5-0).

![](_page_1_Figure_7.jpeg)

## Glycolysis, pyruvate dehydrogenase, and pentose phosphate pathway

Enzymes of the upper glycolytic pathway are present in slightly higher amounts in cells growing in defined medium with glucose as carbon substrate compared to the cells growing in rich medium (Figures [3](#page-3-0) and [4)](#page-5-0). However, the opposite is observed for the lower glycolytic pathway; enzymes belonging to this part of the glycolysis and also the subunits of the pyruvate dehydrogenase complex, the enzyme connecting the glycolytic pathway and the tricarbocyclic acid cycle, are present in significantly lower amounts in cells growing in defined medium (Figures [3](#page-3-0) and [4](#page-5-0)). Also, most identified enzymes belonging to the pentose phosphate pathway (e.g. Gnd, TktA, TalB, and Eda) are found in lower amounts in cells growing in defined medium (Figures [3](#page-3-0) and [4](#page-5-0)). Thus, the higher amounts of enzymes of the lower glycolytic and pentose phosphate pathways as well as of pyruvate dehydrogenase during rapid growth in rich medium point to an increased capacity for carbon flux

<span id="page-2-0"></span>![](_page_2_Figure_1.jpeg)

leading towards connecting pathways (e.g. tricarbocyclic acid cycle) under these growth conditions.

## Tricarbocyclic acid (TCA) cycle and metabolism of acetate and fermentative by-products

In contrast to the enzymes of the lower glycolytic and the pentose phosphate pathways, the enzymes of the TCA cycle accumulate to higher levels in cells growing in defined medium, respectively, to lower levels in cells growing in rich medium (Figures [3](#page-3-0) and [4](#page-5-0)). Thus, larger amounts of enzymes of the lower glycolytic and pentose phosphate pathways as well as of pyruvate dehydrogenase and lower amounts of TCA cycle enzymes during rapid growth in rich medium point here to a potential bottleneck leading to off-pathway carbon flux at this junction and to acetate formation. In this line, enzymes catalyzing the formation of acetate from acetyl-CoA (Pta and AckA) are present at elevated level in cells growing exponentially in rich medium (Figure [4](#page-5-0)). The enzymes responsible for acetate formation directly from pyruvate (PoxB) and acetate uptake through acetyl-CoA (Acs) are neither in defined nor in rich medium present in large amounts (Figure [4,](#page-5-0) Additional file [1](#page-14-0): Table S1). Thus, acetate uptake through Acs but also acetate formation through PoxB does not seem to play a significant role during exponential growth in both media. Interestingly, cells growing exponentially in rich medium also exhibit higher levels of AdhE (Figure [4,](#page-5-0) Additional file [1](#page-14-0): Table S1), the enzyme catalyzing the anaerobic formation of ethanol from acetyl-CoA [\[9](#page-14-0)[-11\]](#page-15-0), despite the absence of cell external anaerobic or even oxygen limiting conditions (Figure [1](#page-1-0)) confirming the bottleneck at the junction of the pyruvate dehydrogenase and the TCA cycle, or more general, at the pyruvate node.

#### Anaplerotic reactions

Enzymes catalyzing anaplerotic reactions between glycolysis and TCA cycle intermediates in both directions (e.g. Ppc, PckA, MaeB) accumulate to slightly higher amounts in cells growing exponentially in defined medium (Figures [3](#page-3-0) and [4](#page-5-0)) also pointing to a more balanced (flux) connection between both pathways under these growth conditions. On the contrary, lower amounts of anaplerotic and TCA cycle enzymes and higher amounts of lower glycolytic and pentose phosphate pathway enzymes as well as higher amounts of the pyruvate dehydrogenase are in line with the elevated flux towards acetate observed during rapid growth in rich medium.

#### Respiratory energy generation

The majority of NADH is generated in the TCA cycle and reoxidized through electron transfer to oxidized quinones catalyzed by NADH dehydrogenases. Concurrently exported protons are reimported driving energy generation through ATPases. Identified proteins belonging to both types of enzymes (e.g. NuoB/C/G/F and AtpA/D/H) do not constitute a large part of the bacterial proteome and do not show significant changes in response to nutrient availability (Figure [3,](#page-3-0) Additional file [1](#page-14-0): Table S1) supporting previous assumptions that growth rates of E. coli under aerobic conditions are limited by respiration and the concomitant rate of ATP generation through oxidative phosphorylation [[12\]](#page-15-0).

#### Amino acid biosynthesis and degradation

The most prominent differences in the proteome of cells growing exponentially in rich and glucose supplemented mineral salt medium are related to enzymes catalyzing the formation of amino acids from glucose and ammonia

<span id="page-3-0"></span>![](_page_3_Figure_2.jpeg)

(and sulfate). Most striking is the exceptional strong abundance of MetE and IlvC, both proteins accumulating to 3.6 and 2.8%, respectively, of the relative protein mass (RPM) of E. coli BL21 growing in defined medium (Figure [2,](#page-2-0) Additional file [1](#page-14-0): Table S1). Altogether, identified enzymes involved in amino acid synthesis (57 proteins) constitute almost 20% of the relative protein mass of cells growing in defined medium but accumulate only to 6% in the proteome of cells growing exponentially in rich medium (Figure 3). Thus, a large fraction of resources is directed to synthesize the enzymes required for amino acid synthesis during growth in glucose

supplemented mineral salt medium while these resources can be redirected to other tasks in cells growing rapidly in rich medium.

In contrast, some enzymes required for amino acid salvage, e.g. enzymes required for the degradation of arginine, threonine, and proline (AstD, Tdh, Kbl, and PutA, respectively) are present in higher amounts during rapid growth in rich medium (Additional file [1:](#page-14-0) Table S1) suggesting that some amino acids are not only used for protein synthesis but also serve as precursors for other biomass building blocks or are even fed into central catabolic pathways.

![](_page_4_Figure_2.jpeg)

#### <span id="page-5-0"></span>(See figure on previous page.)

Figure 4 Central metabolic pathway proteome at different growth conditions. Comparative abundances of (lumped) central metabolic pathway enzymes of E. coli BL21 grown in defined medium at exponential phase (A1 and B1) and in rich medium at exponential (A2 and B2) and stationary phases (A3 and B3). The thickness of the arrows and dotted borders represents the relative (lumped) abundances of enzymes belonging to the pathways at these growth conditions. A detailed direct comparison of pathway regulation at the proteomic level at different growth conditions (C1, defined versus rich media at exponential phase and C2, stationary versus exponential phase in rich medium) is given in a pathway map using different color coded arrows for the representation of the comparative Log2 ratios of the single enzyme abundances [code given in C1, C1: Log2(defined/rich), C2: Log2(stationary/exponential)]. An average of single enzyme abundances from cells grown in LB and TB media was calculated to represent the single enzyme abundance in cells grown in rich medium. The entire comparative proteomic (pathway) data are visualized in Additional file [1](#page-14-0): Figures S7-S9 and the corresponding values are given in the Additional file [1:](#page-14-0) Table S1. The list of abbreviations is found at the end of the Additional file [1](#page-14-0): Figure S9.

For the detailed analysis of the abundance levels of the identified enzymes of the central carbon metabolism please refer to Additional file [1:](#page-14-0) Table S1.

#### Transport systems

Substrates need to be transported into cells prior to their catabolic breakdown or employment for anabolic purposes. The most important transport system for carbohydrates, in particular glucose, is the phosphotransferase system (PTS). All identified enzymes of the PTS (e.g. PtsIH, Crr, and ManX) are present in larger amounts in cells growing in defined medium with glucose as carbon substrate compared to the cells growing in rich medium (Figure 4). In contrast, the proteins for maltose transport (MalEK and LamB) accumulate to higher levels in cells growing rapidly in rich medium (Figure 4). Interestingly, proteins involved in amino acid and peptide transport are present in larger amounts in cells growing in defined medium which does not contain amino acid and peptide supplements (Figure 4). The unspecific transport channels of the outer membrane (OmpA/F, OmpC absent in BL21 [[13](#page-15-0)-[15\]](#page-15-0)) are present in similar amounts during exponential growth in all media (Additional file [1](#page-14-0): Table S1).

For the detailed analysis of the abundance levels of the identified proteins involved in transport please refer to Additional file [1:](#page-14-0) Table S1.

#### Protein synthesis and folding

Proteins and enzymes involved in transcription, translation, and folding form a substantial part of the bacterial proteome under both growth conditions, however, they accumulate to larger quantities in cells growing exponentially in rich medium compared to the cells growing in defined medium (Figures [3](#page-3-0) and [5,](#page-7-0) Additional file [1](#page-14-0): Table S1). In particular, the fraction of proteins and enzymes directly related to translation, e.g. ribosomal proteins, aminoacyl-tRNA synthetases and elongation factors, is considerably higher in cells growing rapidly in rich medium (~20% versus ~12% of the relative protein mass, Figure [3](#page-3-0) and Additional file [1](#page-14-0): Table S1). Enzymes involved in transcription and most heat shock proteins chaperoning protein folding also accumulate to higher levels during exponential growth in rich medium but not as pronounced as those proteins directly involved in translation (Figure [3](#page-3-0)). However, chaperones of the heat shock protein family involved in aggregate dissolution (e.g. IbpA [[16](#page-15-0),[17](#page-15-0)] and ClpB [[18,19](#page-15-0)]) are present in lower amounts during growth in rich medium (Figure [5,](#page-7-0) Additional file [1](#page-14-0): Table S1) reflecting a different level of regulation as for the majority of heat shock proteins. Other enzymes catalyzing protein folding reactions (e.g. peptidyl prolyl cistrans isomerases) are present in relatively low amounts during exponential growth in both media (Additional file [1](#page-14-0): Table S1) suggesting that these folding reactions could represent potential bottlenecks on the path to correctly folded protein chains.

For the detailed analysis of the abundance levels of the identified enzymes/proteins involved in protein synthesis and folding please refer to Additional file [1](#page-14-0): Table S1.

#### Stationary phase response

When entering into stationary phase, cells growing in rich and glucose supplemented mineral salt medium exhibit a different behavior, apparent through a gradual or a sudden decline of the growth rate (Figure [1)](#page-1-0), respectively, which is also reflected in different alterations of the cellular proteome (Figures [2](#page-2-0) and [3,](#page-3-0) Additional file [1](#page-14-0): Figure S10, Table S1). The proteome of cells growing in defined medium does not reveal big changes during entry into stationary phase (Figure [3,](#page-3-0) Additional file [1](#page-14-0): Figure S10, Table S1), however, the proteome of cells growing in rich medium exhibits a more drastic change in composition (Figures [2](#page-2-0) and [3,](#page-3-0) Additional file [1:](#page-14-0) Figure S10, Table S1).

#### Central (carbon) metabolism

The enzymes of the upper and lower glycolytic pathway, the pentose phosphate pathway and the amino acid synthesis pathways reveal a slight to moderate increase and the enzymes of the TCA cycle increase strongly in quantity during entry into stationary phase in rich medium (Figures [3](#page-3-0) and 4). The pyruvate dehydrogenase, the multimeric enzyme connecting the glycolytic pathway and the TCA cycle, however, reveals a contrary behavior. AceE and AceF, subunits of pyruvate dehydrogenase

![](_page_6_Figure_1.jpeg)

#### <span id="page-7-0"></span>(See figure on previous page.)

Figure 5 The transcription, translation, and protein folding machinery: proteomic comparison at different growth conditions. A detailed direct comparison of the transcription, translation, and protein folding machinery at the proteomic level at different growth conditions (A, defined versus rich media at exponential phase; B, stationary versus exponential phase in rich medium; and C, stationary versus exponential phase in defined medium) is given using different color coded arrows for the representation of the comparative Log2 ratios of single protein abundances [code on top, A: Log2(defined/rich), B: Log2(stationary/exponential)rich, C: Log2(stationary/exponential)defined]. An average of single protein abundances from cells grown in LB and TB media was calculated to represent the single protein abundance in cells grown in rich medium. The entire comparative proteomic (pathway) data are visualized in Additional file [1](#page-14-0): Figures S7-S9 and the corresponding values are given in the Additional file [1](#page-14-0): Table S1. The list of abbreviations is found at the end of the Additional file [1](#page-14-0): Figure S9.

complex, decrease considerably in quantity (Figure [4](#page-5-0)). Simultaneously, enzymes catalyzing other enzymatic conversions of pyruvate, e.g. from pyruvate to PEP (PpsA) or even to acetate (PoxB) and lactate (Dld and LdhA), increase in quantity (Figure [4)](#page-5-0). In addition, elevated amounts of Acs, the enzyme responsible for acetate utilization, and anaplerotic enzymes such as the glyoxylate shunt enzyme GlcB, and the enzymes catalyzing anaplerotic reactions between glycolysis and TCA cycle such as PckA and MaeB are observed (Figure [4)](#page-5-0). These findings show that during entry into stationary phase cells are getting attuned to metabolize acetate (via Acs) in agreement with the observed reutilization of acetate (Figure [1](#page-1-0)). A lower abundance of Pta and AckA in stationary phase cells (Figure [4](#page-5-0)) corroborates the previous finding that the Pta-AckA pathway is not utilized for acetate uptake but serves catabolic purposes leading to acetate secretion under carbon excess conditions [[20,21\]](#page-15-0). Increased levels of PoxB and Acs and reduced levels of the pyruvate dehydrogenase in stationary phase cells (Figure [4](#page-5-0)) also suggest that a substantial fraction of a potential carbon flux from the glycolytic pathway to the TCA cycle may not pass through pyruvate dehydrogenase but through the PoxB and Acs dependent route.

### Amino acid biosynthesis, transport systems, and salvage pathways

Enzymes of the amino acid biosynthesis pathways increase slightly on average during entry into stationary phase in rich medium but clearly do not reach the quantities found during growth in glucose-supplemented defined medium (Figures [2,](#page-2-0) [3](#page-3-0) and [4,](#page-5-0) Additional file [1:](#page-14-0) Table S1). Most proteins of the PTS system, but also those involved in maltose, amino acid, and peptide transport, increase in quantity during entry into stationary phase in cells grown in rich medium (Figure [4](#page-5-0), Additional file [1](#page-14-0): Table S1). Also, the unspecifc transport channels of the outer membrane (OmpA/F) slightly increase in quantity (Additional file [1:](#page-14-0) Table S1) facilitating the influx of remaining substrates and preparing cells for potential future feast conditions. Moreover, enzymes utilized for amino acid degradation (e.g. for the degradation of tryptophan, TnaA; arginine, AstA/B/D; and threonine, Tdh and Kbl) increase in abundance during entry into stationary phase (Additional file [1](#page-14-0): Table S1). In general, enzymes required for the uptake and degradation of less favorable substrates increase to a higher level during stationary phase.

Protein synthesis, degradation, folding, and stress responses Proteins involved in transcription and translation show a strong decrease during entry into stationary phase in cells growing in rich medium (Figures [3](#page-3-0) and 5, Additional file [1](#page-14-0): Table S1). Most prominent is the severe decline of ribosomal proteins but also aminoacyl-tRNA-synthetases and enzymes involved in peptide chain elongation and RNA degradation decrease strongly in abundance. In contrast, heat shock proteins involved in chaperoning protein folding increase modestly and the disaggregation chaperones, ClpB and IbpA, as well as proteolytic enzymes increase strongly during entry into stationary phase in rich medium (Figure 5, Additional file [1](#page-14-0): Table S1). These findings clearly emphasize that during entry into stationary phase cellular resources are redirected from proliferation to preservation by increasing the cellular potential to remove aggregated or otherwise damaged proteins by disaggregation and degradation and to reutilize released amino acids for the generation of protective enzymes. In this line, enzymes involved in the elimination of reactive oxygen species (e.g. SodB, KatG), DNA protection (e.g. Ssb, Dps, CbpA), and general stress responses (UspA) also show a significant increase during entry into stationary phase (Additional file [1:](#page-14-0) Table S1). Altogether, the changes observed during entry into stationary phase in cells growing in rich medium in the abundance of enzymes related to central carbon metabolism, transport systems and salvage pathways, protein synthesis and folding as well as stress responses are also observed for cells entering stationary phase during growth in glucose-supplemented mineral salt medium but not as pronounced (Figure [3](#page-3-0) and 5, Additional file [1](#page-14-0): Table S1). During the slower exponential growth in the defined medium cells already exhibit a more "stationary-phase-like" phenotype compared to the rapidly growing cells in rich medium and thus, the reconstruction of the proteome during entry into stationary phase during growth in glucose-supplemented mineral salt medium is not that prominent (Additional file [1](#page-14-0): Figure S10).

For the detailed analysis of the abundance levels of all identified proteins in cells from exponential and stationary phase in defined and rich media please refer to Additional file [1:](#page-14-0) Table S1.

## Discussion

Our studies illustrate that the most significant alterations in the proteome of E. coli BL21 at the different growth conditions studied are related to the enzymes of the central metabolic pathways, amino acid biosynthesis, the translational machinery, and general cell protection (Table [1)](#page-9-0).

## Antagonistic regulation of central metabolic pathways and acetate formation

Although studied for decades, acetate formation is still a puzzling peculiarity of E. coli carbon metabolism. E. coli is a "Crabtree-positive" bacterium secreting acetate not only under conditions of oxygen limitation but also under "carbon excess" conditions presumably as a result of an unbalanced coupling of catabolic and anabolic pathways. Many transcriptional factors are involved in the regulation of central metabolic pathways (Figure [6](#page-11-0), Additional file [1](#page-14-0): Table S2), but the predominant ones are the catabolite repressor activator (Cra or FruR) [\[22](#page-15-0)], the cAMP receptor protein (CRP) [\[23](#page-15-0)], and the aerobic respiration control protein (ArcA) [[24,25\]](#page-15-0). These global transcription factors are known to have opposing effects on the expression of metabolic pathway genes. For example, Cra causes repression of the majority of glycolytic pathway genes [[26\]](#page-15-0) as well as the genes of the pyruvate dehydrogenase complex [[22\]](#page-15-0) (Figure [6,](#page-11-0) Additional file [1](#page-14-0): Table S2). Inactivation of Cra is caused by high levels of fructose-1-phosphate (F1P) or fructose-1,6-bisphosphate (FBP) [\[27,28\]](#page-15-0) leading, most likely, to elevated levels of the respective enzymes. On the other hand, high levels of glucose are known to cause low CRP-cAMP levels [[29\]](#page-15-0) and, thus, expression of genes dependent on CRPcAMP, e.g. the first gene of the lower glycolytic pathway, namely gapA [\[30](#page-15-0)], and the genes of the pyruvate dehydrogenase complex [\[31,32\]](#page-15-0) will be low. Thus, the corresponding enzymes may exhibit opposing changes in their abundance than the majority of glycolytic pathway enzymes when cells are subjected to a change in growth conditions. CRP-cAMP does not only activate gapA and the genes of the pyruvate dehydrogenase complex but also the genes of the TCA cycle [[33-36\]](#page-15-0) which are additionally controlled by ArcA [\[25](#page-15-0)] (Figure [6,](#page-11-0) Additional file [1:](#page-14-0) Table S2). ArcA control on TCA cycle genes is mediated by the pool of oxidized quinones; a high level of oxidized quinones prevents phosphorylation of ArcA which in turn relieves repression of TCA cycle genes [[24\]](#page-15-0). Opposite, a high level of reduced quinones leads to phosphorylation of ArcA and resultant repression of TCA cycle genes. Thus, limitations in the respiratory pathway, in particular in the reoxidation of quinones by terminal oxidases, and high rates of NAD(P)H generation may shift the equilibrium to reduced quinones and may cause a downregulation of the TCA cycle genes and a resulting decrease in the abundance of TCA cycle enzymes.

According to our study, the formation of acetate, or more general, the observed regulation of central metabolic pathways, appears to result mainly from the opposing effects of these three different transcription factors. During exponential growth in glucose-supplemented mineral salt medium high levels of upper glycolytic and low levels of lower glycolytic pathway enzymes as well as low levels of the enzymes of the pyruvate dehydrogenase complex are observed. The opposite is true for exponential growth in rich medium, under these conditions upper glycolytic pathway enzymes show a slight decrease while lower glycolytic pathway enzymes and the enzymes of the pyruvate dehydrogenase complex strongly increase in abundance. Thus, during exponential growth in rich medium a repressing influence of Cra on the lower glycolytic pathway as well as on the genes of the pyruvate dehydrogenase complex might be outcompeted by an activating influence of CRP-cAMP leading to increased levels of the corresponding enzymes, e.g. to an increased level of the pyruvate dehydrogenase complex (Figure [4,](#page-5-0) Table [1](#page-9-0)). On the other hand decreased levels of TCA cycle enzymes during rapid growth in rich medium indicate that the activating influence of CRPcAMP on the TCA cycle genes is outcompeted by a repressing influence of high levels of phosphorylated ArcA (ArcA-P) (Figure [4](#page-5-0), Table [1](#page-9-0), Additional file [1](#page-14-0): Table S2). Thus, the dominating role of ArcA-P over CRP-cAMP most likely leads to the observed unbalance at the junction of the glycolytic pathway and the TCA cycle in cells growing rapidly in rich medium (Figure [4](#page-5-0), Table [1](#page-9-0)). In summary, the observed differences in the abundance of enzymes of the central metabolic pathways can be explained by the antagonistic and competitive pathway regulation through these three global transcriptional regulators, Cra, CRP-cAMP, and ArcA-P: Cra inhibits the expression of genes of the upper and lower glycolytic pathway and the genes of the pyruvate dehydrogenase complex, CRP-cAMP activates the expression of genes of the lower glycolytic pathway (e.g. gapA), the pyruvate dehydrogenase complex, and the TCA cycle, and ArcA-P inhibits the expression of TCA cycle genes (Figure [6,](#page-11-0) Additional file [1](#page-14-0): Table S2). In nutrient excess conditions, ArcA-P appears to be the dominating transcription factor leading to the well-known bottleneck at the pyruvate node, the junction between the glycolytic pathway and the TCA cycle, and the observed off-pathway carbon flux towards acetate (Figures [1](#page-1-0) and [4](#page-5-0)).

## <span id="page-9-0"></span>Table 1 Compendium of quantitative proteomic changes at different growth conditions

|                    |                                            | Hunger<br>Feast<br>Famine                         |                                        |                                       |                             |             |              |
|--------------------|--------------------------------------------|---------------------------------------------------|----------------------------------------|---------------------------------------|-----------------------------|-------------|--------------|
| Group1             |                                            | Mineral salt<br>medium at<br>exponential<br>phase | Rich medium<br>at exponential<br>phase | Rich medium<br>at stationary<br>phase | Transcriptional regulation2 |             |              |
| Protein1           | Function1                                  |                                                   | Relative Protein Mass (RPM)1 -%        |                                       | Activation                  | Repression  | Sigma factor |
| Upper glycolysis   |                                            |                                                   |                                        |                                       |                             |             |              |
| FbaB               | FBP → DHAP + G3P                           | 0.03                                              | 0.00                                   | 0.08                                  |                             | Cra         | RpoS         |
| TpiA               | DHAP → G3P                                 | 0.26                                              | 0.21                                   | 0.39                                  |                             | Cra         | RpoS, RpoD   |
| Lower glycolysis   |                                            |                                                   |                                        |                                       |                             |             |              |
| GapA               | G3P → 1,3-BPG                              | 0.70                                              | 0.97                                   | 1.23                                  | CRP-cAMP                    | Cra         | RpoS, RpoD   |
| GpmA               | 3PG → 2PG                                  | 0.27                                              | 0.38                                   | 0.62                                  |                             | Fur         | RpoS, RpoD   |
|                    | Pyruvate dehydrogenase                     |                                                   |                                        |                                       |                             |             |              |
| AceE               | Pyruvate → Acetyl-CoA + CO2                | 0.82                                              | 2.15                                   | 1.49                                  | CRP-cAMP                    | Cra, PdhR   | RpoS, RpoD   |
| AceF               |                                            | 0.34                                              | 0.82                                   | 0.58                                  | CRP-cAMP                    | Cra, PdhR   | RpoS, RpoD   |
|                    | Acetate metabolism                         |                                                   |                                        |                                       |                             |             |              |
| Acs                | Acetate → Acetyl-CoA                       | 0.10                                              | 0.06                                   | 0.35                                  | CRP-cAMP                    | Fis         | RpoS, RpoD   |
| PoxB               | Pyruvate → Acetate + CO2                   | 0.02                                              | 0.02                                   | 0.08                                  |                             |             | RpoS, RpoD   |
| Pta                | Acetyl-CoA → Acetyl-Phosphate              | 0.14                                              | 0.35                                   | 0.17                                  | ArcA-P                      |             |              |
| AckA               | Acetyl-Phosphate → Acetate                 | 0.12                                              | 0.36                                   | 0.15                                  | ArcA-P                      |             | RpoH         |
| TCA cycle          |                                            |                                                   |                                        |                                       |                             |             |              |
| GltA               | Oxaloacetate + Acetyl-CoA →<br>Citrate     | 1.12                                              | 0.29                                   | 1.25                                  | CRP-cAMP                    | ArcA-P      | RpoD         |
| AcnB               | Citrate → Isocitrate                       | 0.88                                              | 0.60                                   | 1.06                                  | CRP-cAMP                    | ArcA-P, Fis | RpoD         |
| SucD               | Succinyl-CoA → Succinate                   | 0.49                                              | 0.22                                   | 0.60                                  | CRP-cAMP                    | ArcA-P      | RpoS, RpoD   |
| SdhA               | Succinate → Fumarate                       | 0.64                                              | 0.26                                   | 0.59                                  | CRP-cAMP                    | ArcA-P      | RpoD         |
| FumA               | Fumarate → (S)-Malate                      | 0.07                                              | 0.03                                   | 0.10                                  | CRP-cAMP                    | ArcA-P      | RpoD         |
|                    | Amino acid biosynthesis                    |                                                   |                                        |                                       |                             |             |              |
| IlvC               | Branched-chain amino acids<br>biosynthesis | 2.83                                              | 0.16                                   | 0.13                                  |                             | Leucine3    | RpoD         |
| MetE               | Methionine biosynthesis                    | 3.60                                              | 0.10                                   | 0.16                                  |                             | Methionine3 |              |
| CysK               | Cysteine biosynthesis                      | 0.67                                              | 0.19                                   | 0.45                                  |                             | Cysteine3   | RpoD         |
| ThrB               | Threonine biosynthesis                     | 0.08                                              | 0.03                                   | 0.06                                  | DksA-ppGpp                  |             | RpoD         |
| HisC               | Histidine biosynthesis                     | 0.09                                              | 0.00                                   | 0.04                                  | DksA-ppGpp                  |             | RpoD         |
| RNA polymerases    |                                            |                                                   |                                        |                                       |                             |             |              |
| RpoA               | RNA polymerase, core enzyme                | 0.51                                              | 0.74                                   | 0.37                                  |                             | DksA-ppGpp  | RpoD         |
|                    | Ribosomal proteins                         |                                                   |                                        |                                       |                             |             |              |
| RplL               | 50S ribosomal subunit protein              | 0.70                                              | 0.95                                   | 0.53                                  |                             | DksA-ppGpp  |              |
| RpsA               | 30S ribosomal subunit protein              | 0.86                                              | 2.11                                   | 0.88                                  |                             | DksA-ppGpp  | RpoD         |
| Elongation factors |                                            |                                                   |                                        |                                       |                             |             |              |
| FusA               | Elongation factor G                        | 1.49                                              | 2.18                                   | 1.39                                  |                             | DksA-ppGpp  | RpoD, RpoE   |
| TufA               | Elongation factor Tu (EF-Tu)               | 3.51                                              | 5.66                                   | 3.98                                  |                             | DksA-ppGpp  | RpoD, RpoE   |
|                    | Amino acid degradation                     |                                                   |                                        |                                       |                             |             |              |
| AstA               | Arginine (to succinate)                    | 0.03                                              | 0.00                                   | 0.10                                  |                             |             | RpoS         |
| AstB               | degradation                                | 0.03                                              | 0.03                                   | 0.06                                  |                             |             | RpoS         |
|                    | (Metabolite) degradation                   |                                                   |                                        |                                       |                             |             |              |
| GabD               |                                            | 0.10                                              | 0.09                                   | 0.17                                  | CRP-cAMP                    |             | RpoS         |
|                    |                                            |                                                   |                                        |                                       |                             |             |              |

| GabT      | 4-Aminobutyrate (GABA)<br>degradation                              | 0.00 | 0.00 | 0.10 | CRP-cAMP |          | RpoS |
|-----------|--------------------------------------------------------------------|------|------|------|----------|----------|------|
| TreA      | Trehalose degradation                                              | 0.06 | 0.03 | 0.05 |          |          | RpoS |
| dismutase | Hydroperoxide reductases and superoxide                            |      |      |      |          |          |      |
| Tpx       | Antioxidant under anaerobic<br>condition                           | 0.08 | 0.06 | 0.17 |          | ArcA-P   | RpoD |
| SodB      | Superoxide dismutase                                               | 0.14 | 0.12 | 0.91 |          | CRP-cAMP | RpoD |
|           | DNA protection and repair                                          |      |      |      |          |          |      |
| Dps       | Protection of the DNA from<br>damage                               | 0.03 | 0.05 | 0.65 |          | Fis      | RpoS |
| UspA      | A member of the RecA-dependent<br>DNA protection and repair system | 0.11 | 0.07 | 0.20 |          | FadR     | RpoD |

|  |  | Table 1 Compendium of quantitative proteomic changes at different growth conditions (Continued) |  |  |  |  |  |  |
|--|--|-------------------------------------------------------------------------------------------------|--|--|--|--|--|--|
|--|--|-------------------------------------------------------------------------------------------------|--|--|--|--|--|--|

1 Functional classification is mostly according to EcoCyc database [(http://ecocyc.org/](http://ecocyc.org/)) and confirmed by KEGG database ([http://www.genome.jp/kegg/)](http://www.genome.jp/kegg/). Only examples from important groups are given the detailed analysis of the abundance levels of all identified proteins is found in Additional file [1:](#page-14-0) Table S1 or Additional file [2](#page-14-0): Table S5 for the entire data set including standard deviation. The RPM (%) of up-regulated proteins in bold letters for better recognition. 2 The information of transcriptional regulation is from RegulonDB Version 7.0 [(http://regulondb.ccg.unam.mx)](http://regulondb.ccg.unam.mx/). According to the differences of each protein's Relative Protein Mass at hunger, feast and famine conditions, the most (feasible) dominating transcription and sigma factors controlling the expression of the corresponding gene are given in bold letters.

3 Feed-back inhibition of amino acid biosynthesis (enzymes) by the end products of the corresponding pathways.

#### More details about the pyruvate node and acetate dilemma

Starting from pyruvate, the formation of acetate can occur via two different routes (Figure [6B](#page-11-0)). The most important route involves the pyruvate dehydrogenase catalyzed generation of acetyl-CoA followed by the formation of acetylphosphate and, finally, acetate, involving the two enzymes phosphotransacetylase (Pta) and acetate kinase (AckA), the Pta-AckA pathway [[20\]](#page-15-0). Regulation of genes encoding the enzymes of the pyruvate dehydrogenase is complex and involves many factors (Figure [6](#page-11-0)B, Additional file [1](#page-14-0): Table S2). In addition to regulation through Cra and CRPcAMP, the pyruvate dehydrogenase transcriptional repressor (PdhR) mediated repression of aceEF is inhibited by elevated concentrations of pyruvate [[31,37](#page-15-0)] (Figure [6](#page-11-0)B, Additional file [1:](#page-14-0) Table S2) presumably leading to increased levels of pyruvate dehydrogenase. Little is known about the regulation of the following Pta-AckA pathway, amongst others, transcriptional control seems to be mainly exerted through Fnr [[38\]](#page-15-0) (no functional fnr gene in E. coli BL21(DE3) [[39](#page-15-0)]) but also, at least moderately, through ArcA-(P) [[38,40](#page-15-0)] (Figure [6](#page-11-0)B). The other, though minor route towards acetate is directly from pyruvate catalyzed by pyruvate oxidase B (PoxB) [[21,41\]](#page-15-0). The Pta-AckA pathway is the dominating pathway in the exponential growth phase, and the PoxB pathway dominates in the stationary phase [[42](#page-15-0)]. PoxB is a unidirectional acting enzyme [[43,44](#page-15-0)], but the Pta-AckA pathway can, in principle, operate in both directions [[20\]](#page-15-0). Acetate uptake, however, occurs almost exclusively through acetyl coenzyme A synthetase (Acs) [[20,45](#page-15-0)]. Acs expression is controlled by many factors [[46](#page-15-0)] (Figure [6B](#page-11-0)) including Fis, a nucleoid associated protein [\[47,48](#page-15-0)] which accumulates to high levels during rapid growth in rich medium [[49,50](#page-15-0)]. Fis is the most important factor negatively controlling acs expression during exponential growth [[46](#page-15-0)]. It has been suggested that acetate accumulation at high growth rates occurs mainly as a result of the reduced assimilation capacity due to the downregulation of Acs [\[51\]](#page-15-0). On the other hand, increased acs expression occurs during entry into stationary phase [[52](#page-15-0)] and, moreover, increased acs expression as well as elevated Acs levels are observed at low growth rates in continuous culture experiments [\[51,53](#page-15-0)].

During rapid growth in rich medium we observed a significant increase in the abundance of the subunits of the pyruvate dehydrogenase complex (AceEF) which does not match with the more moderate increase observed for the lower glycolytic pathway enzymes (Figure [4](#page-5-0), Table [1](#page-9-0), Additional file [1](#page-14-0): Table S1). Most likely, in addition to the CRP-cAMP dependent activation elevated levels of intracellular pyruvate additionally contribute through the inhibition of PdhR to elevated expression of the genes of the pyruvate dehydrogenase complex, respectively to an increased abundance of the corresponding subunits. These findings suggest that in rich medium carbon overflow does not only cause limitations at the end of the catabolic pathways, e.g. in reoxidizing reduced quinones, but also upstream, e.g. at the end of the glycolytic pathway by accumulation of pyruvate. Remarkably, cells do not activate pyruvate catabolism through PoxB but try to enhance the capacity for pyruvate catabolism through pyruvate dehydrogenase during rapid growth in rich medium (Figure [4](#page-5-0)B, Table [1](#page-9-0)). Thus, formation of acetate from pyruvate obviously involves the pyruvate dehydrogenase catalyzed reaction and the Pta-AckA pathway and not the

<span id="page-11-0"></span>![](_page_11_Figure_2.jpeg)

(+, line ending arrow) and deactivation (−, line ending bar). The list of abbreviations is found at the end of the Additional file [1](#page-14-0): Figure S9. The construction of this simplified model of transcriptional regulation is based on data found in RegulonDB ([http://regulondb.ccg.unam.mx/)](http://regulondb.ccg.unam.mx/) [\[58](#page-16-0)].

PoxB dependent route. The observed low levels of TCA cycle enzymes as well as the strong abundance of Pta and AckA under these growth conditions (Figure [4](#page-5-0), Table [1](#page-9-0), Additional file [1:](#page-14-0) Table S1) pave the route towards acetate formation and are consistent with a dominating role of the transcriptional regulator ArcA-P (Figure [6)](#page-11-0). Thus, our results strongly suggest that acetate formation – even in the low acetate-producing strain BL21 – is caused by limitations in the respiratory chain, e.g. a limited capacity to reoxidize reduced quinones. The observed almost neglectable amounts of Acs during rapid growth in rich medium (Figures [2](#page-2-0) and [4](#page-5-0), Table [1](#page-9-0), Additional file [1:](#page-14-0) Table S1), presumably caused by high levels of Fis, additionally contribute to the observed extracellular accumulation of acetate (Figure [1)](#page-1-0). Another important issue to consider is the high level of freely available amino acids in rich medium as they additionally contribute to the overload on the lower glycolytic pathway and the pyruvate node because there is no need for flux diversion towards amino acid biosynthesis. Indeed, all enzymes which catalyze the first reactions of amino acid biosynthesis and the majority of the following enzymes starting from glycolytic pathway as well as form TCA cycle intermediates are strongly down-regulated during exponential growth in rich medium (Table [1](#page-9-0), Additional file [1](#page-14-0): Figure S7, Table S1). Thus, acetate formation during rapid growth in rich medium is additionally aggravated through the availability of free amino acids.

## Stringent-like and stationary phase responses

Absence or low levels of free amino acids are known to provoke a stringent-like response by elevating the amounts of the signaling molecule(s) (p)ppGpp [\[54-56\]](#page-15-0). (p)ppGpp binds to RNA polymerase (RNAP) and in conjunction with the RNAP binding protein DksA downregulates the expression of genes encoding ribosomal proteins [\[57\]](#page-16-0) and other enzymes and proteins involved in protein synthesis [[57,58\]](#page-16-0) (Figure [6](#page-11-0), Additional file [1:](#page-14-0) Table S3). Moreover, ribosomal RNA and ribosomal proteins are also being actively degraded during entry into stationary phase [\[59-61](#page-16-0)]. ppGpp exerted control is proposed to occur in a gradual manner: freely available amino acids result in low ppGpp concentrations (feast conditions), no free amino acids available but opportunity for amino acid biosynthesis (hunger conditions) lead to intermediate ppGpp concentrations, and no free amino acids and no opportunity for amino acid biosynthesis (famine conditions) result in the accumulation of high levels of ppGpp [[56](#page-15-0)]. Conditions between feast and famine, e.g. hunger provoking conditions are also known to initiate the induction of biosynthetic pathways [\[56\]](#page-15-0). The induction of some pathways involved in amino acid biosynthesis by DksA-ppGpp has been shown (e.g. thrB, hisC) [[62](#page-16-0)-[64](#page-16-0)], although the majority of amino acid biosynthesis pathways are controlled by complex feedback inhibition mechanisms, including the transcriptional regulation of metE [[65](#page-16-0)], ilvC [\[66-68](#page-16-0)], and cysK [\[69-71\]](#page-16-0). ppGpp bound to DksA also induces the expression of the stationary phase sigma factor (RpoS or sσ ) [[72](#page-16-0)], however, the induction of the RpoS regulon requires high ppGpp concentrations, namely famine conditions [\[56\]](#page-15-0). RpoS controls the expression of many stationary phase genes including the genes encoding transport proteins for better nutrient scavenging [[73,74\]](#page-16-0) and degradation of less favorable substrates [\[75-79](#page-16-0)] as well as genes of the oxidative stress response [[56](#page-15-0)] and other genes related to general cell protection [\[80](#page-16-0)] (Additional file [1](#page-14-0): Table S4). RpoS also exerts positive control on the expression of metabolic pathway genes, namely the genes of the lower and upper glycolytic pathway [\[56,](#page-15-0)[81\]](#page-16-0) and, importantly, also on acs [\[52](#page-15-0)[,81,82](#page-16-0)] and poxB [[56](#page-15-0)[,81,83\]](#page-16-0) (Figure [6,](#page-11-0) Additional file [1](#page-14-0): Table S4). Moreover, acs expression receives additional amplification during entry into stationary phase as the negative control of Fis on acs in exponentially growing cells is reversed through the negative control of Fis through DksA-ppGpp [[84](#page-16-0)] (Figure [6](#page-11-0), Additional file [1](#page-14-0): Table S3).

In the experimental conditions investigated in this study, feast, hunger, and famine conditions are encountered during exponential growth in rich medium, exponential growth in glucose supplemented defined mineral salt medium, and in stationary phase, respectively, and reflected in the observed alterations of the bacterial proteome of E. coli BL21 (Table [1)](#page-9-0). Proteins of the translational machinery reach their highest abundance during rapid growth in rich medium, intermediate levels at exponential growth in glucose-supplemented mineral salt medium, and lowest levels during stationary phase (Figures [3](#page-3-0) and [5,](#page-7-0) Table [1](#page-9-0), Additional file [1:](#page-14-0) Table S1). The high levels of enzymes of the amino acid synthesis pathway found during growth on glucose-supplemented defined mineral salt medium (Figures [2](#page-2-0) and [3](#page-3-0), Table [1](#page-9-0), Additional file [1](#page-14-0): Table S1) are in line with hunger conditions known to provoke the induction of biosynthetic pathways. However, the exceptionally high levels of MetE and IlvC found as part of the E. coli BL21 proteome during growth on glucose supplemented defined mineral salt medium (Figure [2A](#page-2-0) and Table [1)](#page-9-0) are presumably mainly caused by feedback derepression of amino acid synthesis pathways. The RpoS controlled response is most obviously reflected in the alterations of the proteome of E. coli BL21 after transition from exponential growth to stationary phase in rich medium. For example, the observed increase in proteins devoted to degradation of less favorable substrates (e.g. AstA/B, GabD/T and TreA) and general cell protection (e.g. DpS) (Table [1](#page-9-0), Additional file [1](#page-14-0): Figures S8 and S9, Table S1) is consistent with the expected RpoS dependent alterations of the bacterial proteome of stationary phase cells. In addition, the control of central metabolic pathways through RpoS becomes also evident through the general increase in the abundance of glycolytic pathway enzymes but also in the increased levels of PoxB and Acs in stationary phase cells (Figures [2](#page-2-0), [4](#page-5-0) and [6,](#page-11-0) Table [1](#page-9-0), Additional file [1:](#page-14-0) Table S1) and the concurrent uptake of acetate during stationary phase (Figure [1)](#page-1-0). However, there are also other proteins important for stationary phase survival of E. coli with increasing abundance during stationary phase in rich medium whose genes are not controlled by RpoS but by other factors (e.g. SodB [\[32](#page-15-0)], UspA [\[85](#page-16-0)], Tpx [[86](#page-16-0)], SsB [\[87\]](#page-16-0)) (Table [1](#page-9-0), Additional file [1](#page-14-0): Figure S8, Table S1). Interestingly, the proteomic reconstruction during entry into stationary phase is more prominent in cells growing in rich medium compared to cells growing in glucose-supplemented mineral salt medium. It seems the large amount of ribosomal proteins and other proteins involved in protein synthesis present during rapid growth in rich medium serve as the major reservoir for recycling to generate the proteins required for starvation survival. The lower abundance of ribosomal proteins in exponentially growing cells in glucosesupplemented mineral salt medium represents a more limited pool for proteomic reconstruction during entry into stationary phase. The large portion of proteins involved in amino acid biosynthesis in exponentially growing cells from defined medium does not serve as recycling reservoir as these proteins do not decrease during entry into stationary phase but slightly increase in quantity.

## Conclusions

In summary the observed changes in the proteome during entry into stationary phase are consistent with stringentlike and stationary phase responses and a dominating control through DksA-ppGpp and RpoS. The proteomic changes during entry into stationary phase are more prominent in rich medium as the cells have access to a larger reservoir of recyclable proteins, e.g. the proteins of the translational machinery. During rapid growth in rich medium ArcA-P appears to be the dominant metabolic pathway controlling transcriptional factor most likely the result of the limited capacity of E. coli to reoxidize the reduced quinones leading to the well-known phenomenon of acetate secretion. Certainly, transcriptional regulation is more complex than outlined above but these simplified interpretations adequately address the observed compositional alterations of the E. coli BL21 proteome. It seems also that the observed regulatory pattern is a good example of combinatorial control which can be suboptimal for cells under certain growth conditions [[88](#page-16-0)] in particular under laboratory conditions which may substantially deviate from the conditions of the ecological niche in which the cells have originally evolved.

## Materials and methods

Strain, culture conditions, and basic analytical procedures E. coli BL21 (DE3) (Novagen, Germany) was grown in rich (Luria Bertani or Terrific Broth) or defined mineral salt medium with glucose as carbon source. The composition of the rich media was as follows: Luria Bertani (LB): 10 g L−1 tryptone (Becton, Dickinson and Company-BD, USA), 5 g L−1 yeast extract (Becton, Dickinson and Company-BD, USA), and 5 g L−1 NaCl; Terrific broth (TB): 12 g L−1 tryptone (Becton, Dickinson and Company-BD, USA), 24 g L−1 yeast extract (Becton, Dickinson and Company-BD, USA), 5 g L−1 glycerol, 2.31 g L−1 KH2PO4, and 12.54 g L−1 K2HPO4. The composition of the Defined Non-inducing Broth (DNB) was as follows: 10.91 g L−1 glucose, 4 g L−1 (NH4)2HPO4, 13.3 g L−1 KH2PO4, 1.55 g L−1 citric acid, 0.59 g L−1 MgSO4, 100.8 mg L−1 Fe(III) citrate, 2.1 mg L−1 Na2MoO4.2H2O, 2.5 mg L−1 CoCl2.6H2O, 15 mg L−1 MnCl2.4H2O, 1.5 mg L−1 CuCl2.2H2O, 3 mg L−1 H3BO3, 33.8 mg L−1 Zn (CH3COOH)2.2H2O, 14.1 mg L−1 Titriplex III. The pH of all media was adjusted to pH 6.8 with NaOH prior to sterilization. Details about medium preparation were as described previously [\[3,](#page-14-0)[89](#page-16-0),[90](#page-16-0)]. Cultivations were carried out in duplicate using 2 L Erlenmeyer flasks with three baffles containing 200 ml medium at 37°C and 180 rpm (Multitron Cell, INFORS HT, Switzerland). Cell growth was monitored by measurement of the absorbance at 600 nm (WPA CO8000 Cell Density Meter, Biochrom, UK). Acetate was analyzed using an acetic acid kit (Cat. No. K-ACETRM, Megazyme, Ireland). Cell samples for twodimensional (2D) gel electrophoresis were immediately centrifuged at 17,000 g and 4°C for 3 min after sampling. After removal of the supernatant, cell pellets were stored at −80°C prior to gel electrophoresis. The profile of the dissolved oxygen concentration was determined in 500 mL shake flasks with integrated pH and oxygen sensors (SFS) using a SFR Shake Flask Reader (PreSens Precision Sensing GmbH, Germany) and the same cultivation conditions as described above (50 mL medium at 37°C and 180 rpm). Offline sampling was omitted in shake flask cultures during dissolved oxygen measurements to prevent perturbation of the dissolved oxygen profile.

### 2D gel electrophoresis

Cell pellets were disrupted using BugBuster™ Protein Extraction Reagent (Novagen, USA) supplemented with rLysozyme and Benzonase according to manufacturer's instructions. Whole cell protein in the BugBuster suspension was precipitated without prior centrifugation as described previously [\[91\]](#page-16-0). The protein pellets were solubilized in rehydration solution with IPG buffer (GE Healthcare, UK). About 280 μg of protein were loaded onto Immobiline DryStrip gels of pH 3–10 NL (GE Healthcare, UK). The first-dimension using <span id="page-14-0"></span>isoelectric focusing (IEF) and the second dimension using SDS-PAGE (10-15% gradient gel) were carried out using the IPGphor™ Isoelectric Focusing and Hoefer™ DALT systems (GE Healthcare, UK), respectively. The detailed experimental conditions were according to manufacturer's instructions and as described previously [[92](#page-16-0)]. Gels were stained using colloidal Coomassie R250 [[93](#page-16-0)] and analyzed using Proteomweaver™ 4.0 software (Bio-Rad, USA) for protein spot detection, matching, and quantification. For each sample, 2D gels were made in triplicate and the best two gels analyzed using Proteomweaver™ 4.0. Each spot's intensity was normalized by the whole spots intensity of the same 2D gel. The corresponding average intensity of the spot (or the sum of several spots representing the same protein in case of spot multiplicity) taken from the two duplicate gels was used to determine this protein's portion (%) of the relative protein mass (RPM).

#### Protein identification and classification

Protein spots were identified by matrix-assisted laser desorption ionization time-of-flight mass spectrometry (MALDI-TOF MS). Protein spots were excised manually from 2D gels. After washing, reduction, and alkylation, in-gel digestion was carried out by incubation with sequencing grade porcine trypsin modified by reductive methylation (Promega, USA). Obtained peptides were extracted and purified with reversed-phase C18 ZipTips (Millipore, USA). Desalted peptide solutions were mixed with a saturated matrix solution and spotted onto a 384 MTP target and dried at room temperature. A Bruker Ultraflex time-of-flight mass spectrometer (Bruker Daltonics GmbH, Germany) was employed to obtain peptide mass fingerprints. Details of the protocol are given elsewhere [\[92,94\]](#page-16-0). The MAS-COT search program (Matrix Science, UK) was used for protein identification using the annotated E. coli genome as database (Uniprot:<http://www.uniprot.org/>). All proteins with a Mowse score greater than 54 were regarded as significant (p < 0.05). The EcoCyc database was mainly used for classification of identified proteins into functional categories [(http://ecocyc.org/](http://ecocyc.org/), [\[95\]](#page-16-0)). Classification was confirmed using the KEGG database ([http://](http://www.genome.jp/kegg/) [www.genome.jp/kegg/,](http://www.genome.jp/kegg/) [[96](#page-16-0)]). The transcriptional regulation of genes encoding identified proteins were mainly acquired from the RegulonDB ([http://regulondb.ccg.unam.](http://regulondb.ccg.unam.mx/) [mx/,](http://regulondb.ccg.unam.mx/) [[58](#page-16-0)]) and EcoCyc [(http://ecocyc.org/](http://ecocyc.org/), [\[95\]](#page-16-0)) databases.

## Additional files

[Additional file 1:](http://www.biomedcentral.com/content/supplementary/1475-2859-13-45-S1.pdf) Figures S1-S6. Images of 2D gels of the proteome of E. coli BL21 (DE3) growing in defined and rich media (Luria Bertani and Terrific Broth) at exponential and stationary growth phases. Identified proteins are marked and "clickable" to get access to further protein

information (http://www.uniprot.org). Figures S7-S9: Detailed comparative scheme of the (proteomic) pathway regulation at different growth conditions. E. coli BL21 (DE3) growing in defined medium versus rich medium at exponential phase, at stationary phase versus exponential phase in rich medium, and at stationary phase versus exponential phase in defined medium. Figure S10: Comparison of the stationary and exponential phase proteome of E. coli BL21 (DE3) in different media. Table S1: Quantitative data of individual proteins of E. coli BL21 (DE3) growing in defined and rich media (Luria Bertani and Terrific Broth) at exponential and stationary phases. Table S2: Transcriptional control of glycolysis and TCA cycle genes. The information was extracted from the RegulonDB database (http://regulondb.ccg.unam.mx/). Table S3: DksA-ppGpp controlled genes of E. coli. The information was extracted from the RegulonDB (http://regulondb.ccg.unam.mx/) and EcoCyc databases (http://ecocyc.org/). Table S4: RpoS controlled genes of E. coli. The information was extracted from the RegulonDB (http://regulondb.ccg.unam.mx/) and EcoCyc databases (http://ecocyc.org/).

[Additional file 2: Table S5.](http://www.biomedcentral.com/content/supplementary/1475-2859-13-45-S2.xlsx) Quantitative data of individual proteins (obtained from duplicate gels showing standard deviation) of E. coli BL21 (DE3) growing in defined and rich media (Luria Bertani and Terrific Broth) at exponential and stationary phases.

#### Competing interests

The authors declare that they have no competing interests.

#### Authors' contributions

ZL did the experimental work, analyzed the data and prepared the first draft of the manuscript. MN contributed to protein identification by Maldi-ToF. UR directed the study and prepared the final manuscript. All authors read and approved the final manuscript.

#### Acknowledgements

Partial financial support from the German Ministry of Education and Research (BMBF) within the FORSYS-Partner program (grant FKZ 0315285D) is gratefully acknowledged.

#### Received: 19 November 2013 Accepted: 14 March 2014 Published: 23 March 2014

## References

- 1. Shiloach J, Rinas U: Glucose and acetate metabolism in E. coli System level analysis and biotechnological applications in protein production processes. In Systems biology and biotechnology of Escherichia coli. Edited by Lee SY. Berlin, Heidelberg, New York: Springer-Verlag; 2009:377–400.
- 2. Yoon SH, Han MJ, Jeong H, Lee CH, Xia XX, Lee DH, Shim JH, Lee SY, Oh TK, Kim JF: Comparative multi-omics systems analysis of Escherichia coli strains B and K-12. Genome Biol 2012, 13:R37.
- 3. Korz DJ, Rinas U, Hellmuth K, Sanders EA, Deckwer WD: Simple fed-batch technique for high cell density cultivation of Escherichia coli. J Biotechnol 1995, 39:59–65.
- 4. Martin VJ, Pitera DJ, Withers ST, Newman JD, Keasling JD: Engineering a mevalonate pathway in Escherichia coli for production of terpenoids. Nat Biotechnol 2003, 21:796–802.
- 5. Tsuruta H, Paddon CJ, Eng D, Lenihan JR, Horning T, Anthony LC, Regentin R, Keasling JD, Renninger NS, Newman JD: High-level production of amorpha-4,11-diene, a precursor of the antimalarial agent artemisinin, in Escherichia coli. PLoS ONE 2009, 4:e4489.
- 6. Chavez-Bejar MI, Balderas-Hernandez VE, Gutierrez-Alejandre A, Martinez A, Bolivar F, Gosset G: Metabolic engineering of Escherichia coli to optimize melanin synthesis from glucose. Microb Cell Fact 2013, 12:108.
- 7. Park JH, Lee SY: Fermentative production of branched chain amino acids: a focus on metabolic engineering. Appl Microbiol Biotechnol 2010, 85:491–506.
- 8. Julsing MK, Kuhn D, Schmid A, Buhler B: Resting cells of recombinant E. coli show high epoxidation yields on energy source and high sensitivity to product inhibition. Biotechnol Bioeng 2012, 109:1109–1119.
- 9. Leonardo MR, Cunningham PR, Clark DP: Anaerobic regulation of the adhE gene, encoding the fermentative alcohol dehydrogenase of Escherichia coli. J Bacteriol 1993, 175:870–878.
- 10. Leonardo MR, Dailly Y, Clark DP: Role of NAD in regulating the adhE gene of Escherichia coli. J Bacteriol 1996, 178:6013–6018.
- <span id="page-15-0"></span>11. Holland-Staley CA, Lee K, Clark DP, Cunningham PR: Aerobic activity of Escherichia coli alcohol dehydrogenase is determined by a single amino acid. J Bacteriol 2000, 182:6049–6054.
- 12. Anderson KB, von Meyenburg K: Are growth rates of Escherichia coli in batch cultures limited by respiration? J Bacteriol 1980, 144:114–123.
- 13. Mizuno T, Shinkai A, Matsui K, Mizushima S: Osmoregulatory expression of porin genes in Escherichia coli: a comparative study on strains B and K-12. FEMS Microbiol Lett 1990, 68:289–293.
- 14. Schneider D, Duperchy E, Depeyrot J, Coursange E, Lenski R, Blot M: Genomic comparisons among Escherichia coli strains B, K-12, and O157: H7 using IS elements as molecular markers. BMC Microbiol 2002, 2:18.
- 15. Studier FW, Daegelen P, Lenski RE, Maslov S, Kim JF: Understanding the differences between genome sequences of Escherichia coli B strains REL606 and BL21(DE3) and comparison of the E. coli B and K-12 genomes. J Mol Biol 2009, 394:653–680.
- 16. Matuszewska M, Kuczynska-Wisnik D, Laskowska E, Liberek K: The small heat shock protein IbpA of Escherichia coli cooperates with IbpB in stabilization of thermally aggregated proteins in a disaggregation competent state. J Biol Chem 2005, 280:12292–12298.
- 17. Ratajczak E, Zietkiewicz S, Liberek K: Distinct activities of Escherichia coli small heat shock proteins IbpA and IbpB promote efficient protein disaggregation. J Mol Biol 2009, 386:178–189.
- 18. Goloubinoff P, Mogk A, Ben Zvi AP, Tomoyasu T, Bukau B: Sequential mechanism of solubilization and refolding of stable protein aggregates by a bichaperone network. Proc Natl Acad Sci U S A 1999, 96:13732–13737.
- 19. Rosenzweig R, Moradi S, Zarrine-Afsar A, Glover JR, Kay LE: Unraveling the mechanism of protein disaggregation through a ClpB-DnaK interaction. Science 2013, 339:1080–1083.
- 20. Brown TDK, Jones-Mortimer MC, Kornberg HL: The enzymic interconversion of acetate and acetyl-coenzyme in Escherichia coli. J Gen Microbiol 1977, 102:327–336.
- 21. Phue JN, Shiloach J: Transcription levels of key metabolic genes are the cause for different glucose utilization pathways in E. coli B (BL21) and E. coli K (JM109). J Biotechnol 2004, 109:21–30.
- 22. Shimada T, Yamamoto K, Ishihama A: Novel members of the Cra regulon involved in carbon metabolism in Escherichia coli. J Bacteriol 2011, 193:649–659.
- 23. Fic E, Bonarek P, Gorecki A, Kedracka-Krok S, Mikolajczak J, Polit A, Tworzydlo M, Dziedzicka-Wasylewska M, Wasylewski Z: cAMP receptor protein from Escherichia coli as a model of signal transduction in proteins - a review. J Mol Microbiol Biotechnol 2009, 17:1–11.
- 24. Georgellis D, Kwon O, Lin EC: Quinones as the redox signal for the arc two-component system of bacteria. Science 2001, 292:2314–2316.
- 25. Malpica R, Sandoval GR, Rodriguez C, Franco B, Georgellis D: Signaling by the arc two-component system provides a link between the redox state of the quinone pool and gene expression. Antioxid Redox Signal 2006, 8:781–795.
- 26. Shimada T, Fujita N, Maeda M, Ishihama A: Systematic search for the Cra-binding promoters using genomic SELEX system. Genes Cells 2005, 10:907–918.
- 27. Ramseier TM, Negre D, Cortay JC, Scarabel M, Cozzone AJ, Saier MH Jr: In vitro binding of the pleiotropic transcriptional regulatory protein, FruR, to the fru, pps, ace, pts and icd operons of Escherichia coli and Salmonella typhimurium. J Mol Biol 1993, 234:28–44.
- 28. Nègre D, Bonod-Bidaud C, Geourjon C, Deleage G, Cozzone AJ, Cortay JC: Definition of a consensus DNA-binding site for the Escherichia coli pleiotropic regulatory protein, FruR. Mol Microbiol 1996, 21:257–266.
- 29. Ishizuka H, Hanamura A, Inada T, Aiba H: Mechanism of the down-regulation of cAMP receptor by glucose in Escherichia coli: role of autoregulation of the crp gene. EMBO J 1994, 13:3077–3082.
- 30. Charpentier B, Branlant C: The Escherichia coli gapA gene is transcribed by the vegetative RNA polymerase holoenzyme Eσ70 and by the heat shock RNA polymerase Eσ32. J Bacteriol 1994, 176:830–839.
- 31. Quail MA, Guest JR: Purification, characterization and mode of action of PdhR, the transcriptional repressor of the pdhR-aceEF-lpd operon of Escherichia coli. Mol Microbiol 1995, 15:519–529.
- 32. Zhang Z, Gosset G, Barabote R, Gonzalez CS, Cuevas WA, Saier MH Jr: Functional interactions between the carbon and iron utilization regulators, Crp and Fur, in Escherichia coli. J Bacteriol 2005, 187:980–990.
- 33. Wilde RJ, Guest JR: Transcript analysis of the citrate synthase and succinate dehydrogenase genes of Escherichia coli K12. J Gen Microbiol 1986, 132:3239–3251.
- 34. Cunningham L, Gruer MJ, Guest JR: Transcriptional regulation of the aconitase genes (acnA and acnB) of Escherichia coli. Microbiology 1997, 143:3795–3805.
- 35. Cunningham L, Guest JR: Transcription and transcript processing in the sdhCDAB-sucABCD operon of Escherichia coli. Microbiology 1998, 144:2113–2123.
- 36. Chen YP, Lin HH, Yang CD, Huang SH, Tseng CP: Regulatory role of cAMP receptor protein over Escherichia coli fumarase genes. J Microbiol 2012, 50:426–433.
- 37. Göhler A-G, Kökpinar Ö, Schmidt-Heck W, Geffers R, Guthke R, Rinas U, Schuster S, Jahreis K, Kaleta C: More than just a metabolic regulator elucidation and validation of new targets of PdhR in Escherichia coli. BMC Syst Biol 2011, 5:197.
- 38. Shalel-Levanon S, San KY, Bennett GN: Effect of ArcA and FNR on the expression of genes related to the oxygen regulation and the glycolysis pathway in Escherichia coli under microaerobic growth conditions. Biotechnol Bioeng 2005, 92:147–159.
- 39. Pinske C, Bonn M, Kruger S, Lindenstrauss U, Sawers RG: Metabolic deficiences revealed in the biotechnologically important model bacterium Escherichia coli BL21(DE3). PLoS ONE 2011, 6:e22830.
- 40. Salmon KA, Hung SP, Steffen NR, Krupp R, Baldi P, Hatfield GW, Gunsalus RP: Global gene expression profiling in Escherichia coli K12: effects of oxygen availability and ArcA. J Biol Chem 2005, 280:15084–15096.
- 41. Phue JN, Noronha SB, Hattacharyya R, Wolfe AJ, Shiloach J: Glucose metabolism at high density growth of E. coli B and E. coli K: differences in metabolic pathways are responsible for efficient glucose utilization in E. coli B as determined by microoarrys and northern blot analyses. Biotechnol Bioeng 2005, 90:805–820.
- 42. Dittrich CR, Bennett GN, San KY: Characterization of the acetate-producing pathways in Escherichia coli. Biotechnol Prog 2005, 21:1062–1067.
- 43. Mather MW, Gennis RB: Kinetic studies of the lipid-activated pyruvate oxidase flavoprotein of Escherichia coli. J Biol Chem 1985, 260:16148–16155.
- 44. Marchal D, Pantigny J, Laval JM, Moiroux J, Bourdillon C: Rate constants in two dimensions of electron transfer between pyruvate oxidase, a membrane enzyme, and ubiquinone (coenzyme Q8), its water-insoluble electron carrier. Biochemistry 2001, 40:1248–1256.
- 45. Kumari S, Beatty CM, Browning DF, Busby SJ, Simel EJ, Hovel-Miner G, Wolfe AJ: Regulation of acetyl coenzyme A synthetase in Escherichia coli. J Bacteriol 2000, 182:4173–4179.
- 46. Wolfe AJ: The acetate switch. Microbiol Mol Biol Rev 2005, 69:12–50.
- 47. Browning DF, Beatty CM, Wolfe AJ, Cole JA, Busby SJ: Independent regulation of the divergent Escherichia coli nrfA and acsP1 promoters by a nucleoprotein assembly at a shared regulatory region. Mol Microbiol 2002, 43:687–701.
- 48. Browning DF, Beatty CM, Sanstad EA, Gunn KE, Busby SJ, Wolfe AJ: Modulation of CRP-dependent transcription at the Escherichia coli acsP2 promoter by nucleoprotein complexes: anti-activation by the nucleoid proteins FIS and IHF. Mol Microbiol 2004, 51:241–254.
- 49. Ball CA, Osuna R, Ferguson KC, Johnson RC: Dramatic changes in Fis levels upon nutrient upshift in Escherichia coli. J Bacteriol 1992, 174:8043–8056.
- 50. Nafissi M, Chau J, Xu J, Johnson RC: Robust translation of the nucleoid protein Fis requires a remote upstream AU element and is enhanced by RNA secondary structure. J Bacteriol 2012, 194:2458–2469.
- 51. Valgepea K, Adamberg K, Nahku R, Lahtvee PJ, Arike L, Vilu R: Systems biology approach reveals that overflow metabolism of acetate in Escherichia coli is triggered by carbon catabolite repression of acetyl-CoA synthetase. BMC Syst Biol 2010, 4:166.
- 52. Shin S, Song SG, Lee DS, Pan JG, Park C: Involvement of iclR and rpoS in the induction of acs, the gene for acetyl coenzyme A synthetase of Escherichia coli K-12. FEMS Microbiol Lett 1997, 146:103–108.
- 53. Nahku R, Valgepea K, Lahtvee PJ, Erm S, Abner K, Adamberg K, Vilu R: Specific growth rate dependent transcriptome profiling of Escherichia coli K12 MG1655 in accelerostat cultures. J Biotechnol 2010, 145:60–65.
- 54. Nyström T: Stationary-phase physiology. Annu Rev Microbiol 2004, 58:161–181.
- 55. Magnusson LU, Farewell A, Nyström T: ppGpp: a global regulator in Escherichia coli. Trends Microbiol 2005, 13:236–242.
- 56. Traxler MF, Zacharia VM, Marquardt S, Summers SM, Nguyen HT, Stark SE, Conway T: Discretely calibrated regulatory loops controlled by ppGpp partition gene induction across the 'feast to famine' gradient in Escherichia coli. Mol Microbiol 2011, 79:830–845.
- <span id="page-16-0"></span>57. Lemke JJ, Sanchez-Vazquez P, Burgos HL, Hedberg G, Ross W, Gourse RL: Direct regulation of Escherichia coli ribosomal protein promoters by the transcription factors ppGpp and DksA. Proc Natl Acad Sci U S A 2011, 108:5712–5717.
- 58. Gama-Castro S, Salgado H, Peralta-Gil M, Santos-Zavaleta A, Muniz-Rascado L, Solano-Lira H, Jimenez-Jacinto V, Weiss V, Garcia-Sotelo JS, Lopez-Fuentes A, Porron-Sotelo L, Alquicira-Hernandez S, Medina-Rivera A, Martinez-Flores I, Alquicira-Hernandez K, Martinez-Adame R, Bonavides-Martinez C, Miranda-Rios J, Huerta AM, Mendoza-Vargas A, Collado-Torres L, Taboada B, Vega-Alvarado L, Olvera M, Olvera L, Grande R, Morett E, Collado-Vides J: RegulonDB version 7.0: transcriptional regulation of Escherichia coli K-12 integrated within genetic sensory response units (Gensor Units). Nucleic Acids Res 2011, 39:D98–105.
- 59. Rinas U, Hellmuth K, Kang R, Seeger A, Schlieker H: Entry of Escherichia coli into stationary phase is indicated by endogenous and exogenous accumulation of nucleobases. Appl Environ Microbiol 1995, 61:4147–4151.
- 60. Zundel MA, Basturea GN, Deutscher MP: Initiation of ribosome degradation during starvation in Escherichia coli. RNA 2009, 15:977–983.
- 61. Piir K, Paier A, Liiv A, Tenson T, Maivali U: Ribosome degradation in growing bacteria. EMBO Rep 2011, 12:458–462.
- 62. Paul BJ, Berkmen MB, Gourse RL: DksA potentiates direct activation of amino acid promoters by ppGpp. Proc Natl Acad Sci U S A 2005, 102:7823–7828.
- 63. Srivatsan A, Wang JD: Control of bacterial transcription, translation and replication by (p)ppGpp. Curr Opin Microbiol 2008, 11:100–105.
- 64. Lee JH, Lennon CW, Ross W, Gourse RL: Role of the coiled-coil tip of Escherichia coli DksA in promoter control. J Mol Biol 2012, 416:503–517.
- 65. Weissbach H, Brot N: Regulation of methionine synthesis in Escherichia coli. Mol Microbiol 1991, 5:1593–1597.
- 66. Gollop N, Chipman DM, Barak Z: Inhibition of acetohydroxy acid synthase by leucine. Biochim Biophys Acta 1983, 748:34–39.
- 67. Rhee KY, Senear DF, Hatfield GW: Activation of gene expression by a ligand-induced conformational change of a protein-DNA complex. J Biol Chem 1998, 273:11257–11266.
- 68. Opel ML, Hatfield GW: DNA supercoiling-dependent transcriptional coupling between the divergently transcribed promoters of the ilvYC operon of Escherichia coli is proportional to promoter strengths and transcript lengths. Mol Microbiol 2001, 39:191–198.
- 69. Monroe RS, Ostrowski J, Hryniewicz MM, Kredich NM: In vitro interactions of CysB protein with the cysK and cysJIH promoter regions of Salmonella typhimurium. J Bacteriol 1990, 172:6919–6929.
- 70. Hindson VJ: Serine acetyltransferase of Escherichia coli: substrate specificity and feedback control by cysteine. Biochem J 2003, 375:745–752.
- 71. Yamamoto K, Oshima T, Nonaka G, Ito H, Ishihama A: Induction of the Escherichia coli cysK gene by genetic and environmental factors. FEMS Microbiol Lett 2011, 323:88–95.
- 72. Brown L, Gentry D, Elliott T, Cashel M: DksA affects ppGpp induction of RpoS at a translational level. J Bacteriol 2002, 184:4455–4465.
- 73. Rajkumari K, Kusano S, Ishihama A, Mizuno T, Gowrishankar J: Effects of H-NS and potassium glutamate on σS - and σ70-directed transcription in vitro from osmotically regulated P1 and P2 promoters of proU in Escherichia coli. J Bacteriol 1996, 178:4176–4181.
- 74. Lacour S, Landini P: σS -dependent gene expression at the onset of stationary phase in Escherichia coli: function of sS -dependent genes and identification of their promoter sequences. J Bacteriol 2004, 186:7186–7195.
- 75. Hengge-Aronis R, Klein W, Lange R, Rimmele M, Boss W: Trehalose synthesis genes are controlled by the putative sigma factor encoded by rpoS and are involved in stationary-phase thermotolerance in Escherichia coli. J Bacteriol 1991, 173:7918–7924.
- 76. Kiupakis AK, Reitzer L: ArgR-independent induction and ArgR-dependent superinduction of the astCADBE operon in Escherichia coli. J Bacteriol 2002, 184:2940–2950.
- 77. Waterman SR, Small PL: Transcriptional expression of Escherichia coli glutamate-dependent acid resistance genes gadA and gadBC in an hns rpoS mutant. J Bacteriol 2003, 185:4644–4647.
- 78. Metzner M, Germer J, Hengge R: Multiple stress signal integration in the regulation of the complex σS -dependent csiD-ygaF-gabDTP operon in Escherichia coli. Mol Microbiol 2004, 51:799–811.
- 79. Dong T, Schellhorn HE: Control of RpoS in global gene expression of Escherichia coli in minimal media. Mol Genet Genomics 2009, 281:19–33.
- 80. Hengge-Aronis R: Survival of hunger and stress: the role of rpoS in early stationary phase gene regulation in Escherichia coli. Cell 1993, 72:165–168.
- 81. Olvera L, Mendoza-Vargas A, Flores N, Olvera M, Sigala JC, Gosset G, Morett E, Bolivar F: Transcription analysis of central metabolism genes in Escherichia coli. Possible roles of sigma38 in their expression, as a response to carbon limitation. PLoS ONE 2009, 4:e7466.
- 82. Rahman M, Hasan MR, Shimizu K: Growth phase-dependent changes in the expression of global regulatory genes and associated metabolic pathways in Escherichia coli. Biotechnol Lett 2008, 30:853–860.
- 83. Chang YY, Wang AY, Cronan J: Expression of Escherichia coli pyruvate oxidase (PoxB) depends on the sigma factor encoded by the rpoS(katF) gene. Mol Microbiol 1994, 11:1019–1028.
- 84. Mallik P, Paul BJ, Rutherford ST, Gourse RL, Osuna R: DksA is required for growth phase-dependent regulation, growth rate-dependent control, and stringent control of fis expression in Escherichia coli. J Bacteriol 2006, 188:5775–5782.
- 85. Farewell A, Diez AA, DiRusso CC, Nyström T: Role of the Escherichia coli FadR regulator in stasis survival and growth phase-dependent expression of the uspA, fad, and fab genes. J Bacteriol 1996, 178:6443–6450.
- 86. Kim SJ, Han YH, Kim IH, Kim HK: Involvement of ArcA and Fnr in expression of Escherichia coli thiol peroxidase gene. IUBMB Life 1999, 48:215–218.
- 87. Ogasawara H, Teramoto J, Yamamoto S, Hirao K, Yamamoto K, Ishihama A, Utsumi R: Negative regulation of DNA repair gene (uvrA) expression by ArcA/ArcB two-component system in Escherichia coli. FEMS Microbiol Lett 2005, 251:243–249.
- 88. Price MN, Deutschbauer AM, Skerker JM, Wetmore KM, Ruths T, Mar JS, Kuehl JV, Shao W, Arkin AP: Indirect and suboptimal control of gene expression is widespread in bacteria. Mol Syst Biol 2013, 9:660.
- 89. Li Z, Kessler W, van den Heuvel J, Rinas U: Simple defined autoinduction medium for high-level recombinant protein production for T7-based Escherichia coli expression systems. Appl Microbiol Biotechnol 2011, 91:1203–1213.
- 90. Li Z, Nimtz M, Rinas U: Optimized procedure to generate heavy isotope and selenomethionine-labeled proteins for structure determination using Escherichia coli-based expression systems. Appl Microbiol Biotechnol 2011, 92:823–833.
- 91. Wessel D, Flügge UI: A method for the quantitative recovery of protein in dilute solution in the presence of detergents and lipids. Anal Biochem 1984, 138:141–143.
- 92. Lu X, Sun J, Nimtz M, Wissing J, Zeng AP, Rinas U: The intra- and extracellular proteome of Aspergillus niger growing on defined medium with xylose or maltose as carbon substrate. Microb Cell Fact 2010, 9:23.
- 93. Candiano G, Bruschi M, Musante L, Santucci L, Ghiggeri GM, Carnemolla B, Orecchia P, Zardi L, Righetti PG: Blue silver: a very sensitive colloidal Coomassie G-250 staining for proteome analysis. Electrophoresis 2004, 25:1327–1333.
- 94. Wang W, Sun J, Nimtz M, Deckwer W-D, Zeng A-P: Protein identification from two-dimensional gel electrophoresis analysis of Klebsiella pneumoniae by combined use of mass spectrometry data and raw genome sequences. Proteome Sci 2003, 1:6.
- 95. Keseler IM, Mackie A, Peralta-Gil M, Santos-Zavaleta A, Gama-Castro S, Bonavides-Martinez C, Fulcher C, Huerta AM, Kothari A, Krummenacker M, Latendresse M, Muniz-Rascado L, Ong Q, Paley S, Schroder I, Shearer AG, Subhraveti P, Travers M, Weerasinghe D, Weiss V, Collado-Vides J, Gunsalus RP, Paulsen I, Karp PD: EcoCyc: fusing model organism databases with systems biology. Nucleic Acids Res 2013, 41:D605–D612.
- 96. Kanehisa M, Goto S: KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res 2000, 28:27–30.

## doi:10.1186/1475-2859-13-45

Cite this article as: Li et al.: The metabolic potential of Escherichia coli BL21 in defined and rich medium. Microbial Cell Factories 2014 13:45.