![](_page_0_Picture_0.jpeg)

## ARTICLE

1234567890():,;

https://doi.org/10.1038/s41467-020-16872-8 **OPEN**

![](_page_0_Picture_4.jpeg)

# Diauxie and co-utilization of carbon sources can coexist during bacterial growth in nutritionally complex environments

Elena Perri[n](http://orcid.org/0000-0001-5148-3178) [1](http://orcid.org/0000-0001-5148-3178),7, Veronica Ghini2,7, Michele Giovannini1 , Francesca Di Patti3,4, Barbara Cardazzo5, Lisa Carraro5, Camilla Fagorz[i](http://orcid.org/0000-0002-9996-8868) [1](http://orcid.org/0000-0002-9996-8868) , Paola Turano6, Renato Fani1 & Marco Fond[i](http://orcid.org/0000-0001-9291-5467) [1](http://orcid.org/0000-0001-9291-5467),4✉

It is commonly thought that when multiple carbon sources are available, bacteria metabolize them either sequentially (diauxic growth) or simultaneously (co-utilization). However, this view is mainly based on analyses in relatively simple laboratory settings. Here we show that a heterotrophic marine bacterium, Pseudoalteromonas haloplanktis, can use both strategies simultaneously when multiple possible nutrients are provided in the same growth experiment. The order of nutrient uptake is partially determined by the biomass yield that can be achieved when the same compounds are provided as single carbon sources. Using transcriptomics and time-resolved intracellular 1 H-13C NMR, we reveal specific pathways for utilization of various amino acids. Finally, theoretical modelling indicates that this metabolic phenotype, combining diauxie and co-utilization of substrates, is compatible with a tight regulation that allows the modulation of assimilatory pathways.

<sup>1</sup>Department of Biology, University of Florence, Florence, Italy. 2Consorzio Interuniversitario Risonanze Magnetiche di Metallo Proteine (CIRMMP), Florence, Italy. 3Department of Physics and Astronomy, University of Florence, Florence, Italy. 4CSDC, University of Florence, Florence, Italy. 5Department of Comparative Biomedicine and Food Science, University of Padova, Padova, Italy. 6Center of Magnetic Resonance (CERM), University of Florence, Florence, Italy. 7 These authors contributed equally: Elena Perrin, Veronica Ghini. ✉email: [marco.fondi@uni](mailto:marco.fondi@unifi.it)fi.it

Microorganisms must quickly and efficiently adapt to a variety of possible fluctuations in the surrounding environment. When considering changes in the pool of available nutrients, this is usually achieved by a tight regulation of their metabolic phenotypes by sensing the availability of specific compounds, synthesizing the enzymes required for their catabolism and repressing them after specific metabolites are depleted[1.](#page-13-0) The spectrum of possible bacterial metabolic adaptation strategies can be observed, for example, when growing cells in a medium containing a simple mixture of carbon sources. In this situation bacterial may exhibit different patterns including diauxic growth[2,](#page-13-0) simultaneous consumption[3](#page-13-0), and bistable growth[4,5.](#page-13-0) Further, nutrients concentration and growth medium composition are known to affect other important cellular features such as motility and cell adhesion[6](#page-13-0),[7](#page-13-0) and biofilm formation[8](#page-13-0),[9](#page-13-0). Typically, these phenomena have been studied (both theoretically and experimentally) in model organisms (e.g., Escherichia coli and Lactococcus lactis[)10,](#page-13-0) grown on defined media containing simple mixtures of 2/3 carbohydrates, e.g., glucose and lactos[e11](#page-13-0)–[14.](#page-13-0) In natural conditions, however, bacteria rarely encounter simple combinations of exploitable carbon/energy sources. Rather, complex mixtures of nutrients are common and often colonized by actively growing bacteria. This is the case, for example, of intracellular pathogens that are commonly faced with a diverse set of host nutrients in infected tissues. In these cases, bacteria have been shown to adapt to this situation by the simultaneous exploitation of plastic and flexibles nutrient utilization strategies[15](#page-13-0)–[19](#page-13-0).

Do the same models developed for simplified conditions hold also in real-case scenarios? At present, we witness a knowledge gap concerning the study of these processes in experimental settings that do not involve model organisms and/or defined media and we lack a sound theoretical understanding of the mechanisms driving nutrients assimilation strategies in conditions that are closer to the ones found in natural settings.

Bacterial exploitation of nutrient patches is made up of (at least) two different stages, i.e., physical interaction followed by carbon sources metabolic degradation. The capability of bacteria to interact with transient nutrient sources is well documented and has revealed their high efficiency in exploiting transient nutrient patches[20](#page-13-0),[21](#page-13-0). Little is known, instead, on the molecular aspects regulating and influencing bacterial productivity once micro-scale nutrient hot spots are colonized. At this stage, i.e., when cells start to feed on the available carbon source(s), other cellular mechanisms need to be involved to ensure a systematic exploitation of the resource. Indeed, as nutrient patches are likely composed of complex nutrient mixtures (that may include carbohydrates, amino acids, lipids and nucleic acids) bacteria need to dynamically activate specific degradation pathways according to the kind and concentration of external nutrients. In other words, a continuous and flexible genetic reprogramming needs to be active to ensure that the preferred compound(s) are sequentially or simultaneously taken up from the external environment and properly metabolized. Up to now, this latter aspect has been mostly overlooked despite it might be central in the understanding of micro-scale nutrients dynamics.

In this regard, the marine environment represents a paradigmatic example of the challenges encountered by microorganisms when it comes to the efficient (and rapid) exploitation of complex nutritional inputs. Such a habitat is thought to be characterized by a low average nutrient level (e.g., the concentration of amino acids is in the range of ∼10−9 M) and nutrients in general appear and disappear in a sporadic fashion, demanding a precise chemical response, a fast swimming speed, and ability to localize and exploit a nutrient patch once it is found[6](#page-13-0). These are the conditions that are commonly faced by marine heterotrophic bacteria, i.e., those microorganisms relying on the assimilation of external biomass for both energy generation and nutrition. Their metabolism is pivotal for the maintenance and the correct balance of oceanic biogeochemical cycles as they are central to the so-called microbial loop, i.e., the trophic pathway of the marine food web responsible for the microbial assimilation of dissolved organic matter, by transforming phytoplankton-derived organic matter and fuelling the entire ocean biogeochemical nutrient cycle. In this regard, the metabolism of Pseudoalteromonas haloplanktis TAC125 (PhTAC125), a heterotrophic marine bacterium isolated from Antarctica, has recently gained a certain attention due to its potential biotechnological exploitation[22](#page-13-0), its capability to synthesize anti-biofilm compounds[23](#page-13-0), the necessity to set up efficient culture conditions[24](#page-13-0),[25](#page-13-0) and the metabolic reprogramming during growth in complex environments[26.](#page-13-0) In particular, the analysis of its growth phenotype in an amino acid rich medium has shown the presence of metabolic switches among different groups of amino acids[25,](#page-13-0) although nothing could be said about the molecular mechanisms underlying such phenotype and the possible regulation involved. Using constraint-based metabolic modelling we attempted to provide a systems level scheme of PhTAC125 metabolic re-wiring as a consequence of carbon source switching in such a nutritionally complex medium. Our simulations highlighted an efficient reprogramming of PhTAC125 metabolic machinery to quickly adapt to a nutritionally unstable environment, compatible with adaptation to fast growth in a highly competitive environment[26.](#page-13-0)

Here we have investigated the global regulation of a marine heterotrophic bacterium when grown in both a complex and a defined rich medium (i.e., including multiple possible carbon sources) using and integrating a set of complementary-omics techniques (i.e., transcriptomics and 1H and 13C NMR metabolomics) with measured growth parameters. We show that the two main nutritional strategies commonly observed (co-utilization and sequential uptake of multiple substrates) can coexist in the same growth experiment, leading to an efficient exploitation of the available carbon sources. We also developed two theoretical models accounting for nutrients switching in a nutritionally rich environment in the presence and absence of cell regulation acting at the level of resource allocation in the synthesis of nutrient assimilation pathways. We show that a model taking into consideration an overall regulatory control on the sequence of nutrients uptake produces a better fit with available experimental data with respect to a purely Michaelis–Menten kinetic model.

### Results

Global regulation of a triauxic growth. P. haloplanktis TAC125 (hereinafter PhTAC125) cells were grown in shaken flasks in a complex medium composed of Schatz salt[s27](#page-14-0) and peptone as their C source. Optical density (OD) was measured every hour and cellular RNA was sampled in five different time points of their growth (Fig. [1a](#page-2-0)). To increase the time points for a better growth rate estimation, we also used an interpolation technique on the data generated at this stage. Details on the specific interpolation approach used are reported in "Methods" and the resulting plot is reported in Supplementary Fig. 1. The growth curve displays a triauxic pattern (Fig. [1a](#page-2-0)). An initial growth phase (growth rate of 0.023 h−1, 0.021 considering interpolated data) is interrupted by a lag phase between min. 180 and min. 240; afterwards, cells start growing over but such growth is interrupted by another lag phase between min. 280 and min. 340. Cells then started growing again, until the end of the experiment (growth rate of 0.004 h−1, 0.006 h−1 considering interpolated data). The average growth rate across all the time points was estimated to be 0.01 h−1. To identify transcriptional changes during cell growth total RNA was extracted and

<span id="page-2-0"></span>![](_page_2_Figure_2.jpeg)

Fig. 1 Transcriptomic data of the multi-auxic growth. a PhTAC125 growth curves in peptone. Asterisks represent hypothetical switch points and T1–T5 indicate the sampled time points. b Clustering of PhTAC125 genes according to their expression values (normalized according to clust algorithm, see "Methods") throughout the growth curve. c COG functional categories that are significantly over- ("+" sign) or under- ("−" sign) represented inside each cluster. Source data are provided as a Source Data file.

sequenced (two biological and two technical replicates for a total of 20 samples) using Illumina MiSeq (Genomix4Life, Naples, Italy). The main features of the 20 sequenced samples are reported in Supplementary Table 1. We clustered the genes according to their expression during the growth and identified six major trends (clusters C1–C6, Fig. 1b). Overall, we were able to cluster 2045 genes out of the 3448 encoded by the PhTAC125 genome (roughly 60%). We then performed a functional annotation and a functional enrichment analysis for the genes embedded in each cluster. One of these clusters (C6) did not include any significantly enriched functional category and thus it was discarded. Cluster C1 includes genes that display a decrease in their expression between the first two time points (T1 and T2) and a constant (low) expression across the rest of the growth curve.

Over-represented genes embedded in this cluster included those involved in basic housekeeping functions such as translation, DNA replication and transcription (Fig. 1c). Genes embedded in Cluster C2 displayed a decreasing trend throughout the growth curve and mainly included genes involved in RNA processing, metabolism of coenzymes and amino acids transport and metabolism. The expression of the 83 genes included in Cluster C3 was characterized by an abrupt increase between T1 and T2 and then an overall decrease until the end of the curve. This cluster significantly included genes involved in posttranslational modification, protein turnover and chaperons. Clusters C4 and C5 included genes whose expression tended to increase in the later stages of the growth; over-represented genes in C4 mainly belonged to lipid metabolism, cell motility and amino acids transport and metabolism. The expression of genes included in C5 decreased during the first stages of the growth and is then increased for the rest part of the curve. The cluster of genes included those involved in signal transduction mechanisms and cell motility.

Whole-genome transcriptomics data depict a scenario in which PhTAC125 is active and fast-growing mainly during the first stages of the curve, as reflected by the relatively high expression of translation, transcription replication and coenzyme metabolism genes. Genes embedded in these categories are under-represented among those increasing their expression in the last stages of the growth (Fig. 1c) and over-represented among those with high expression values in the first stages of the growth. Metabolically, PhTAC125 cells seem to rely more on amino acids metabolism in the initial stages of their growth, consistently with their progressive exhaustion in the medium. The last part of the growth experiment was also characterized by an increase in gene expression of cell motility-related genes (over-represented in C4 and C5). Finally, genes generally related to post-processing mechanisms peak their expression at T2.

#### A non-E. coli-like regulatory response to nutrients exhaustion.

The triauxic growth curve reported in Fig. 1a (and in Supplementary Fig. 1 using interpolated data) suggests the presence of a dynamic control on the adjustment of cell physiology. Here we sought to quantify the regulatory effort required to growing cells for modulating such cellular response. We focused on transcriptional factors (TFs) and two-component response systems (TCRSs) and analyzed differentially expressed genes among three points of PhTAC125 growth curve, namely T1 vs. T3 and T3 vs. T5. These points should capture PhTAC125 cells during exponential growth after the first growth lag (T1), in-between the two growth lags (T3) and after the final growth lag but before getting to plateau (T5).

First, we checked whether PhTAC125 regulation system somehow resembled the model scheme of the known overall metabolic regulation (i.e., the one characterized in E. coli). Of the 81 transcription factors known to directly or indirectly control central metabolic enzymes[28](#page-14-0), we found a reliable homologue (E-value < 1e−20) only for 34 of them (Supplementary Note 1 and Supplementary Data 1). PhTAC125, for example, lacks key players in bacterial diauxic shifts as the major global regulator of catabolite-sensitive operons (when complexed to cAMP) crp and the genes responsible for the synthesis of cyclic AMP (adenylate cyclase, cyaA). Among the 34 global regulators identified, only ten (roughly 25% of the shared ones and 12% of the entire E. coli set) displayed a significantly altered expression following the first transition (T1–T3) and none of them was differentially expressed following the second one (T3–T5). Details on the shared, differentially expressed TFs are provided in Supplementary Table 2.

<span id="page-3-0"></span>![](_page_3_Figure_2.jpeg)

Fig. 2 Differentially expressed genes. a–d Number of up- and downregulated genes in the two contrasts considered, for different gene categories. e Fold change of each gene responsible for the first degradative step of each amino acid. Differentially expressed genes are marked with an asterisk. Source data are provided as a Source Data file.

A similar situation was observed for eight selected sigmafactors that control gene expression globall[y28.](#page-14-0) In this case, as expected, an ortholog was found for each of them, but only two of them showed an altered expression following T1–T3 transition and the expression of none of them was significantly altered following T3–T5 one. The two genes displaying a significant change in gene expression were rpoS and rpoD. RpoS is the primary regulator of stationary phase genes, whereas RpoD is the primary sigma factor during exponential growth. Expectedly, the first resulted to be upregulated following the T1–T3 transition, whereas the second was downregulated (Supplementary Table 3).

With the exception of RpoS and RpoD, whose expression is in line with the global control of exponential vs. stationary phases, it appears that growth lags are regulated by mechanisms that poorly overlap with our current knowledge.

For this reason, we evaluated the expression of the entire repertoire of PhTAC125 TFs across the two points that involved the ceasing of cellular growth in our experiment, i.e., T1–T3 and T3–T5. Overall, we identified 41 differentially expressed TFs, 22 downregulated and 19 upregulated (Fig. 2b, Supplementary Fig. 2 and Supplementary Data 2) following the first growth interruption. The second growth lag was characterized by the significant change in expression (upregulation) of just one TF. Together with TFs, TCRSs are a basic stimulus-response coupling mechanism to sense and react to changes in environmental conditions, e.g., nutrient concentration. We identified differentially expressed TCRSs in the two selected contrasts. Overall, we found 21 TCRSsrelated genes that were differentially expressed in T1 vs. T3 and none in the T3 vs. T5 transition (Fig. 2c and Supplementary Data 3).

We conclude that the two growth lags observed in the curve apparently point to different reprogramming efforts that, in turn, may underpin distinct nutrients uptake strategies. The first growth interruption seems to have a deeper impact on the entire metabolic system, whereas the second could imply a fine tuning of the catabolic machinery. This is further confirmed by the overall number of DEGs across the selected contrasts (Fig. [2](#page-3-0)a), 1280 between T1 and T3 (633 and 647 up- and downregulated genes, respectively) and only 101 between T3 and T5.

Amino acid assimilation pathways and their (dis)regulation. Previous experiments have shown that PhTAC125 displays a coordinated sequence of amino acids degradation when grown in medium embedding complex mixtures of such molecules (i.e., peptone or casamino acid-based media, Supplementary Note 2)[25.](#page-13-0) In other words, some amino acids are preferred over others and are metabolized early in PhTAC125 growth curve. This switching among nutrients suggests that an active and modulated reprogramming occurs during PhTAC125 growth in a nutritionally complex environment. Similarly, differentially expressed amino acid metabolic genes (hereinafter AA-genes) are unevenly distributed among the two contrasts considered (Fig. [2](#page-3-0)d). T1 vs. T3 displays a higher number of DEGs (85, 33 downregulated and 55 upregulated) with respect to T3 vs. T5 (8, 5 downregulated and 3 upregulated). Considering the amino acid assimilation pathways of differentially expressed AA-genes in the T1 vs. T3 contrast (Supplementary Fig. 3), we did not observe a clear functional bias towards specific routes. Almost all the pathways are represented, both in terms of up- and downregulated genes. Similarly, the switch between T3 and T5 included downregulated genes involved in a broad spectrum of metabolic pathways including Val, Leu and Ile degradation, Tyr metabolism and Ala, Asp and Glu metabolism (one gene for each pathway) and upregulated genes in Gly, Ser and Thr metabolism (1 gene), Lys biosynthesis (1 gene) and Arg biosynthesis (3 genes).

To unravel the faith of each amino acid inside the cell, we analyzed the expression of the genes involved in all their possible first assimilatory step (Fig. [2](#page-3-0)e and Supplementary Data 4). We found 13 DEGs in the T1–T3 contrasts and 2 DEGs in the T3–T5 contrast. The first set comprised seven upregulated and six downregulated genes; upregulated genes were involved in Glu, Phe, Met, Gly Ile/Val/Leu, Asn/Asp and Ser degradation, whereas downregulated genes were responsible for the first assimilatory step of Met, Asp, Glu, Tyr, Ser and Ala. DEGs identified in the second contrast included genes involved in Glu and Gly degradation. Taking the DEGs indicated above as a proxy for the entire assimilatory process of the corresponding amino acids, we noticed a good overlap with available PhTAC125 physiological data[25.](#page-13-0)

DEGs analysis also allowed the identification of the major amino acid entry points into PhTAC125 metabolism. We counted, for example, 12 alternative possibilities steps to metabolize Asp in PhTAC125 (Fig. [2e](#page-3-0)), but the expression of only two of them (purA and asnB) appeared to be significantly modified during PhTAC125 growth. Similarly, six alternative steps can convert Glu to other cellular intermediates following its uptake. At T1, only two of these genes showed an altered expression level, suggesting that these may represent the most relevant players in Glu assimilation and usage. Nearly the same holds for Ser, with five distinct entry points and only two of them being differentially regulated.

Overall, we have identified possible key players, both in the switch among the set of metabolized amino acids, and in the entrance of amino acids into PhTAC125 entire metabolic network. However, nutrients switching requires an efficient genetic regulation to ensure that each catabolic pathway is active at the right moment, allowing a correct proteome allocation. For this reason, we analyzed the co-expression of genes belonging to the same metabolic pathway (Supplementary Note 3) and identified an overall dis-regulation of such genes (average Fisher's Z transformation average of Pearson correlation coefficient 0.49, Fig. [3](#page-5-0)a, b, Supplementary Fig. 4 and Supplementary Table 4). Focusing on the known regulons including AA-genes (Supplementary Table 5), we noticed that nearly half of them (three out of seven) displayed a relatively low (0.47, ArgR) or almost absent (0.26 and 0.11, MetJ and TyrR1, respectively) correlation among the expression values of the corresponding genes (Table [1](#page-6-0)). Figure [3c](#page-5-0)–e summarizes the details of the correlation existing among each gene of each pathway. In the case of ArgR regulon, for example, the major contribution to the low intra-regulon correlation is due to astA and astD (PSHAa0195 and 0196, respectively), showing an almost opposite expression pattern compared with the other ArgR regulated genes, especially with PSHAa2287-91 (Fig. [3](#page-5-0)C). astA and astD are involved in the conversion of Arg to Glu, whereas ArgHA, B, C, F, G (encoded by PSHAa2287-91, respectively) are involved in the synthesis of Arg from Glu through the formation of citrulline and fumarate.

Concerning MetJ regulon, we noticed a group of genes (including PSHAa2222, PSHAa2223, PSHAa0287 and PSHAa2292) whose expression values are negatively correlated with those of genes PSHAa2274-76 and PSHAa1226 (Fig. [3](#page-5-0)d). This group of coregulated genes include those involved in the conversion of homocysteine to Met (PSHAa2222 and PSHAa2223), an Lalanine-DL-glutamate epimerase (PSHAa0287) and a Methylthioribulose-1-phosphate dehydratase involved in the Met salvage pathway (PSHAa2292). Finally, as for TyrR1regulon, PSHAa2042-43, coding for 4a-hydroxytetrahydrobiopterin dehydratase and phenylalanine-4-hydroxylase are negatively correlated to the other genes in the same regulon (Fig. [3](#page-5-0)e). PSHAa2043 encodes phhA the gene responsible for the synthesis of Tyr from Phe, whereas PSHAa2042 (phhB) encodes a Pterin-4-alphacarbinolamine dehydratase responsible for the conversion of 4ahydroxytetrahydrobiopterin to dihydrobiopterin.

Upstream of most of the genes belonging to the three regulons considered, we were able to identify a conserved motif for each regulon (Fig. [3](#page-5-0)f–h), partially overlapping with their known TF binding site. Finally, a closer inspection to the metabolic steps encoded by the differentially regulated genes of these regulons revealed that they usually belong to different and symmetric regions of the same metabolic pathway. PSHAa0195 and PSHAa0196 respectively encodes for astA and astB, responsible for the first steps of the route leading to the formation of Glu from Arg. The other genes of the ArgR regulon are mostly involved in the production of Arg starting from Glu (Fig. [3i](#page-5-0)). Similarly, phhA (encoded by PSHAa2043) is involved in the formation of Tyr (from Phe), whereas all the other genes are responsible for the formation of Tyr from a set of different precursors (e.g., prephenate) (Fig. [3](#page-5-0)k). In the case of MetR regulon, among the genes that could be reliably assigned to the methionine metabolic pathway, one of the two group of coregulated genes belong to the upper part of the pathway (upstream the main product methionine), whereas members the other one are in its close proximity (PSHAa2222) or involved in the methionine salvage pathway (PSHAa2292), the set of reactions responsible for the recycle of the thiomethyl group of S-adenosylmethionine from methylthioadenosine (Fig. [3j](#page-5-0)).

In a previous wor[k26](#page-13-0), we have simulated the growth of PhTAC125 in a nutritionally complex environment (peptone) and derived the overall metabolic reprogramming occurring during growth in a rich undefined medium using constraintbased metabolic modelling (i.e., flux balance analysis, FBA) (Supplementary Note 4). We observed a good correlation between the measured changes in the expression of AA gene regulons and the predicted changes in metabolic fluxes of their encoded reactions using FBA (Pearson's product-moment correlation = 0.89, p value = 0.019). See Supplementary Table 6 for further details.

<span id="page-5-0"></span>![](_page_5_Figure_2.jpeg)

Fig. 3 Deregulation of amino acid metabolic pathways. a Cumulative density plot of Fisher's Z transformation average of Pearson correlation expressing the co-regulation of genes belonging to the same pathway. b Log2-transformed TPM values of the genes belonging to the same regulon. c–e Graphic representation of the Pearson correlation matrix for the genes belonging to the regulons showing the lowest average correlation, Arginine metabolism/ biosynthesis, Methionine biosynthesis and Aromatic amino acids metabolism. Locus tags in bold indicate those of the regulator of the corresponding regulon. f–h Show the conserved upstream motif found for each gene of the considered regulon. The bar above each conserved motif indicates the overlap existing with the known TF binding site according to the RegPrecise database. Finally, in (i–k) the reactions encoded by the genes included in the same regulon are schematically represented. Red and blue lines schematically represent the correlation coefficient of Fig. 3c, d. Source data are provided as a Source Data file.

*PSHAa2922*

Taken together these results suggest that (i) the two growth lags observed (Fig. [1](#page-2-0)a) may be the same phenotypic representation of two different cellular states (i.e., assimilation strategies) and that (ii) a rather complex genetic regulation is at work to ensure a correct decision-making process in nutritionally dynamic environments. In the next sections these two aspects will be elucidated using controlled growth conditions and a combination of NMR experiments and theoretical modelling.

Combination of simultaneous and sequential amino acids uptake. Up to now, we have analyzed the behaviour of bacterial cells in a complex medium, using gene expression as a proxy for amino acids assimilation pathways. The medium used (peptone) is a complex mixture of nutrients whose exact composition is unknown.

Accordingly, it is not possible to conclude that the observed growth features (i.e., triauxic growth) are due to the exhaustion of certain preferred amino acids in the medium. For this reason, we assembled a medium including 19 amino acids, 0.2 mM each (named 19 AA medium, cysteine was not included in the list because of difficulties in its unambiguous quantitation during the experiments due to its spontaneous oxidation, as also reported in ref. [29](#page-14-0)) and determined the kinetics of their usage during PhTAC125 growth by analyzing the growing media using 1H NMR (Supplementary Note 5 and Supplementary Fig. 5). Data obtained revealed that an important fraction of all the provided amino acids (16 out of 19) are consumed in the first 7 h of the growth. Afterwards, the remaining three amino acids (His, Met and Trp) are (slowly) metabolized (Fig. [4](#page-6-0)a). Clustering the amino acids assimilation profiles allowed a clearer visualization of the order in which amino acids are used by PhTAC125 during its <span id="page-6-0"></span>growth (Fig. 4b). This analysis divided the set of metabolized compounds into four main, non-overlapping clusters. Gln, Glu and Arg are the first amino acids to be consumed in the medium. Their concentration reaches (negligible) values close to 0.01 mM after 4.5 h of growth, remaining constant afterwards. The second set of amino acids is composed of Asn, Asp, Leu and Pro. Their degradation starts with a small delay with respect to the one of the first cluster, and they are completely removed from the

| Table 1 Fisher's Z transformation average of Pearson |  |  |  |  |  |  |
|------------------------------------------------------|--|--|--|--|--|--|
| correlation of the same-regulon genes.               |  |  |  |  |  |  |

| Regulon | Process                                  | Fisher's Z transformation<br>of Pearson correlation<br>coefficient |
|---------|------------------------------------------|--------------------------------------------------------------------|
| HutC    | His degradation                          | 2.55                                                               |
| HmgS    | Tyr degradation                          | 1.92                                                               |
| MetR    | Met biosynthesis                         | 1.22                                                               |
| LiuR    | Branched-chain amino<br>acid degradation | 0.97                                                               |
| ArgR    | Arg biosynthesis/<br>degradation         | 0.47                                                               |
| MetJ    | Met metabolism, Met<br>degradation       | 0.26                                                               |
| TyrR1   | Aromatic amino acids<br>metabolism       | 0.11                                                               |

medium only between 6 and 6.5 h. The third cluster includes nine amino acids (Fig. 4b). Their consumption is rather slow in the first 3 h of growth; afterwards, it accelerates leading to negligible concentration of the corresponding amino acids at 7.5 h. The concentration of amino acids belonging to the fourth cluster remains overall constant for the first 6 h of growth. After that moment, corresponding to the point in which all the other amino acids are consumed, it starts decreasing. Importantly, this pattern of amino acids assimilation results in a triauxic growth curve (Fig. 4c). Indeed, (short) growth lag phases are observed after 4 and 6 h of growth, in correspondence with the major transition in amino acids assimilation pattern.

Overall, this behaviour highlights a balanced mix between simultaneous and sequential uptake of nutrients. Amino acids belonging to the same group (Fig. 4d) are simultaneously metabolized by the cells but the assimilation of different groups occurs with different dynamics and is responsible for growth lags in the curve. Finally, a typical diauxic nutrient shift is observed when all the main (preferred) sources are exhausted and the degradation of the other (previously ignored) compounds begins. Available growth phenotypes[24,](#page-13-0)[30](#page-14-0) (Supplementary Note 6) seem to suggest that the order in which nutrients are used by PhTaC125 during the growth depends both on the final biomass and on the specific growth rate achievable when grown with amino acids as sole carbon sources (Supplementary Fig. 6A, B). Also, the order of amino acids uptake, partially reflects their entry point into the TCA cycle (Supplementary Fig. 7).

![](_page_6_Figure_7.jpeg)

Fig. 4 Amino acids assimilation profiles. a Degradation dynamics for each of the 19 amino acids included in the defined AA medium. b Clustering of timeresolved concentration values for the 19 amino acids analyzed. Error bars represent SD of two different cell cultures in two independent experiments. c Growth curve of the 19 amino acids experiment. Asterisks indicate growth lags. d Degradation dynamics for each of the four identified clusters of amino acids included in the defined AA medium. Colour codes as in b. Grey shaded area includes the 95% confidence of the linear regression (coloured) line over the concentrations of the amino acids belonging to the same group. Source data are provided as a Source Data file.

Despite the trend seems to be quite clear, a certain variability was observed in the results shown in Supplementary Fig. 6A, B. This is the case, for example, of Leu in cluster C2 that is preferred to Ala (C3) despite the growth rate of PhTAC125 growth on Leu is about twofold lower than that on Ala. This might be accounted for by promiscuous uptake of nutrients. Indeed, the broad spectrum of some amino acids transporters is well described in bacteria[31,32](#page-14-0). Thus, some amino acids might be taken up from the medium not as the result of an active cellular control over the most efficient carbon sources, but as the result of the broadspectrum activity of a membrane transporter.

An additional explanation to the composition of the clusters might involve the overlap among the catabolic pathways of the substrate that are co-consumed (i.e., belonging to the same cluster, Supplementary Note 7). Indeed, we noticed that C1 amino acids (Arg, Glu and Gln) share part of their catabolic pathways and are all converted to alpha-ketoglutarate before entering the TCA cycle. Two (out of four, i.e., Asp and Asn) amino acids of cluster C2 are converted to oxaloacetate before entering the TCA cycle. Most of the C3 amino acids (six out of nine, i.e., Ala, Ser, Gly, Thr, Phe and Tyr) are converted to pyruvate, thus not being directly catabolized into one of the TCA cycle intermediates. Other three C3 amino acids (Thr, Ile and Val) can be catabolized to form pyruvate and a TCA intermediate (i.e., Succinyl-CoA). Finally, C4 includes those amino acids that are negligibly used during PhTAC125 growth, so no degradation pathways overlap is required to explain their inclusion in the same group (Supplementary Fig. 7).

In order to check whether the sequential or co-consumption of substrates also depends on their own concentrations, we performed a set of additional experiments aimed at evaluating the effect of higher concentrations of nutrients on the pattern of nutrients assimilation (Supplementary Note 8). We both tested the effects of (i) providing all the 19 amino acids at concentrations five and ten times higher (1 and 2 mM, respectively) than those used in the 19 AA experiment described above and (ii) increasing the concentration of late-metabolized amino acids (His, Met, Trp, 2 mM) on the timing of the last metabolic switch observed during PhTAC125 growth. The results of these analyses are reported in (Supplementary Figs. 8 and 9) and revealed that metabolic phenotype identified in the original 19 AA experiment is poorly affected by the concentration of the available carbon sources. Also, we compared the phenotype observed in PhTAC125 with the one of a model organism (E. coli, Supplementary Note 9), when grown in the same nutritional environment (19 AA medium, 0.2 mM each). A poor overlap was observed between the specific response to nutrients switching of the two microorganisms (Supplementary Figs. 10 and 11).

The different fate of the catabolized amino acids. Next, we investigated the different fate of amino acids belonging to the same cluster once entered inside PhTAC125 cells. To this aim, we used uniformly labelled 13C amino acids (Glu, Asp, Ala and Met, belonging to C1, C2, C3 and C4, respectively) and followed the path of their labelled carbon atoms inside the cells by NMR. More in detail, for each labelled amino acid used, we prepared four different parallel cultures, each containing 18 amino acids plus the 13C labelled one as the only carbon source for PhTAC125. These four cultures were run in parallel and each of them was sacrificed at a different time point. Specifically, we analyzed four time points, i.e., early and late exponential growth (3 and 6 h) and early and late stationary phase (8 h and 30 min, 24 h). For each time point we analyzed both growing media and the cell lysates by acquiring mono-dimensional 1H NMR spectra and bidimensional 1H-13C HSQC NMR spectra. A scheme of the structure of this experiment is reported in Supplementary Fig. 12. This experiment allowed us to study the metabolic fluxes in a timedependent fashion and provided hints on the fate of the metabolized amino acids inside the cell (Fig. [5](#page-8-0)).

First, by using 1H NMR spectra of growing media, we confirmed the same growth dynamics for all the replicates of the experiment and the same overall growth features observed in the unlabelled 19 AA experiment (compare Fig. [5](#page-8-0)a, c). Similarly, amino acids were consumed in the same order and with the same overall rates previously observed (compare Figs. [5b](#page-8-0) and [4a](#page-6-0)).

A principal component analysis (PCA) performed on the 1H spectra of cell lysates clustered the samples according to their sampling time, thus confirming the consistency and the high reproducibility among the different replicates (Supplementary Fig. 13), besides the occurrence of significantly different metabolic profiles of PhTAC125 cells along the four sampled time points.

1H-13C NMR spectra of cell lysates were used to have an overview of the fate of the metabolized amino acids inside the cells. Each of the four selected amino acids showed a specific path of labelled carbon atoms inside the cells, as revealed by the PCA reported in Fig. [4c](#page-6-0) (1H-13C HSQC NMR spectra). While the spectra of Asp and Glu almost overlap, the spectra of Ala and Met are very well separated. This trend probably reflects the fact that both Glu and Asp are used to directly feed the TCA cycle, whereas Met and Ala are used to feed different pathways inside the cell (see below).

In particular, we were able to show that Ala is mostly used to feed many important pathways inside the cell, namely glycolysis/ gluconeogenesis, nucleotide precursors metabolism and (partly) TCA cycle. At T3 we found 13C coming from Ala degradation in most of the key compounds that are the input/output of the aforementioned pathways, i.e., pyruvate, phosphoenolpyruvate (PEP), fatty acids and AXP/GXP (Fig. [5](#page-8-0)d, Supplementary Figs. 14 and 15). Oxaloacetate is the only TCA cycle intermediate that displays Ala labelling at T3 (Fig. [5d](#page-8-0) and Supplementary Fig. 16). At T3, we found Asp-derived 13C labelling on all the TCA cycle intermediates identified and on purine biosynthesis intermediates, AXP and GXP (Fig. [5](#page-8-0)d, Supplementary Figs. 15 and 16). At later growth stages Asp labelling appeared also on PEP and NAD precursors (Fig. [5](#page-8-0)d, Supplementary Figs. 14 and 17). Glu-derived 13C labelling appears in all the TCA intermediates analyzed (with the exception of oxaloacetate) starting from T3 (Fig. [5](#page-8-0)d and Supplementary Fig. 16), suggesting that Glu is readily redirected towards the TCA upon its uptake. Interestingly, Glu seems also to be used as a substrate for NAD (and precursors) biosynthesis from the early stage of growth (Fig. [5D](#page-8-0) and Supplementary Fig. 17). Finally, this experiment confirmed that Met is incorporated into the PhTAC125 metabolic network at a later stage of its growth, in that no identified compounds was labelled with 13C of Met at T3. Met labelled 13C appear to be included into homocysteine starting from T6. After 24 h, we found Met labelled 13C on a compound whose NMR pattern corresponds to that of trimethylamine (Fig. [5d](#page-8-0) and Supplementary Fig. 18). Interestingly, in a previous wor[k33](#page-14-0), we had characterized PhTAC125 as a methylamine producer and also showed that adding Met to the growth medium was pivotal to allow PhTAC125 to produce this compound (and inhibit the growth of human opportunistic pathogens). The metabolic flux analysis performed here suggests a link between the production of methylamine from the degradation of Met, possibly through the formation of trimethylamine.

Overall, using 13C-labelled precursors provided evidence that the different amino acids used by PhTAC125 during its growth have different and complementary roles within the cell. Ala is readily converted into pyruvate and this is used to feed both gluconeogenic pathways and (possibly through the formation of

<span id="page-8-0"></span>![](_page_8_Figure_2.jpeg)

Fig. 5 The fate of catabolized amino acids. a PhTAC125 growth curves across all replicates in the 19 AA medium with one of them being 13C labelled. Each colour indicates which was the 13C-labelled amino acids for each replicate. Error bars represent SD of four different cell cultures. b Extracellular concentration of all the amino acids in time. Colour codes as in a. c PCA on the 13C spectra. Colour codes as in a. Grey shaded area includes the 95% confidence of the linear regression (coloured) line over the concentrations of the amino acids belonging to the same group. d Central PhTAC125 metabolic network, highlighting the possible entry points of the labelled amino acids. Circles close to compounds indicate the 13C signal recovered at the different time points (T3–T24) and coming from different amino acids (colour codes as in a) corresponding. Source data are provided as a Source Data file.

acetyl-CoA) fatty acids synthesis. Only at later growth stages, TCA intermediates started to display Ala-derived labelling. Conversely, Glu is immediately used to feed the TCA cycle as all its intermediates but one (oxaloacetate) carry Glu labelling at T3. No other compounds in the upper part of Fig. 5d displayed Glu-derived 13C atoms, thus suggesting a clear separation between Ala and Glu assimilation pathways. The primary assimilation pathway of Asp seems to be through the TCA cycle, although part of its carbon skeleton is diverted to PEP formation (and possibly to gluconeogenesis) at T6. Quite interestingly, 13C atoms derived from Asp degradation are observed on purine metabolism product AXP and GXP from the first sampled time point to the last, despite the gluconeogenic metabolism precursor PEP does not display the same signal. We thus argue that the origin of Asp-derived labelling on AXP and GXP might be due to TCA intermediates playing a role in purine metabolism, e.g., 5 phosphoribosylamine, Finally, we confirmed the late entrance of Met into PhTAC125 metabolic network, being initially converted to homocysteine and then also rerouted towards the production of trimethylamine (at T24), with a possible but still undisclosed pathway involving the formation of betaine or carnitine[34](#page-14-0)–[36](#page-14-0).

To further characterize the metabolic response of PhTAC125 to nutrients consumption, the whole intracellular metabolome was evaluated through untargeted 1H NMR across four time points, i.e., early and late exponential growth (3 and 6 h) and early and late stationary phase (8 h and 30 min, 24 h). In the spectra, the signals of 26 metabolites could be unambiguously assigned and quantified (Supplementary Fig. 19 and 20). These metabolites were representative of six major metabolic modules, i.e., amino acids metabolism, purine and pyrimidine metabolism, sugars, amino sugars metabolism, nucleotide precursors and TCA. The trend of metabolites' concentration in time revealed that the relative concentration of purine and pyrimidines intermediates (namely uridine and inosine) decreases in time. A similar trend is observed for ribose and for the detected TCA intermediate (fumarate). This scenario is compatible with the decrease in growth rate and overall biomass production of PhTAC125 along the growth curve (Fig. [4c](#page-6-0)) and with a consequent decrease of DNA synthesis activity (purine and pyrimidine precursors) and energy demand (TCA cycle). Conversely, sugars (e.g., glucose) and amino sugars (e.g., UDP-NAG) increase their cellular concentration in the latest growth stages, reaching values that are up to three times the initial ones. This latter finding can be interpreted in the light of two considerations: first, we have already shown that carbohydrates metabolic genes increase their expression at the final stages of PhTAC125 growth on peptones, suggesting the activation of sugar metabolism-related pathways at later growth stages. Second, the increase of sugar/amino sugars intracellular concentrations might have a role in PhTAC125 cell aggregation when nutrients concentration starts to deplete given that: (i) such metabolites are known to be involved in cell–cell contact[s37](#page-14-0), (ii) PhTAC125 was shown to produce a biofilm that incorporates amino sugars and (iii) this is supposed to be a strategy to survive in poor nutrient condition[s37](#page-14-0). Our findings are in line with a scenario in which the presence of a lower availability of nutrients induces a greater production of biofilm since the biofilm matrix can improve the capture of nutrient[s38](#page-14-0).

Modelling simultaneous and sequential amino acids uptake. At least two different explanations may account for the mixed sequential/diauxic nutrient uptake. Either this phenotype is "simply" determined by different uptake kinetics of the different compounds or the assimilation pattern is actively regulated by the cells. To discern between these two scenarios, we implemented two mathematical models accounting for cell growth and nutrients uptake during the 19 AA experiment. To reduce the complexity of the problem, the 19 amino acids were lumped into the four corresponding clusters shown in Fig. [4](#page-6-0)b, d. In this way we modelled the growth of PhTAC125 in a hypothetical growth medium embedding four different groups of carbon sources ideally representing the 19 AA medium. The first model is based on the Michaelis–Menten–Monod kinetics (MMM model) and is formulated as follows:

$$P + \text{S}_i \xrightarrow{r} \text{2P},\tag{1}$$

where P represents bacterial cells, Si (with i = 1, 2, 3, 4) represents each of the four groups of pooled C sources and r the rate at which the reaction occurs. Specifically, r was modelled according to a canonical (Michaelis–Menten-derived) Monod kinetics with:

$$r = \frac{\mathcal{S}_i \cdot \phi \mathcal{S}_i}{k_i + \phi \mathcal{S}_i},\tag{2}$$

where βi , ϕSi and ki , represent the maximum rate constant for cell production, the concentration and the Michaelis–Menten constant for the ith group of amino acids, respectively. According to these formulas, the state variables model can be written as:

$$\frac{d\phi P}{dt} = \sum_{i=1}^{4} \frac{\beta_i \cdot \phi S_i^2}{k_i + \phi S_i} \cdot \phi P - d \cdot \phi P,\tag{3}$$

$$\frac{d\phi \mathbf{S}_i}{dt} = -\frac{\beta_i \cdot \phi \mathbf{S}_i^2}{k_i + \phi \mathbf{S}_i} \cdot \phi P,\tag{4}$$

where Si (with i = 1, 2, 3, 4) represents each of the four lumped substrates and d the bacterial cells death rate.

The second model implemented here accounts for the effect of the regulatory processes of catabolite inhibition and activation that can be observed during microbial growth on multiple substrates. The model is a modified version of the cybernetic model proposed by ref. [39](#page-14-0), overall resembling the one proposed in ref. [40](#page-14-0).

The cybernetic modelling framework takes into account the (yet) unknown regulatory processes regulating the microorganisms' uptake kinetics. It assumes that microorganisms have evolved under the selective pressure to become optimal with respect to certain cellular objectives (in our case, maximization of biomass production) and achieve this task by actively modulating the induction/repression and activation/inhibition of the key enzymes of substrates available in their external environment. Cybernetic variables (see below) are introduced in the model to account for the induction/repression and activation/inhibition of the key, bottleneck enzymes regulating cell growth, substrate consumption, and key enzyme production[41,42.](#page-14-0)

According to this model, the assimilation of substrate Si by cells P (Eq. (1)) is assumed to be catalyzed by the set of enzymes Ei (with i = 1, 2, 3, 4). The assumption here is that enzymes responsible for the assimilatory pathway of each pool of nutrients are induced by the presence of Si (and repressed by the presence of the other nutrients). This alternative model can be written as:

$$P \xrightarrow{r_l \nu_l} \mathbf{2P},\tag{5}$$

$$
\varnothing \xrightarrow{r_{\mathbb{E}_l} u_l} E_i,\tag{6}
$$

where Ei represents the key assimilatory enzyme. The rate equations for biomass production (Eq. (5)) and for enzyme synthesis (Eq. (6)) can be written as a modified form of Monod's equation and are respectively expressed as follows:

$$r_i = V_{\,\,max,i} \cdot \phi E_i \cdot \frac{\phi S_i}{K_{S_i} + \phi S_i},\tag{7}$$

$$r_{E_i} = \,^\cdot V_i \cdot \frac{\phi \mathcal{S}_i}{\mathcal{K}_{E_i} + \phi \mathcal{S}_i},\tag{8}$$

where ϕEi represents enzyme concentration, VEi is the maximum rate constant for enzyme's biosynthesis and Vmax;i is the is the maximum rate constant for bacterial production P on the ith substrate. KSi and KEi are the Michaelis–Menten constant for the ith substrate and the synthesis of the ith enzyme, respectively.

The inhibition/activation effect due to the concentration of the different substrates is accounted for by two (control) variables, ui and vi, representing the fractional allocation of resource for the synthesis of Ei and the mechanism of controlling enzymes Ei activity, respectively. ui is expressed as:

$$
\mu_i = \frac{r_i}{\sum_{j=1}^4 r_j},
\tag{9}
$$

with 0 ≤ ui ≤ 1 and P4 j¼1 ui ¼ 1. The other control parameter, vi is expressed as:

$$\nu_i = \frac{r_i}{\max\{r_1, r_2, r_3, r_4\}},\tag{10}$$

where the denominator accounts for the observation that priority is given to the consumption of the substrate(s) that guarantee the highest growth rate[43.](#page-14-0) The model further considers constitutive enzyme production rate (βi), the effect of dilution of the specific enzyme level due to cell growth (α), constant protein decay in the cells and bacterial death rate (d) and can be written as follows:

$$\frac{d\phi \mathcal{S}_i}{dt} = -r_i \cdot \nu_i \cdot \phi P,\tag{11}$$

$$\frac{d\phi P}{dt} = \sum_{j=1}^{4} \nu_j \cdot r_j \cdot \phi P - d \cdot \phi P,\tag{12}$$

$$\frac{d\phi E_i}{dt} = \mu_i \cdot r_{E_i} - \sum_{j=1}^4 \nu_j \cdot r_j \cdot \phi P - \alpha \cdot \phi E_i + \beta_i. \tag{13}$$

The model parameters were determined by fitting the experimental data (shown in Fig. [4](#page-6-0)a–d) with model simulations and their values are reported in Supplementary Tables 7 and 8. As shown in Fig. 6, the cybernetic model accurately reproduces the dynamics of all the species considered. This is even clearer in the case of nutrient concentration dynamics where the model implementing the MMM model, is not capable of producing a satisfactory approximation of the real data. Indeed, R2 calculation indicates that the cybernetic modelling framework is performing better on four out of five of the species included in the model (Table 2). Likewise, when using the AIC metric, the comparison between cybernetic model and MMM was shown to follow the same trend.

These results suggest that, in the conditions tested, the uptake of nutrients is tightly regulated, leading to the simultaneous presence of diauxic and co-utilization strategies within the same growth curve. Hints on the main catabolic players involved in such assimilation patterns were obtained combining transcriptomic data from the complex-medium experiment and RT-PCR on specific targets (see Supplementary Note 10, Supplementary Fig. 21 and Supplementary Table 9).

#### Discussion

Our knowledge on the possible bacterial strategies for nutrients assimilation when multiple sources are available is biased by the fact that it has been mainly studied in a few model organisms, providing them with a reduced number of possible inputs (compared with those available in their source environment). Here, we have characterized a non-model response to nutrients switching and studied the process of bacterial nutrients uptake in experimental conditions that more closely resemble a natural setting, in terms of the availability of many different substrates simultaneously. Using a marine heterotrophic bacterium (P. haloplanktis TAC125) as a case study strain, we have shown that its response during growth lags do not resemble the one currently known for E. coli. Only 10% of the E. coli metabolic regulators and two (out of eight) main generic controllers (rpoS and rpoD) displayed an altered expression level in our experiments. Also, we showed that when the two microorganisms were independently cultivated in the same defined medium embedding 19 different amino acids, differences arose in the choice of the amino acids to utilize, in the timing of uptake and in the presence/absence of overall growth lags.

The poor overlap between PhTAC1215 and E. coli transcriptional response suggests that, in marine bacteria, the response to nutritional switches and/or multi-auxic growth patterns may involve still untapped genetic circuits. As a matter of fact, PhTAC125 is known to lack the CCR syste[m44,](#page-14-0) which is currently referred to as the main driver in metabolic switches and diauxic phenotypes. Using time-resolved transcriptomics we have shown that growth lags in a nutritionally complex environment are probably due to the exhaustion of specific carbon sources and that such event has a deep impact also on other important gene categories including, for example, motility-related genes. In the second part of the curve, i.e., when nutrient concentration decreases, cell motility genes increase their expression, probably reflecting the need to explore the surrounding environment for other potential sources. This is in line with the observation that many bacteria become motile when nutrients are scarce[45.](#page-14-0) Moreover, among the genes peaking their expression in correspondence of the first growth lag, those involved in posttranslational modification, protein turnover and chaperon are

![](_page_10_Figure_8.jpeg)

Fig. 6 Modelling the multi-auxic growth. Simulation outcomes for the two models implemented here for biomass (a) and lumped nutrients (b–e). Dashed lines represent the prediction for the MMM model. Solid lines represent the prediction for the cybernetic model. Open circles represent experimental data (same data shown in Fig. [4](#page-6-0)d). Error bars represent SD of two different cell cultures in two independent experiments.

| Table 2 R2 and<br>AIC calculation for the different simulations using the two different models. |           |           |           |           |         |  |  |
|-------------------------------------------------------------------------------------------------|-----------|-----------|-----------|-----------|---------|--|--|
| Model                                                                                           | Cluster A | Cluster B | Cluster C | Cluster D | Biomass |  |  |
| Pearson—MMM model                                                                               | 0.9844    | 0.9922    | 0.9584    | 0.9584    | 0.9959  |  |  |
| Pearson—Cybernetic                                                                              | 0.9920    | 0.9973    | 0.9953    | 0.9193    | 0.9967  |  |  |
| AIC—MMM model                                                                                   | −408.53   | −418.46   | −338.17   | −476.82   | −378.46 |  |  |
| AIC—Cybernetic                                                                                  | −439.48   | −475.38   | −438.75   | −395.97   | −372.37 |  |  |

NATURE COMMUNICATIONS | (2020) 11:3135 | https://doi.org/10.1038/s41467-020-16872-8 | www.nature.com/n[aturecommunications](www.nature.com/naturecommunications) [11](www.nature.com/naturecommunications)

significantly over-represented. Generally speaking, proteins belonging to this functional category can be easily associated to the stress encountered by an exponentially growing batch culture that exhaust (part of) the readily available and preferred nutrients in the growth medium. At this moment cells undergo a regulated transition into stasis by activating a stereotypic stress response. Post-translational modifications have been shown to play a role in the starvation‐induced growth arrest, for example in S. coelicolor[28,46](#page-14-0). It is to be noticed that these proteins do not increase their expression in the later stages of the growth and, in particular, in the second growth arrest experienced by PhTAC125. This, together with the observation that both the overall number of DEGs and of those involved in many other functional processes (amino acids degradation, TFs and TCRS) is much higher in correspondence of the first time contrast, suggested that the same phenotype (the two growth lags) were mirrored by profoundly different cellular reprogramming patterns. This prompted us to investigate more in depth the regulation of amino acid assimilation pathways. This analysis highlighted the tight regulation required to efficiently exploit complex and amino acid enriched, nutritional conditions. A paradigmatic example of this capability is the (dis)regulation of several genes belonging to the same regulon (ArgR, TyrR1 and MetR), responsible for the activity of distinct and conflicting functional metabolic modules inside the same metabolic pathway. In principle, all the genes belonging to the same regulon are under the control the same TF. It is known that some TFs may function as either activators or repressors, often according to the positioning of the TF binding site in the target promoter, although this feature had not been described, to date, for amino acid assimilation pathways[47.](#page-14-0) Apparently, such a mechanism is at work in some of the amino acid catabolic pathways of PhTAC125, probably ensuring an efficient and correct exploitation of the amino acid mixture available in the surrounding environment.

Despite being closer to the environmental natural setting, growth in complex medium does not allow a precise understanding of the usage of all the available carbon sources. For this reason, we have assembled a defined but nutritionally rich medium containing 19 amino acids as the sole carbon sources for the bacterial cells. Tracking their concentration in time, we showed that the two main feeding strategies commonly thought to be exclusive to each other (i.e., sequential and simultaneous uptake of nutrients) can coexist in the same growth curve. Clustering metabolized amino acids into four major groups revealed that amino acids belonging to the same group are coutilized, whereas the switch among the different clusters is tightly modulated. A "canonical" diauxic shift is finally observed at the end of the growth, when the consumption of a set of previously untapped nutrients begins. Thus, similar to the complex-medium growth experiment (Fig. [1](#page-2-0)), the two growth lags apparently underlay different cellular states. Indeed, despite the exhaustion of nutrients is common to both growth lags, in one case (the first lag) cells are already metabolizing alternative compounds when the exhaustion of the preferred sources occurs.

The order in which nutrients are utilized can be partially explained by the biomass yield and growth rates obtained when each single amino acid is provided as single carbon sources. Those amino acids allowing the highest growth rate and biomass production are those that are consumed first in the 19 AA experiment.

Using 13C-labelled amino acids and time-resolved NMR spectra, we were able to follow the degradation pathways of four selected amino acids (Glu, Ala, Asp and Met) and derive their fate inside the cell. We showed that Ala is readily converted into pyruvate and then probably used to fuel sugar metabolism (i.e., gluconeogenesis and pentose phosphate pathway), leading to the production of ribonucleotides. Asp and Glu are instead promptly used to fuel the TCA cycle, with two notable exceptions, i.e., the entrance of Glu-derived carbons into the biosynthesis of NAD precursors and the conversion of (part of) the initial amount of Asp into PEP and its usage for ribonucleotide biosynthesis. The fate of Met inside the metabolic network remained hard to decipher, and the actual contribution to the growth of PhTAc125 will require further investigation. Further, results coming from the untargeted evaluation of the overall intracellular metabolome is in line with an increased importance of sugar metabolism/ intermediates upon exhaustion of the available amino acids in the medium, probably reflected in the characterized production of biofilm in poor nutrients conditions.

Finally, we have shown that the dynamics of nutrients degradation can be explained using a theoretical model that accounts for gene regulation and, in general, for the proper resource allocation for the synthesis of the main assimilatory pathways. This modelling framework can accurately interpret the pattern of nutrients degradation in a nutritionally rich environment.

In conclusion, we would like to stress the importance of cultivating and studying microorganisms in nutritional conditions that more closely resemble the ones most found in nature, for example for what concerns the contemporary availability of many distinct possible carbon sources as done here. By doing so and using a combination of computational and experimental (transcriptomics and NMR-based metabolomics) approaches, we have shown that, despite diauxie and co-utilization strategies have been usually thought as conflicting phenotypes, they can coexist in the same growth curve and give rise to a diversified ensemble of feeding strategies.

The use of different sources depending on the phase of cell growth and, most of all, a distinct metabolic fate inside the cell for each of the metabolized compounds, is a common feature of intracellular bacteria (e.g., Legionella pneumophila, Listeria monocytogenes or Coxiella burnetii[17](#page-13-0),[19](#page-13-0),[48](#page-14-0)–[50](#page-14-0)) suggesting that plastic strategies for carbon assimilation might be evolved in response to nutritionally poor and highly variable conditions.

It will be interesting to investigate which are the molecular mechanisms allowing the implementation of this mixed and apparently unconventional feeding strategy and, in particular, the fine-tuned regulatory circuits that are probably responsible for the efficient switching among all the available carbon sources. Future efforts will be also devoted to understanding the effect of fluctuations (in the number of cells and/or in nutrients concentration) and of possible population heterogenicity[10](#page-13-0),[11](#page-13-0) on the resulting growth dynamics of heterotrophic marine bacteria.

#### Methods

Bacterial strain, media and growth condition. P. haloplanktis TAC125[51](#page-14-0) cells were routinely grown in Marine Agar (MA) or Broth (MB) (Condalab, Spain) under aerobic condition at 21 °C. The stock suspension of the strain was stored in 20% [v/v] glycerol solution at −80 °C. For growth curves experiment, Schatz salts[52](#page-14-0) (1 g/l KH2PO4, 1 g/l NH4NO3, 10 g/l NaCl, 0.2 g/l MgSO4 × 7H2O, 0.01 g/l FeSO4 × 7H2O, 0.01 g/l CaCl2 × 2H2O) were supplemented with 5 g/l Peptone N-Z-Soy BL 7 (Sigma-Aldrich S.r.l) (complex medium) or with 19 amino acids (19 AA medium) each one at a final concentration of 0.2 mM (cysteine was not included due to its rapid oxidation to cystin[29](#page-14-0)). To confirm the order of AA we used Schatz salts supplemented with (i) 19 AA at a final concentration of 1 mM each, (ii) 19 AA at a final concentration of 2 mM each and (iii) 16 AA at a final concentration of 0.2 mM and histidine, methionine and tryptophan at a final concentration of 2 mM. The experiments with 13C AA were performed using Schatz salts supplemented with 18 standard AA and one of the four marked amino acid (L-Alanine-13C3, L-Aspartic acid-13C4, L-Glutamic acid-13C5 and L-Methionine-13C5) all at a final concentration of 0.2 mM. In all cases the pH was adjusted to 7.0. All the amino acids were purchased from Sigma-Aldrich S.r.l. All the experiments were performed under aerobic condition at 21 °C.

E. coli Dh5α (laboratory stock suspension stored in 20% [v/v] glycerol solution at −80 °C) were grown in Luria Bertani (LB)[53](#page-14-0), agar or broth, and in M9 media[53](#page-14-0) supplemented with 19 AA at a final concentration of 0.2 mM (pH 7.0) (19 AA M9 medium). The experiments were performed under aerobic condition at 37 °C.

Growth curve experiments. The growth curve experiments with PhTAC125 were performed after two pre-cultures, as in Wilmes et al.[25](#page-13-0) with some adaptations. For the complex medium growth curves, a first preculture was grown for 20 h, in 20 ml MB medium in a 100 ml flask. Then this preculture was diluted 1:1000 in a final volume of 100 ml of the complex medium in a 1 l flask. After 20 h of growth, the optical density (OD600) of this second preculture was measured to be used to inoculate the final flask (1 l) in a final volume of 150 ml of Schatz salts and Peptone with a starting OD600 ~ 0.1. For the growth curves in the 19 AA medium, for all the concentrations used, the first preculture in MB was diluted 1:100 in a final volume of 100 ml of the 19 AA medium (for each experiment the AA concentration used for the final growth curves was used) in a 1 l flask. After 22 h of growth, the second preculture was washed, resuspend and used to inoculate the final flask (1l) in a final volume of 200 ml of the 19 AA medium with a starting OD600 ~ 0.1. In all experiments, the pre-cultures and the final growth cultures were incubated at 21 °C with shaking. Each experiment was performed in duplicate. Cell growth was monitored measuring the OD600 every hour in the experiments in complex medium, and every half an hour or an hour, depending on the concentration of AA used, in the experiments with the 19 AA medium. Three different measures were performed at each time point for each biological replicate.

The growth curve experiments with E. coli were performed after a first preculture of 20 h in 20 ml of LB broth in a 100 ml flask and a second preculture in 100 ml of 19 AA M9 medium.

After 20 h of growth, the optical density (OD600) was measured to be used to inoculate the final flask (1 l) in a final volume of 200 ml of 19 AA M9 medium. The cells were grown at 37 °C with shaking in duplicate and the growth was monitored measuring the OD600 every hour.

Sampling. Two biological replicates of the growth curves performed in complex medium were used for RNA-seq experiment. Every hour, in correspondence of the OD600 measurements, two replicates of 500 µl each for each curve, were treated with the RNA protect bacteria reagent (Qiagen S.r.l.) and conserved at −80 °C.

During the experiment in the 0.2 mM 19 AA medium, at each time point, two replicates of 500 µl each for each curve, were treated with the RNA protect bacteria reagent like above. During all the experiment in the 19 AA medium, regardless of the concentration of AA used, and in the growth curve with E.coli, two replicates of 1 ml each for each curve were filtered at each time point (Filtropur 0.2 µm, SARSTED AG & Co. KG) to remove bacterial cells and conserved at −20 °C for NMR metabolomic.

Growth curves with 13C amino acids. Four different experiments were performed using one uniformly 13C-labelled amino acid each time (13C-Glu, 13C-Ala, 13C-Asp and 13C-Met). For each experiment, two pre-cultures as described above were used, while the final growth experiments were performed in quadruplicate. Cell growth were monitored measuring the OD600 every hour. At four time points, early and late exponential growth (3 and 6 h) and early and late stationary phase (8 h and 30 min, 24 h), one of the four replicates was analyzed. Overall 1 ml of the medium were filtered (Filtropur 0.2 µm, SARSTED AG & Co. KG) to remove bacterial cells and conserved at −20 °C. The remaining culture (199 ml), was pelleted by centrifuging for 10 min at 11,000 rpm at 4 °C and resuspended in 500 µl of PBS[54.](#page-14-0) Then, cells were sonicated for 20 min, with cycle of 1 s of activity and 9 s of rest (292.5 W, 13 mm tip), with contemporary cooling on ice. After lysis, the samples were centrifugated for 25 min at 4 °C, at 8000 g, as described in ref. [54.](#page-14-0)

RNA extraction and sequencing. For RNA-seq, a preliminary sequencing (data not shown) was performed on an Illumina Hiseq 50 platform (Genomix4Life S.r.l., Italy). Total RNA was extracted with a RNeasy Tissue Mini Kit (Qiagen S.r.l.) following manufacturer's instructions. For improving the lysis step proteinase k and lysozyme were added to the lysis solution and the samples were homogenized using Tissue lyser II (Qiagen S.r.l.). The concentration and purity of RNA were analyzed using a NanoDrop ND-1000 (Thermo Fisher Scientific) and a Bioanalyzer (Agilent Technologies, Inc.). rRNA was removed from the sample using the Ribo-Zero Magnetic Kit (Bacteria) (Illumina, Inc.). The quality of the RNA depletion was then checked using Bioanalyzer (Agilent RNA 6000 PICO Assay, Agilent Technologies, Inc.). The ScriptSeq v2 RNA-Seq Library Preparation Kit (Illumina, Inc.) is then used to make the RNA-Seq library from the Ribo-Zero treated RNA. For each library 1 µg of RNA (rRNA depleted) was used following manufacturer's instructions. The quality of the libraries was evaluated using Bioanalyzer (Agilent Technologies, Inc.).

For the final experiment total RNA was extracted from a total of 20 samples of the growth curve in complex medium (five time points, two technical and two biological replicates) and library sequencing has been carried out at Genomix4Life S.r.l. (Italy) on an Illumina NextSeq500 (single-end sequencing strategy, 1 × 75 bp, ~25 reads/sample).

For real-time PCR (qRT-PCR), total RNA was extracted from the samples of five time points (T4, T6, T8, T10 and T12) for each biological replicate of the growth curve performed in the 19 AA medium, using a RNeasy Mini Kit (Qiagen S. r.l.), following the manufacturer's instruction. DNA was then removed from the samples using a RNase-free DNase (Qiagen S.r.l.). Overall 10 µl of the extracted RNA was reverse-transcribed using a Superscript II Reverse Transcriptase (Invitrogen) with Random primers (Invitrogen) following the manufacturer's instruction.

Quantitative real-time PCR (qRT-PCR). qRT-PCR reactions were performed in a final volume of 10 μl containing 1 μl of a 1:10 dilution of each cDNA, 5 μl of Powrup Sybr Master Mix (Life Technologies) and 1 μM of each primer (Supplementary Table 9). Primers were designed using the Primer3 software[55.](#page-14-0) Each sample was spotted in triplicate.

A first experiment using known amounts of DNA of the PhTAC125 strain (1-0.1-0.01-0.001 ng) were performed to obtain a standard curve and calculate the amplification efficiency for each primer pairs (data not shown). rplM and dnaA genes were used as internal references to normalize mRNA content.

All the reactions were performed on a QuantStudio™ 7 Flex Real-Time PCR System (Applied Biosystems by Life Technologies). Cycling conditions were: hold stage [50 °C for 2′ and 95 °C for 10′], PCR stage [40 cycles of: 95 °C for 30′′, 59 °C for 1′, 72 °C for 15′′], melt curve stage [95 °C for 15′′ 60 °C for 1′, 95 °C for 15′].

RNA-seq data analysis. Bowtie 2 (v2.2.3[)56](#page-14-0) was used to align raw reads to P. haloplanktis TAC125 reference genome (GCA_000026085.1_ASM2608v1). rRNA depletion, strand specificity and gene coverage were evaluated using BEDTools (v2.20.1)[57](#page-14-0) and SAMtools (v0.1.19)[58](#page-14-0) to verify the library preparation and sequencing performances. Raw read counts were then used to calculate TPM values for each PhTAC125 gene. Clusters of co-regulated genes were identified using the Clust tool[59](#page-14-0) using the following parameters: k-means clustering method, tightness weight equal to 0.3 and Q3s outliers threshold equal to 2.0.

Differentially expressed genes between the various contrasts were identified using the R (R Development Core Team, 2012, [https://www.r-project.org/)](https://www.r-project.org/) package DeSeq[260](#page-14-0) using default parameters and the following thresholds: adjusted p value < 0.01 and log2FC > 0.75 or log2FC < −0.75. The clustering of genes based on their FC was performed using the Pheatmap R package. Visualization of Pearson correlation was performed using "corplot" R package.

Functional enrichment analysis and regulon identification. To conduct functional enrichment, each gene whose upstream intergenic region was clustered in one of the three clusters was assigned to a specific functional category using a BLAST[61](#page-14-0) search against the COG database[62,](#page-14-0) with default parameters and considering a hit as significant if E-value < 1e−20. The exact binomial test implemented in the R package was used to assess over- and under-represented functional categories against the corresponding genomic background. Available information on PhTAC125 amino acid metabolism regulons were retrieved using RegPrecise database[63,64.](#page-14-0) The RegPrecise includes information for 7 PhTAC125 regulons (Supplementary Table 5).

Motif finding. Shared, conserved upstream motifs were searched up to 200 bp upstream of the genes belonging to the same regulon. These sequences were retrieved were retrieved from the P. haloplanktis TAC125 reference genome and fed into the MEME suite[65](#page-14-0) v. 5.1.1. MEME was used in combination with MAST (version 5.0.5)[66](#page-14-0) for identifying the most plausible shared motifs upstream of the selected genes. MEME was used setting the following parameters: -nmotifs 5, -minw 6, -maxw 30, -objfun classic, -revcomp, -markov_order 0, -minsites 1, -maxsites 3. All the other parameters were set as default. MAST was used using default parameters. In all cases, only the best scoring motif was considered for further analyses, provided that the search produced a significant result (e-value < 0.05). The conservation of identified shared motifs was represented using WebLogo[67](#page-14-0).

Modelling. The deterministic system was simulated by numerically integrating differential equations using the Matlab built-in function ode45 v. 2019a. To estimate the unknown parameters of the model from experimental data we used a stochastic curve-fitting in-house Matlab software. The algorithm is based on the paper by Cardoso et al.[68](#page-14-0) and consists in the combination of the non-linear simplex and the simulated annealing approach to minimize the squared deviation function. To increase the points available for curve-fitting, we used the spline interpolation function implemented in MATLAB on the measured values of nutrients and biomass concentration. The same function was adopted to increase the points available for growth rate estimation during the experiment reported in Fig. [1](#page-2-0)a.

The codes used to perform the simulations reported in this work and the details about the options of the curve-fitting environment, are available at [https://](https://multiauxic.sourceforge.io) [multiauxic.sourceforge.io.](https://multiauxic.sourceforge.io)

To assess the quality of the fit of the MMM vs. the cybernetic model R2 and AIC were computed. R2 values between experimental data and model predictions were <span id="page-13-0"></span>computed using the built-in cor.test function in R[69.](#page-14-0) AIC values were computed as:

$$AIC = N \times \log\left(\frac{\text{SSE}}{N}\right) + 2 \times p_{\text{-}}$$

With p being the number of parameters in the models (19 and 9 for cybernetic and MMM models, respectively), N being the number of training cases and SSE being the sum of squared errors for each training set. The interpolated datasets used to compute the fit with the models were also used when computing AIC (i.e., N = 49). The AIC values were computed using MATLAB 2019a.

Growth rates estimation on interpolated growth data. To increase the points available for curve-fitting, we used the spline interpolation function implemented in MATLAB on the measured values of biomass concentration (OD) obtained from growing PhTAC125 cells on peptone.

Metabolomic assay and data analysis. NMR spectra were acquired on (i) cell media to monitor the uptake of the various amino acids by measuring their levels in samples collected at different time points of cell growth; (ii) cell lysates to characterize the intracellular metabolome and its variation over time.

Medium samples were prepared in 5.00 mm NMR tubes by mixing 60 μL of a potassium phosphate buffer (1.5 M K2HPO4, 100% (v/v) 2H2O, 10 mM sodium trimethylsilyl [2,2,3,3−2H4] propionate (TMSP), pH 7.4) and 540 μL of sample. Cell lysate samples were prepared in 5.00 NMR tubes by mixing 60 μL of 2H2O and 540 μL of samples.

Spectral acquisition and processing were performed according to procedures developed at CERM[54,70](#page-14-0)–[74](#page-14-0). All the spectra were recorded using a Bruker 600 MHz spectrometer (Bruker BioSpin) operating at 600.13 MHz proton Larmor frequency and equipped with a 5 mm PATXI 1H-13C-15N and 2H-decoupling probe including a z axis gradient coil, an automatic tuning matching and an automatic and refrigerate sample changer (SampleJet). A BTO 2000 thermocouple served for temperature stabilization at the level of ~0.1 K at the sample. Before measurement, samples were kept for 5 min inside the NMR probe head, for temperature equilibration at 300 K.

For media, NMR spectra were acquired with water peak suppression and (i) one-dimensional (1D) 1H standard NOESY pulse sequence[75](#page-14-0) using 128 scans, 65,536 data points, a spectral width of 12,019 Hz, an acquisition time of 2.7 s, a relaxation delay of 4 s and a mixing time of 0.01 s; (ii) two-dimensional (2D) 1H- 13C heteronuclear single quantum coherence spectroscopy (HSQC) pulse sequence (hsqcetgpsisp2, Bruker). A total of 80 scans were collected using a spectral width of 12,019 for f2 and of 30,178 for f1, f2 and f1 acquisition time of 0.085 sand 0.002 s, respectively, and a relaxation delay of 2 s.

For lysates, NMR spectra were acquired with water peak suppression and (i) 1D 1H standard NOESY pulse sequence using 64 scans, 98,304 data points, a spectral width of 18,028 Hz, an acquisition time of 2.7 s, a relaxation delay of 4 s and a mixing time of 0.01; (ii) 1D 1H Carr–Purcell–Meiboom–Gill sequence using 64 scans[76,](#page-14-0) 73,728 data points, a spectral width of 12,019 Hz, an acquisition time of 3.07 s and a relaxation delay of 4 s; (iii) 2D 1H-13C HSQC pulse sequence (hsqcetgpsisp2, Bruker)[77.](#page-14-0) A total of 80 scans were collected using a spectral width of 12,019 for f2 and of 30,178 for f1, f2 and f1 acquisition time of 0.085 sand 0.002 s, respectively, and a relaxation delay of 2 s.

The raw data were multiplied by a 0.3 Hz exponential line broadening before applying Fourier transformation. Transformed spectra were automatically corrected for phase and baseline distortions. All the spectra were then calibrated to the reference signal of TMSP at δ 0.00 1H chemical shift (ppm) using TopSpin 3.5 (Bruker BioSpin srl).1H-13C HSQC spectra were also calibrated to the methyl signal of alanine at δ 19.03 13C chemical shift (ppm).

The signals deriving from each metabolite were assigned using an internal NMR spectral library of pure organic compounds, spiking NMR experiments and literature data. Matching between the present NMR spectra and the NMR spectral library was performed using the AMIX and Assure software (Bruker BioSpin srl). The relative concentrations of the various metabolites were calculated by integrating the corresponding signals in defined spectral range, using a home-made R 3.0.2 script.

Reporting summary. Further information on research design is available in the Nature Research Reporting Summary linked to this article.

#### Data availability

RNA-seq data that support the findings of this study have been deposited in NCBI SRA archive with the accession codes SAMN12207305 to SAMN12207324. Metabolomics data have been deposited at MetaboLights (<https://www.ebi.ac.uk/metabolights/>) under code unique identifier MTBLS1699 ([www.ebi.ac.uk/metabolights/MTBLS1699](http://www.ebi.ac.uk/metabolights/MTBLS1699)).

Databases used in this work: COG ([https://www.ncbi.nlm.nih.gov/COG/)](https://www.ncbi.nlm.nih.gov/COG/), KEGG ([https://www.genome.jp/kegg/)](https://www.genome.jp/kegg/), RegPrecise ([http://regprecise.sbpdiscovery.org:8080/](http://regprecise.sbpdiscovery.org:8080/WebRegPrecise/) [WebRegPrecise/](http://regprecise.sbpdiscovery.org:8080/WebRegPrecise/)).

The authors declare that the other data supporting the findings of this study are available within the paper. No restrictions apply to data availability. The source data underlying Figs. 1a–c, 2a–d, 2b–e, 4a, c, d, 5a, b, S6a, S8a–d, S9a, b, 10a, b, S20 and S21 are provided as a Source Data files.

#### Code availability

Code and scripts used to implement the model are available at [https://multiauxic.](https://multiauxic.sourceforge.io) [sourceforge.io.](https://multiauxic.sourceforge.io)

Received: 11 September 2019; Accepted: 26 May 2020;

#### References

- 1. Lambert, G. & Kussell, E. Memory and fitness optimization of bacteria under fluctuating environments. PLoS Genet. 10, e1004556 (2014).
- 2. Monod, J. The growth of bacterial cultures. Annu. Rev. Microbiol. 3, 371–394 (1949).
- 3. Joshua, C. J., Dahl, R., Benke, P. I. & Keasling, J. D. Absence of diauxie during simultaneous utilization of glucose and Xylose by Sulfolobus acidocaldarius. J. Bacteriol. 193, 1293–1301 (2011).
- 4. Novick, A. & Weiner, M. Enzyme induction as an all-or-none phenomenon. Proc. Natl Acad. Sci. USA 43, 553 (1957).
- 5. Kim, H. U. et al. Integrative genome-scale metabolic analysis of Vibrio vulnificus for drug targeting and discovery. Mol. Syst. Biol. 7, 460 (2011).
- 6. Xie, L. & Wu, X. L. Bacterial motility patterns reveal importance of exploitation over exploration in marine microhabitats. Part I Theory Biophys. J. 107, 1712–1720 (2014).
- 7. Son, K., Menolascina, F. & Stocker, R. Speed-dependent chemotactic precision in marine bacteria. Proc. Natl Acad. Sci. USA 113, 8624–8629 (2016).
- 8. Rochex, A. & Lebeault, J. M. Effects of nutrients on biofilm formation and detachment of a Pseudomonas putida strain isolated from a paper machine. Water Res. 41, 2885–2892 (2007).
- 9. Petrova, O. E. & Sauer, K. Sticky situations: key components that control bacterial surface attachment. J. Bacteriol. 194, 2413–2425 (2012).
- 10. Solopova, A. et al. Bet-hedging during bacterial diauxic shift. Proc. Natl Acad. Sci. USA 111, 7427–7432 (2014).
- 11. Succurro, A., Segre, D. & Ebenhoh, O. Emergent subpopulation behavior uncovered with a community dynamic metabolic model of Escherichia coli diauxic growth. mSystems 4, e00230-18 (2019).
- 12. Narang, A. & Pilyugin, S. S. Bacterial gene regulation in diauxic and nondiauxic growth. J. Theor. Biol. 244, 326–348 (2007).
- 13. Wang, X., Xia, K., Yang, X. & Tang, C. Growth strategy of microbes on mixed carbon sources. Nat. Commun. 10, 1279 (2019).
- 14. Hermsen, R., Okano, H., You, C., Werner, N. & Hwa, T. A growth-rate composition formula for the growth of E.coli on co-utilized carbon substrates. Mol. Syst. Biol. 11, 801 (2015).
- 15. Chen, F. et al. Differential substrate usage and metabolic fluxes in Francisella tularensis subspecies holarctica and Francisella novicida. Front Cell Infect. Microbiol 7, 275 (2017).
- 16. Steeb, B. et al. Parallel exploitation of diverse host nutrients enhances Salmonella virulence. PLoS Pathog. 9, e1003301 (2013).
- 17. Abu Kwaik, Y. & Bumann, D. Host delivery of favorite meals for intracellular pathogens. PLoS Pathog. 11, e1004866 (2015).
- 18. Abu Kwaik, Y. & Bumann, D. Microbial quest for food in vivo: 'nutritional virulence' as an emerging paradigm. Cell Microbiol 15, 882–890 (2013).
- 19. Eisenreich, W., Rudel, T., Heesemann, J. & Goebel, W. To eat and to be eaten: mutual metabolic adaptations of immune cells and intracellular bacterial pathogens upon infection. Front Cell Infect. Microbiol. 7, 316 (2017).
- 20. Enke, T. N., Leventhal, G. E., Metzger, M., Saavedra, J. T. & Cordero, O. X. Microscale ecology regulates particulate organic matter turnover in model marine microbial communities. Nat. Commun. 9, 2743 (2018).
- 21. Seymour, J. R., Marcos & Stocker, R. Chemotactic response of marine microorganisms to micro-scale nutrient layers. J. Vis. Exp. 4, e203 (2007).
- 22. Qi, W. et al. New insights on Pseudoalteromonas haloplanktis TAC125 genome organization and benchmarks of genome assembly applications using next and third generation sequencing technologies. Sci. Rep. 9, 16444 (2019).
- 23. Casillo, A. et al. Anti-biofilm activity of a long-chain fatty aldehyde from Antarctic Pseudoalteromonas haloplanktis TAC125 against Staphylococcus epidermidis Biofilm. Front. Cell Infect. Microbiol. 7, 46 (2017).
- 24. Giuliani, M. et al. Process optimization for recombinant protein production in the psychrophilic bacterium Pseudoalteromonas haloplanktis. Process Biochem. 46, 953–959 (2011).
- 25. Wilmes, B. et al. Fed-batch process for the psychrotolerant marine bacterium Pseudoalteromonas haloplanktis. Micro. Cell Fact. 9, 72 (2010).
- 26. Fondi, M., Bosi, E., Presta, L., Natoli, D. & Fani, R. Modelling microbial metabolic rewiring during growth in a complex medium. BMC Genomics 17, 970 (2016).
- <span id="page-14-0"></span>27. Vigentini, I., Merico, A., Tutino, M. L., Compagno, C. & Marino, G. Optimization of recombinant human nerve growth factor production in the psychrophilic Pseudoalteromonas haloplanktis. J. Biotechnol. 127, 141–150 (2006).
- 28. Haverkorn van Rijsewijk, B. R., Nanchen, A., Nallet, S., Kleijn, R. J. & Sauer, U. Large-scale 13C-flux analysis reveals distinct transcriptional control of respiratory and fermentative metabolism in Escherichia coli. Mol. Syst. Biol. 7, 477 (2011).
- 29. Yang, Y. et al. Relation between chemotaxis and consumption of amino acids in bacteria. Mol. Microbiol. 96, 1272–1282 (2015).
- 30. Mocali, S. et al. Ecology of cold environments: new insights of bacterial metabolic adaptation through an integrated genomic-phenomic approach. Sci. Rep. 7, 839 (2017).
- 31. Hosie, A. H., Allaway, D., Galloway, C. S., Dunsby, H. A. & Poole, P. S. Rhizobium leguminosarum has a second general amino acid permease with unusually broad substrate specificity and high similarity to branched-chain amino acid transporters (Bra/LIV) of the ABC family. J. Bacteriol. 184, 4071–4080 (2002).
- 32. Hosie, A. H. & Poole, P. S. Bacterial ABC transporters of amino acids. Res. Microbiol. 152, 259–270 (2001).
- 33. Sannino, F. et al. Pseudoalteromonas haloplanktis produces methylamine, a volatile compound active against Burkholderia cepacia complex strains. N. Biotechnol. 35, 13–18 (2017).
- 34. Bazire, P. et al. Characterization of l-carnitine metabolism in Sinorhizobium meliloti. J. Bacteriol. 201, e00772-18 (2019).
- 35. Marciani, P. et al. L-carnitine and carnitine ester transport in the rat small intestine. Pharmacol. Res. 23, 157–162 (1991).
- 36. Unemoto, T., Hayashi, M., Miyaki, K. & Hayashi, M. Formation of trimethylamine from DL-carnitine by Serratia marcescens. Biochim. Biophys. Acta 121, 220–222 (1966).
- 37. Plumbridge, J. Regulation of the utilization of amino sugars by Escherichia coli and Bacillus subtilis: same genes, different control. J. Mol. Microbiol. Biotechnol. 25, 154–167 (2015).
- 38. Ricciardelli, A. et al. Environmental conditions shape the biofilm of the Antarctic bacterium Pseudoalteromonas haloplanktis TAC125. Microbiol Res 218, 66–75 (2019).
- 39. Kompala, D. S., Ramkrishna, D. & Tsao, G. T. Cybernetic modeling of microbial growth on multiple substrates. Biotechnol. Bioeng. 26, 1272–1281 (1984).
- 40. Boianelli, A. et al. A non-linear deterministic model for regulation of diauxic lag on cellobiose by the pneumococcal multidomain transcriptional regulator CelR. PLoS One 7, e47393 (2012).
- 41. Mandli, A. R. & Modak, J. M. Cybernetic modeling revisited: a method for inferring the cybernetic variables ui from experimental data. Ind. Eng. Chem. Res. 54, 10190–10196 (2015).
- 42. Ramkrishna, D. & Song, H. S. Dynamic models of metabolism: review of the cybernetic approach. AIChE J. 58, 986–997 (2012).
- 43. Kompala, D. S., Ramkrishna, D., Jansen, N. B. & Tsao, G. T. Investigation of bacterial growth on mixed substrates: experimental evaluation of cybernetic models. Biotechnol. Bioeng. 28, 1044–1055 (1986).
- 44. Medigue, C. et al. Coping with cold: the genome of the versatile marine Antarctica bacterium Pseudoalteromonas haloplanktis TAC125. Genome Res 15, 1325–1335 (2005).
- 45. Koirala, S. et al. A nutrient-tunable bistable switch controls motility in Salmonella enterica serovar Typhimurium. mBio 5, e01611–e01614 (2014).
- 46. Novotna, J. et al. Proteomic studies of diauxic lag in the differentiating prokaryote Streptomyces coelicolor reveal a regulatory network of stressinduced proteins and central metabolic enzymes. Mol. Microbiol 48, 1289–1303 (2003).
- 47. van Hijum, S. A., Medema, M. H. & Kuipers, O. P. Mechanisms and evolution of control logic in prokaryotic transcriptional regulation. Microbiol Mol. Biol. Rev. 73, 481–509 (2009). Table of Contents.
- 48. Fuchs, T. M., Eisenreich, W., Kern, T. & Dandekar, T. Toward a systemic understanding of Listeria monocytogenes metabolism during infection. Front. Microbiol. 3, 23 (2012).
- 49. Häuslein, I. et al. Multiple substrate usage of Coxiella burnetii to feed a bipartite-type metabolic network. Front. Cell. Infect. Microbiol. 7, 285 (2017).
- 50. Omsland, A., Hackstadt, T. & Heinzen, R. A. Bringing culture to the uncultured: Coxiella burnetii and lessons for obligate intracellular bacterial pathogens. PLoS Pathog. 9, e1003540 (2013).
- 51. Birolo, L. et al. Aspartate aminotransferase from the Antarctic bacterium Pseudoalteromonas haloplanktis TAC 125: cloning, expression, properties, and molecular modelling. Eur. J. Biochem. 267, 2790–2802 (2000).
- 52. Papa, R., Rippa, V., Sannia, G., Marino, G. & Duilio, A. An effective cold inducible expression system developed in Pseudoalteromonas haloplanktis TAC125. J. Biotechnol. 127, 199–210 (2007).
- 53. Miller, J. et al. A short course. Bacterial Genetics (Cold Spring Harbor Lab Press, Cold Spring Harbor, NY, 1992).
- 54. Checcucci, A. et al. Creation and characterization of a genomically hybrid strain in the nitrogen-fixing symbiotic bacterium Sinorhizobium meliloti. ACS Synth. Biol. 7, 2365–2378 (2018).
- 55. Koressaar, T. et al. Primer3_masker: integrating masking of template sequence with primer design software. Bioinformatics 34, 1937–1938 (2018).
- 56. Langmead, B., Trapnell, C., Pop, M. & Salzberg, S. L. Ultrafast and memoryefficient alignment of short DNA sequences to the human genome. Genome Biol. 10, R25 (2009).
- 57. Quinlan, A. R. & Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010).
- 58. Li, H. et al. The sequence alignment/map format and SAMtools. Bioinformatics 25, 2078–2079 (2009).
- 59. Abu-Jamous, B. & Kelly, S. Clust: automatic extraction of optimal coexpressed gene clusters from gene expression data. Genome Biol. 19, 172 (2018).
- 60. Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014).
- 61. Altschul, S. F. et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25, 3389–3402 (1997).
- 62. Tatusov, R. L., Koonin, E. V. & Lipman, D. J. A genomic perspective on protein families. Science 278, 631–637 (1997).
- 63. Leyn, S. A., Maezato, Y., Romine, M. F. & Rodionov, D. A. Genomic reconstruction of carbohydrate utilization capacities in microbial-mat derived consortia. Front. Microbiol. 8, 1304 (2017).
- 64. Leyn, S. A. et al. Comparative genomics and evolution of transcriptional regulons in Proteobacteria. Micro. Genom. 2, e000061 (2016).
- 65. Bailey, T. L. et al. MEME SUITE: tools for motif discovery and searching. Nucleic Acids Res. 37, W202–W208 (2009).
- 66. Bailey, T. L. & Gribskov, M. Combining evidence using p-values: application to sequence homology searches. Bioinformatics 14, 48–54 (1998).
- 67. Crooks, G. E., Hon, G., Chandonia, J. M. & Brenner, S. E. WebLogo: a sequence logo generator. Genome Res. 14, 1188–1190 (2004).
- 68. Cardoso, M. F., Salcedo, R. L. & De Azevedo, S. F. The simplex-simulated annealing approach to continuous non-linear optimization. Comput. Chem. Eng. 20, 1065–1080 (1996).
- 69. R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria (2018). Available online at [https://www.R-project.org/.](https://www.R-project.org/)
- 70. Vignoli, A. et al. High-throughput metabolomics by 1D NMR. Angew. Chem. Int. Ed. Engl. 58, 968–994 (2019).
- 71. Takis, P. G., Ghini, V., Tenori, L., Turano, P. & Luchinat, C. Uniqueness of the NMR approach to metabolomics. TrAC Trends Anal. Chem. 120, 115300 (2019).
- 72. Bernacchioni, C. et al. NMR metabolomics highlights sphingosine kinase-1 as a new molecular switch in the orchestration of aberrant metabolic phenotype in cancer cells. Mol. Oncol. 11, 517–533 (2017).
- 73. Ghini, V. et al. Evidence of a DHA signature in the lipidome and metabolome of human hepatocytes. Int. J. Mol. Sci. 18, 359 (2017).
- 74. D'Alessandro, G. et al. (1)H-NMR metabolomics reveals the Glabrescione B exacerbation of glycolytic metabolism beside the cell growth inhibitory effect in glioma. Cell Commun. Signal. 17, 108 (2019).
- 75. Mckay, R. T. How the 1D‐NOESY suppresses solvent signal in metabonomics NMR spectroscopy: an examination of the pulse sequence components and evolution. Concepts Magn. Reson. Part A 38, 197–220 (2011).
- 76. Carr, H. Y. & Purcell, E. M. Effects of diffusion on free precession in nuclear magnetic resonance experiments. Phys. Rev. 94, 630 (1954).
- 77. Castañar, L., Parella, T. Recent advances in small molecule NMR: improved HSQC and HSQMBC experiments. In: Annual Reports on NMR Spectroscopy. (Elsevier, 2015).

#### Acknowledgements

P.T. and V.G. acknowledge the support and the use of resources of Instruct-ERIC, a Landmark ESFRI project, and specifically the CERM/CIRMMP Italy Centre. M.F. and E.P. would like to thank Prof. Alessio Mengoni for the help in designing the experiments and his critical evaluation of the work. This project was supported by a PNRA (Programma Nazionale di Ricerca in Antartide) grant (grant PNRA16_00246).

#### Author contributions

M.F., E.P. and R.F. initiated the project. M.F. and E.P. designed all the experiments. E.P. performed and/or supervised to all the experiments performed in this work. P.T. and V.G. performed the metabolomic experiments. B.C. and L.C. performed preliminary

transcriptomic experiments. MG contributed during growth and metabolomic experiments and RNA-seq data analysis under the supervision of E.P. and M.F. C.F. contributed to the realization of growth experiments. MF performed/supervised to all the computation reported in this work. M.F. and F.D.P. wrote the theoretical kinetic models and performed the simulations. M.F. wrote the paper. All the authors contributed to the editing of the paper. All the authors have read and approved the final version of the paper.

#### Competing interests

The authors declare no competing interests.

#### Additional information

Supplementary information is available for this paper at [https://doi.org/10.1038/s41467-](https://doi.org/10.1038/s41467-020-16872-8) [020-16872-8.](https://doi.org/10.1038/s41467-020-16872-8)

Correspondence and requests for materials should be addressed to M.F.

Peer review information Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available.

#### Reprints and permission information is available at <http://www.nature.com/reprints>

Publisher's note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit [http://creativecommons.org/](http://creativecommons.org/licenses/by/4.0/) [licenses/by/4.0/](http://creativecommons.org/licenses/by/4.0/).

© The Author(s) 2020