## **OPEN**

Received: 3 October 2017 Accepted: 15 December 2017 Published: xx xx xxxx

# **Regulation of metabolism in**  *Escherichia coli* **during growth on mixtures of the non-glucose sugars: arabinose, lactose, and xylose**

**Ehab M. Ammar1,2, Xiaoyi Wang1 & Christopher V. Rao1**

**Catabolite repression refers to the process where the metabolism of one sugar represses the genes involved in metabolizing another sugar. While glucose provides the canonical example, many other sugars are also known to induce catabolite repression. However, less is known about the mechanism for catabolite repression by these non-glucose sugars. In this work, we investigated the mechanism of catabolite repression in the bacterium** *Escherichia coli* **during growth on lactose, L-arabinose, and D-xylose. The metabolism of these sugars is regulated in a hierarchical manner, where lactose is the preferred sugar, followed by L-arabinose, and then D-xylose. Previously, the preferential utilization of L-arabinose over D-xylose was found to result from transcriptional crosstalk. However, others have proposed that cAMP governs the hierarchical regulation of many non-glucose sugars. We investigated whether lactose-induced repression of L-arabinose and D-xylose gene expression is due to transcriptional crosstalk or cAMP. Our results demonstrate that it is due to cAMP and not transcriptional crosstalk. In addition, we found that repression is reciprocal, where both L-arabinose and D-xylose also repress the lactose gene expression, albeit to a lesser extent and also through a mechanism involving cAMP. Collectively, the results further our understanding of metabolism during growth on multiple sugars.**

During growth on a mixture of sugars, bacteria will ofen consume the sugars sequentially through a process known as catabolite repression[1,](#page-9-0)[2](#page-9-1) . Usually, the cells frst consume the sugar yielding the highest growth rate, followed by the sugar yielding the next highest growth rate, and so o[n3](#page-9-2) . Tey do this by repressing the expression of the genes involved in metabolizing the less preferred sugar[s2](#page-9-1) . Tis mechanism maximizes the growth rate by ensuring that cells devote their limited metabolic resources towards the preferred sugar[s2](#page-9-1) . Te classic example involves the growth of the bacterium *Escherichia coli* on glucose and lactose, where *E. coli* will consume glucose before lactos[e1](#page-9-0)[,4](#page-9-3),[5](#page-9-4) . Te sequential utilization of these two sugars results in diauxic growth, where the cells frst grow on glucose and then, following a short period of no growth, proceed to grow on lactose. Tis two-phase pattern of growth is also observed when *E. coli* is grown on mixtures of glucose and some other sugars[2](#page-9-1) .

In *E. coli*, glucose catabolite repression is regulated by the phosphotransferase system, a multi-protein phosphorylation cascade that couples glucose uptake and metabolis[m4,](#page-9-3)[6,](#page-9-5)[7](#page-9-6) . One protein in this cascade, EIIAGlc, plays a central role in glucose catabolite repression. When EIIAGlc is phosphorylated, it activates adenylate cyclase, which increases the concentration of cAMP in the cells. cAMP activates the expression of many genes involved in the metabolism of non-glucose sugars through the action of the cAMP receptor protein CRP. In addition, the expression of these genes is also induced by their cognate sugars. Because EIIAGlc phosphorylation increases when the cells are not metabolizing glucose, these CRP-regulated metabolic genes are expressed only in the absence of glucose and presence of their cognate sugar. In addition, unphosphorylated EIIAGlc can bind and inhibit the ability of some sugar transporters to uptake their cognate sugars through a process known as inducer exclusio[n4](#page-9-3)[,6](#page-9-5)[–10.](#page-9-7) Tus, when the cells are metabolizing glucose and EIIAGlc is predominantly in the unphosphorylated form, these sugars are not transported into the cell and, therefore, are unable to induce the expression of their cognate metabolic genes. Tese two mechanisms are believed to be the main factors regulating glucose catabolite repression in *E. coli*.

1 Department of Chemical and Biomolecular Engineering, University of Illinois at Urbana-Champaign, Urbana, IL, 61801, USA. 2Genetic Engineering and Biotechnology Research Institute, University of Sadat City, El-Sadat City, Egypt. Correspondence and requests for materials should be addressed to C.V.R. (email: [cvrao@illinois.edu](mailto:cvrao@illinois.edu))

Catabolite repression has also been observed with sugar mixtures not involving glucose. One prominent example is the growth of *E. coli* on mixtures of L-arabinose and D-xylose (hereafer referred to simply as arabinose and xylose[)11.](#page-9-8) In this case, the cells will consume arabinose before xylose, though the extent of repression is less pronounced than observed with glucose and arabinose or glucose and xylose. Arabinose catabolite repression results from regulatory crosstalk between these two sugar utilization system[s12](#page-9-9). Te arabinose transport and metabolic genes are regulated by the transcription factors AraC and CRP. When AraC is bound with arabinose and CRP with cAMP, they activate the expression of the arabinose genes. Similarly, the xylose transport and metabolic genes are regulated by the transcription factors XylR and CRP. When XylR is bound with xylose and CRP with cAMP, they activate the expression of the xylose transport and metabolic genes. Arabinose catabolite repression results from arabinose-bound AraC binding to the promoters of the xylose genes and inhibiting their expression through a competitive mechanism. In addition, xylose weakly represses the expression of the arabinose genes using an analogous mechanism involving xylose-bound XylR. Tis reciprocal mechanism generates a small subpopulation of the cells that selectively consume xylose even in the presence of arabinos[e13.](#page-9-10)

A recent study by Aidelberg and coworkers investigated the utilization of six non-glucose sugars (lactose, arabinose, xylose, D-sorbitol, L-rhamnose and D-ribose) in *E. coli*[14.](#page-9-11) Tey observed hierarchical expression of the metabolic genes for these sugars. In particular, they found that sugars higher up in the hierarchy repress the expression of the genes involved in the metabolism of the sugars lower down in the hierarchy. Tese results suggest that the cells selectively utilize these six sugars in a similar hierarchical manner. In addition, they proposed that the hierarchical expression of these genes is regulated by CRP through the diferential production of cAMP. A number of studies have shown that cAMP levels are inversely proportional to the growth rate of the cells, irrespective of the sugar, due to inhibition of adenylate cyclase by the downstream metabolites[15](#page-9-12)[–17.](#page-9-13) In support of this hypothesis, Aidelberg and coworkers found that the expression hierarchy matches the growth rates associated with these sugars. Furthermore, they found that the promoters for genes associated with sugars at the top of the hierarchy are more sensitive to cAMP than those at the bottom of the hierarchy. In other words, when the cell is growing on a higher-growth yielding sugar such as lactose, insufcient cAMP is produced to induce the expression of genes for lower-growth yielding sugars such as rhamnose. Tis mechanism is appealing as it readily explains how cells selectively utilize sugars in accords with their growth yield. Tese results also suggest that multiple mechanisms are involved in the selective utilization of non-glucose sugars, where some involve transcriptional crosstalk, as is the case with arabinose and xylose, and others do not.

In this work, we investigated the selective utilization of lactose, arabinose, and xylose. When *E. coli* is grown on a mixture of lactose and arabinose or lactose and xylose, it will consume the lactose frst. Motivated by the study of Aidelberg and coworkers, we sought to determine whether the preferential utilization of lactose is due to cAMP or regulatory crosstalk. Using a number of diferent approaches, we conclude that the selective utilization of lactose is mainly due to cAMP. Tese results indicate that *E. coli* employs multiple mechanisms to regulate the utilization of non-glucose sugars.

#### **Results**

**Lactose represses arabinose and xylose gene expression.** As a first step towards investigating catabolite repression in sugar mixtures involving lactose and arabinose or lactose and xylose, we measured how these sugars individually afect the expression of each other's metabolic genes. To measure gene expression, we employed single-copy transcriptional fusions to the fuorescent protein Venus. Briefy, we grew the cells in TB7 medium containing 1 mM of the cognate sugar, which is sufcient to fully activate the expression of the corresponding metabolic genes, and increasing concentrations of a second sugar. As shown in Fig. [1a](#page-2-0), lactose inhibited expression from both the *araB* and *xylA* promoters, which respectively control the expression of the arabinose and xylose metabolic genes. Tese results are consistent with the results reported by Aidelberg and coworkers[14.](#page-9-11) In addition, we found that lactose represses the *xylA* promoter more strongly than the *araB* promoter.

We also tested whether arabinose and xylose inhibit the *lacZ* promoter, which controls the expression of the lactose metabolic genes. As shown in Fig. [1b,](#page-2-0) arabinose represses expression from the *lacZ* promoter. Tese results indicate that the repression is reciprocal. In other words, lactose represses arabinose metabolism and arabinose represses lactose metabolism. However, repression by lactose is somewhat greater than repression by arabinose. In the case of xylose (Fig. [1c](#page-2-0)), we found that it weakly represses expression from the *lacZ* promoter. While repression is also reciprocal during growth on lactose and xylose, lactose is clearly the preferred sugar.

We also performed identical experiments in M9 minimal medium[13](#page-9-10),[18](#page-9-14). Similar to what was observed in TB7, we found that lactose inhibits the expression of the arabinose and xylose metabolic genes in M9 minimal medium (Figure S1), though the efect is stronger in this medium. In addition, we found that arabinose again weakly represses the expression of the lactose metabolic genes. However, xylose was found to have no signifcant efect on the expression of the lactose genes in M9 minimal medium.

**Lactose reduces the rate of arabinose and xylose consumption.** We next investigated sugar utilization during growth on arabinose or xylose in the presence of lactose. We grew the cells in sugar mixtures containing equal concentrations on a molar basis of the two sugars (2mM). As shown in Fig. [2](#page-3-0), the cells individually consume lactose and arabinose at roughly the same rates. However, the rate of utilization for both sugars was decreased somewhat when the cells were grown in the presence of both. Tese results are consistent with a mechanism of reciprocal repression. Only during growth on lactose and xylose was catabolite repression observed. In this case, lactose signifcantly reduced the rate of xylose consumption. However, xylose did not reduce lactose consumption. Unlike arabinose, xylose is intrinsically consumed more slowly than lactose. Moreover, the rate of xylose consumption is further reduced in the presence of lactose, most likely due to repression of the xylose metabolic genes. Te results are consistent with our gene expression results (Fig. [1)](#page-2-0), where lactose exhibits a greater efect on xylose genes than on arabinose genes.

![](_page_2_Figure_1.jpeg)

<span id="page-2-0"></span>![](_page_2_Figure_2.jpeg)

We also measured the sugar utilization in M9 minimal medium. As shown in Figure S2, lactose delayed the consumption of both arabinose and xylose. Tese results are consistent with those obtained using TB7 (Fig. [2](#page-3-0)). However, in M9 minimal medium, arabinose did not delay the utilization of lactose, unlike the case with TB7 medium. In addition, both lactose and xylose were utilized at similar rates when they were individually added to the growth medium. Tese results demonstrate that lactose represses both the utilization of arabinose and xylose in M9 minimal medium. However, reciprocal repression does not occur in this medium.

**Lactose catabolite repression does not involve transcriptional crosstalk.** Previous work demonstrated that arabinose represses xylose metabolism and vice versa through a mechanism involving transcriptional crosstal[k12](#page-9-9). In particular, arabinose-bound AraC binds the *xylA* promoter and represses transcription. Similarly, xylose-bound XylR binds the *araB* promoter and represses transcription, albeit to a lesser extent. One possibility is that lactose represses arabinose and xylose gene expression also using competitive mechanism. Briefy, the

![](_page_3_Figure_1.jpeg)

<span id="page-3-0"></span>**Figure 2.** Lactose inhibits the utilization of arabinose and xylose. (**a**) Sugar utilization in mixtures of lactose and arabinose. Diamonds refer to arabinose concentrations in the presence (black) or absence (white) of lactose; circles refer to lactose concentrations in the presence (black) or absence (white) of arabinose. (**b**) Sugar utilization in mixtures of lactose and xylose. Diamonds refer to xylose concentrations in the presence (black) or absence (white) of lactose; circles refer to lactose concentrations in the presence (black) or absence (white) of xylose. Cells were grown in TB7. Error bars denote the standard deviation of three experiments performed on separated days.

transcriptional repressor LacI regulates the expression of the lactose genes. When LacI is bound with allolactose, a by-product of lactose metabolism, it no longer binds the *lacZ* promoter and represses transcription. We therefore hypothesized that allolactose-bound LacI binds to the *araB* and *xylA* promoters and represses transcription of the respective metabolic genes. To test this hypothesis, we measured expression from the *araB* and *xylA* promoters in a Δ*lacI* mutant. As shown in Fig. [3](#page-4-0), lactose was still able to repress *araB* and *xylA* promoters in the absence of LacI. Tese results demonstrate that lactose catabolite repression does not involve transcriptional crosstalk by LacI.

**Lactose-mediated repression is independent of arabinose and xylose metabolism.** We next tested whether lactose is able to repress arabinose and xylose gene expression in the absence of arabinose and xylose metabolism. Te goal here was to see whether lactose was afecting some downstream metabolic process that somewhat would afect arabinose and xylose gene expression. As shown in Figs [4a](#page-5-0) and [5a](#page-6-0), lactose still inhibited arabinose and xylose gene expression in strains unable to metabolize these respective sugars. Tese results demonstrate that lactose-mediated repression of the arabinose and xylose genes is independent of arabinose and xylose metabolism.

**Arabinose and xylose-mediated repression of the lactose metabolism genes is respectively dependent on arabinose and xylose metabolism.** We next tested whether the repression of the lactose genes by arabinose and xylose was respectively dependent on arabinose and xylose metabolism. As shown in Figs [4b](#page-5-0) and [5b,](#page-6-0) we observed no repression of lactose gene expression in the absence of arabinose and xylose metabolism. Te results demonstrate that arabinose or xylose metabolism is necessary for the repression of the lactose genes. Tey also suggest that the mechanism for repression involves cAMP. As the respective regulators, AraC and XylR, were still present in these mutants, these results also indicate that the mechanism does not involve transcriptional crosstalk.

We also performed the reciprocal experiment, where we tested whether lactose is able to repress arabinose and xylose gene expression in the absence of lactose metabolism using Δ*lacZ* mutant. Not surprisingly, we observed no repression (data not shown). However, these experiments were uninformative, because they provide no insight regarding the mechanism of lactose-mediated repression. Te reason is that LacZ, the enzyme that cleaves lactose

![](_page_4_Figure_1.jpeg)

<span id="page-4-0"></span>**Figure 3.** Lactose catabolite repression does not involve transcriptional crosstalk by LacI. (**a**) Efect of increasing concentrations of lactose on *araB* promoter activity in Δ*lacI* mutant (Δ*lacI* P*araB*-Venus). Cells were grown in TB7 with 1mM arabinose. (**b**) Efect of increasing concentrations of lactose on *xylA* promoter activity in Δ*lacI* mutant (Δ*lacI* P*xylA*-Venus). Cells were grown in TB7 with 1mM xylose. Error bars denote the standard deviation of three experiments performed on separated days.

into glucose and galactose, also produces allolactose, the inducer for LacI[19](#page-9-15). Tus, we cannot determine whether loss of repression in a Δ*lacZ* mutant results from the loss of the LacI inducer or the inability of the cells to utilize lactose.

**Repression by lactose can be explained by preferential promoter activation by cAMP-CRP.** Aidelberg and coworkers previously proposed that the hierarchy among lactose, arabinose and xylose is due to diferential promoter activation by the CRP global regulato[r14.](#page-9-11) To further test this hypothesis, we measured whether the addition of cAMP would relieve lactose inhibition of the arabinose and xylose promoters. As shown in Fig. [6,](#page-7-0) the addition of external cAMP increased *araB* and *xylA* promoter activities in presence of lactose. In the case of the *araB* promoter, repression was relieved by the addition of 5 or 10mM cAMP. However, in the case of the *xylA* promoter, repression was only slightly relieved, even though similar concentrations of cAMP were employed. We were unable to test higher concentrations of cAMP (>10mM), because they hindered cell growth. Nonetheless, these results are consistent with a mechanism whereby lactose inhibits arabinose and xylose metabolism through CRP.

We also tested whether the addition of cAMP would relieve the repression of the *lacZ* promoter by arabinose. As shown in Figure S3, we found that repression was partially relieved by the addition of 5 mM cAMP. Higher concentrations of cAMP (10 mM), however, did not further increase the activity of the *lacZ* promoter. Tese results demonstrate that arabinose-mediated repression of the *lacZ* promoter is due in part to cAMP. Whether it represents the sole mechanism is unknown.

**Swapping the CRP binding site of** *xylA* **promoter with that of** *lacZ* **promoter reduces lactose-mediated repression of xylose metabolism.** Previous results suggest that the CRP binding site within the *lacZ* promoter has a higher afnity for cAMP-CRP than the CRP binding site within the *xylA* promoter[14](#page-9-11). If catabolite repression is due to diferential activation by CRP, then replacing the weak CRP binding site within the *xylA* promoter with a stronger one from the *lacZ* promoter should relieve repression. Te *xylA* promoter has a single CRP binding site[20.](#page-9-16) Te *lacZ* promoter has two CRP binding site[s21,](#page-9-17)[22.](#page-9-18) Te distal site is considered to be the main CRP binding site for the *lacZ* promoter. Terefore, we replaced the native CRP binding site within the *xylA* promoter with the distal CRP binding site from the *lacZ* promoter. As shown in Fig. [7,](#page-7-1) less repression by lactose was observed with the *xylA* promoter containing the distal *lacZ* CRP binding site than with the one containing the native CRP binding site. Tese results further support a mechanism where lactose-mediated repression of xylose metabolism is due to cAMP.

![](_page_5_Figure_1.jpeg)

<span id="page-5-0"></span>**Figure 4.** Lactose-mediated repression is independent of arabinose metabolism. (**a**) Efect of increasing concentrations of lactose on *araB* promoter activity in Δ*araBAD* mutant (Δ*araBAD* P*araB*-Venus). Cells were grown in TB7 with 1mM arabinose. (**b**) Efect of increasing concentrations of arabinose on *lacZ* promoter activity in Δ*araBAD* mutant (Δ*araBAD* P*lacZ*-Venus). Cells were grown in TB7 with 1mM lactose. Error bars denote the standard deviation of three experiments performed on separated days.

#### **Discussion**

*E. coli* is capable of growing on a number of diferent sugars. While much is known about how *E. coli* regulates its metabolism when grown on a single sugar, far less is known about how this bacterium regulates its metabolism when grown on mixtures of sugars, particularly when the mixture does not involve glucose. A number of studies have shown that *E. coli* will selectively utilize non-glucose sugars, frst consuming the sugar that yields the highest growth rate and then consuming the sugar that yields the next highest growth and so on[11](#page-9-8),[12](#page-9-9),[14](#page-9-11). Still many questions remain about the mechanisms governing the selective utilization of these sugars. Two mechanisms have recently been proposed. One involves transcriptional crosstalk between the regulatory systems governing the metabolism of individual sugars. Tis mechanism was shown to govern the preferential utilization of arabinose over xylose[12](#page-9-9). Te other mechanism involves cAMP, a second messenger involved in glucose catabolite repressio[n4](#page-9-3)[,6](#page-9-5) . Tis mechanism was proposed to govern the preferential utilization of multiple non-glucose sugars[14.](#page-9-11) However, this study did not consider whether transcriptional crosstalk was also involved. Indeed, cAMP does not appear to govern the preferential utilization of arabinose over xylos[e12](#page-9-9). Furthermore, repression is reciprocal in the case of arabinose and xylose, where arabinose strongly represses the expression of the xylose metabolic genes and xylose weakly represses the expression of the arabinose metabolic gene[s12.](#page-9-9) Tese results indicate that the hierarchy among sugars is not necessarily strict but rather depends on the relative concentrations of the two sugars.

In this work, we investigated the selective utilization of three sugars in *E. coli*: lactose, arabinose, and xylose. Our goal was to determine whether transcriptional crosstalk or cAMP/CRP governed the selective utilization of these sugars. In rich medium, we found that both lactose and arabinose inhibited the utilization of the other (Fig. [2b](#page-3-0)). Consistent with these results, we found that lactose inhibits the expression of the arabinose metabolic genes and that arabinose inhibits the expression of the lactose metabolic genes. In minimal medium, however, lactose is the preferred sugar. Tese results are also consistent with gene expression data (Figure S1), where lactose is a stronger repressor of arabinose gene expression than arabinose is of lactose gene expression. Te origin of these diferences on rich and minimal media is not known, though may refect the complex growth patterns observed in peptide-based media[s23,](#page-9-19)[24.](#page-9-20) In the case of xylose and lactose, *E. coli* prefers the latter in both complex and minimal media. Even though xylose inhibits the expression of the lactose genes, the efect is relatively weak.

Tis hierarchy matches the growth rates associated with these sugar[s14.](#page-9-11) One question is why does *E. coli* grow more rapidly on lactose than on arabinose or xylose. Te increased growth rate associated with lactose is likely due to its metabolism requiring less steps: intracellular lactose is degraded by the enzyme LacZ into galactose and glucose, where the latter can directly enter the Embden–Meyerhof–Parnas (glycolytic) pathway. Arabinose and

![](_page_6_Figure_1.jpeg)

<span id="page-6-0"></span>**Figure 5.** Lactose-mediated repression is independent of xylose metabolism. (**a**) Efect of increasing concentrations of lactose on *xylA* promoter activity in Δ*xylAB* mutant (Δ*xylAB* P*xylA*-Venus). Cells were grown in TB7 with 1mM xylose. (**b**) Efect of increasing concentrations of xylose on *lacZ* promoter activity in Δ*xylAB* mutant (Δ*xylAB* P*lacZ*-Venus). Cells were grown in TB7 with 1mM lactose. Error bars denote the standard deviation of three experiments performed on separated days.

xylose metabolism involves many more steps, as they both enter the glycolytic pathway through the non-oxidative branch of the pentose phosphate pathway. Tis likely explains why their metabolism is more inefcient, in addition to their lower energetic yield. Terefore, one would expect that lactose inhibits the metabolism of arabinose and xylose, as this would refect a more efcient allocation of limited metabolic resources.

Regarding the mechanism, our data support one involving cAMP. Indeed, we found that lactose was still able to repress the expression of the arabinose and xylose metabolic genes when LacI, the cognate regulator of the lactose genes[25](#page-9-21), was deleted (Fig. [3](#page-4-0)). Had the mechanism involved transcriptional crosstalk, we would expect repression to be eliminated in a Δ*lacI* mutant, contrary to what was observed. Tese results indicate that lactose metabolism alone is sufcient for repressing the expression of the arabinose and xylose genes. Likewise, we found that arabinose and xylose metabolism were necessary for arabinose and xylose to repress lactose gene expression (Figs [4](#page-5-0) and [5](#page-6-0)).

Tat metabolism is necessary for repression strongly suggests that cAMP is involved. In support of this mechanism, we found that the addition of exogenous cAMP partially relieved lactose-mediated repression of arabinose and xylose gene expression (Fig. [6)](#page-7-0). In the case of the arabinose genes, exogenous cAMP was nearly able to counteract the repressive efects of lactose. In the case of the xylose genes, however, exogenous cAMP was only able to partially relieve the repression by lactose, possibly due to our inability to explore higher cAMP concentration[s15](#page-9-12)[–17.](#page-9-13) As a further test, we also replaced the native CRP binding site within the *xylA* promoter with one from the *lacZ* promoter. Consistent with a mechanism involving cAMP, we found that lactose-mediated repression was reduced when the CRP site was replaced in the *xylA* promoter (Fig. [7](#page-7-1)).

While our results demonstrate that cAMP regulates the sugar utilization in mixtures of lactose and arabinose or xylose, it may not be the only mechanism involved. One possibility is that allolactose, a byproduct of lactose metabolism and the inducer for Lac[I19](#page-9-15), could inhibit AraC and XylR, the respective regulators of arabinose and xylose gene expression[20,](#page-9-16)[26,](#page-10-0)[27.](#page-10-1) Previous work has shown that isopropyl β-D-1-thiogalactopryanoside (IPTG), a non-metabolizable lactose analog, competitively inhibits Ara[C28.](#page-10-2) We tested whether it competitively inhibits XylR. However, we found that it had no efect on xylose gene expression (data not shown). We also tested whether thiomethyl galactoside (TMG), another non-metabolizable lactose analog, inhibits arabinose and xylose gene expression (Figure S4). Consistent with the IPTG results, we found that TMG competitively inhibits arabinose gene expression but not xylose gene expression (Figure S4). Tese results suggest that allolactose may competitively inhibit AraC, though whether these non-metabolizable analogs truly mimic the efect of allolactose is unknown. Unfortunately, we were unable to directly test allolactose as this sugar is difcult to obtain.

![](_page_7_Figure_1.jpeg)

<span id="page-7-0"></span>**Figure 6.** cAMP mitigates lactose-mediated repression of arabinose and xylose gene expression. (**a**) Efect of diferent concentrations of cAMP on *araB* promoter activity. Cells were grown in M9 minimal medium containing 1mM arabinose. (**b**) Efect of diferent concentrations of cAMP on *xylA* promoter activity. Cells were grown in M9 minimal medium containing 1mM xylose. Error bars denote the standard deviation of three experiments performed on separated days.

![](_page_7_Figure_3.jpeg)

<span id="page-7-1"></span>**Figure 7.** Swapping the CRP binding site of *xylA* promoter with that of *lacZ* promoter reduces lactose-mediated repression of xylose metabolism. Efect of increasing concentrations of lactose on the activity of *xylA* promoter with native (grey bars) or *lacZ* (white bars) CRP binding site. Cells were grown on M9 minimal medium with 1mM xylose. Error bars denote the standard deviation of three experiments performed on separated days.

Nevertheless, these results indicate that allolactose likely does not repress XylR. Furthermore, as lactose represses xylose gene expression more strongly than arabinose gene expression, it seems unlikely that inhibition of AraC by allolactose, assuming it occurs, is a signifcant contributor to catabolite repression.

Why does *E. coli* employ diferent mechanisms for selective sugar utilization? As noted above, the mechanism difers for a mixture involving arabinose and xylose and those involving lactose and arabinose or lactose and xylose. One possibility is that these diferences refect the relative frequency of encountering these diferent sugar mixtures in nature. In particular, *E. coli* is unlikely to encounter sugar mixtures involving lactose and arabinose

<span id="page-8-0"></span>

| Strain  | Characteristics                                                                                             | Source or Reference   |
|---------|-------------------------------------------------------------------------------------------------------------|-----------------------|
| MG1655  | λ rph-1 (wild type)                                                                                         | CGSC, Yale University |
| SK459   | attλ::[kan ParaB-Venus oriR6K]                                                                              | 13                    |
| XW215   | attλ::[kan PxylA-Venus oriR6K]                                                                              |                       |
| XW330   | attλ::[kan PlacZ-Venus oriR6K]                                                                              |                       |
| EA037   | ΔlacI::FRT attλ::[kan ParaB-Venus oriR6K]                                                                   |                       |
| EA039   | ΔlacI::FRT attλ::[kan PxylA-Venus oriR6K]                                                                   |                       |
| EA040   | ΔlacZ::FRT attλ::[kan ParaB-Venus oriR6K]                                                                   |                       |
| EA041   | ΔlacZ::FRT attλ::[kan PxylA-Venus oriR6K]                                                                   |                       |
| EA046   | ΔaraBAD::FRT attλ::[kan ParaB-Venus oriR6K]                                                                 |                       |
| EA047   | ΔaraBAD::FRT attλ::[kan PlacZ-Venus oriR6K]                                                                 |                       |
| EA049   | ΔxylAB::FRT attλ::[kan PlacZ-Venus oriR6K]                                                                  |                       |
| EA050   | ΔxylAB::FRT attλ::[kan PxylA-Venus oriR6K]                                                                  |                       |
| EA051   | attλ::[kan PxylA-Venus oriR6K] (CRP binding site of xylA promoter<br>is swapped with that of lacZ promoter) |                       |
| Plasmid |                                                                                                             |                       |
| pVenus  | kan att Venus oriR6K (CRIM plasmid with Venus reporter)                                                     |                       |
| pInt-ts | bla int oriR6K (helper plasmid for att λ integration)                                                       | 33                    |
| pXW211  | kan att PxylA-Venus oriR6K (CRIM plasmid with PxylA-Venus<br>reporter)                                      |                       |
| pXW238  | pXW211 in which (CRP binding site of xylA promoter is swapped<br>with that of lacZ promoter)                |                       |
| pSK460  | kan att PlacZ-Venus oriR6K (CRIM plasmid with PlacZ-Venus<br>reporter)                                      |                       |

**Table 1.** Strains and plasmids used in this study.

or lactose and xylose. Lactose is found in milk whereas arabinose and xylose are found in the complex polysaccharides that comprise the plant cell wall[29](#page-10-3). In other words, cells are unlikely to be exposed to mixture of lactose and arabinose or lactose and xylose whereas they will likely be exposed to mixture of arabinose and xylose. Tis would suggest that cAMP is a general mechanism for catabolite repression whereas transcriptional crosstalk is a more specialized one. In particular, mechanisms involving transcriptional crosstalk likely evolved to regulate more frequent events than the generalized mechanism involving cAMP, which appears not to be adapted for any specifc sugars.

In summary, we investigated the mechanism for catabolite repression in sugar mixtures of lactose, arabinose, and xylose. Building on previous work, we established that cAMP principally regulates the selective utilization of lactose and arabinose; and lactose and xylose. Moreover, we found that repression is reciprocal in the sense that both sugars repress each other's metabolism, though lactose is clearly the dominant sugar. Tese results further our understanding of metabolism in *E. coli*. Tey also demonstrate that multiple mechanisms are involved in regulating the metabolism of sugar mixtures not involving glucose.

## **Methods**

**Media and growth conditions.** Luria-Bertani (LB) medium (10g/L tryptone, 5g/L yeast extract, and 10g/L NaCl) was used for strain construction. All experiments were performed in either bufered tryptone broth (TB7: 10 g/L tryptone and 1mM MgSO4, bufered at pH 7 with 100mM potassium phosphate) or M9 minimal medium (6.8 g/L Na2HPO4, 3 g/L KH2PO4, 1 g/L NH4Cl, 0.5 g/L NaCl, 2 mM MgSO4, 100 μM CaCl2, 0.001% thiamine hydrochloride and 4 g/L glycerol). All experiments were performed at 37 °C. Antibiotics were added as needed at the following concentrations: ampicillin, 100μg/mL; chloramphenicol, 20μg/mL; kanamycin, 40μg/mL.

**Strain and plasmid construction.** Table [1](#page-8-0) lists all strains and plasmids used in this study. Table S1 lists all oligonucleotides used in strain and plasmid construction. All strains are isogenic derivatives of *E. coli* MG1655. Genes were deleted using the method of Datsenko and Wanne[r30.](#page-10-4) Prior to removing the antibiotic resistant cassette using the FLP recombinase, P1 transduction was used to move the cassette to a clean parental background. Te plasmid pXW211 contains a transcriptional fusion of the *xylA* promoter to the fuorescent protein Venus. To construct this plasmid, *xylA* promoter (genomic region: 3730691–3731015) was PCR amplifed using the primers SK435F and XW435R, and then the PCR fragment was cloned into the integrative plasmid pVenus using SalI and EcoRI restriction sites. Te plasmid pSK460 contains a transcriptional fusion of the *lacZ* promoter to the fuorescent protein Venus. To construct this plasmid, the *lacZ* promoter (genomic region: 366222–366545) was PCR amplifed using the primers SK418F and XW418R, and then the PCR fragment was cloned into the integrative plasmid pVenus using SalI and EcoRI restriction sites. Te plasmid pXW238 contains a transcriptional fusion of the *xylA* promoter to the fuorescent protein Venus, where the native CRP binding site of the *xylA* promoter (TTTTGCGAGCGAGCGCACACTT) (genomic region: 3730899–3730878[)20](#page-9-16)[,31](#page-10-5) was replaced with the CRP binding site of the *lacZ* promoter (TAATGTGAGTTAGCTCACTCAT) (genomic region: 366394–366415[)32.](#page-10-6) To construct this plasmid, the plasmid pXW211 was PCR amplifed using the primers EA061F and EA062R, which contain the CRP binding site from the *lacZ* promoter as 5′ overhangs. Te resulting PCR fragment was then phosphorylated using T4 polynucleotide kinase and ligated back into a circular plasmid using T4 DNA ligase. Te plasmid was then integrated into λ phage attachment sites using the CRIM integration syste[m33.](#page-10-7) Afer confrming by PCR that a single integration took place, the integrated plasmid was moved into a clean background using P1 transduction[34](#page-10-8),[35](#page-10-9).

**Fluorescence assays.** In the experiments involving TB7 as the base medium, the cultures were frst grown overnight in TB7. Te overnight culture was then diluted 1:100 into fresh media and grown for 2 h to an OD600 of ~0.2. Te cultures were then subcultured into 1mL of fresh TB7 medium containing the specifed sugar to an initial OD600 of 0.01 in 96 deep-well plates. Te cultures were then incubated on a microplate shaker at 600 rpm for 5h. In the experiments involving M9 as the base media, the cells were grown overnight in M9 minimal media. Te overnight culture was diluted 1:50 into fresh media and grown for 5h to an OD600 of ~0.2. One mL aliquots were then distributed to individual wells in a 96 deep-well plate. Te specifed sugars were then added to each well, and then the plate was incubated on the microplate shaker at 600 rpm for 4h. At the end of every experiment, 100 µL samples were taken from each well and transferred to a black, clear-bottom Costar 96-well microplate. Fluorescence (excitation at 515 nm; emission at 528 nm) and absorbance (OD600) were measured using Tecan Safre2 microplate reader. Te fuorescence readings were normalized with the OD600 to account for cell density. All experiments were performed on three separate days, and the average values with the standard deviations are reported.

**Analytical measurements.** Cells were frst grown overnight in the specifed media. Te overnight culture was then sub-cultured into 60mL of fresh media containing the specifed sugars in a 125mL shake fask with an initial OD600 of 0.05. Te fasks were then incubated on a standard orbital shaker at 250 rpm and 37 °C. Samples (1mL) were taken at specifed time intervals and fltered through 0.22μm flters (Millipore) for subsequent analysis. Arabinose, lactose and xylose concentrations were measured using a Shimadzu HPLC system equipped with Shimadzu RID-10A refractive index detector (Shimadzu Corp., Kyoto, Japan), and an Aminex HPX-87H column (300 mm ×7.8 mm) (Bio-Rad Laboratories, Hercules, CA, USA). Te HPLC oven was adjusted to 65 °C and a mobile phase of 0.5mM H2SO4 was run at 0.6mL/min for 15min per sample. Kinetic experiments were run in duplicates with the averages and standard deviations reported.

**Data availability.** Any other datasets generated during and/or analyzed during the current study are available from the corresponding author upon request.

#### **References**

- <span id="page-9-0"></span>1. Görke, B. & Stülke, J. Carbon catabolite repression in bacteria: many ways to make the most out of nutrients. *Nature Reviews Microbiology* **6**, 613–624 (2008).
- <span id="page-9-1"></span>2. Magasanik, B. In *Cold Spring Harbor symposia on quantitative biology*. 249–256, (Cold Spring Harbor Laboratory Press).
- <span id="page-9-2"></span>3. Stülke, J. & Hillen, W. Carbon catabolite repression in bacteria. *Current opinion in microbiology* **2**, 195–201 (1999).
- <span id="page-9-3"></span>4. Deutscher, J. Te mechanisms of carbon catabolite repression in bacteria. *Current opinion in microbiology* **11**, 87–93 (2008).
- <span id="page-9-4"></span>5. Kremling, A., Geiselmann, J., Ropers, D. & De Jong, H. Understanding carbon catabolite repression in *Escherichia coli* using quantitative models. *Trends in microbiology* **23**, 99–109 (2015).
- <span id="page-9-5"></span>6. Deutscher, J., Francke, C. & Postma, P. W. How phosphotransferase system-related protein phosphorylation regulates carbohydrate metabolism in bacteria. *Microbiology and Molecular Biology Reviews* **70**, 939–1031 (2006).
- <span id="page-9-6"></span>7. Postma, P., Lengeler, J. & Jacobson, G. Phosphoenolpyruvate: carbohydrate phosphotransferase systems of bacteria. *Microbiological Reviews* **57**, 543–594 (1993).
- 8. Chen, S., Oldham, M. L., Davidson, A. L. & Chen, J. Carbon catabolite repression of the maltose transporter revealed by X-ray crystallography. *Nature* **499**, 364–368 (2013).
- 9. Hariharan, P. & Guan, L. Insights into the inhibitory mechanisms of the regulatory protein IIAGlc on melibiose permease activity. *Journal of Biological Chemistry* **289**, 33012–33019 (2014).
- <span id="page-9-7"></span>10. Hariharan, P., Balasubramaniam, D., Peterkofsky, A., Kaback, H. R. & Guan, L. Termodynamic mechanism for inhibition of lactose permease by the phosphotransferase protein IIAGlc. *Proceedings of the National Academy of Sciences* **112**, 2407–2412 (2015).
- <span id="page-9-8"></span>11. Kang, H., Song, S. & Park, C. Priority of pentose utilization at the level of transcription: arabinose, xylose, and ribose operons. *Molecules and cells* **8**, 318–323 (1998).
- <span id="page-9-9"></span>12. Desai, T. A. & Rao, C. V. Regulation of arabinose and xylose metabolism in *Escherichia coli*. *Appl Environ Microb* **76**, 1524–1532 (2010).
- <span id="page-9-10"></span>13. Koirala, S., Wang, X. & Rao, C. V. Reciprocal regulation of L-arabinose and D-xylose metabolism in. *Escherichia coli. J Bacteriol* **198**, 386–393 (2016).
- <span id="page-9-11"></span>14. Aidelberg, G. *et al*. Hierarchy of non-glucose sugars in *Escherichia coli*. *BMC systems biology* **8**, 1 (2014).
- <span id="page-9-12"></span>15. You, C. *et al*. Coordination of bacterial proteome with metabolism by cyclic AMP signalling. *Nature* **500**, 301–306 (2013).
- 16. Doucette, C. D., Schwab, D. J., Wingreen, N. S. & Rabinowitz, J. D. α-Ketoglutarate coordinates carbon and nitrogen utilization via enzyme I inhibition. *Nature chemical biology* **7**, 894–901 (2011).
- <span id="page-9-13"></span>17. Daniel, J. & Danchin, A. 2-Ketoglutarate as a possible regulatory metabolite involved in cyclic AMP-dependent catabolite repression in *Escherichia coli* K12. *Biochimie* **68**, 303–310 (1986).
- <span id="page-9-14"></span>18. Afroz, T., Biliouris, K., Kaznessis, Y. & Beisel, C. L. Bacterial sugar utilization gives rise to distinct single‐cell behaviours. *Molecular microbiology* **93**, 1093–1103 (2014).
- <span id="page-9-15"></span>19. Jobe, A. & Bourgeois, S. *lac* repressor-operator interaction: VI. Te natural inducer of the *lac* operon. *Journal of molecular biology* **69**, 397–408 (1972).
- <span id="page-9-16"></span>20. Song, S. & Park, C. Organization and regulation of the D-xylose operons in *Escherichia coli* K-12: XylR acts as a transcriptional activator. *J Bacteriol* **179**, 7025–7032 (1997).
- <span id="page-9-17"></span>21. Schmitz, A. Cyclic AMP receptor protein interacts with lactose operator DNA. *Nucleic acids research* **9**, 277–292 (1981).
- <span id="page-9-18"></span>22. Dickson, R. C., Abelson, J., Barnes, W. M. & Reznikof, W. S. Genetic regulation: the Lac control region. *Science* **187**, 27–35 (1975).
- <span id="page-9-19"></span>23. Sezonov, G., Joseleau-Petit, D. & D'Ari, R. *Escherichia coli* physiology in Luria-Bertani broth. *J Bacteriol* **189**, 8746–8749 (2007).
- <span id="page-9-20"></span>24. Christensen, D. G., Orr, J. S., Rao, C. V. & Wolfe, A. J. Increasing growth yield and decreasing acetylation in *Escherichia coli* by optimizing the carbon-to-magnesium ratio in peptide-based media. *Appl Environ Microb* **83**, e03034–03016 (2017).
- <span id="page-9-21"></span>25. Kercher, M. A., Pace, H. C., Schumacher, M. A., Brennan, R. G. & Lu, P. Crystal structure of the lactose operon repressor and its complexes with DNA and inducer. *Science* **271**, 1247 (1996).
- <span id="page-10-0"></span>26. Englesberg, E., Irr, J., Power, J. & Lee, N. Positive control of enzyme synthesis by gene C in the L-arabinose system. *J Bacteriol* **90**, 946–957 (1965).
- <span id="page-10-1"></span>27. Casadaban, M. J. Regulation of the regulatory gene for the arabinose pathway, *araC*. *Journal of molecular biology* **104**, 557–566 (1976).
- <span id="page-10-2"></span>28. Lee, S. K. *et al*. Directed evolution of AraC for improved compatibility of arabinose- and lactose-inducible promoters. *Appl Environ Microb* **73**, 5711–5715 (2007).
- <span id="page-10-3"></span>29. Bajpai, P. in *Pretreatment of lignocellulosic biomass for biofuel Production* 7–12 (Springer, 2016).
- <span id="page-10-4"></span>30. Datsenko, K. A. & Wanner, B. L. One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. *Proceedings of the National Academy of Sciences* **97**, 6640–6645 (2000).
- <span id="page-10-5"></span>31. Laikova, O. N., Mironov, A. A. & Gelfand, M. S. Computational analysis of the transcriptional regulation of pentose utilization systems in the gamma subdivision of *Proteobacteria*. *FEMS microbiology letters* **205**, 315–322 (2001).
- <span id="page-10-6"></span>32. Xiong, X., De la Cruz, N. & Reznikof, W. Downstream deletion analysis of the *lac* promoter. *J Bacteriol* **173**, 4570–4577 (1991).
- <span id="page-10-7"></span>33. Haldimann, A. & Wanner, B. L. Conditional-replication, integration, excision, and retrieval plasmid-host systems for gene structurefunction studies of bacteria. *J Bacteriol* **183**, 6384–6393 (2001).
- <span id="page-10-8"></span>34. Lennox, E. Transduction of linked genetic characters of the host by bacteriophage P1. *Virology* **1**, 190–206 (1955).
- <span id="page-10-9"></span>35. Tomason, L. C., Costantino, N. & Court, D. L. E. *coli* genome manipulation by P1 transduction. *Current protocols in molecular biology*, 1.17. 11–11.17. 18 (2007).

#### **Acknowledgements**

Tis work was supported by the Energy Biosciences Institute. We thank Dr. Santosh Koirala for help with the fuorescence assays.

#### **Author Contributions**

E.M.A. and C.V.R. conceived and designed the experiments. E.M.A. and X.W. performed the experiments. E.M.A., X.W. and C.V.R. analyzed the data. E.M.A. and C.V.R. wrote the manuscript. E.M.A. and C.V.R edited the manuscript.

### **Additional Information**

**Supplementary information** accompanies this paper at [https://doi.org/10.1038/s41598-017-18704-0.](http://dx.doi.org/10.1038/s41598-017-18704-0)

**Competing Interests:** Te authors declare that they have no competing interests.

**Publisher's note:** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional afliations.

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. Te images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit [http://creativecommons.org/licenses/by/4.0/.](http://creativecommons.org/licenses/by/4.0/)

© Te Author(s) 2018